<!DOCTYPE html>
<html xmlns="http://www.w3.org/1999/xhtml" lang="en" xml:lang="en"><head>

<meta charset="utf-8">
<meta name="generator" content="quarto-1.4.549">

<meta name="viewport" content="width=device-width, initial-scale=1.0, user-scalable=yes">

<meta name="author" content="Sarah Tanja">
<meta name="dcterms.date" content="2024-02-22">

<title>Sarah Tanja’s Open Lab Notebook - Daily Lab Log</title>
<style>
code{white-space: pre-wrap;}
span.smallcaps{font-variant: small-caps;}
div.columns{display: flex; gap: min(4vw, 1.5em);}
div.column{flex: auto; overflow-x: auto;}
div.hanging-indent{margin-left: 1.5em; text-indent: -1.5em;}
ul.task-list{list-style: none;}
ul.task-list li input[type="checkbox"] {
  width: 0.8em;
  margin: 0 0.8em 0.2em -1em; /* quarto-specific, see https://github.com/quarto-dev/quarto-cli/issues/4556 */ 
  vertical-align: middle;
}
/* CSS for syntax highlighting */
pre > code.sourceCode { white-space: pre; position: relative; }
pre > code.sourceCode > span { line-height: 1.25; }
pre > code.sourceCode > span:empty { height: 1.2em; }
.sourceCode { overflow: visible; }
code.sourceCode > span { color: inherit; text-decoration: inherit; }
div.sourceCode { margin: 1em 0; }
pre.sourceCode { margin: 0; }
@media screen {
div.sourceCode { overflow: auto; }
}
@media print {
pre > code.sourceCode { white-space: pre-wrap; }
pre > code.sourceCode > span { text-indent: -5em; padding-left: 5em; }
}
pre.numberSource code
  { counter-reset: source-line 0; }
pre.numberSource code > span
  { position: relative; left: -4em; counter-increment: source-line; }
pre.numberSource code > span > a:first-child::before
  { content: counter(source-line);
    position: relative; left: -1em; text-align: right; vertical-align: baseline;
    border: none; display: inline-block;
    -webkit-touch-callout: none; -webkit-user-select: none;
    -khtml-user-select: none; -moz-user-select: none;
    -ms-user-select: none; user-select: none;
    padding: 0 4px; width: 4em;
  }
pre.numberSource { margin-left: 3em;  padding-left: 4px; }
div.sourceCode
  {   }
@media screen {
pre > code.sourceCode > span > a:first-child::before { text-decoration: underline; }
}
/* CSS for citations */
div.csl-bib-body { }
div.csl-entry {
  clear: both;
  margin-bottom: 0em;
}
.hanging-indent div.csl-entry {
  margin-left:2em;
  text-indent:-2em;
}
div.csl-left-margin {
  min-width:2em;
  float:left;
}
div.csl-right-inline {
  margin-left:2em;
  padding-left:1em;
}
div.csl-indent {
  margin-left: 2em;
}</style>


<script src="../../site_libs/quarto-nav/quarto-nav.js"></script>
<script src="../../site_libs/quarto-nav/headroom.min.js"></script>
<script src="../../site_libs/clipboard/clipboard.min.js"></script>
<script src="../../site_libs/quarto-search/autocomplete.umd.js"></script>
<script src="../../site_libs/quarto-search/fuse.min.js"></script>
<script src="../../site_libs/quarto-search/quarto-search.js"></script>
<meta name="quarto:offset" content="../../">
<link href="../../images/favicon.png" rel="icon" type="image/png">
<script src="../../site_libs/quarto-html/quarto.js"></script>
<script src="../../site_libs/quarto-html/popper.min.js"></script>
<script src="../../site_libs/quarto-html/tippy.umd.min.js"></script>
<script src="../../site_libs/quarto-html/anchor.min.js"></script>
<link href="../../site_libs/quarto-html/tippy.css" rel="stylesheet">
<script src="../../site_libs/bootstrap/bootstrap.min.js"></script>
<link href="../../site_libs/bootstrap/bootstrap-icons.css" rel="stylesheet">
<link href="../../site_libs/bootstrap/bootstrap.min.css" rel="stylesheet" class="quarto-color-scheme" id="quarto-bootstrap" data-mode="light">
<link href="../../site_libs/bootstrap/bootstrap-dark.min.css" rel="prefetch" class="quarto-color-scheme quarto-color-alternate" id="quarto-bootstrap" data-mode="dark">
<script id="quarto-search-options" type="application/json">{
  "location": "navbar",
  "copy-button": false,
  "collapse-after": 3,
  "panel-placement": "end",
  "type": "overlay",
  "limit": 50,
  "keyboard-shortcut": [
    "f",
    "/",
    "s"
  ],
  "show-item-context": false,
  "language": {
    "search-no-results-text": "No results",
    "search-matching-documents-text": "matching documents",
    "search-copy-link-title": "Copy link to search",
    "search-hide-matches-text": "Hide additional matches",
    "search-more-match-text": "more match in this document",
    "search-more-matches-text": "more matches in this document",
    "search-clear-button-title": "Clear",
    "search-text-placeholder": "",
    "search-detached-cancel-button-title": "Cancel",
    "search-submit-button-title": "Submit",
    "search-label": "Search"
  }
}</script>
<style>html{ scroll-behavior: smooth; }</style>
<link href="../../site_libs/htmltools-fill-0.5.8.1/fill.css" rel="stylesheet">
<script src="../../site_libs/htmlwidgets-1.6.4/htmlwidgets.js"></script>
<script src="../../site_libs/jquery-3.6.0/jquery-3.6.0.min.js"></script>
<link href="../../site_libs/leaflet-1.3.1/leaflet.css" rel="stylesheet">
<script src="../../site_libs/leaflet-1.3.1/leaflet.js"></script>
<link href="../../site_libs/leafletfix-1.0.0/leafletfix.css" rel="stylesheet">
<script src="../../site_libs/proj4-2.6.2/proj4.min.js"></script>
<script src="../../site_libs/Proj4Leaflet-1.0.1/proj4leaflet.js"></script>
<link href="../../site_libs/rstudio_leaflet-1.3.1/rstudio_leaflet.css" rel="stylesheet">
<script src="../../site_libs/leaflet-binding-2.2.2/leaflet.js"></script>
<script src="../../site_libs/leaflet-providers-2.0.0/leaflet-providers_2.0.0.js"></script>
<script src="../../site_libs/leaflet-providers-plugin-2.2.2/leaflet-providers-plugin.js"></script>


<meta property="og:title" content="Sarah Tanja’s Open Lab Notebook - Daily Lab Log">
<meta property="og:description" content="Enter at your peril; This one is for me, not you">
<meta property="og:image" content="https://sarahtanja.github.io/quarto-blog/posts/daily-log/log.png">
<meta property="og:site_name" content="Sarah Tanja's Open Lab Notebook">
<meta property="og:image:height" content="2500">
<meta property="og:image:width" content="2143">
</head>

<body class="floating nav-fixed slimcontent">

<div id="quarto-search-results"></div>
  <header id="quarto-header" class="headroom fixed-top quarto-banner">
    <nav class="navbar navbar-expand-lg " data-bs-theme="dark">
      <div class="navbar-container container-fluid">
      <div class="navbar-brand-container mx-auto">
    <a href="linktree.html" class="navbar-brand navbar-brand-logo">
    <img src="../../images/favicon.png" alt="" class="navbar-logo">
    </a>
    <a class="navbar-brand" href="linktree.html">
    <span class="navbar-title">Sarah Tanja’s Open Lab Notebook</span>
    </a>
  </div>
            <div id="quarto-search" class="" title="Search"></div>
          <button class="navbar-toggler" type="button" data-bs-toggle="collapse" data-bs-target="#navbarCollapse" aria-controls="navbarCollapse" aria-expanded="false" aria-label="Toggle navigation" onclick="if (window.quartoToggleHeadroom) { window.quartoToggleHeadroom(); }">
  <span class="navbar-toggler-icon"></span>
</button>
          <div class="collapse navbar-collapse" id="navbarCollapse">
            <ul class="navbar-nav navbar-nav-scroll ms-auto">
  <li class="nav-item">
    <a class="nav-link" href="../../learning.qmd"> 
<span class="menu-text">Learning</span></a>
  </li>  
  <li class="nav-item">
    <a class="nav-link" href="../../projects.html"> 
<span class="menu-text">Projects</span></a>
  </li>  
  <li class="nav-item">
    <a class="nav-link" href="../../posts.html"> 
<span class="menu-text">Posts</span></a>
  </li>  
  <li class="nav-item">
    <a class="nav-link" href="../../about.html"> 
<span class="menu-text">About</span></a>
  </li>  
  <li class="nav-item compact">
    <a class="nav-link" href="https://www.linkedin.com/in/sarah-tanja-595722146/"> <i class="bi bi-linkedin" role="img">
</i> 
<span class="menu-text"></span></a>
  </li>  
  <li class="nav-item dropdown ">
    <a class="nav-link dropdown-toggle" href="#" id="nav-menu-bi-github" role="button" data-bs-toggle="dropdown" aria-expanded="false">
      <i class="bi bi-github" role="img">
</i> 
 <span class="menu-text"></span>
    </a>
    <ul class="dropdown-menu dropdown-menu-end" aria-labelledby="nav-menu-bi-github">    
        <li>
    <a class="dropdown-item" href="https://github.com/sarahtanja">
 <span class="dropdown-text">Sarah’s GitHub</span></a>
  </li>  
        <li>
    <a class="dropdown-item" href="https://github.com/sarahtanja/coral-embryo-leachate">
 <span class="dropdown-text">Coral Embryology Project Repository</span></a>
  </li>  
        <li>
    <a class="dropdown-item" href="https://github.com/sarahtanja/coral-leachate-heat">
 <span class="dropdown-text">Coral Photophysiology Project Repository</span></a>
  </li>  
        <li>
    <a class="dropdown-item" href="https://github.com/sarahtanja/anemone-leachate-heat">
 <span class="dropdown-text">Anemone Photophysiology Project Repository</span></a>
  </li>  
        <li>
    <a class="dropdown-item" href="https://github.com/sarahtanja/coral-pae-temp">
 <span class="dropdown-text">Coral Microbiome Project Repository</span></a>
  </li>  
    </ul>
  </li>
  <li class="nav-item dropdown ">
    <a class="nav-link dropdown-toggle" href="#" id="nav-menu-bi-envelope-paper" role="button" data-bs-toggle="dropdown" aria-expanded="false">
      <i class="bi bi-envelope-paper" role="img">
</i> 
 <span class="menu-text"></span>
    </a>
    <ul class="dropdown-menu dropdown-menu-end" aria-labelledby="nav-menu-bi-envelope-paper">    
        <li class="dropdown-header">stanja@uw.edu</li>
        <li>
    <a class="dropdown-item" href="../../contact.html">
 <span class="dropdown-text">Contact</span></a>
  </li>  
    </ul>
  </li>
</ul>
          </div> <!-- /navcollapse -->
          <div class="quarto-navbar-tools">
  <a href="" class="quarto-color-scheme-toggle quarto-navigation-tool  px-1" onclick="window.quartoToggleColorScheme(); return false;" title="Toggle dark mode"><i class="bi"></i></a>
</div>
      </div> <!-- /container-fluid -->
    </nav>
</header>
<!-- content -->
<header id="title-block-header" class="quarto-title-block default toc-left page-columns page-full">
  <div class="quarto-title-banner page-columns page-full">
    <div class="quarto-title column-body">
      <div class="quarto-title-block"><div><h1 class="title">Daily Lab Log</h1><button type="button" class="btn code-tools-button" id="quarto-code-tools-source"><i class="bi"></i> Code</button></div></div>
            <p class="subtitle lead">Enter at your peril; This one is for me, not you</p>
                                <div class="quarto-categories">
                <div class="quarto-category">daily log</div>
              </div>
                  </div>
  </div>
    
  
  <div class="quarto-title-meta">

      <div>
      <div class="quarto-title-meta-heading">Author</div>
      <div class="quarto-title-meta-contents">
               <p>Sarah Tanja </p>
            </div>
    </div>
      
      <div>
      <div class="quarto-title-meta-heading">Published</div>
      <div class="quarto-title-meta-contents">
        <p class="date">February 22, 2024</p>
      </div>
    </div>
    
      <div>
      <div class="quarto-title-meta-heading">Modified</div>
      <div class="quarto-title-meta-contents">
        <p class="date-modified">October 23, 2025</p>
      </div>
    </div>
      
    </div>
    
  
  </header><div id="quarto-content" class="quarto-container page-columns page-rows-contents page-layout-article page-navbar">
<!-- sidebar -->
  <nav id="quarto-sidebar" class="sidebar collapse collapse-horizontal quarto-sidebar-collapse-item sidebar-navigation floating overflow-auto">
    <nav id="TOC" role="doc-toc" class="toc-active" data-toc-expanded="99">
    <h2 id="toc-title">Contents <i class="bi bi-bookmark-heart"></i></h2>
   
  <ul>
  <li><a href="#september-2024" id="toc-september-2024" class="nav-link active" data-scroll-target="#september-2024"><span class="header-section-number">1</span> September 2024</a>
  <ul>
  <li><a href="#sep-2024" id="toc-sep-2024" class="nav-link" data-scroll-target="#sep-2024"><span class="header-section-number">1.0.0.1</span> 03-SEP-2024</a></li>
  <li><a href="#sep-2024-1" id="toc-sep-2024-1" class="nav-link" data-scroll-target="#sep-2024-1"><span class="header-section-number">1.0.0.2</span> 02-SEP-2024</a></li>
  </ul></li>
  <li><a href="#august-2024" id="toc-august-2024" class="nav-link" data-scroll-target="#august-2024"><span class="header-section-number">2</span> August 2024</a>
  <ul>
  <li><a href="#aug-2024" id="toc-aug-2024" class="nav-link" data-scroll-target="#aug-2024"><span class="header-section-number">2.0.0.1</span> 28-AUG-2024</a></li>
  <li><a href="#aug-2024-1" id="toc-aug-2024-1" class="nav-link" data-scroll-target="#aug-2024-1"><span class="header-section-number">2.0.0.2</span> 27-AUG-2024</a></li>
  <li><a href="#aug-2024-2" id="toc-aug-2024-2" class="nav-link" data-scroll-target="#aug-2024-2"><span class="header-section-number">2.0.0.3</span> 15-AUG-2024</a></li>
  </ul></li>
  <li><a href="#march-2024" id="toc-march-2024" class="nav-link" data-scroll-target="#march-2024"><span class="header-section-number">3</span> March 2024</a>
  <ul>
  <li><a href="#mar-2024" id="toc-mar-2024" class="nav-link" data-scroll-target="#mar-2024"><span class="header-section-number">3.0.0.1</span> 28-MAR-2024</a></li>
  <li><a href="#mar-2024-1" id="toc-mar-2024-1" class="nav-link" data-scroll-target="#mar-2024-1"><span class="header-section-number">3.0.0.2</span> 21-MAR-2024</a></li>
  <li><a href="#mar-2024-2" id="toc-mar-2024-2" class="nav-link" data-scroll-target="#mar-2024-2"><span class="header-section-number">3.0.0.3</span> 20-MAR-2024</a></li>
  </ul></li>
  <li><a href="#february-2024" id="toc-february-2024" class="nav-link" data-scroll-target="#february-2024"><span class="header-section-number">4</span> February 2024</a>
  <ul>
  <li><a href="#feb-2024" id="toc-feb-2024" class="nav-link" data-scroll-target="#feb-2024"><span class="header-section-number">4.0.0.1</span> 22-FEB-2024</a></li>
  <li><a href="#feb-2024-1" id="toc-feb-2024-1" class="nav-link" data-scroll-target="#feb-2024-1"><span class="header-section-number">4.0.0.2</span> 21-FEB-2024</a></li>
  <li><a href="#feb-2024-2" id="toc-feb-2024-2" class="nav-link" data-scroll-target="#feb-2024-2"><span class="header-section-number">4.0.0.3</span> 20-FEB-2024</a></li>
  <li><a href="#feb-2024-3" id="toc-feb-2024-3" class="nav-link" data-scroll-target="#feb-2024-3"><span class="header-section-number">4.0.0.4</span> <strong>02-FEB-2024</strong></a></li>
  <li><a href="#feb-2024-4" id="toc-feb-2024-4" class="nav-link" data-scroll-target="#feb-2024-4"><span class="header-section-number">4.0.0.5</span> 08-FEB-2024</a></li>
  <li><a href="#feb-2024-5" id="toc-feb-2024-5" class="nav-link" data-scroll-target="#feb-2024-5"><span class="header-section-number">4.0.0.6</span> 07-FEB-2024</a></li>
  <li><a href="#feb-2024-6" id="toc-feb-2024-6" class="nav-link" data-scroll-target="#feb-2024-6"><span class="header-section-number">4.0.0.7</span> 05-FEB-2024</a></li>
  <li><a href="#january-2024" id="toc-january-2024" class="nav-link" data-scroll-target="#january-2024"><span class="header-section-number">4.1</span> January 2024</a></li>
  <li><a href="#december-2023" id="toc-december-2023" class="nav-link" data-scroll-target="#december-2023"><span class="header-section-number">4.2</span> December 2023</a></li>
  <li><a href="#november-2023" id="toc-november-2023" class="nav-link" data-scroll-target="#november-2023"><span class="header-section-number">4.3</span> November 2023</a>
  <ul>
  <li><a href="#nov-2023" id="toc-nov-2023" class="nav-link" data-scroll-target="#nov-2023"><span class="header-section-number">4.3.0.1</span> 27-NOV-2023</a></li>
  <li><a href="#nov-2023-1" id="toc-nov-2023-1" class="nav-link" data-scroll-target="#nov-2023-1"><span class="header-section-number">4.3.0.2</span> <strong>21-NOV-2023</strong></a></li>
  <li><a href="#nov-2023-2" id="toc-nov-2023-2" class="nav-link" data-scroll-target="#nov-2023-2"><span class="header-section-number">4.3.0.3</span> 11-NOV-2023</a></li>
  <li><a href="#nov-2023-3" id="toc-nov-2023-3" class="nav-link" data-scroll-target="#nov-2023-3"><span class="header-section-number">4.3.0.4</span> 10-NOV-2023</a></li>
  <li><a href="#nov-2023-4" id="toc-nov-2023-4" class="nav-link" data-scroll-target="#nov-2023-4"><span class="header-section-number">4.3.0.5</span> 09-NOV 2023</a></li>
  <li><a href="#nov-2023-5" id="toc-nov-2023-5" class="nav-link" data-scroll-target="#nov-2023-5"><span class="header-section-number">4.3.0.6</span> 08-NOV-2023</a></li>
  <li><a href="#nov-2023-6" id="toc-nov-2023-6" class="nav-link" data-scroll-target="#nov-2023-6"><span class="header-section-number">4.3.0.7</span> 05-NOV-2023</a></li>
  <li><a href="#nov-2023-7" id="toc-nov-2023-7" class="nav-link" data-scroll-target="#nov-2023-7"><span class="header-section-number">4.3.0.8</span> 03-NOV-2023</a></li>
  <li><a href="#nov-2023-8" id="toc-nov-2023-8" class="nav-link" data-scroll-target="#nov-2023-8"><span class="header-section-number">4.3.0.9</span> 02-NOV-2023</a></li>
  <li><a href="#nov-2023-9" id="toc-nov-2023-9" class="nav-link" data-scroll-target="#nov-2023-9"><span class="header-section-number">4.3.0.10</span> 01-NOV-2023</a></li>
  </ul></li>
  <li><a href="#october-2023" id="toc-october-2023" class="nav-link" data-scroll-target="#october-2023"><span class="header-section-number">4.4</span> October 2023</a>
  <ul>
  <li><a href="#oct-2023" id="toc-oct-2023" class="nav-link" data-scroll-target="#oct-2023"><span class="header-section-number">4.4.0.1</span> <strong>27-OCT-2023</strong></a></li>
  <li><a href="#oct-2023-1" id="toc-oct-2023-1" class="nav-link" data-scroll-target="#oct-2023-1"><span class="header-section-number">4.4.0.2</span> <strong>26-OCT-2023</strong></a></li>
  <li><a href="#oct-2023-2" id="toc-oct-2023-2" class="nav-link" data-scroll-target="#oct-2023-2"><span class="header-section-number">4.4.0.3</span> 24-OCT-2023</a></li>
  <li><a href="#oct-2023-3" id="toc-oct-2023-3" class="nav-link" data-scroll-target="#oct-2023-3"><span class="header-section-number">4.4.0.4</span> 23-OCT-2023</a></li>
  <li><a href="#oct-2023-4" id="toc-oct-2023-4" class="nav-link" data-scroll-target="#oct-2023-4"><span class="header-section-number">4.4.0.5</span> 16-OCT-2023</a></li>
  <li><a href="#oct-2023-5" id="toc-oct-2023-5" class="nav-link" data-scroll-target="#oct-2023-5"><span class="header-section-number">4.4.0.6</span> 11-OCT-2023</a></li>
  <li><a href="#oct-2023-6" id="toc-oct-2023-6" class="nav-link" data-scroll-target="#oct-2023-6"><span class="header-section-number">4.4.0.7</span> 10-OCT-2023</a></li>
  <li><a href="#oct-2023-7" id="toc-oct-2023-7" class="nav-link" data-scroll-target="#oct-2023-7"><span class="header-section-number">4.4.0.8</span> 03-OCT-2023</a></li>
  <li><a href="#oct-2023-8" id="toc-oct-2023-8" class="nav-link" data-scroll-target="#oct-2023-8"><span class="header-section-number">4.4.0.9</span> 02-OCT-2023</a></li>
  </ul></li>
  <li><a href="#september-2023" id="toc-september-2023" class="nav-link" data-scroll-target="#september-2023"><span class="header-section-number">4.5</span> September 2023</a>
  <ul>
  <li><a href="#sep-2023" id="toc-sep-2023" class="nav-link" data-scroll-target="#sep-2023"><span class="header-section-number">4.5.0.1</span> 20-SEP-2023</a></li>
  <li><a href="#sep-2023-1" id="toc-sep-2023-1" class="nav-link" data-scroll-target="#sep-2023-1"><span class="header-section-number">4.5.0.2</span> 05-SEP-2023</a></li>
  <li><a href="#sep-2023-2" id="toc-sep-2023-2" class="nav-link" data-scroll-target="#sep-2023-2"><span class="header-section-number">4.5.0.3</span> 01-SEP-2023</a></li>
  </ul></li>
  <li><a href="#august-2023" id="toc-august-2023" class="nav-link" data-scroll-target="#august-2023"><span class="header-section-number">4.6</span> August 2023</a>
  <ul>
  <li><a href="#aug-2023" id="toc-aug-2023" class="nav-link" data-scroll-target="#aug-2023"><span class="header-section-number">4.6.0.1</span> 31-AUG-2023</a></li>
  <li><a href="#aug-2023-1" id="toc-aug-2023-1" class="nav-link" data-scroll-target="#aug-2023-1"><span class="header-section-number">4.6.0.2</span> 29-AUG-2023</a></li>
  <li><a href="#aug-2023-2" id="toc-aug-2023-2" class="nav-link" data-scroll-target="#aug-2023-2"><span class="header-section-number">4.6.0.3</span> 28-AUG-2023</a></li>
  <li><a href="#aug-2023-3" id="toc-aug-2023-3" class="nav-link" data-scroll-target="#aug-2023-3"><span class="header-section-number">4.6.0.4</span> 24-AUG-2023</a></li>
  <li><a href="#aug-2023-4" id="toc-aug-2023-4" class="nav-link" data-scroll-target="#aug-2023-4"><span class="header-section-number">4.6.0.5</span> 17-AUG-2023</a></li>
  <li><a href="#aug-2023-5" id="toc-aug-2023-5" class="nav-link" data-scroll-target="#aug-2023-5"><span class="header-section-number">4.6.0.6</span> 16-AUG-2023</a></li>
  <li><a href="#aug-2023-6" id="toc-aug-2023-6" class="nav-link" data-scroll-target="#aug-2023-6"><span class="header-section-number">4.6.0.7</span> 15-AUG-2023</a></li>
  <li><a href="#aug-2023-7" id="toc-aug-2023-7" class="nav-link" data-scroll-target="#aug-2023-7"><span class="header-section-number">4.6.0.8</span> 03-AUG-2023</a></li>
  </ul></li>
  <li><a href="#july-2023" id="toc-july-2023" class="nav-link" data-scroll-target="#july-2023"><span class="header-section-number">4.7</span> July 2023</a>
  <ul>
  <li><a href="#jul-2023" id="toc-jul-2023" class="nav-link" data-scroll-target="#jul-2023"><span class="header-section-number">4.7.0.1</span> 20-JUL-2023</a></li>
  <li><a href="#jul-2023-1" id="toc-jul-2023-1" class="nav-link" data-scroll-target="#jul-2023-1"><span class="header-section-number">4.7.0.2</span> 19-JUL-2023</a></li>
  </ul></li>
  <li><a href="#june-2023" id="toc-june-2023" class="nav-link" data-scroll-target="#june-2023"><span class="header-section-number">4.8</span> June 2023</a>
  <ul>
  <li><a href="#jun-2023" id="toc-jun-2023" class="nav-link" data-scroll-target="#jun-2023"><span class="header-section-number">4.8.0.1</span> 07-JUN-2023</a></li>
  <li><a href="#jun-2023-1" id="toc-jun-2023-1" class="nav-link" data-scroll-target="#jun-2023-1"><span class="header-section-number">4.8.0.2</span> 06-JUN-2023</a></li>
  <li><a href="#jun-2023-2" id="toc-jun-2023-2" class="nav-link" data-scroll-target="#jun-2023-2"><span class="header-section-number">4.8.0.3</span> 05-JUN-2023</a></li>
  <li><a href="#jun-2023-3" id="toc-jun-2023-3" class="nav-link" data-scroll-target="#jun-2023-3"><span class="header-section-number">4.8.0.4</span> 02-JUN-2023</a></li>
  <li><a href="#jun-2023-4" id="toc-jun-2023-4" class="nav-link" data-scroll-target="#jun-2023-4"><span class="header-section-number">4.8.0.5</span> 01-JUN-2023</a></li>
  </ul></li>
  <li><a href="#may-2023" id="toc-may-2023" class="nav-link" data-scroll-target="#may-2023"><span class="header-section-number">4.9</span> May 2023</a>
  <ul>
  <li><a href="#may-2023-1" id="toc-may-2023-1" class="nav-link" data-scroll-target="#may-2023-1"><span class="header-section-number">4.9.0.1</span> 31-MAY-2023</a></li>
  <li><a href="#may-2023-2" id="toc-may-2023-2" class="nav-link" data-scroll-target="#may-2023-2"><span class="header-section-number">4.9.0.2</span> 29-MAY-2023</a></li>
  <li><a href="#may-2023-3" id="toc-may-2023-3" class="nav-link" data-scroll-target="#may-2023-3"><span class="header-section-number">4.9.0.3</span> 25-MAY-2023</a></li>
  <li><a href="#may-2023-4" id="toc-may-2023-4" class="nav-link" data-scroll-target="#may-2023-4"><span class="header-section-number">4.9.0.4</span> 24-MAY-2023</a></li>
  <li><a href="#may-2023-5" id="toc-may-2023-5" class="nav-link" data-scroll-target="#may-2023-5"><span class="header-section-number">4.9.0.5</span> 23-MAY-2023</a></li>
  <li><a href="#may-2023-6" id="toc-may-2023-6" class="nav-link" data-scroll-target="#may-2023-6"><span class="header-section-number">4.9.0.6</span> 19-MAY-2023</a></li>
  <li><a href="#may-2023-7" id="toc-may-2023-7" class="nav-link" data-scroll-target="#may-2023-7"><span class="header-section-number">4.9.0.7</span> 18-MAY-2023</a></li>
  <li><a href="#may-2023-8" id="toc-may-2023-8" class="nav-link" data-scroll-target="#may-2023-8"><span class="header-section-number">4.9.0.8</span> 04-MAY-2023</a></li>
  </ul></li>
  <li><a href="#april-2023" id="toc-april-2023" class="nav-link" data-scroll-target="#april-2023"><span class="header-section-number">4.10</span> April 2023</a>
  <ul>
  <li><a href="#apr-2023" id="toc-apr-2023" class="nav-link" data-scroll-target="#apr-2023"><span class="header-section-number">4.10.0.1</span> 25-APR-2023</a></li>
  <li><a href="#apr-2023-1" id="toc-apr-2023-1" class="nav-link" data-scroll-target="#apr-2023-1"><span class="header-section-number">4.10.0.2</span> 24-APR-2023</a></li>
  <li><a href="#apr-2023-2" id="toc-apr-2023-2" class="nav-link" data-scroll-target="#apr-2023-2"><span class="header-section-number">4.10.0.3</span> 21-APR-2023</a></li>
  <li><a href="#apr-2023-3" id="toc-apr-2023-3" class="nav-link" data-scroll-target="#apr-2023-3"><span class="header-section-number">4.10.0.4</span> 20-APR-2023</a></li>
  <li><a href="#apr-2023-4" id="toc-apr-2023-4" class="nav-link" data-scroll-target="#apr-2023-4"><span class="header-section-number">4.10.0.5</span> 19-APR-2023</a></li>
  <li><a href="#apr-2023-5" id="toc-apr-2023-5" class="nav-link" data-scroll-target="#apr-2023-5"><span class="header-section-number">4.10.0.6</span> 18-APR-2023</a></li>
  <li><a href="#apr-2023-6" id="toc-apr-2023-6" class="nav-link" data-scroll-target="#apr-2023-6"><span class="header-section-number">4.10.0.7</span> 17-APR-2023</a></li>
  <li><a href="#apr-2023-7" id="toc-apr-2023-7" class="nav-link" data-scroll-target="#apr-2023-7"><span class="header-section-number">4.10.0.8</span> 14-APR-2023</a></li>
  <li><a href="#apr-2023-8" id="toc-apr-2023-8" class="nav-link" data-scroll-target="#apr-2023-8"><span class="header-section-number">4.10.0.9</span> 11-APR-2023</a></li>
  <li><a href="#apr-2023-9" id="toc-apr-2023-9" class="nav-link" data-scroll-target="#apr-2023-9"><span class="header-section-number">4.10.0.10</span> 10-APR-2023</a></li>
  </ul></li>
  <li><a href="#march-2023" id="toc-march-2023" class="nav-link" data-scroll-target="#march-2023"><span class="header-section-number">4.11</span> March 2023</a>
  <ul>
  <li><a href="#mar-2023" id="toc-mar-2023" class="nav-link" data-scroll-target="#mar-2023"><span class="header-section-number">4.11.0.1</span> 29-MAR-2023</a></li>
  <li><a href="#mar-2023-1" id="toc-mar-2023-1" class="nav-link" data-scroll-target="#mar-2023-1"><span class="header-section-number">4.11.0.2</span> 28-MAR-2023</a></li>
  <li><a href="#mar-2023-2" id="toc-mar-2023-2" class="nav-link" data-scroll-target="#mar-2023-2"><span class="header-section-number">4.11.0.3</span> 27-MAR-2023</a>
  <ul class="collapse">
  <li><a href="#section" id="toc-section" class="nav-link" data-scroll-target="#section"><span class="header-section-number">4.11.0.3.1</span> </a></li>
  </ul></li>
  <li><a href="#mar-2023-3" id="toc-mar-2023-3" class="nav-link" data-scroll-target="#mar-2023-3"><span class="header-section-number">4.11.0.4</span> 21-MAR-2023</a></li>
  <li><a href="#mar-2023-4" id="toc-mar-2023-4" class="nav-link" data-scroll-target="#mar-2023-4"><span class="header-section-number">4.11.0.5</span> 20-MAR-2023</a></li>
  <li><a href="#mar-2023-5" id="toc-mar-2023-5" class="nav-link" data-scroll-target="#mar-2023-5"><span class="header-section-number">4.11.0.6</span> 16-MAR-2023</a></li>
  <li><a href="#mar-2023-6" id="toc-mar-2023-6" class="nav-link" data-scroll-target="#mar-2023-6"><span class="header-section-number">4.11.0.7</span> 15-MAR-2023</a></li>
  <li><a href="#mar-2023-7" id="toc-mar-2023-7" class="nav-link" data-scroll-target="#mar-2023-7"><span class="header-section-number">4.11.0.8</span> 14-MAR-2023</a></li>
  <li><a href="#mar-2023-8" id="toc-mar-2023-8" class="nav-link" data-scroll-target="#mar-2023-8"><span class="header-section-number">4.11.0.9</span> 13-MAR-2023</a></li>
  <li><a href="#mar-2023-9" id="toc-mar-2023-9" class="nav-link" data-scroll-target="#mar-2023-9"><span class="header-section-number">4.11.0.10</span> 10-MAR-2023</a></li>
  <li><a href="#mar-2023-10" id="toc-mar-2023-10" class="nav-link" data-scroll-target="#mar-2023-10"><span class="header-section-number">4.11.0.11</span> 09-MAR-2023</a></li>
  <li><a href="#mar-2023-11" id="toc-mar-2023-11" class="nav-link" data-scroll-target="#mar-2023-11"><span class="header-section-number">4.11.0.12</span> 07-MAR-2023</a></li>
  <li><a href="#mar-2023-12" id="toc-mar-2023-12" class="nav-link" data-scroll-target="#mar-2023-12"><span class="header-section-number">4.11.0.13</span> 06-MAR-2023</a></li>
  <li><a href="#mar-2023-13" id="toc-mar-2023-13" class="nav-link" data-scroll-target="#mar-2023-13"><span class="header-section-number">4.11.0.14</span> 03-MAR-2023</a></li>
  <li><a href="#mar-2023-14" id="toc-mar-2023-14" class="nav-link" data-scroll-target="#mar-2023-14"><span class="header-section-number">4.11.0.15</span> 02-MAR-2023</a></li>
  </ul></li>
  <li><a href="#january-2023" id="toc-january-2023" class="nav-link" data-scroll-target="#january-2023"><span class="header-section-number">4.12</span> January 2023</a>
  <ul>
  <li><a href="#jan-2023" id="toc-jan-2023" class="nav-link" data-scroll-target="#jan-2023"><span class="header-section-number">4.12.0.1</span> 31-JAN-2023</a></li>
  <li><a href="#jan-2023-1" id="toc-jan-2023-1" class="nav-link" data-scroll-target="#jan-2023-1"><span class="header-section-number">4.12.0.2</span> 20-JAN-2023</a></li>
  <li><a href="#jan-2023-2" id="toc-jan-2023-2" class="nav-link" data-scroll-target="#jan-2023-2"><span class="header-section-number">4.12.0.3</span> 19-JAN-2023</a></li>
  <li><a href="#jan-2023-3" id="toc-jan-2023-3" class="nav-link" data-scroll-target="#jan-2023-3"><span class="header-section-number">4.12.0.4</span> 18-JAN-2023</a></li>
  <li><a href="#jan-2023-4" id="toc-jan-2023-4" class="nav-link" data-scroll-target="#jan-2023-4"><span class="header-section-number">4.12.0.5</span> 17-JAN-2023</a></li>
  <li><a href="#jan-2023-5" id="toc-jan-2023-5" class="nav-link" data-scroll-target="#jan-2023-5"><span class="header-section-number">4.12.0.6</span> 13-JAN-2023</a></li>
  <li><a href="#jan-2023-6" id="toc-jan-2023-6" class="nav-link" data-scroll-target="#jan-2023-6"><span class="header-section-number">4.12.0.7</span> 12-JAN-2023</a></li>
  <li><a href="#jan-2023-7" id="toc-jan-2023-7" class="nav-link" data-scroll-target="#jan-2023-7"><span class="header-section-number">4.12.0.8</span> 09-JAN-2023</a></li>
  </ul></li>
  </ul></li>
  </ul>
</nav>
</nav>
<div id="quarto-sidebar-glass" class="quarto-sidebar-collapse-item" data-bs-toggle="collapse" data-bs-target=".quarto-sidebar-collapse-item"></div>
<!-- margin-sidebar -->
    <div id="quarto-margin-sidebar" class="sidebar margin-sidebar">
    </div>
<!-- main -->
<main class="content quarto-banner-title-block page-columns page-full" id="quarto-document-content">





<section id="september-2024" class="level1" data-number="1">
<h1 data-number="1"><span class="header-section-number">1</span> September 2024</h1>
<section id="sep-2024" class="level4" data-number="1.0.0.1">
<h4 data-number="1.0.0.1" class="anchored" data-anchor-id="sep-2024"><span class="header-section-number">1.0.0.1</span> 03-SEP-2024</h4>
<ul>
<li><p>Received confirmation that RNA is in line for QAQC</p></li>
<li><p>Read manual for Nikon compound scope, setup for making embryology preservation slides &amp; taking photos</p></li>
</ul>
</section>
<section id="sep-2024-1" class="level4" data-number="1.0.0.2">
<h4 data-number="1.0.0.2" class="anchored" data-anchor-id="sep-2024-1"><span class="header-section-number">1.0.0.2</span> 02-SEP-2024</h4>
<ul>
<li><p>Submitted Sample Form to Azenta</p></li>
<li><p>Writing &amp; editing chapter 1 methods</p></li>
</ul>
</section>
</section>
<section id="august-2024" class="level1" data-number="2">
<h1 data-number="2"><span class="header-section-number">2</span> August 2024</h1>
<section id="aug-2024" class="level4" data-number="2.0.0.1">
<h4 data-number="2.0.0.1" class="anchored" data-anchor-id="aug-2024"><span class="header-section-number">2.0.0.1</span> 28-AUG-2024</h4>
<ul>
<li>Received email confirmation of receipt of RNA at Azenta!</li>
</ul>
</section>
<section id="aug-2024-1" class="level4" data-number="2.0.0.2">
<h4 data-number="2.0.0.2" class="anchored" data-anchor-id="aug-2024-1"><span class="header-section-number">2.0.0.2</span> 27-AUG-2024</h4>
<ul>
<li><p>Shipped Total RNA to Azenta in 96 well plate on ~16lbs of dry ice, FedEx Priority Overnight from SeaTac FedEx Office Ship Center</p></li>
<li><p>Tracking #: 278809181237</p></li>
</ul>
</section>
<section id="aug-2024-2" class="level4" data-number="2.0.0.3">
<h4 data-number="2.0.0.3" class="anchored" data-anchor-id="aug-2024-2"><span class="header-section-number">2.0.0.3</span> 15-AUG-2024</h4>
<ul>
<li><p>Met with Jefferey Kim of Azenta GeneWiz</p>
<ul>
<li><p>Got a quote for 60 TotalRNA-seq with PolyA selection, 20M reads per sample</p></li>
<li><p>For Total RNA in nuclease-free water on dry ice, need 100ng of Total RNA… so if I have 10ng/uL in 30 uL… that means there is 300 ng of Total RNA in that sample. Jeff said 10ng/uL is usually the cutoff. I can send Total eluted RNA in 30uL without normalizing or Nanodropping… they will run RIN scores when they receive the samples. I just have to extract and Qubit!</p></li>
<li><p>23-28 business days turnaround time</p></li>
<li><p>With discount, price per sample is $98.50! Total cost is $5,910.00 in quote :D</p></li>
</ul></li>
</ul>
</section>
</section>
<section id="march-2024" class="level1" data-number="3">
<h1 data-number="3"><span class="header-section-number">3</span> March 2024</h1>
<section id="mar-2024" class="level4" data-number="3.0.0.1">
<h4 data-number="3.0.0.1" class="anchored" data-anchor-id="mar-2024"><span class="header-section-number">3.0.0.1</span> 28-MAR-2024</h4>
<ul>
<li><p>met with Kip and made calendar of assignments due</p></li>
<li><p>changed out filters on seawater filter rack in PBB</p>
<blockquote class="blockquote">
<p>To measure impacts of microplastic exposure on anemone size, oral disk diameters were calculated using the average of two perpendicular measurements across the oral surface intersecting at the mouth. Measurements were made using Image J software (Rasband 19972018) with digital photographs of each anemone …. - Beck, 2022 WWU Dissertation</p>
</blockquote></li>
<li><p>How do we get scaled images of the oral disc?</p></li>
</ul>
<!-- -->
<ul>
<li><p>downloaded AmScope software for MU503B that is attached to the dissecting scope in PBB: <a href="https://amscope.com/pages/software-downloads" class="uri">https://amscope.com/pages/software-downloads</a></p></li>
<li><p>downloaded AmScope manual: <a href="https://storage.googleapis.com/software-download-d79bb.appspot.com/Manual%20Download%20Files/Manual%20Files/Cameras/MU_MD_Series_Camera_Software_Manual.pdf" class="uri">https://storage.googleapis.com/software-download-d79bb.appspot.com/Manual%20Download%20Files/Manual%20Files/Cameras/MU_MD_Series_Camera_Software_Manual.pdf</a></p></li>
</ul>
</section>
<section id="mar-2024-1" class="level4" data-number="3.0.0.2">
<h4 data-number="3.0.0.2" class="anchored" data-anchor-id="mar-2024-1"><span class="header-section-number">3.0.0.2</span> 21-MAR-2024</h4>
<ul>
<li><p>spent nearly all day drying an edge of each petri dish, and gluing bee-tags to them with superglue, then letting it dry by floating the petri dish on a floating rack in the seatable. Took all day..</p></li>
<li><p>swapped anemones from upper seatable to lower, changed water in upper, and letting it come down to temp (when doing water changes, cannot do it directly, as header water is at room temp and seatables are at 10C…. How will I deal with this during the experiment when each anemone is in a 200mL jar and I’ll be doing water changes every other day</p>
<div class="cell">
<div class="sourceCode cell-code" id="cb1"><pre class="sourceCode r code-with-copy"><code class="sourceCode r"><span id="cb1-1"><a href="#cb1-1" aria-hidden="true" tabindex="-1"></a><span class="co"># 72 anemones each in 200mL volume... how much volume I will need to change every other day</span></span>
<span id="cb1-2"><a href="#cb1-2" aria-hidden="true" tabindex="-1"></a>ml <span class="ot">&lt;-</span> <span class="dv">72</span><span class="sc">*</span><span class="dv">200</span></span>
<span id="cb1-3"><a href="#cb1-3" aria-hidden="true" tabindex="-1"></a><span class="fu">print</span>(ml)</span></code><button title="Copy to Clipboard" class="code-copy-button"><i class="bi"></i></button></pre></div>
<div class="cell-output cell-output-stdout">
<pre><code>[1] 14400</code></pre>
</div>
<div class="sourceCode cell-code" id="cb3"><pre class="sourceCode r code-with-copy"><code class="sourceCode r"><span id="cb3-1"><a href="#cb3-1" aria-hidden="true" tabindex="-1"></a><span class="co"># how much of each treatment (seawater vs. leachate)</span></span>
<span id="cb3-2"><a href="#cb3-2" aria-hidden="true" tabindex="-1"></a>ml<span class="sc">/</span><span class="dv">2</span></span></code><button title="Copy to Clipboard" class="code-copy-button"><i class="bi"></i></button></pre></div>
<div class="cell-output cell-output-stdout">
<pre><code>[1] 7200</code></pre>
</div>
</div>
<p>The amount of microplastic may be our limiting factor for how long the exposure is.</p></li>
</ul>
<p>We have 3 grams of each microplastic type</p>
<p>To make a batch of leachate according to the Saliu Lab protocol, we have 250 mg of microplastic.</p>
<div class="cell">
<div class="sourceCode cell-code" id="cb5"><pre class="sourceCode r code-with-copy"><code class="sourceCode r"><span id="cb5-1"><a href="#cb5-1" aria-hidden="true" tabindex="-1"></a><span class="co"># 3 grams is 3000 mg, how many batches of leachate will each microplastic type make?</span></span>
<span id="cb5-2"><a href="#cb5-2" aria-hidden="true" tabindex="-1"></a><span class="dv">3000</span><span class="sc">/</span><span class="dv">250</span></span></code><button title="Copy to Clipboard" class="code-copy-button"><i class="bi"></i></button></pre></div>
<div class="cell-output cell-output-stdout">
<pre><code>[1] 12</code></pre>
</div>
</div>
<p>12 batches… that’s a lot, and should be plenty since it is a highly concentrated amount that will be diluted.</p>
<p>How will it be diluted?</p>
</section>
<section id="mar-2024-2" class="level4" data-number="3.0.0.3">
<h4 data-number="3.0.0.3" class="anchored" data-anchor-id="mar-2024-2"><span class="header-section-number">3.0.0.3</span> 20-MAR-2024</h4>
<ul>
<li>cleaned anemones by picking rocks and debris from them, and squirting with chilled seawater from a lab squeezie bottle. Two were ‘escapees’, one was fully out of the petri dish! I popped it back in. The ones that are nudging around the edge/lip of the petri dish I use my finger to just push their pedal disc back inside the dish… not sure what I can use to deter them from escape that won’t expose them to more chemicals (i.e.&nbsp;tape ahdesive, etc.)</li>
</ul>
</section>
</section>
<section id="february-2024" class="level1 page-columns page-full" data-number="4">
<h1 data-number="4"><span class="header-section-number">4</span> February 2024</h1>
<section id="feb-2024" class="level4" data-number="4.0.0.1">
<h4 data-number="4.0.0.1" class="anchored" data-anchor-id="feb-2024"><span class="header-section-number">4.0.0.1</span> 22-FEB-2024</h4>
<ul>
<li><p>Roberts Lab meeting, received feedback on anemone research proposal: suggested I do initial response curves to find at what temp A. elegantissima experience stress, at what microplastic leachate concentration do they experience stress? Measure response variables ‘quickly’ by respirometry and PAM. Can I make the single stress, multiple response variable, the first chapter of the Master’s thesis? Then, two stressors together with `omics response variables (once we find the inflection points of stress) can be the second chapter…. Ariana pointed out that the paper reviewers are going to want to know why I chose certain concentrations of microplastic leachate. I have to justify that by doing a response curve first. Learned about Roberts Lab pubathon.</p></li>
<li><p>Took FireSting O2 to IT to try and get correct drivers on <code>mariscos</code> ruggedized laptop to run the program…</p></li>
<li><p>pyroscience FireStingO2 deemed ‘dead’ by IT specialist Michael Shriver… the unit won’t connect. I sent the following to the Walz product support team Todd Kana (kana@bayinstruments.com) &amp; Erich Keim (ekeim@walz.com):</p>
<blockquote class="blockquote">
<p>Good afternoon&nbsp; Erich and Todd,&nbsp;</p>
<p>I’m a graduate student researcher in the Padilla-Gamino Lab at the University of Washington, and I’m working with a Walz Mini PAM-II and FireStingO2 oxygen meter that our lab purchased from you as a set. The instrument was previously used by Miranda Roethler, who passed your contact information on to me.&nbsp;</p>
<p>The FireStingO2 oxygen meter is not powering up, and won’t connect to any computer that we’ve tried it on. It was just with our IT specialist for the last few hours (he was attempting to get it to connect by downloading&nbsp;old drivers), and he shared that the laser-cut `FiresStingO2` light on the instrument would ‘pulse’ red and then remain off. He deemed the instrument ‘dead’…&nbsp;</p>
<p>This instrument has only been used a handful of times, and it has been kept in good condition (not dropped, or been exposed/submerged in water etc.).&nbsp;</p>
<p>Have you had this issue before? If so, are there any solutions you can help provide?</p>
<p>Kindy,&nbsp;</p>
<p>- Sarah Tanja&nbsp;</p>
</blockquote></li>
</ul>
</section>
<section id="feb-2024-1" class="level4" data-number="4.0.0.2">
<h4 data-number="4.0.0.2" class="anchored" data-anchor-id="feb-2024-1"><span class="header-section-number">4.0.0.2</span> 21-FEB-2024</h4>
<ul>
<li><p>pyroscience FireStingO2 only works on Windows 10!?</p></li>
<li><p>Ammonia levels in PBB header tank are at 1ppm, something green is growing in there. When was the last time we checked that the UV sterilizer was working? I should change out all the filters on that seawater purification rig, and the reverse osmosis deionized water filter cascade.</p></li>
<li><p>There is algae growing in the PBB header/holding tank</p></li>
<li><p>Met with Kip, Added Kip to WDFW scientific collection permit</p></li>
<li><p>9 permit locations!</p></li>
<li><p>Messed around with mapping the collection sites, from R Graph Gallery, <a href="https://r-graph-gallery.com/180-change-background-in-leaflet-map.html" class="uri">https://r-graph-gallery.com/180-change-background-in-leaflet-map.html</a></p></li>
<li><p>All map background by leaflet can be found here: <a href="https://leaflet-extras.github.io/leaflet-providers/preview/index.html" class="uri">https://leaflet-extras.github.io/leaflet-providers/preview/index.html</a></p></li>
</ul>
<p>Interactive Map with <code>leaflet</code></p>
<div class="cell" data-hash="daily-log_cache/html/unnamed-chunk-3_ac4992d46475ddd714d0de6505015e42">
<div class="sourceCode cell-code" id="cb7"><pre class="sourceCode r code-with-copy"><code class="sourceCode r"><span id="cb7-1"><a href="#cb7-1" aria-hidden="true" tabindex="-1"></a><span class="do">## set CRAN mirror</span></span>
<span id="cb7-2"><a href="#cb7-2" aria-hidden="true" tabindex="-1"></a><span class="fu">options</span>(<span class="at">repos =</span> <span class="fu">c</span>(<span class="at">CRAN =</span> <span class="st">"https://cran.r-project.org/"</span>))</span></code><button title="Copy to Clipboard" class="code-copy-button"><i class="bi"></i></button></pre></div>
</div>
<div class="cell" data-hash="daily-log_cache/html/unnamed-chunk-4_149cb63dfbb50c68fffc34acda1e3125">
<div class="sourceCode cell-code" id="cb8"><pre class="sourceCode r code-with-copy"><code class="sourceCode r"><span id="cb8-1"><a href="#cb8-1" aria-hidden="true" tabindex="-1"></a><span class="do">## install packages</span></span>
<span id="cb8-2"><a href="#cb8-2" aria-hidden="true" tabindex="-1"></a><span class="cf">if</span> (<span class="sc">!</span><span class="fu">require</span>(<span class="st">"tidyverse"</span>, <span class="at">character.only =</span> <span class="cn">TRUE</span>)) { <span class="fu">install.packages</span>(<span class="st">"tidyverse"</span>)}</span></code><button title="Copy to Clipboard" class="code-copy-button"><i class="bi"></i></button></pre></div>
<div class="cell-output cell-output-stderr">
<pre><code>Loading required package: tidyverse</code></pre>
</div>
<div class="cell-output cell-output-stderr">
<pre><code>Warning: package 'tidyverse' was built under R version 4.2.3</code></pre>
</div>
<div class="cell-output cell-output-stderr">
<pre><code>Warning: package 'ggplot2' was built under R version 4.2.3</code></pre>
</div>
<div class="cell-output cell-output-stderr">
<pre><code>Warning: package 'tidyr' was built under R version 4.2.3</code></pre>
</div>
<div class="cell-output cell-output-stderr">
<pre><code>Warning: package 'readr' was built under R version 4.2.3</code></pre>
</div>
<div class="cell-output cell-output-stderr">
<pre><code>Warning: package 'purrr' was built under R version 4.2.3</code></pre>
</div>
<div class="cell-output cell-output-stderr">
<pre><code>Warning: package 'dplyr' was built under R version 4.2.3</code></pre>
</div>
<div class="cell-output cell-output-stderr">
<pre><code>Warning: package 'stringr' was built under R version 4.2.3</code></pre>
</div>
<div class="cell-output cell-output-stderr">
<pre><code>Warning: package 'forcats' was built under R version 4.2.3</code></pre>
</div>
<div class="cell-output cell-output-stderr">
<pre><code>Warning: package 'lubridate' was built under R version 4.2.3</code></pre>
</div>
<div class="cell-output cell-output-stderr">
<pre><code>── Attaching core tidyverse packages ──────────────────────── tidyverse 2.0.0 ──
✔ dplyr     1.1.4     ✔ readr     2.1.5
✔ forcats   1.0.0     ✔ stringr   1.5.1
✔ ggplot2   3.5.1     ✔ tibble    3.2.1
✔ lubridate 1.9.3     ✔ tidyr     1.3.1
✔ purrr     1.0.2     </code></pre>
</div>
<div class="cell-output cell-output-stderr">
<pre><code>── Conflicts ────────────────────────────────────────── tidyverse_conflicts() ──
✖ dplyr::filter() masks stats::filter()
✖ dplyr::lag()    masks stats::lag()
ℹ Use the conflicted package (&lt;http://conflicted.r-lib.org/&gt;) to force all conflicts to become errors</code></pre>
</div>
<div class="sourceCode cell-code" id="cb21"><pre class="sourceCode r code-with-copy"><code class="sourceCode r"><span id="cb21-1"><a href="#cb21-1" aria-hidden="true" tabindex="-1"></a><span class="cf">if</span> (<span class="sc">!</span><span class="fu">require</span>(<span class="st">"leaflet"</span>, <span class="at">character.only =</span> <span class="cn">TRUE</span>)){ <span class="fu">install.packages</span>(<span class="st">"leaflet"</span>)}</span></code><button title="Copy to Clipboard" class="code-copy-button"><i class="bi"></i></button></pre></div>
<div class="cell-output cell-output-stderr">
<pre><code>Loading required package: leaflet</code></pre>
</div>
<div class="cell-output cell-output-stderr">
<pre><code>Warning: package 'leaflet' was built under R version 4.2.3</code></pre>
</div>
</div>
<div class="cell" data-hash="daily-log_cache/html/unnamed-chunk-5_433bdbbefda76dd9ccd1bdf2fd27b51b">
<div class="sourceCode cell-code" id="cb24"><pre class="sourceCode r code-with-copy"><code class="sourceCode r"><span id="cb24-1"><a href="#cb24-1" aria-hidden="true" tabindex="-1"></a><span class="do">## load libraries</span></span>
<span id="cb24-2"><a href="#cb24-2" aria-hidden="true" tabindex="-1"></a><span class="fu">library</span>(tidyverse)</span>
<span id="cb24-3"><a href="#cb24-3" aria-hidden="true" tabindex="-1"></a><span class="fu">library</span>(leaflet)</span></code><button title="Copy to Clipboard" class="code-copy-button"><i class="bi"></i></button></pre></div>
</div>
<div class="cell">
<div class="sourceCode cell-code" id="cb25"><pre class="sourceCode r code-with-copy"><code class="sourceCode r"><span id="cb25-1"><a href="#cb25-1" aria-hidden="true" tabindex="-1"></a>grey <span class="ot">&lt;-</span> <span class="fu">leaflet</span>() <span class="sc">%&gt;%</span> </span>
<span id="cb25-2"><a href="#cb25-2" aria-hidden="true" tabindex="-1"></a>   <span class="fu">addTiles</span>() <span class="sc">%&gt;%</span> </span>
<span id="cb25-3"><a href="#cb25-3" aria-hidden="true" tabindex="-1"></a>   <span class="fu">setView</span>( <span class="at">lng =</span> <span class="sc">-</span><span class="fl">122.8</span>, <span class="at">lat =</span> <span class="fl">47.9</span>, <span class="at">zoom =</span> <span class="dv">8</span> ) <span class="sc">%&gt;%</span> </span>
<span id="cb25-4"><a href="#cb25-4" aria-hidden="true" tabindex="-1"></a>   <span class="fu">addProviderTiles</span>(<span class="st">"Esri.WorldGrayCanvas"</span>)</span>
<span id="cb25-5"><a href="#cb25-5" aria-hidden="true" tabindex="-1"></a>grey</span></code><button title="Copy to Clipboard" class="code-copy-button"><i class="bi"></i></button></pre></div>
<div class="cell-output-display">
<div class="leaflet html-widget html-fill-item" id="htmlwidget-884c744460d1d86fb1d8" style="width:100%;height:464px;"></div>
<script type="application/json" data-for="htmlwidget-884c744460d1d86fb1d8">{"x":{"options":{"crs":{"crsClass":"L.CRS.EPSG3857","code":null,"proj4def":null,"projectedBounds":null,"options":{}}},"calls":[{"method":"addTiles","args":["https://{s}.tile.openstreetmap.org/{z}/{x}/{y}.png",null,null,{"minZoom":0,"maxZoom":18,"tileSize":256,"subdomains":"abc","errorTileUrl":"","tms":false,"noWrap":false,"zoomOffset":0,"zoomReverse":false,"opacity":1,"zIndex":1,"detectRetina":false,"attribution":"&copy; <a href=\"https://openstreetmap.org/copyright/\">OpenStreetMap<\/a>,  <a href=\"https://opendatacommons.org/licenses/odbl/\">ODbL<\/a>"}]},{"method":"addProviderTiles","args":["Esri.WorldGrayCanvas",null,null,{"errorTileUrl":"","noWrap":false,"detectRetina":false}]}],"setView":[[47.9,-122.8],8,[]]},"evals":[],"jsHooks":[]}</script>
</div>
</div>
<div class="cell">
<div class="sourceCode cell-code" id="cb26"><pre class="sourceCode r code-with-copy"><code class="sourceCode r"><span id="cb26-1"><a href="#cb26-1" aria-hidden="true" tabindex="-1"></a>terrain <span class="ot">&lt;-</span> <span class="fu">leaflet</span>() <span class="sc">%&gt;%</span> </span>
<span id="cb26-2"><a href="#cb26-2" aria-hidden="true" tabindex="-1"></a>   <span class="fu">addTiles</span>() <span class="sc">%&gt;%</span> </span>
<span id="cb26-3"><a href="#cb26-3" aria-hidden="true" tabindex="-1"></a>   <span class="fu">setView</span>( <span class="at">lng =</span> <span class="sc">-</span><span class="fl">122.8</span>, <span class="at">lat =</span> <span class="fl">47.9</span>, <span class="at">zoom =</span> <span class="dv">8</span> ) <span class="sc">%&gt;%</span> </span>
<span id="cb26-4"><a href="#cb26-4" aria-hidden="true" tabindex="-1"></a>   <span class="fu">addProviderTiles</span>(<span class="st">"Esri.WorldTerrain"</span>)</span>
<span id="cb26-5"><a href="#cb26-5" aria-hidden="true" tabindex="-1"></a>terrain</span></code><button title="Copy to Clipboard" class="code-copy-button"><i class="bi"></i></button></pre></div>
<div class="cell-output-display">
<div class="leaflet html-widget html-fill-item" id="htmlwidget-d41a82b49f2c8291edc2" style="width:100%;height:464px;"></div>
<script type="application/json" data-for="htmlwidget-d41a82b49f2c8291edc2">{"x":{"options":{"crs":{"crsClass":"L.CRS.EPSG3857","code":null,"proj4def":null,"projectedBounds":null,"options":{}}},"calls":[{"method":"addTiles","args":["https://{s}.tile.openstreetmap.org/{z}/{x}/{y}.png",null,null,{"minZoom":0,"maxZoom":18,"tileSize":256,"subdomains":"abc","errorTileUrl":"","tms":false,"noWrap":false,"zoomOffset":0,"zoomReverse":false,"opacity":1,"zIndex":1,"detectRetina":false,"attribution":"&copy; <a href=\"https://openstreetmap.org/copyright/\">OpenStreetMap<\/a>,  <a href=\"https://opendatacommons.org/licenses/odbl/\">ODbL<\/a>"}]},{"method":"addProviderTiles","args":["Esri.WorldTerrain",null,null,{"errorTileUrl":"","noWrap":false,"detectRetina":false}]}],"setView":[[47.9,-122.8],8,[]]},"evals":[],"jsHooks":[]}</script>
</div>
</div>
<div class="cell">
<div class="sourceCode cell-code" id="cb27"><pre class="sourceCode r code-with-copy"><code class="sourceCode r"><span id="cb27-1"><a href="#cb27-1" aria-hidden="true" tabindex="-1"></a>Stadia.AlidadeSmooth <span class="ot">&lt;-</span> <span class="fu">leaflet</span>() <span class="sc">%&gt;%</span> </span>
<span id="cb27-2"><a href="#cb27-2" aria-hidden="true" tabindex="-1"></a>   <span class="fu">addTiles</span>() <span class="sc">%&gt;%</span> </span>
<span id="cb27-3"><a href="#cb27-3" aria-hidden="true" tabindex="-1"></a>   <span class="fu">setView</span>( <span class="at">lng =</span> <span class="sc">-</span><span class="fl">122.8</span>, <span class="at">lat =</span> <span class="fl">47.9</span>, <span class="at">zoom =</span> <span class="dv">8</span> ) <span class="sc">%&gt;%</span> </span>
<span id="cb27-4"><a href="#cb27-4" aria-hidden="true" tabindex="-1"></a>   <span class="fu">addProviderTiles</span>(<span class="st">"Stadia.AlidadeSmooth"</span>)</span>
<span id="cb27-5"><a href="#cb27-5" aria-hidden="true" tabindex="-1"></a>Stadia.AlidadeSmooth</span></code><button title="Copy to Clipboard" class="code-copy-button"><i class="bi"></i></button></pre></div>
<div class="cell-output-display">
<div class="leaflet html-widget html-fill-item" id="htmlwidget-0d9e51ddc9d12734e1d7" style="width:100%;height:464px;"></div>
<script type="application/json" data-for="htmlwidget-0d9e51ddc9d12734e1d7">{"x":{"options":{"crs":{"crsClass":"L.CRS.EPSG3857","code":null,"proj4def":null,"projectedBounds":null,"options":{}}},"calls":[{"method":"addTiles","args":["https://{s}.tile.openstreetmap.org/{z}/{x}/{y}.png",null,null,{"minZoom":0,"maxZoom":18,"tileSize":256,"subdomains":"abc","errorTileUrl":"","tms":false,"noWrap":false,"zoomOffset":0,"zoomReverse":false,"opacity":1,"zIndex":1,"detectRetina":false,"attribution":"&copy; <a href=\"https://openstreetmap.org/copyright/\">OpenStreetMap<\/a>,  <a href=\"https://opendatacommons.org/licenses/odbl/\">ODbL<\/a>"}]},{"method":"addProviderTiles","args":["Stadia.AlidadeSmooth",null,null,{"errorTileUrl":"","noWrap":false,"detectRetina":false}]}],"setView":[[47.9,-122.8],8,[]]},"evals":[],"jsHooks":[]}</script>
</div>
</div>
<div class="cell">
<div class="sourceCode cell-code" id="cb28"><pre class="sourceCode r code-with-copy"><code class="sourceCode r"><span id="cb28-1"><a href="#cb28-1" aria-hidden="true" tabindex="-1"></a>Stadia.OSMBright <span class="ot">&lt;-</span> <span class="fu">leaflet</span>() <span class="sc">%&gt;%</span> </span>
<span id="cb28-2"><a href="#cb28-2" aria-hidden="true" tabindex="-1"></a>   <span class="fu">addTiles</span>() <span class="sc">%&gt;%</span> </span>
<span id="cb28-3"><a href="#cb28-3" aria-hidden="true" tabindex="-1"></a>   <span class="fu">setView</span>( <span class="at">lng =</span> <span class="sc">-</span><span class="fl">122.8</span>, <span class="at">lat =</span> <span class="fl">47.9</span>, <span class="at">zoom =</span> <span class="dv">8</span> ) <span class="sc">%&gt;%</span> </span>
<span id="cb28-4"><a href="#cb28-4" aria-hidden="true" tabindex="-1"></a>   <span class="fu">addProviderTiles</span>(<span class="st">"Stadia.OSMBright"</span>)</span>
<span id="cb28-5"><a href="#cb28-5" aria-hidden="true" tabindex="-1"></a>Stadia.OSMBright</span></code><button title="Copy to Clipboard" class="code-copy-button"><i class="bi"></i></button></pre></div>
<div class="cell-output-display">
<div class="leaflet html-widget html-fill-item" id="htmlwidget-27e01c2d147544d4e577" style="width:100%;height:464px;"></div>
<script type="application/json" data-for="htmlwidget-27e01c2d147544d4e577">{"x":{"options":{"crs":{"crsClass":"L.CRS.EPSG3857","code":null,"proj4def":null,"projectedBounds":null,"options":{}}},"calls":[{"method":"addTiles","args":["https://{s}.tile.openstreetmap.org/{z}/{x}/{y}.png",null,null,{"minZoom":0,"maxZoom":18,"tileSize":256,"subdomains":"abc","errorTileUrl":"","tms":false,"noWrap":false,"zoomOffset":0,"zoomReverse":false,"opacity":1,"zIndex":1,"detectRetina":false,"attribution":"&copy; <a href=\"https://openstreetmap.org/copyright/\">OpenStreetMap<\/a>,  <a href=\"https://opendatacommons.org/licenses/odbl/\">ODbL<\/a>"}]},{"method":"addProviderTiles","args":["Stadia.OSMBright",null,null,{"errorTileUrl":"","noWrap":false,"detectRetina":false}]}],"setView":[[47.9,-122.8],8,[]]},"evals":[],"jsHooks":[]}</script>
</div>
</div>
<div class="cell">
<div class="sourceCode cell-code" id="cb29"><pre class="sourceCode r code-with-copy"><code class="sourceCode r"><span id="cb29-1"><a href="#cb29-1" aria-hidden="true" tabindex="-1"></a>Jawg.Lagoon <span class="ot">&lt;-</span> <span class="fu">leaflet</span>() <span class="sc">%&gt;%</span> </span>
<span id="cb29-2"><a href="#cb29-2" aria-hidden="true" tabindex="-1"></a>   <span class="fu">addTiles</span>() <span class="sc">%&gt;%</span> </span>
<span id="cb29-3"><a href="#cb29-3" aria-hidden="true" tabindex="-1"></a>   <span class="fu">setView</span>( <span class="at">lng =</span> <span class="sc">-</span><span class="fl">122.8</span>, <span class="at">lat =</span> <span class="fl">47.9</span>, <span class="at">zoom =</span> <span class="dv">8</span> ) <span class="sc">%&gt;%</span> </span>
<span id="cb29-4"><a href="#cb29-4" aria-hidden="true" tabindex="-1"></a>   <span class="fu">addProviderTiles</span>(<span class="st">"Jawg.Lagoon"</span>)</span>
<span id="cb29-5"><a href="#cb29-5" aria-hidden="true" tabindex="-1"></a>Jawg.Lagoon</span></code><button title="Copy to Clipboard" class="code-copy-button"><i class="bi"></i></button></pre></div>
<div class="cell-output-display">
<div class="leaflet html-widget html-fill-item" id="htmlwidget-75ae8ed32e757a3f7c2d" style="width:100%;height:464px;"></div>
<script type="application/json" data-for="htmlwidget-75ae8ed32e757a3f7c2d">{"x":{"options":{"crs":{"crsClass":"L.CRS.EPSG3857","code":null,"proj4def":null,"projectedBounds":null,"options":{}}},"calls":[{"method":"addTiles","args":["https://{s}.tile.openstreetmap.org/{z}/{x}/{y}.png",null,null,{"minZoom":0,"maxZoom":18,"tileSize":256,"subdomains":"abc","errorTileUrl":"","tms":false,"noWrap":false,"zoomOffset":0,"zoomReverse":false,"opacity":1,"zIndex":1,"detectRetina":false,"attribution":"&copy; <a href=\"https://openstreetmap.org/copyright/\">OpenStreetMap<\/a>,  <a href=\"https://opendatacommons.org/licenses/odbl/\">ODbL<\/a>"}]},{"method":"addProviderTiles","args":["Jawg.Lagoon",null,null,{"errorTileUrl":"","noWrap":false,"detectRetina":false}]}],"setView":[[47.9,-122.8],8,[]]},"evals":[],"jsHooks":[]}</script>
</div>
</div>
<div class="cell">
<div class="sourceCode cell-code" id="cb30"><pre class="sourceCode r code-with-copy"><code class="sourceCode r"><span id="cb30-1"><a href="#cb30-1" aria-hidden="true" tabindex="-1"></a>Stadia.StamenWatercolor <span class="ot">&lt;-</span> <span class="fu">leaflet</span>() <span class="sc">%&gt;%</span> </span>
<span id="cb30-2"><a href="#cb30-2" aria-hidden="true" tabindex="-1"></a>   <span class="fu">addTiles</span>() <span class="sc">%&gt;%</span> </span>
<span id="cb30-3"><a href="#cb30-3" aria-hidden="true" tabindex="-1"></a>   <span class="fu">setView</span>( <span class="at">lng =</span> <span class="sc">-</span><span class="fl">122.8</span>, <span class="at">lat =</span> <span class="fl">47.9</span>, <span class="at">zoom =</span> <span class="dv">8</span> ) <span class="sc">%&gt;%</span> </span>
<span id="cb30-4"><a href="#cb30-4" aria-hidden="true" tabindex="-1"></a>   <span class="fu">addProviderTiles</span>(<span class="st">"Stadia.StamenWatercolor"</span>)</span>
<span id="cb30-5"><a href="#cb30-5" aria-hidden="true" tabindex="-1"></a>Stadia.StamenWatercolor</span></code><button title="Copy to Clipboard" class="code-copy-button"><i class="bi"></i></button></pre></div>
<div class="cell-output-display">
<div class="leaflet html-widget html-fill-item" id="htmlwidget-148aa71818bd85d80641" style="width:100%;height:464px;"></div>
<script type="application/json" data-for="htmlwidget-148aa71818bd85d80641">{"x":{"options":{"crs":{"crsClass":"L.CRS.EPSG3857","code":null,"proj4def":null,"projectedBounds":null,"options":{}}},"calls":[{"method":"addTiles","args":["https://{s}.tile.openstreetmap.org/{z}/{x}/{y}.png",null,null,{"minZoom":0,"maxZoom":18,"tileSize":256,"subdomains":"abc","errorTileUrl":"","tms":false,"noWrap":false,"zoomOffset":0,"zoomReverse":false,"opacity":1,"zIndex":1,"detectRetina":false,"attribution":"&copy; <a href=\"https://openstreetmap.org/copyright/\">OpenStreetMap<\/a>,  <a href=\"https://opendatacommons.org/licenses/odbl/\">ODbL<\/a>"}]},{"method":"addProviderTiles","args":["Stadia.StamenWatercolor",null,null,{"errorTileUrl":"","noWrap":false,"detectRetina":false}]}],"setView":[[47.9,-122.8],8,[]]},"evals":[],"jsHooks":[]}</script>
</div>
</div>
<ul>
<li></li>
<li><p>emailed Hannah at Seattle Aquarium about seawater pickup for Friday FEB 23 from 10-4</p></li>
<li><p>reserved SAFS Truck for seawater pickup</p></li>
</ul>
</section>
<section id="feb-2024-2" class="level4" data-number="4.0.0.3">
<h4 data-number="4.0.0.3" class="anchored" data-anchor-id="feb-2024-2"><span class="header-section-number">4.0.0.3</span> 20-FEB-2024</h4>
<ul>
<li><p>Downloaded pyroscience FireStingO2 Pyro Oxygen Logger</p>
<ul>
<li><p>First tried on my computer Owl, USB was not recognizing, so we’re trying it on QuelpoToo</p></li>
<li><p>The USB connection doesn’t connect to QuelpoToo either (is it the cable?)</p></li>
</ul></li>
</ul>
</section>
<section id="feb-2024-3" class="level4" data-number="4.0.0.4">
<h4 data-number="4.0.0.4" class="anchored" data-anchor-id="feb-2024-3"><span class="header-section-number">4.0.0.4</span> <strong>02-FEB-2024</strong></h4>
<ul>
<li><p>Follow-up with UW office of veterans affairs about debt</p></li>
<li><p>Lab meeting notes:</p>
<ul>
<li>move shaker table to lab 242 hood</li>
<li>move nitrogen needle lipid extractor to methanol/chloroform cabinet in 242</li>
<li>ok to buy organizational bins for PBB, keep receipts &amp; get reimbursed</li>
<li>respirometry chamber battery operation problematic… reduced flow, battery drain over time (30mins); Nuria left a plug-in chamber somewhere in PBB</li>
<li></li>
</ul></li>
<li><p>Writing your resume for federal job webinar</p>
<ul>
<li><p>schedule A: disability accommodation</p></li>
<li><p>Federal hiring resume, there is no page limit</p></li>
<li><p>The resume gets you to the hiring manager, show in the resume that you qualify before it goes to a hiring manager</p></li>
<li><p>hiring managers in Federal hiring do not click on links (github, linkedIn, etc.) they won’t be viewed…</p></li>
<li><p>‘Your resume is the movie trailer…’ Highlight the level of complexity of your work</p></li>
<li><p>Itemize how how manage something, on top of listing your role… you have to explain what you did in that role. The role ‘president of xx’ doesn’t translate to the person reviewing the resume knowing what you can do… leave nothing to the imagination. Be explicit.</p></li>
<li><p>Spoken presentations: how many presentations you have given, what &amp; when</p></li>
<li><p>The first person that looks at your resume is a human resources specialist, not a scientist!</p></li>
<li><p>show that you are an ‘innovator’ and that when you see a problem, you take initiative to fix it.</p></li>
<li><p>where you go, people are impacted positively, there is recognition, things change or improve, things get better, you save money, you meet deadlines, you bring about conflict resolution.</p></li>
<li><p>the cover letter is not used to qualify you, ONLY THE RESUME!</p></li>
<li><p>Vet team to assist navigating federal hiring process: fedshirevets@opm.gov</p></li>
<li><p>You get shuttled into qualified, well-qualified, best-qualified. Qualifying doesn’t mean that you get referred to the hiring manager… you can qualify but not get referred to the hiring manager. You can be referred, but not get an interview.</p></li>
<li><p>wrp workforce recruitment program</p></li>
</ul></li>
</ul>
</section>
<section id="feb-2024-4" class="level4" data-number="4.0.0.5">
<h4 data-number="4.0.0.5" class="anchored" data-anchor-id="feb-2024-4"><span class="header-section-number">4.0.0.5</span> 08-FEB-2024</h4>
<ul>
<li><p>attended Roberts Lab meeting</p>
<ul>
<li>Will be presenting experiment design &amp; proposal on Feb 22nd</li>
</ul></li>
<li><p>worked on dropbox-uploader app in raven server to try and get data from dropbox directly to raven *not working</p></li>
</ul>
</section>
<section id="feb-2024-5" class="level4" data-number="4.0.0.6">
<h4 data-number="4.0.0.6" class="anchored" data-anchor-id="feb-2024-5"><span class="header-section-number">4.0.0.6</span> 07-FEB-2024</h4>
<ul>
<li><p>Identified anemones from Alki Beach have <em>Symbiodinium muscatinei</em></p></li>
<li><p>Identified anemones from Teddy Bear Cove have <em>Symbiodinium muscatinei</em></p></li>
</ul>
</section>
<section id="feb-2024-6" class="level4" data-number="4.0.0.7">
<h4 data-number="4.0.0.7" class="anchored" data-anchor-id="feb-2024-6"><span class="header-section-number">4.0.0.7</span> 05-FEB-2024</h4>
<ul>
<li><p>Lab meeting cancelled</p></li>
<li><p>emailed Francesco Saliu asking about sending their lab SPME fibers for analysis</p>
<ul>
<li>still no package or protocol, starting to think about plan B/C/D…etc</li>
</ul></li>
<li><p>found rnoaa package and zenodo by Wendell Raymond https://zenodo.org/records/6555076 from paper “<a href="https://esajournals.onlinelibrary.wiley.com/doi/10.1002/ecy.3798">Assessment of the impacts of an unprecedented heatwave on intertidal shellfish of the Salish Sea</a>” recommended by Bob Oxborrow</p>
<ul>
<li>explored rnoaa package</li>
</ul></li>
<li><p>13:30 - shadow fish 427 lab on photosynthesis and respiration with Miranda</p></li>
</ul>
<p>Goal for the week:</p>
<ul>
<li>Get PAM photosynthesis initial data</li>
</ul>
</section>
<section id="january-2024" class="level2" data-number="4.1">
<h2 data-number="4.1" class="anchored" data-anchor-id="january-2024"><span class="header-section-number">4.1</span> January 2024</h2>
</section>
<section id="december-2023" class="level2" data-number="4.2">
<h2 data-number="4.2" class="anchored" data-anchor-id="december-2023"><span class="header-section-number">4.2</span> December 2023</h2>
</section>
<section id="november-2023" class="level2" data-number="4.3">
<h2 data-number="4.3" class="anchored" data-anchor-id="november-2023"><span class="header-section-number">4.3</span> November 2023</h2>
<section id="nov-2023" class="level4" data-number="4.3.0.1">
<h4 data-number="4.3.0.1" class="anchored" data-anchor-id="nov-2023"><span class="header-section-number">4.3.0.1</span> 27-NOV-2023</h4>
<ul>
<li>Lab Meeting:
<ul>
<li>Requested to order cultured abalone sea urchins</li>
<li>Reminded JPG about meeting with Alison Gardell Tues Nov 28th at 1:30PM via Zoom</li>
<li>JPG requested I put together slides for meeting with Alison Gardell</li>
</ul></li>
<li>Got seawater from Seattle Aquarium</li>
<li>Met with Kip and got them setup with safety trainings</li>
<li>Must order sea urchins on Mon Dec 11th for a Mon Dec 18th/Tues Dec 19th delivery</li>
</ul>
</section>
<section id="nov-2023-1" class="level4" data-number="4.3.0.2">
<h4 data-number="4.3.0.2" class="anchored" data-anchor-id="nov-2023-1"><span class="header-section-number">4.3.0.2</span> <strong>21-NOV-2023</strong></h4>
<ul>
<li><p><a href="../committee-2/index.qmd">2nd Master’s Thesis Committee Meeting</a></p>
<ul>
<li>Trying to come up with a solution to lack of solvent control.. how to move forward</li>
</ul></li>
</ul>
</section>
<section id="nov-2023-2" class="level4" data-number="4.3.0.3">
<h4 data-number="4.3.0.3" class="anchored" data-anchor-id="nov-2023-2"><span class="header-section-number">4.3.0.3</span> 11-NOV-2023</h4>
<ul>
<li>WSN day 4</li>
</ul>
</section>
<section id="nov-2023-3" class="level4" data-number="4.3.0.4">
<h4 data-number="4.3.0.4" class="anchored" data-anchor-id="nov-2023-3"><span class="header-section-number">4.3.0.4</span> 10-NOV-2023</h4>
<ul>
<li>WSN day 3</li>
</ul>
</section>
<section id="nov-2023-4" class="level4" data-number="4.3.0.5">
<h4 data-number="4.3.0.5" class="anchored" data-anchor-id="nov-2023-4"><span class="header-section-number">4.3.0.5</span> 09-NOV 2023</h4>
<ul>
<li>WSN day 2</li>
</ul>
</section>
<section id="nov-2023-5" class="level4" data-number="4.3.0.6">
<h4 data-number="4.3.0.6" class="anchored" data-anchor-id="nov-2023-5"><span class="header-section-number">4.3.0.6</span> 08-NOV-2023</h4>
<ul>
<li>WSN day 1</li>
</ul>
</section>
<section id="nov-2023-6" class="level4" data-number="4.3.0.7">
<h4 data-number="4.3.0.7" class="anchored" data-anchor-id="nov-2023-6"><span class="header-section-number">4.3.0.7</span> 05-NOV-2023</h4>
<ul>
<li><p>Undoing large file commit on coral-pae-temp for ~2hrs 🤬, tried to just undo the one commit, but couldn’t get it to work. Had to execute <code>git reset –hard origin/main</code></p></li>
<li><p>Added the following to the <code>git/hooks/pre-commit</code> file:</p>
<pre><code>#!/bin/bash

echo "automatically ignoring large files"
find . -size 5M | sed 's|^\./||g' &gt;&gt; .gitignore
cat .gitignore | sort | uniq &gt; .gitignore

git diff --exit-code .gitignore
exit_status=$?
if [ $exit_status -eq 1 ]
then
    set +e
    for i in `cat .gitignore`
    do
    set +e
        git rm --cached $i
    done

    git add .gitignore
    git commit .gitignore --no-verify -m"ignoring large files"

    echo "ignored new large files"
fi</code></pre></li>
</ul>
</section>
<section id="nov-2023-7" class="level4" data-number="4.3.0.8">
<h4 data-number="4.3.0.8" class="anchored" data-anchor-id="nov-2023-7"><span class="header-section-number">4.3.0.8</span> 03-NOV-2023</h4>
<ul>
<li><p>Finished extraction, nanodrop &amp; qubit for 12 samples</p>
<ul>
<li><p>6MH1a</p></li>
<li><p>10OH1a - greenish, bad! no DNA/RNA, degraded!</p></li>
<li><p>6PA2a</p></li>
<li><p>5HH1a</p></li>
<li><p>5HA1a</p></li>
<li><p>9OA2a</p></li>
<li><p>7HA1a</p></li>
<li><p>3CA1a</p></li>
<li><p>8CA1a</p></li>
<li><p>2OA1a</p></li>
<li><p>2MH1a</p></li>
<li><p>5CA2a</p></li>
</ul></li>
</ul>
</section>
<section id="nov-2023-8" class="level4" data-number="4.3.0.9">
<h4 data-number="4.3.0.9" class="anchored" data-anchor-id="nov-2023-8"><span class="header-section-number">4.3.0.9</span> 02-NOV-2023</h4>
<ul>
<li><p>Started extraction of 12 samples, paused after adding 350uL of DNA/RNA Lysis Buffer to 350uL of PK-Digested sample supernatent, and frozen in -80 freezer:</p>
<ul>
<li>6MH1a</li>
<li>10OH1a - green</li>
<li>6PA2a</li>
<li>5HH1a</li>
<li>5HA1a</li>
<li>9OA2a</li>
<li>7HA1a</li>
<li>3CA1a</li>
<li>8CA1a</li>
<li>2OA1a</li>
<li>2MH1a</li>
<li>5CA2a - green</li>
</ul></li>
<li><p>Finished qubit of 10 samples:</p>
<ul>
<li><p>1MH2a</p></li>
<li><p>2LA1a</p></li>
<li><p>1MA1a</p></li>
<li><p>1OH2a</p></li>
<li><p>6PH2a</p></li>
<li><p>10LH1a</p></li>
<li><p>9LA2a</p></li>
<li><p>8CH2a</p></li>
<li><p>8OA1a</p></li>
<li><p>5OH1a</p></li>
</ul></li>
</ul>
</section>
<section id="nov-2023-9" class="level4" data-number="4.3.0.10">
<h4 data-number="4.3.0.10" class="anchored" data-anchor-id="nov-2023-9"><span class="header-section-number">4.3.0.10</span> 01-NOV-2023</h4>
<ul>
<li><p>Finished extraction &amp; nanodrop of 10 samples:</p>
<ul>
<li>1MH2a</li>
<li>2LA1a</li>
<li>1MA1a</li>
<li>1OH2a</li>
<li>6PH2a</li>
<li>10LH1a</li>
<li>9LA2a</li>
<li>8CH2a</li>
<li>8OA1a</li>
<li>5OH1a</li>
</ul></li>
<li><p>GSS Presentation Design Workshop with Tim Essington</p>
<ul>
<li><p>Presentations should have ONE THING they are communicating: ‘Here’s a nugget’</p></li>
<li><p>One Word Theme</p></li>
<li><p>One Sentence Summary</p></li>
<li><p>Add motion to a few slides to break things up visually</p></li>
<li><p>Use a blank slide and pause at a key pivot in your story</p></li>
<li><p>build the slide iteratively when you have a complex figure</p></li>
<li><p>story arc:</p>
<ul>
<li><p>ordinary world</p></li>
<li><p>challenge</p></li>
<li><p>series of actions to overcome challenge</p></li>
<li><p>resolution</p></li>
<li><p>new world</p></li>
</ul></li>
<li><p>‘Lean into your ’buts’</p></li>
<li><p>leave silence + space between story arcs</p></li>
</ul></li>
</ul>
</section>
</section>
<section id="october-2023" class="level2 page-columns page-full" data-number="4.4">
<h2 data-number="4.4" class="anchored" data-anchor-id="october-2023"><span class="header-section-number">4.4</span> October 2023</h2>
<section id="oct-2023" class="level4" data-number="4.4.0.1">
<h4 data-number="4.4.0.1" class="anchored" data-anchor-id="oct-2023"><span class="header-section-number">4.4.0.1</span> <strong>27-OCT-2023</strong></h4>
<ul>
<li><p>QA/QC’d samples from 26-OCT-2023</p></li>
<li><p>Met with Kip &amp; moved experiment setup to PBB</p></li>
<li><p>Collected glycerol from Jon Bakker at Horticultural Center Merill Hall</p></li>
</ul>
</section>
<section id="oct-2023-1" class="level4" data-number="4.4.0.2">
<h4 data-number="4.4.0.2" class="anchored" data-anchor-id="oct-2023-1"><span class="header-section-number">4.4.0.2</span> <strong>26-OCT-2023</strong></h4>
<ul>
<li><p>extracted samples:</p>
<ul>
<li><p>6LH1a</p></li>
<li><p>8LA2a</p></li>
<li><p>10CH2a</p></li>
<li><p>5CH2a</p></li>
<li><p>4HA2a</p></li>
<li><p>7OA2a</p></li>
<li><p>2MA2a</p></li>
<li><p>3LH1a</p></li>
<li><p>5LA1a</p></li>
<li><p>8MH2a</p></li>
</ul></li>
</ul>
</section>
<section id="oct-2023-2" class="level4" data-number="4.4.0.3">
<h4 data-number="4.4.0.3" class="anchored" data-anchor-id="oct-2023-2"><span class="header-section-number">4.4.0.3</span> 24-OCT-2023</h4>
<ul>
<li>working on WSN presentation!</li>
</ul>
</section>
<section id="oct-2023-3" class="level4" data-number="4.4.0.4">
<h4 data-number="4.4.0.4" class="anchored" data-anchor-id="oct-2023-3"><span class="header-section-number">4.4.0.4</span> 23-OCT-2023</h4>
<ul>
<li><p>JPG 1on1</p></li>
<li><p>Bought Amtrak tickets to WSN (refundable)</p></li>
</ul>
</section>
<section id="oct-2023-4" class="level4 page-columns page-full" data-number="4.4.0.5">
<h4 data-number="4.4.0.5" class="anchored" data-anchor-id="oct-2023-4"><span class="header-section-number">4.4.0.5</span> 16-OCT-2023</h4>
<ul>
<li>attended lab meeting via Zoom</li>
</ul>
<p>“Total RNA was extracted using the Duet DNA/RNA Miniprep Plus Kit (Zymo Research, Irvine, CA, USA, Cat #D7003) with modifications to the manufacturer’s sample preparation steps, as described here. Sterile clippers were used to remove an approximately 1-cm diameter region from the coral tissue. The tissue clipping was placed in 500&nbsp;μL of DNA/RNA shield (Zymo Cat #R1100-50) with 0.5&nbsp;mm glass beads (Fisher Scientific Cat #15-340-152) and homogenized by vortexing for 1&nbsp;minute. The supernatant (450&nbsp;μl) was transferred to a new tube and centrifuged at 9000&nbsp;rcf for 3&nbsp;min to remove any debris. The supernatant (300&nbsp;μl) was mixed with 30&nbsp;μl of Proteinase K (Zymo Cat #D3001-2-20) digestion buffer and 15&nbsp;μl of Proteinase K in a new tube using a brief vortex step. Equal volume of Zymo lysis buffer (Zymo Cat #D7001-1-200) was added (345&nbsp;μl) and extraction was completed as outlined in the manufacturer’s protocol. RNA concentration was quantified using the Qubit RNA Broad Range kit (Invitrogen Cat #Q10211) and integrity was assayed with an Agilent TapeStation 4200 system and the RNA Integrity Number (RIN) values ranged from 7.0–8.9. RNA libraries were prepared using the Zymo-Seq RiboFree Total RNA Library Kit (Zymo Cat #R3000) according to the manufacturer’s instructions, starting with 125&nbsp;ng of RNA as input. Prepared libraries were sent to Zymo Research Corporation for sequencing using the Illumina HiSeq for paired end sequencing (2&nbsp;×&nbsp;100&nbsp;base pairs [bp]).” - Wong &amp; Putnam 2022 <span class="citation" data-cites="wong2022">(<a href="#ref-wong2022" role="doc-biblioref">Wong and Putnam 2022</a>)</span></p>
<div class="no-row-height column-margin column-container"><div id="ref-wong2022" class="csl-entry" role="listitem">
Wong, Kevin H., and Hollie M. Putnam. 2022. <span>“The Genome of the Mustard Hill Coral, Porites Astreoides.”</span> <em>Gigabyte</em> 2022 (July): 1–12. <a href="https://doi.org/10.46471/gigabyte.65">https://doi.org/10.46471/gigabyte.65</a>.
</div></div></section>
<section id="oct-2023-5" class="level4" data-number="4.4.0.6">
<h4 data-number="4.4.0.6" class="anchored" data-anchor-id="oct-2023-5"><span class="header-section-number">4.4.0.6</span> 11-OCT-2023</h4>
<ul>
<li><p>Attended <a href="https://isbscience.org/">ISB</a> workshop day 1</p>
<ul>
<li><p>Gibbons-Lab github repo: <a href="https://github.com/Gibbons-Lab/isb_course_2023/tree/main" class="uri">https://github.com/Gibbons-Lab/isb_course_2023/tree/main</a></p></li>
<li><p>Course presentation slides: <a href="https://gibbons-lab.github.io/isb_course_2023/16S/" class="uri">https://gibbons-lab.github.io/isb_course_2023/16S/</a></p></li>
<li><p>Multinomial Naive Bayes <code>qiime feature-classifier classify-sklearn</code> trained on V4 region of the 16S marker gene</p></li>
<li><p>center log ratio transform abundance counts</p></li>
<li><p>recommended papers:</p>
<ul>
<li><p><a href="https://www.nature.com/articles/s41467-022-28034-z"><strong>Microbiome differential abundance methods produce different results across 38 datasets</strong></a></p></li>
<li><p><a href="https://www.futurelearn.com/info/courses/exploring-the-landscape-of-antibiotic-resistance-in-microbiomes/0/steps/272960#:~:text=PCoA%20and%20NMDS%20methods%20take,to%20fit%20a%20normal%20distribution"><strong>Beta diversity (or Ordination) analysis</strong></a>“PCoA and NMDS methods take the distance matrix as an input and are sensitive to distance method used. The PCoA maximises the linear correlation between samples, wherein NMDS maximises the rank-order correlation between samples. Additionally, in case of NMDS, data is not required to fit a normal distribution.” This link has some good visualizations: <a href="https://www.davidzeleny.net/anadat-r/doku.php/en:pcoa_nmds" class="uri">https://www.davidzeleny.net/anadat-r/doku.php/en:pcoa_nmds</a></p></li>
<li><p>Why is the mean being used instead of the median or the modal? Bacteria distribution is not a normal distribution across populations…. This is a log ratio of (relative abundance/geometric mean). It’s a common method for transforming compositional data. For sequencing data normalization does not mean making anything normally distributed but rather putting samples on the same scale… Here’s a review on the topic: <a href="https://www.nature.com/articles/s41522-020-00160-w" class="uri">https://www.nature.com/articles/s41522-020-00160-w</a></p>
<p><img src="https://slack-imgs.com/?c=1&amp;o1=wi32.he32.si&amp;url=https%253A%252F%252Fwww.nature.com%252Fstatic%252Fimages%252Ffavicons%252Fnature%252Fapple-touch-icon-f39cb19454.png" class="img-fluid" alt="Nature"><strong>Nature</strong></p>
<p><a href="https://www.nature.com/articles/s41522-020-00160-w"><strong>Analysis of microbial compositions: a review of normalization and differential abundance analysis</strong></a></p></li>
</ul></li>
</ul></li>
</ul>
</section>
<section id="oct-2023-6" class="level4" data-number="4.4.0.7">
<h4 data-number="4.4.0.7" class="anchored" data-anchor-id="oct-2023-6"><span class="header-section-number">4.4.0.7</span> 10-OCT-2023</h4>
<ul>
<li>Bought WSN flights</li>
<li>Sent list of 123 samples to UW mim_c for formal quote</li>
<li>filled out GI Bill enrollment verification</li>
<li>updated all extraction notebook posts from handwritten notes</li>
</ul>
</section>
<section id="oct-2023-7" class="level4" data-number="4.4.0.8">
<h4 data-number="4.4.0.8" class="anchored" data-anchor-id="oct-2023-7"><span class="header-section-number">4.4.0.8</span> 03-OCT-2023</h4>
<ul>
<li>Submitted abstract to WSN, registered for conference!</li>
</ul>
</section>
<section id="oct-2023-8" class="level4" data-number="4.4.0.9">
<h4 data-number="4.4.0.9" class="anchored" data-anchor-id="oct-2023-8"><span class="header-section-number">4.4.0.9</span> 02-OCT-2023</h4>
<ul>
<li><p>Abstratc accepted to SICB for Jan 3rd!</p></li>
<li><div>
<blockquote class="blockquote">
<p>Your abstract Corals in a hot plastic ocean: How heat and phthalates shift a Montipora capitata coral microbiome has been accepted for the 2024 Annual Meeting in Seattle, WA,&nbsp;on&nbsp;2-6 January 2024. Your abstract id is 1230.<br>
<br>
You presentation is in the&nbsp;Response to anthropogenic environmental changes Concurrent session on&nbsp;Wednesday,&nbsp;January 03, 2024.</p>
<p>The online program can be found <a href="https://urldefense.com/v3/__https://mandrillapp.com/track/click/30774040/www.xcdsystem.com?p=eyJzIjoiVHg1NGFGcEZRZmMyYWdNaEdPOTREWjRMcGQwIiwidiI6MSwicCI6IntcInVcIjozMDc3NDA0MCxcInZcIjoxLFwidXJsXCI6XCJodHRwczpcXFwvXFxcL3d3dy54Y2RzeXN0ZW0uY29tXFxcL3NpY2JcXFwvcHJvZ3JhbVxcXC9JMUtyMjN0XFxcL2luZGV4LmNmbT9wZ2lkPTEzNDImUnVuUmVtb3ZlU2Vzc2lvbkZpbHRlcj0xXCIsXCJpZFwiOlwiZWZiNjZmNTg1MGQ2NDFjMzk1NjU5ZjQ2MjJjMmMzZTZcIixcInVybF9pZHNcIjpbXCI4MDZmOTNlZTNiZmY5ZTM3ODhhMDM1MmE4YzhkODkzMmM1OTFlOTgxXCJdfSJ9__;!!K-Hz7m0Vt54!h-p1pLv7irLQW3sPow2nFi6Q2BxoJRfyoVmE8uN9VbspkSR__iihdpN1iPfG-49elEJFsqnVnF5IW-lY$">HERE</a>. If you need to make a change to your abstract or cancel it you MUST fill out this <a href="https://urldefense.com/v3/__https://mandrillapp.com/track/click/30774040/docs.google.com?p=eyJzIjoiZ0VMSHdvazI5OGJDRWMyMV81NEtlcHZuMWgwIiwidiI6MSwicCI6IntcInVcIjozMDc3NDA0MCxcInZcIjoxLFwidXJsXCI6XCJodHRwczpcXFwvXFxcL2RvY3MuZ29vZ2xlLmNvbVxcXC9mb3Jtc1xcXC9kXFxcL2VcXFwvMUZBSXBRTFNkbmRxdTBwRERoLUhySVBZUXdIZUp6S29fYlRldzJsb2xVeEdSV1Q3STZ1ZWwwZndcXFwvdmlld2Zvcm1cIixcImlkXCI6XCJlZmI2NmY1ODUwZDY0MWMzOTU2NTlmNDYyMmMyYzNlNlwiLFwidXJsX2lkc1wiOltcIjNhOTc5YWRhMTZmM2RiNDliODQ4NDdlOTQ2ZmI2NDk5MzllZDY3MDNcIl19In0__;!!K-Hz7m0Vt54!h-p1pLv7irLQW3sPow2nFi6Q2BxoJRfyoVmE8uN9VbspkSR__iihdpN1iPfG-49elEJFsqnVnJHbFgyn$">FORM</a>.&nbsp; This is the only way changes and cancellations will be accepted.<br>
<br>
Information regarding presentations at the SICB meeting can be found <a href="https://urldefense.com/v3/__https://mandrillapp.com/track/click/30774040/www.xcdsystem.com?p=eyJzIjoiTjJ3UFUxQ25vcmR1RWZGN093bUtaZXpBU3l3IiwidiI6MSwicCI6IntcInVcIjozMDc3NDA0MCxcInZcIjoxLFwidXJsXCI6XCJodHRwczpcXFwvXFxcL3d3dy54Y2RzeXN0ZW0uY29tXFxcL3NpY2JcXFwvcHJvZ3JhbVxcXC9JMUtyMjN0XFxcL2luZGV4LmNmbT9wZ2lkPTEzNjJcIixcImlkXCI6XCJlZmI2NmY1ODUwZDY0MWMzOTU2NTlmNDYyMmMyYzNlNlwiLFwidXJsX2lkc1wiOltcIjgwNmY5M2VlM2JmZjllMzc4OGEwMzUyYThjOGQ4OTMyYzU5MWU5ODFcIl19In0__;!!K-Hz7m0Vt54!h-p1pLv7irLQW3sPow2nFi6Q2BxoJRfyoVmE8uN9VbspkSR__iihdpN1iPfG-49elEJFsqnVnMuFu7ui$">HERE</a>.</p>
<p>Please note that all presenters must register for the meeting by the presenter registration deadline of 1 November 2023, otherwise your talk may be cancelled. Click <a href="https://urldefense.com/v3/__https://mandrillapp.com/track/click/30774040/www.xcdsystem.com?p=eyJzIjoiaVNESXlTUjZ6anQ1Mi1FaS1Fa1dBZG50VjRJIiwidiI6MSwicCI6IntcInVcIjozMDc3NDA0MCxcInZcIjoxLFwidXJsXCI6XCJodHRwczpcXFwvXFxcL3d3dy54Y2RzeXN0ZW0uY29tXFxcL3NpY2JcXFwvcHJvZ3JhbVxcXC81WDlPSWJVXFxcL2luZGV4LmNmbT9wZ2lkPTQ0OFwiLFwiaWRcIjpcImVmYjY2ZjU4NTBkNjQxYzM5NTY1OWY0NjIyYzJjM2U2XCIsXCJ1cmxfaWRzXCI6W1wiNzY1MmM2ZTg0NzcxMDQ3MWFiZmNiZDQ0MGE1OGQ2NTFkNTg5MTgyNFwiXX0ifQ__;!!K-Hz7m0Vt54!h-p1pLv7irLQW3sPow2nFi6Q2BxoJRfyoVmE8uN9VbspkSR__iihdpN1iPfG-49elEJFsqnVnACQAtql$">HERE</a> to register. Please visit the meeting web site for complete information on registering and conference activities. If you have registration questions email <a href="mailto:MScarbrough@burkinc.com">registration@sicb.org</a>, for meeting questions email <a href="mailto:lstrong@burkinc.com">lstrong@burkinc.com</a>.</p>
</blockquote>
</div></li>
</ul>
</section>
</section>
<section id="september-2023" class="level2" data-number="4.5">
<h2 data-number="4.5" class="anchored" data-anchor-id="september-2023"><span class="header-section-number">4.5</span> September 2023</h2>
<section id="sep-2023" class="level4" data-number="4.5.0.1">
<h4 data-number="4.5.0.1" class="anchored" data-anchor-id="sep-2023"><span class="header-section-number">4.5.0.1</span> 20-SEP-2023</h4>
<ul>
<li><a href="https://theconversation.com/the-importance-of-shining-a-light-on-hidden-toxic-histories-211657?utm_medium=email&amp;utm_campaign=Daily%20newsletter%20September%2014%202023&amp;utm_content=Daily%20newsletter%20September%2014%202023+CID_1c20b166b5da888ab34925d477cebf8d&amp;utm_source=campaign_monitor_us&amp;utm_term=the%20artists%20and%20activists%20who%20are%20calling%20attention%20to%20toxic%20legacies">Hidden Toxic Histories</a></li>
</ul>
</section>
<section id="sep-2023-1" class="level4" data-number="4.5.0.2">
<h4 data-number="4.5.0.2" class="anchored" data-anchor-id="sep-2023-1"><span class="header-section-number">4.5.0.2</span> 05-SEP-2023</h4>
<ul>
<li><p>WSN newsletter says abstracts close Sep 29th 2023</p></li>
<li><p>5 min lightning talk or 15 minute talk</p></li>
<li><p><a href="https://www.wsn-online.org/presenter-guidelines/">WSN presenter guidelines</a></p></li>
<li><p>Abstract Guidelines</p></li>
</ul>
<!-- -->
<pre><code>The abstract should briefly introduce the project, the goals, the results, and the significance

The presentation title should have no more than 130 characters max (including spaces) in ALL UPPERCASE

The abstract should have no more than 1600 characters max (including spaces); use standard HTML tags for special formatting (e.g., \&lt;b\&gt;**bold text**\&lt;/b\&gt;, \&lt;i\&gt;*italic text*\&lt;/i\&gt;, etc.)</code></pre>
</section>
<section id="sep-2023-2" class="level4" data-number="4.5.0.3">
<h4 data-number="4.5.0.3" class="anchored" data-anchor-id="sep-2023-2"><span class="header-section-number">4.5.0.3</span> 01-SEP-2023</h4>
<ul>
<li><p>crashed my bike and badly injured my hands… can’t do lab work 😭</p></li>
<li><p>met with UW mim_c Steven Salipantes</p>
<ul>
<li><p>I need to send him a sample list for a final quote, but should be $2K for ~100-120 samples (cost of sequencing lane + reagents) .. averages to ~$20/sample, much better than Mr.&nbsp;DNA ~$85/sample, plus no shipping charges.</p></li>
<li><p>bring all 100-120 samples together</p></li>
<li><p>1.5mL microcentrifuge tubes</p></li>
<li><p>50uL volume , 20ng/uL quantity concentration</p></li>
</ul></li>
</ul>
</section>
</section>
<section id="august-2023" class="level2" data-number="4.6">
<h2 data-number="4.6" class="anchored" data-anchor-id="august-2023"><span class="header-section-number">4.6</span> August 2023</h2>
<section id="aug-2023" class="level4" data-number="4.6.0.1">
<h4 data-number="4.6.0.1" class="anchored" data-anchor-id="aug-2023"><span class="header-section-number">4.6.0.1</span> 31-AUG-2023</h4>
<ul>
<li><p>purchased additional kits, qubit assays, tubes, etc. to spend down Hall Grant</p></li>
<li><p>received Qubit and UV-C Sterilizer!</p></li>
<li><p>extracted:</p>
<ul>
<li><p>mock</p></li>
<li><p>blank</p></li>
<li><p>4-LA2a</p></li>
<li><p>1-HH2a</p></li>
<li><p>2-HH2a</p></li>
<li><p>3-HA2a</p></li>
<li><p>3-MA2a</p></li>
<li><p>1-Ha2a</p></li>
<li><p>4-MA2a</p></li>
<li><p>3-HA2a</p></li>
</ul></li>
</ul>
</section>
<section id="aug-2023-1" class="level4" data-number="4.6.0.2">
<h4 data-number="4.6.0.2" class="anchored" data-anchor-id="aug-2023-1"><span class="header-section-number">4.6.0.2</span> 29-AUG-2023</h4>
<ul>
<li><p>submitted SICB abstract</p></li>
<li><p>submitted Kip’s capstone proposal to Greg Jensen</p></li>
</ul>
</section>
<section id="aug-2023-2" class="level4" data-number="4.6.0.3">
<h4 data-number="4.6.0.3" class="anchored" data-anchor-id="aug-2023-2"><span class="header-section-number">4.6.0.3</span> 28-AUG-2023</h4>
<ul>
<li><p>Look into the <a href="https://coast.noaa.gov/nerrs/research/davidson-fellowship.html">Margaret A. Davidson Graduate Fellowship</a></p></li>
<li><p>Wrote abstract for <a href="https://sicb.org/">SICB</a> , Seattle Jan 2-6 … Deadline is Aug 29th!</p></li>
<li><p>Met with UW mim_c Kyle Asfahl</p></li>
<li><p>balanced Hall budget, have ~$5K left to spend</p></li>
<li><p>Michael Shriever reset medusaFTIR and downloaded Nanodrop software to it</p></li>
<li><p>Take Jeremy’s samples out of freezer to thaw at 4PM</p></li>
</ul>
</section>
<section id="aug-2023-3" class="level4" data-number="4.6.0.4">
<h4 data-number="4.6.0.4" class="anchored" data-anchor-id="aug-2023-3"><span class="header-section-number">4.6.0.4</span> 24-AUG-2023</h4>
<ul>
<li>Trial with 2 ambient samples of modified extraction protocol and RNA Clean &amp; Concentrator. I did 2 samples because I had a kit that had 2 preps left over. Most of the time spent is in lab bench sterilizing, prepping, labelling, getting ice! Which takes literally a 30 minute hike across campus… But I started at 930AM and finished with Qubit in the FTR Roberts Lab by 4PM. With 2 samples! Extractions take ALL DAY.</li>
</ul>
</section>
<section id="aug-2023-4" class="level4" data-number="4.6.0.5">
<h4 data-number="4.6.0.5" class="anchored" data-anchor-id="aug-2023-4"><span class="header-section-number">4.6.0.5</span> 17-AUG-2023</h4>
<ul>
<li><p>Are we sure we want to try TriZol for RNA? It isn’t a phenol-free kit like the Quick DNA/RNA Miniprep, and we have RNA, we just need to purify it. Tomorrow lets run all eluted RNA through RNA Clean &amp; Concentrator, and check it on Nanodrop! LFG.</p></li>
<li><p><strong>THE BLANK HAD ‘TOO LOW’ DNA IN QUBIT!</strong></p></li>
<li><p>Organized extraction notes (handwritten) for extractions 1-8</p></li>
<li><p>Reserved 1 King Bed in Hotel Block for WSN</p></li>
<li><p>Applied for WSN Student Travel Grant</p>
<blockquote class="blockquote">
<p><strong>How will attending the WSN meeting help you advance your career as a naturalist and/or scientist (75 words max)?</strong></p>
<p>Attending WSN would be my first opportunity to share my Masters thesis with the scientific community. As an undergrad, I had a bad first conference experience where I was ridiculed for not having results in a methods poster. I hope to attend WSN because its reputation as a welcoming environment for early-career scientists is the kind of scientific community I wish to be a part of and perpetuate in my own career.</p>
<p><strong>What are you working on to improve diversity, equity and inclusion in your community (75 words max)? (Examples may relate to personal identify, interest, experience, etc.)</strong></p>
<p>I recently took part of the Identity, Belonging, and Inquiry in Science Program, and received special training to mentor students from diverse backgrounds. My mentee worked hard to complete research that she will present at the upcoming SACNAS conference, and I couldn’t be more proud. The training gave me confidence to take on another mentee, this time a Capstone student. Mentorship is a key part of DEI efforts that I find extremely rewarding.</p>
</blockquote></li>
</ul>
</section>
<section id="aug-2023-5" class="level4" data-number="4.6.0.6">
<h4 data-number="4.6.0.6" class="anchored" data-anchor-id="aug-2023-5"><span class="header-section-number">4.6.0.6</span> 16-AUG-2023</h4>
<ul>
<li><p>asked JPG to buy benchtop UV Sterilizer for me &amp; Callum, got approval! Purchased by Kat same day :D</p></li>
<li><p>NMDS (non-metric multidimensional scaling) ordination plots</p></li>
<li><p>walked with Callum, plan to ask JPG to buy Qubit too!</p></li>
<li><p>Made some taxonomy bar plots and heatmap dendrograms… still working to understand what they mean but they don’t look like there is separation between treatments. Haven’t gotten any pvalues yet from the composition analyses…</p></li>
<li><p>Wondering if I should break up the Notebook into diversity &amp; composition? It’s getting really long and overwhelming with all the analyses moshed together</p></li>
<li><p>Still can’t <code>quarto render</code> in Jupyter Lab on raven server, need to figure out how to get those interactive jupyter plots in my notebook!</p></li>
</ul>
</section>
<section id="aug-2023-6" class="level4" data-number="4.6.0.7">
<h4 data-number="4.6.0.7" class="anchored" data-anchor-id="aug-2023-6"><span class="header-section-number">4.6.0.7</span> 15-AUG-2023</h4>
<ul>
<li><p>met with Dr.&nbsp;Amy VanCise about microbiome analysis:</p>
<ul>
<li><p>Look into work by WHOI Amy Apprill, Laura Weber, Cynthia Becker</p></li>
<li><p>Focus analysis on <strong>community composition</strong> over <strong>diversity</strong> statistics</p>
<blockquote class="blockquote">
<p>different sampling depths across different samples … subsample the counts without replacement so that each sample has the same number of features</p>
</blockquote></li>
<li><p>Differential abundance analysis on microbial community is similar to differential gene expression analysis!</p></li>
<li><p>Sampling depth is normalizing between samples</p></li>
<li><p>Look into getting a UV Sterilizer</p></li>
<li><p>Do the analysis without blank, mock, &amp; env samples included… get a clean look at only treatments</p></li>
<li><p>Look into R Vegan package</p></li>
<li><p>Phred score: spatial, 30 (or 25) is good to keep</p></li>
</ul></li>
</ul>
</section>
<section id="aug-2023-7" class="level4" data-number="4.6.0.8">
<h4 data-number="4.6.0.8" class="anchored" data-anchor-id="aug-2023-7"><span class="header-section-number">4.6.0.8</span> 03-AUG-2023</h4>
<ul>
<li><p>Received RNA samples back from UT GSAF</p></li>
<li><p>Used PCR Inhibitor Removal Kit on five of the samples (see RNA Woes post <a href="https://sarahtanja.github.io/quarto-blog/posts/rna-woes/#:~:text=A260%20value.%E2%80%9D-,Zymo%20OneStep%20PCR%20Inhibitor%20Removal%20Kit,-I%20first%20tried">here</a>) :</p>
<ul>
<li><p>1-CA2a</p></li>
<li><p>1-CH2a</p></li>
<li><p>1-Ea</p></li>
<li><p>1-PA2a</p></li>
<li><p>1-PH1a</p></li>
</ul></li>
<li><p>PCR Inhibitor Removal Kit said to add 50-200uL of RNA, but I only could add 30uL. I took the 5 samples from the 96-well plate and augmented them with all the leftover diluted normalized RNA.</p></li>
<li><p>Nanodrop note: blank should result in a spectrum that varies no more than 0.050A in the 1mm path. For this batch I used the Roberts Lab Nanodrop and the blank was -0.009/-0.008 10mm path.</p></li>
</ul>
</section>
</section>
<section id="july-2023" class="level2" data-number="4.7">
<h2 data-number="4.7" class="anchored" data-anchor-id="july-2023"><span class="header-section-number">4.7</span> July 2023</h2>
<section id="jul-2023" class="level4" data-number="4.7.0.1">
<h4 data-number="4.7.0.1" class="anchored" data-anchor-id="jul-2023"><span class="header-section-number">4.7.0.1</span> 20-JUL-2023</h4>
<ul>
<li><p><a href="https://forum.qiime2.org/t/tutorial-integrating-qiime2-and-r-for-data-visualization-and-analysis-using-qiime2r/4121">Integrating QIIME2 and R for data viz</a></p></li>
<li><p>RStudio&gt;Project Options&gt; Python&gt; Python interpreter</p></li>
<li><p><a href="https://support.posit.co/hc/en-us/articles/1500007929061-Using-Python-with-the-RStudio-IDE">Using Python with the RStudio IDE</a></p></li>
</ul>
</section>
<section id="jul-2023-1" class="level4" data-number="4.7.0.2">
<h4 data-number="4.7.0.2" class="anchored" data-anchor-id="jul-2023-1"><span class="header-section-number">4.7.0.2</span> 19-JUL-2023</h4>
<ul>
<li><p>Emailed Jackie about getting another P1000</p></li>
<li><p>Helped Bob Oxborrow with pipettes for stable isotope sample prep</p></li>
<li><p>Downloaded QIIME2 to Roberts Lab Raven server for coral-pae-temp repo</p></li>
<li><p>Shadowed Miranda to learn how to acid wash glassware in 10% HCl:</p>
<ul>
<li><p>Scrub and let soak 24hrs in Alconite bath</p></li>
<li><p>Rinse 3X in Ultrapure water &amp; shift to 10% HCl bath, let soak for 24hrs</p></li>
<li><p>Remove &amp; rinse 3X in Ultrapure water and let sit in Ultrapure water for 24hrs</p></li>
<li><p>Remove &amp; let air dry, cover opening with foil</p></li>
</ul></li>
</ul>
</section>
</section>
<section id="june-2023" class="level2" data-number="4.8">
<h2 data-number="4.8" class="anchored" data-anchor-id="june-2023"><span class="header-section-number">4.8</span> June 2023</h2>
<section id="jun-2023" class="level4" data-number="4.8.0.1">
<h4 data-number="4.8.0.1" class="anchored" data-anchor-id="jun-2023"><span class="header-section-number">4.8.0.1</span> 07-JUN-2023</h4>
<ul>
<li>Shipped DNA to Mr.&nbsp;DNA and RNA to UT GSAF!</li>
</ul>
</section>
<section id="jun-2023-1" class="level4" data-number="4.8.0.2">
<h4 data-number="4.8.0.2" class="anchored" data-anchor-id="jun-2023-1"><span class="header-section-number">4.8.0.2</span> 06-JUN-2023</h4>
<ul>
<li>Normalized all 20 RNA samples</li>
</ul>
</section>
<section id="jun-2023-2" class="level4" data-number="4.8.0.3">
<h4 data-number="4.8.0.3" class="anchored" data-anchor-id="jun-2023-2"><span class="header-section-number">4.8.0.3</span> 05-JUN-2023</h4>
<ul>
<li><p>Re-normalized 2-CA2a DNA and successfully got it to 19.5, 19.2 :D</p></li>
<li><p>Worked all day to normalize RNA with mixed results. In the first batch, 7 out of 20 samples were ‘off’ of the target concentration of 25ng/uL by enough to make me want to redo them (one was way high at 230ng/uL, others were too low at ~10ng/uL)</p></li>
<li><p>I first re-ran the qubit (2nd qubit run) and got really poor results, with all the rerun samples too-low, (even the one that was previously 230ng/uL!?)</p></li>
<li><p>I then re-normalized the samples and ran them on qubit again (3rd qubit run). In this run 4-PH1b hit the target, the rest were still too low</p></li>
<li><p>I re-normalized the samples for a third time (with some increasing concentration modifications) and ran them on qubit for a 4th time. In this run 4-PA2a hit the target, the rest were still too low, and 2-PH2a even got worse despite increasing concentration.</p></li>
<li><p>Ended the day unsatisfied with the following samples too low for normalization despite 4 attempts:</p>
<ul>
<li><p>2-PH2a - too low</p></li>
<li><p>2-CA2a - 15.0, 15.4</p></li>
<li><p>1-PA2a - 10.2, 10.0</p></li>
<li><p>1-PH1a - 17.6, 17.6</p></li>
<li><p>3-CH2a - 15.0, 14.6</p></li>
<li><p>4-CA1a - 14.0, 13.0</p></li>
</ul></li>
</ul>
</section>
<section id="jun-2023-3" class="level4" data-number="4.8.0.4">
<h4 data-number="4.8.0.4" class="anchored" data-anchor-id="jun-2023-3"><span class="header-section-number">4.8.0.4</span> 02-JUN-2023</h4>
<ul>
<li>Normalized DNA (still have RNA left to do)</li>
<li>Tested normalized DNA on Qubit BR DNA assay, results overall were good but a smidge lower than my goal</li>
<li>blank + microbial standard extraction</li>
</ul>
</section>
<section id="jun-2023-4" class="level4" data-number="4.8.0.5">
<h4 data-number="4.8.0.5" class="anchored" data-anchor-id="jun-2023-4"><span class="header-section-number">4.8.0.5</span> 01-JUN-2023</h4>
<ul>
<li>Presented Bioinformatics <a href="https://rpubs.com/sarah_tanja/1048844">final project for FISH 546</a></li>
<li>Attended GSR Meeting</li>
<li>Attended Seminar/TGIT</li>
</ul>
</section>
</section>
<section id="may-2023" class="level2" data-number="4.9">
<h2 data-number="4.9" class="anchored" data-anchor-id="may-2023"><span class="header-section-number">4.9</span> May 2023</h2>
<section id="may-2023-1" class="level4" data-number="4.9.0.1">
<h4 data-number="4.9.0.1" class="anchored" data-anchor-id="may-2023-1"><span class="header-section-number">4.9.0.1</span> 31-MAY-2023</h4>
<ul>
<li>wrote <code>dilute</code> function to normalize RNA &amp; DNA in <a href="https://github.com/sarahtanja/coral-DNA-RNA-lab-extractions/blob/main/code/03-normalizing.qmd">coral-DNA-RNA-lab-extractions</a> GitHub repo:</li>
</ul>
<div class="cell">
<div class="sourceCode cell-code" id="cb33"><pre class="sourceCode r code-with-copy"><code class="sourceCode r"><span id="cb33-1"><a href="#cb33-1" aria-hidden="true" tabindex="-1"></a>dilute <span class="ot">&lt;-</span> <span class="cf">function</span>(C1, C2) {</span>
<span id="cb33-2"><a href="#cb33-2" aria-hidden="true" tabindex="-1"></a>  <span class="co"># Initialize variables</span></span>
<span id="cb33-3"><a href="#cb33-3" aria-hidden="true" tabindex="-1"></a>  n <span class="ot">&lt;-</span> <span class="fu">length</span>(C1)</span>
<span id="cb33-4"><a href="#cb33-4" aria-hidden="true" tabindex="-1"></a>  C1 <span class="ot">&lt;-</span> C1</span>
<span id="cb33-5"><a href="#cb33-5" aria-hidden="true" tabindex="-1"></a>  V1 <span class="ot">&lt;-</span> <span class="fu">numeric</span>(n)  <span class="co"># Initialize V1 vector</span></span>
<span id="cb33-6"><a href="#cb33-6" aria-hidden="true" tabindex="-1"></a>  V2 <span class="ot">&lt;-</span> <span class="fu">numeric</span>(n)  <span class="co"># Initialize V2 vector</span></span>
<span id="cb33-7"><a href="#cb33-7" aria-hidden="true" tabindex="-1"></a>  </span>
<span id="cb33-8"><a href="#cb33-8" aria-hidden="true" tabindex="-1"></a>  <span class="co"># Loop through rows</span></span>
<span id="cb33-9"><a href="#cb33-9" aria-hidden="true" tabindex="-1"></a>  <span class="cf">for</span> (i <span class="cf">in</span> <span class="fu">seq_along</span>(C1)) {</span>
<span id="cb33-10"><a href="#cb33-10" aria-hidden="true" tabindex="-1"></a>    <span class="co"># Check for missing data</span></span>
<span id="cb33-11"><a href="#cb33-11" aria-hidden="true" tabindex="-1"></a>    <span class="cf">if</span> (C1[i] <span class="sc">&lt;=</span> <span class="dv">0</span>) {</span>
<span id="cb33-12"><a href="#cb33-12" aria-hidden="true" tabindex="-1"></a>      <span class="fu">cat</span>(<span class="st">"Skipping row "</span>, i, <span class="st">": Invalid data</span><span class="sc">\n</span><span class="st">"</span>)</span>
<span id="cb33-13"><a href="#cb33-13" aria-hidden="true" tabindex="-1"></a>      <span class="cf">next</span></span>
<span id="cb33-14"><a href="#cb33-14" aria-hidden="true" tabindex="-1"></a>    }</span>
<span id="cb33-15"><a href="#cb33-15" aria-hidden="true" tabindex="-1"></a>    </span>
<span id="cb33-16"><a href="#cb33-16" aria-hidden="true" tabindex="-1"></a>    <span class="co"># Initialize V1 and V2 for the current row</span></span>
<span id="cb33-17"><a href="#cb33-17" aria-hidden="true" tabindex="-1"></a>    V1_row <span class="ot">&lt;-</span> <span class="dv">1</span></span>
<span id="cb33-18"><a href="#cb33-18" aria-hidden="true" tabindex="-1"></a>    V2_row <span class="ot">&lt;-</span> (C1[i] <span class="sc">*</span> V1_row) <span class="sc">/</span> C2</span>
<span id="cb33-19"><a href="#cb33-19" aria-hidden="true" tabindex="-1"></a>    </span>
<span id="cb33-20"><a href="#cb33-20" aria-hidden="true" tabindex="-1"></a>    <span class="co"># Loop until conditions are satisfied</span></span>
<span id="cb33-21"><a href="#cb33-21" aria-hidden="true" tabindex="-1"></a>    <span class="cf">while</span> (<span class="sc">!</span><span class="fu">is.na</span>(V1_row) <span class="sc">&amp;&amp;</span> <span class="sc">!</span><span class="fu">is.na</span>(V2_row) <span class="sc">&amp;&amp;</span> (V1_row <span class="sc">&lt;</span> <span class="dv">1</span> <span class="sc">||</span> V2_row <span class="sc">&lt;</span> <span class="dv">50</span>)) {</span>
<span id="cb33-22"><a href="#cb33-22" aria-hidden="true" tabindex="-1"></a>      V1_row <span class="ot">&lt;-</span> V1_row <span class="sc">+</span> <span class="dv">1</span></span>
<span id="cb33-23"><a href="#cb33-23" aria-hidden="true" tabindex="-1"></a>      V2_row <span class="ot">&lt;-</span> (C1[i] <span class="sc">*</span> V1_row) <span class="sc">/</span> C2</span>
<span id="cb33-24"><a href="#cb33-24" aria-hidden="true" tabindex="-1"></a>      </span>
<span id="cb33-25"><a href="#cb33-25" aria-hidden="true" tabindex="-1"></a>      <span class="co"># Break the loop if V1 exceeds a threshold value</span></span>
<span id="cb33-26"><a href="#cb33-26" aria-hidden="true" tabindex="-1"></a>      <span class="cf">if</span>(V1_row <span class="sc">&gt;</span> <span class="dv">49</span>) {</span>
<span id="cb33-27"><a href="#cb33-27" aria-hidden="true" tabindex="-1"></a>        <span class="fu">cat</span>(<span class="st">"Skipping row "</span>, i, <span class="st">": No valid solution found</span><span class="sc">\n</span><span class="st">"</span>)</span>
<span id="cb33-28"><a href="#cb33-28" aria-hidden="true" tabindex="-1"></a>        <span class="cf">next</span></span>
<span id="cb33-29"><a href="#cb33-29" aria-hidden="true" tabindex="-1"></a>      }</span>
<span id="cb33-30"><a href="#cb33-30" aria-hidden="true" tabindex="-1"></a>    }</span>
<span id="cb33-31"><a href="#cb33-31" aria-hidden="true" tabindex="-1"></a>    </span>
<span id="cb33-32"><a href="#cb33-32" aria-hidden="true" tabindex="-1"></a>    <span class="co"># Store optimized V1 and V2 values in the corresponding vectors</span></span>
<span id="cb33-33"><a href="#cb33-33" aria-hidden="true" tabindex="-1"></a>    V1[i] <span class="ot">&lt;-</span> V1_row</span>
<span id="cb33-34"><a href="#cb33-34" aria-hidden="true" tabindex="-1"></a>    V2[i] <span class="ot">&lt;-</span> V2_row</span>
<span id="cb33-35"><a href="#cb33-35" aria-hidden="true" tabindex="-1"></a>  }</span>
<span id="cb33-36"><a href="#cb33-36" aria-hidden="true" tabindex="-1"></a>  </span>
<span id="cb33-37"><a href="#cb33-37" aria-hidden="true" tabindex="-1"></a><span class="co"># Create a data frame with C1 column</span></span>
<span id="cb33-38"><a href="#cb33-38" aria-hidden="true" tabindex="-1"></a>  result <span class="ot">&lt;-</span> <span class="fu">data.frame</span>(<span class="at">C1 =</span> C1)</span>
<span id="cb33-39"><a href="#cb33-39" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb33-40"><a href="#cb33-40" aria-hidden="true" tabindex="-1"></a><span class="co"># Add V1 and V2 columns to the result data frame</span></span>
<span id="cb33-41"><a href="#cb33-41" aria-hidden="true" tabindex="-1"></a>  result<span class="sc">$</span>V1 <span class="ot">&lt;-</span> V1</span>
<span id="cb33-42"><a href="#cb33-42" aria-hidden="true" tabindex="-1"></a>  result<span class="sc">$</span>V2 <span class="ot">&lt;-</span> V2</span>
<span id="cb33-43"><a href="#cb33-43" aria-hidden="true" tabindex="-1"></a>  result<span class="sc">$</span>C2 <span class="ot">&lt;-</span> C1<span class="sc">*</span>V1<span class="sc">/</span>V2</span>
<span id="cb33-44"><a href="#cb33-44" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb33-45"><a href="#cb33-45" aria-hidden="true" tabindex="-1"></a>  <span class="fu">return</span>(result)</span>
<span id="cb33-46"><a href="#cb33-46" aria-hidden="true" tabindex="-1"></a>}</span></code><button title="Copy to Clipboard" class="code-copy-button"><i class="bi"></i></button></pre></div>
</div>
<ul>
<li>SAFS Spring Picnic!</li>
</ul>
</section>
<section id="may-2023-2" class="level4" data-number="4.9.0.2">
<h4 data-number="4.9.0.2" class="anchored" data-anchor-id="may-2023-2"><span class="header-section-number">4.9.0.2</span> 29-MAY-2023</h4>
<ul>
<li>Worked on Bioinformatics Final Project, RNA expression heat map with Gene Ontology Terms (Protein Functions)</li>
<li>IBIS Celebration of <a href="https://norahess.quarto.pub/uw-lab-note-book/">Nora’s Blog</a>!</li>
</ul>
</section>
<section id="may-2023-3" class="level4" data-number="4.9.0.3">
<h4 data-number="4.9.0.3" class="anchored" data-anchor-id="may-2023-3"><span class="header-section-number">4.9.0.3</span> 25-MAY-2023</h4>
<ul>
<li>Hosted Dr.&nbsp;Nicola Smith as the GSIS</li>
</ul>
</section>
<section id="may-2023-4" class="level4" data-number="4.9.0.4">
<h4 data-number="4.9.0.4" class="anchored" data-anchor-id="may-2023-4"><span class="header-section-number">4.9.0.4</span> 24-MAY-2023</h4>
<ul>
<li><p>University of Texas at Austin <a href="https://wikis.utexas.edu/display/GSAF/Home+Page">Genomic Sequencing and Analysis Facility (GSAF)</a>:</p>
<ul>
<li><p><a href="https://wikis.utexas.edu/display/GSAF/Sample+Input+Guidelines">Sample Input Guidelines</a></p>
<blockquote class="blockquote">
<p>TagSeq: 25ng/ul concentration in greater than or equal to 25ul volume (submitting RNA as low as 10ng/ul or as high as 100ng/ul is acceptable BUT ALL samples MUST be submitted at the same or very similar concentrations)&nbsp;</p>
</blockquote></li>
<li><p><a href="https://lockbox-604.my.site.com/s/login/">GSAF Requester Portal Login</a></p></li>
<li><p><a href="https://platform.login.illumina.com/platform-services-manager/?rURL=https://basespace.login.illumina.com/platform-home/&amp;redirectMethod=GET&amp;clientId=ps-home#/">Illumina BaseSpace Login</a></p></li>
</ul></li>
</ul>
</section>
<section id="may-2023-5" class="level4" data-number="4.9.0.5">
<h4 data-number="4.9.0.5" class="anchored" data-anchor-id="may-2023-5"><span class="header-section-number">4.9.0.5</span> 23-MAY-2023</h4>
<ul>
<li>completed UW driver safety training</li>
<li>met with Steven Roberts, confirmed that no blank/standard is used for RNA-seq</li>
<li>emailed with Jesse Zaneveld, learned about Zymo mock <a href="https://www.zymoresearch.com/collections/zymobiomics-microbial-community-standards/products/zymobiomics-microbial-community-standard">microbial community standard</a> and <a href="https://www.zymoresearch.com/collections/zymobiomics-microbial-community-standards/products/zymobiomics-microbial-community-dna-standard">microbial community DNA standards</a> to be used as positive controls for the extraction (community standard cells) and the PCR library prep and sequencing (community standard DNA) to assess bias in each process.</li>
<li>Put both Zymo standards on purcahsing log, emailed Kat and she said she would buy them tonight with the pcard</li>
<li>Met with Nora, she has made an <a href="https://norahess.quarto.pub/uw-lab-note-book/">awesome quarto blog</a> for the IBIS celebration to showcase what she has learned. We also talked a bit about stats and need to follow up with what to do when your data is not normally distributed (moving from paired T test to Wilcoxon paired-sample T test)</li>
<li>Tried using <code>quarto publish gh-pages</code> command to publish blog</li>
</ul>
</section>
<section id="may-2023-6" class="level4" data-number="4.9.0.6">
<h4 data-number="4.9.0.6" class="anchored" data-anchor-id="may-2023-6"><span class="header-section-number">4.9.0.6</span> 19-MAY-2023</h4>
<ul>
<li><p>Ran Qubit on ex7 samples</p></li>
<li><p>Extraction #7 (the redo) on samples:</p>
<ul>
<li>3-Eb</li>
<li>3-PH1b</li>
<li>2-PA2b</li>
<li>2-CH1b</li>
<li>4-PH1b</li>
<li>3-CA1b</li>
</ul></li>
<li><p>found out how to use quarto to do <a href="https://quarto.org/docs/blog/posts/2022-02-17-advanced-layout/#:~:text=.-,Margin%20References,-Footnotes%20and%20the">Margin References</a>!</p></li>
<li><p>found cool article on <a href="https://themockup.blog/posts/2021-08-27-displaying-verbatim-code-chunks-in-xaringan-presentations/">Verbatim code chunks</a> in R (want to render the code chunk options and show what engine is being used?)</p></li>
<li><p>dry-ice at biochem stockroom is low, have to check that there will be a replenishment next week so I have enough dry ice to ship samples!</p></li>
</ul>
</section>
<section id="may-2023-7" class="level4" data-number="4.9.0.7">
<h4 data-number="4.9.0.7" class="anchored" data-anchor-id="may-2023-7"><span class="header-section-number">4.9.0.7</span> 18-MAY-2023</h4>
<ul>
<li>wrote script to normalize eluted RNA/DNA</li>
</ul>
</section>
<section id="may-2023-8" class="level4" data-number="4.9.0.8">
<h4 data-number="4.9.0.8" class="anchored" data-anchor-id="may-2023-8"><span class="header-section-number">4.9.0.8</span> 04-MAY-2023</h4>
<ul>
<li>received Qubit DNA Broad Range (BR) Assay Kit!</li>
<li>Gotta get P2 DNA/RNA free tips!</li>
<li>ran qubit on DNA &amp; RNA, got great results!</li>
</ul>
</section>
</section>
<section id="april-2023" class="level2" data-number="4.10">
<h2 data-number="4.10" class="anchored" data-anchor-id="april-2023"><span class="header-section-number">4.10</span> April 2023</h2>
<section id="apr-2023" class="level4" data-number="4.10.0.1">
<h4 data-number="4.10.0.1" class="anchored" data-anchor-id="apr-2023"><span class="header-section-number">4.10.0.1</span> 25-APR-2023</h4>
<ul>
<li>sorted pipettes with Callum</li>
<li>submitted Bioinformatics Assignment</li>
<li>ordered Qubit DNA BR assay kit</li>
<li>ordered film sealing plate roller</li>
</ul>
</section>
<section id="apr-2023-1" class="level4" data-number="4.10.0.2">
<h4 data-number="4.10.0.2" class="anchored" data-anchor-id="apr-2023-1"><span class="header-section-number">4.10.0.2</span> 24-APR-2023</h4>
<ul>
<li><p>sent draft letter for NAH experiential learning fund to JPG on Monday April 24th</p></li>
<li><p>found <a href="https://microbe.med.umich.edu/sites/default/files/downloads/2021_ConsolidatedShippingGuide_0.pdf">96-well PCR shipping guide</a></p></li>
<li><p>emailed Mr.DNA about 20 16S samples</p></li>
<li><p>read up on NanoDrop vs.&nbsp;Qubit</p></li>
<li><p>reached out to Sam White about Ubuntu/Windows dual-boot computer in lab 236, which is connected to the Thermo NanoDrop 2000c Sam shared:</p>
<blockquote class="blockquote">
<p>FYI, if you’re solely using the NanoDrop for quantification (i.e.&nbsp;not worried about 260/280), then I’d highly recommend just relying on the Qubit, with both DNA and RNA. The reason for this is that the NanoDrop will almost always over-“calculate” your sample concentrations (both RNA and DNA). The reason for this is that the NanoDrop is a spectrophotometer and measures light absorbance. With almost all DNA/RNA isolations, there will be residual DNA or RNA in your sample. The NanoDrop cannot distinguish between the two molecules, because they’re both made up of the same material; nucleotides. Nucleotides, regardless of their of their origin will absorb light at an average wavelength of 260nm.Since the Qubit assays use molecule-specific dyes, the dye will only bind to DNA or only to RNA. The dyes will not bind to any carryover of other types of molecules in the sample, so the accuracy (and sensitivity) is much, much higher than the NanoDrop.</p>
</blockquote></li>
</ul>
</section>
<section id="apr-2023-2" class="level4" data-number="4.10.0.3">
<h4 data-number="4.10.0.3" class="anchored" data-anchor-id="apr-2023-2"><span class="header-section-number">4.10.0.3</span> 21-APR-2023</h4>
<ul>
<li><p>met with JPG April 1on1</p>
<ul>
<li>talked about travel from Sept.&nbsp;12 - Oct 9th, missing first 2 weeks of Autumn ’23 quarter, focusing that quarter on writing &amp; analysis</li>
<li>GOAL: have all sequences (16S &amp; Transcriptomes) by September 1st 2023</li>
<li>Initial 20 samples extracted &amp; sent for sequencing before FHL</li>
<li>2nd committee meeting to be held once initial 20 samples are run through 1st analysis for 16S &amp; DGE, get advice on strategy for selecting remaining samples</li>
<li>Follow-up samples extracted &amp; sent for sequencing in July/August</li>
<li>Send draft letter for NAH experiential learning fund to JPG on Monday April 24th</li>
</ul></li>
<li><p>Initial test round 2 samples extracted!!!</p>
<ul>
<li><p>3-CA1a</p></li>
<li><p>3-Ea</p>
<ul>
<li>Sampes selected from <a href="https://github.com/sarahtanja/coral-DNA-RNA-lab-extractions/blob/main/rand-sample-processing.Rmd">extr1</a> using a random number generator</li>
<li>details can be found in the <a href="https://github.com/sarahtanja/coral-DNA-RNA-lab-extractions">coral-DNA-RNA-lab-extractions</a> repository</li>
</ul></li>
</ul></li>
</ul>
</section>
<section id="apr-2023-3" class="level4" data-number="4.10.0.4">
<h4 data-number="4.10.0.4" class="anchored" data-anchor-id="apr-2023-3"><span class="header-section-number">4.10.0.4</span> 20-APR-2023</h4>
<ul>
<li>downloaded fastq files for ‘fake data’ to continue script for bioinformatics workflow</li>
<li>Received Mortexer! Whoo!</li>
<li>Received cryo-gloves</li>
<li>got 8lbs of dry-ice with NH</li>
<li>re-organized cryo-vials to wax boxes, with all of the ‘a’ samples in 2 boxes labelled with green lab tape and ‘b’ samples in 2 boxes labelled with blue tape with NH</li>
<li>setup lab bench for dry run dna/rna extraction with NH</li>
<li>NH setup mortexer</li>
<li>submitted 2-year extended warranty for mortexer s/n 22040448 via benchmark scientific in JPG’s name</li>
<li>placed signage on lab 236 to lint roll and lab coat (because I forgot to and that was embarassing)</li>
<li>submitted purchase for mortexer attachment head that will homogenize 12 1.5-2mL tubes at a time horizontal orientation (even better because closer to original lysis protocol for Zymo) Sadly, I should have looked for something like this earlier, but I saw it on the mortexer instruction manual when it was delivered and realized that was what I needed</li>
</ul>
</section>
<section id="apr-2023-4" class="level4" data-number="4.10.0.5">
<h4 data-number="4.10.0.5" class="anchored" data-anchor-id="apr-2023-4"><span class="header-section-number">4.10.0.5</span> 19-APR-2023</h4>
<ul>
<li>continued draft application for ONeill-Quinn</li>
<li>continued editing extraction protocol</li>
<li>expected delivery of mortexer by end of day</li>
</ul>
</section>
<section id="apr-2023-5" class="level4" data-number="4.10.0.6">
<h4 data-number="4.10.0.6" class="anchored" data-anchor-id="apr-2023-5"><span class="header-section-number">4.10.0.6</span> 18-APR-2023</h4>
<ul>
<li>Met with Nora, selected samples for processing using <a href="https://github.com/sarahtanja/coral-DNA-RNA-lab-extractions/blob/main/rand-sample-processing.Rmd">R script to randomize</a></li>
<li>Setup workflow in Hyak</li>
<li>added to extraction protocol</li>
<li>waiting for mortexer delivery, expected by end of day, but FedEx package delivery delayed</li>
<li>emailed asking for details about Neptune controller unit (it was returned after being sent away for RMA maintenance request, but nothing appears to have been done to it)</li>
</ul>
</section>
<section id="apr-2023-6" class="level4" data-number="4.10.0.7">
<h4 data-number="4.10.0.7" class="anchored" data-anchor-id="apr-2023-6"><span class="header-section-number">4.10.0.7</span> 17-APR-2023</h4>
<ul>
<li>added to extraction protocol</li>
<li>read about remote computing, Hyak Mox</li>
<li>enabled Hyak account</li>
</ul>
</section>
<section id="apr-2023-7" class="level4" data-number="4.10.0.8">
<h4 data-number="4.10.0.8" class="anchored" data-anchor-id="apr-2023-7"><span class="header-section-number">4.10.0.8</span> 14-APR-2023</h4>
<ul>
<li>attended Cat Fong chalk talk &amp; SAFS Cafe</li>
<li>gave tour to Edith</li>
</ul>
</section>
<section id="apr-2023-8" class="level4" data-number="4.10.0.9">
<h4 data-number="4.10.0.9" class="anchored" data-anchor-id="apr-2023-8"><span class="header-section-number">4.10.0.9</span> 11-APR-2023</h4>
<ul>
<li><p>NCBI SRA links downloading RNAseq fastq files</p></li>
<li><p>Looking for ‘fake data’ for M. capitata</p>
<ul>
<li><p>DNA, ITS2 to measure symbiont to host cell ratios for ‘Patterns of bleaching and recovery of Montipora capitata in Kaneohe Bay’ hosted at zenodo <a href="https://zenodo.org/record/53231#.ZDWE0HbMKTQ">here</a> https://zenodo.org/record/53231#.ZDWE0HbMKTQ -RNAseq resources from A. Huffmeyer</p>
<ul>
<li>https://ahuffmyer.github.io/ASH_Putnam_Lab_Notebook/E5-Deep-Dive-RNAseq-Count-Matrix-Analysis/</li>
<li>https://github.com/AHuffmyer/EarlyLifeHistory_Energetics &gt; Here is the spreadsheet of my NCBI accession numbers for my sequences: https://docs.google.com/spreadsheets/d/1qDGGpLFcmoO-fIFOPSUhPcxi4ErXIq2PkQbxvCCzI40/edit?pli=1#gid=457642286 If you look at the Mcapitata Ontogeny Timeseries tab it’ll have all the SRR numbers you will need to download the TagSeq files. Instead of downloading the full data set you could just pick like three lifestages for a test comparison - AH</li>
</ul></li>
</ul></li>
</ul>
</section>
<section id="apr-2023-9" class="level4" data-number="4.10.0.10">
<h4 data-number="4.10.0.10" class="anchored" data-anchor-id="apr-2023-9"><span class="header-section-number">4.10.0.10</span> 10-APR-2023</h4>
<ul>
<li>created script to randomize sample processing in <a href="https://github.com/sarahtanja/coral-DNA-RNA-lab-extractions">coral-DNA-RNA-lab-extractions</a> repo</li>
<li>called Zymo and learned that ‘mix’ in protocol step can either mean pipette up and down 10-12 times, or briefly touch to vortex mixer for a pulse</li>
<li>Zymo tech also shared that it’s best to follow protocol and add Lysis buffer to the sample supernatent in a separate rnase-free tube. If you do it in the filter tube some sample may start to percolate thru the filter before being fully buffered.</li>
<li><a href="https://www.zymoresearch.com/products/horizontal-microtube-holder">Horizontal Microtube Holder</a></li>
</ul>
</section>
</section>
<section id="march-2023" class="level2" data-number="4.11">
<h2 data-number="4.11" class="anchored" data-anchor-id="march-2023"><span class="header-section-number">4.11</span> March 2023</h2>
<section id="mar-2023" class="level4" data-number="4.11.0.1">
<h4 data-number="4.11.0.1" class="anchored" data-anchor-id="mar-2023"><span class="header-section-number">4.11.0.1</span> 29-MAR-2023</h4>
<ul>
<li><p>started template for DNA/RNA data extraction sheet in new repo for extractions<br>
</p></li>
<li><p>organized notes for samples in metadata</p></li>
<li><p>installed git on WSL via <a href="https://www.git-tower.com/blog/git-wsl/">this tutorial</a></p></li>
<li><p>finished homework 1 for FISH546</p></li>
<li><p>lab-meeting notes</p>
<ul>
<li><p>purchasing</p>
<ul>
<li><p>no purchases in June!</p>
<blockquote class="blockquote">
<p>The University of Washington is transitioning from their old DOS-based system to Workday Finance, with a Go Live date of July 5th, 2023. During the transition period, there will be some disruptions to normal operations, and the continuity of research and teaching and minimizing the impact on faculty, staff, and students is the priority. There will be Restrictive Periods during which no one will be able to process purchase orders or obtain reimbursements in ARIBA. Exceptions may be made for urgent or emergency purchases. Travel reimbursements and non-travel-related reimbursements should be submitted before June 9th, and processing will resume on July 5th. Blanket Purchase Orders that are expected to be active past June 30th, 2023, will not be automatically transferred to Workday, and all continuing BPOs must be manually reentered by the College of the Environment. Navigating the transition will require patience and close partnerships between research and administrative groups</p>
</blockquote></li>
<li><p>space survey</p>
<ul>
<li>% of time spent in spaces from July ’22 - March ’23</li>
<li>2% 242</li>
<li>8% 236</li>
<li>30% HIMB</li>
<li>60% 264, 264B</li>
<li>undergrad Nora Hessen 2hrs per week in 264, 264B since January ’23</li>
</ul></li>
</ul></li>
</ul></li>
</ul>
</section>
<section id="mar-2023-1" class="level4" data-number="4.11.0.2">
<h4 data-number="4.11.0.2" class="anchored" data-anchor-id="mar-2023-1"><span class="header-section-number">4.11.0.2</span> 28-MAR-2023</h4>
<ul>
<li><p>read papers:</p>
<ul>
<li><p><a href="https://www.annualreviews.org/doi/pdf/10.1146/annurev-micro-102215-095440">Bourne et al.&nbsp;2016</a></p></li>
<li><p><a href="https://ami-journals.onlinelibrary.wiley.com/doi/epdf/10.1111/1758-2229.12541">Sogin et al.&nbsp;2017</a></p></li>
<li><p><a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC4104888/">Kelly et al.&nbsp;2014</a></p></li>
</ul></li>
<li><p>called Zymo tech Christian Camacho ccamacho@zymoresearch.com to get advice on working with coral samples and the Zymo DNA/RNA miniprep kit:</p>
<ul>
<li><p>bead bash for 40 minutes for microbes on mortexer vortex-mixer! This is still tested to result in good RNA, though there may be a trade-off between bead-bashing for lysis and RNA interity… see Zymo optimized lysis protocol</p></li>
<li><p>benchmark mortexer is not a tested method for the kit.. but it’s closest to the vortex genie approved method for lysis</p>
<ul>
<li><a href="https://files.zymoresearch.com/documents/bead_beating_short_protocol_tables.pdf">optimized lysis protocols</a></li>
</ul></li>
<li><p>mortar &amp; pestle with liquid nitrogen, then add powder to bead bashing tube with RNA shield</p></li>
<li><p>lyophilized (freeze-dried) proteinase K is stable at room temp, it should be stored in the -20 freezer only after resuspending in the buffer</p></li>
<li><p>the DNAse 1 is similarly a lyophilized powder, but a very small amount, such that the vial looks empty</p></li>
</ul></li>
<li><p>Gathered lab supplies</p>
<ul>
<li>mortar &amp; pestles</li>
<li>alcohol burner for use with denatured ethyl alcohol</li>
<li>flint spark torch ignitor</li>
<li>found expired RNA later</li>
<li>DNAse away</li>
<li>DEPC water (does that expire?)</li>
<li>weigh papers (for powdered samples)</li>
</ul>
<p>NEED</p>
<ul class="task-list">
<li><label><input type="checkbox">scoopulas (to scoop powder from mortar to weigh paper)</label></li>
<li><label><input type="checkbox">denatured ethyl alcohol (disinfect labware between samples)</label></li>
</ul></li>
</ul>
</section>
<section id="mar-2023-2" class="level4" data-number="4.11.0.3">
<h4 data-number="4.11.0.3" class="anchored" data-anchor-id="mar-2023-2"><span class="header-section-number">4.11.0.3</span> 27-MAR-2023</h4>
<ul>
<li><p>read papers:</p>
<ul>
<li>https://environmentalevidencejournal.biomedcentral.com/articles/10.1186/s13750-023-00298-y</li>
</ul></li>
<li><p>worked on coral collection report</p></li>
</ul>
<section id="section" class="level5" data-number="4.11.0.3.1">
<h5 data-number="4.11.0.3.1" class="anchored" data-anchor-id="section"><span class="header-section-number">4.11.0.3.1</span> </h5>
</section>
</section>
<section id="mar-2023-3" class="level4" data-number="4.11.0.4">
<h4 data-number="4.11.0.4" class="anchored" data-anchor-id="mar-2023-3"><span class="header-section-number">4.11.0.4</span> 21-MAR-2023</h4>
<ul>
<li>interviewed for WSG student assistant position</li>
<li>began work on himb-2022 coral collection report</li>
</ul>
</section>
<section id="mar-2023-4" class="level4" data-number="4.11.0.5">
<h4 data-number="4.11.0.5" class="anchored" data-anchor-id="mar-2023-4"><span class="header-section-number">4.11.0.5</span> 20-MAR-2023</h4>
<ul>
<li>installed Sophos anti-virus on Owl</li>
<li>attended Robert’s Lab meeting</li>
<li>checked out Qubit buffer + reagents in Roberts lab with Olivia &amp; Zach</li>
<li>put mortexer, DEPC-treated water, Qubit assay kit, on purchase log</li>
<li>wrote <a href="https://sarahtanja.github.io/lab-book/protocols/protocol-qbit/">Qubit RNA Broad Range Protocol</a> post</li>
</ul>
</section>
<section id="mar-2023-5" class="level4" data-number="4.11.0.6">
<h4 data-number="4.11.0.6" class="anchored" data-anchor-id="mar-2023-5"><span class="header-section-number">4.11.0.6</span> 16-MAR-2023</h4>
<ul>
<li>finished SMEA/FISH 539 final paper</li>
</ul>
</section>
<section id="mar-2023-6" class="level4" data-number="4.11.0.7">
<h4 data-number="4.11.0.7" class="anchored" data-anchor-id="mar-2023-6"><span class="header-section-number">4.11.0.7</span> 15-MAR-2023</h4>
<ul>
<li>out sick</li>
<li>worked on SMEA/FISH 539 final paper</li>
</ul>
</section>
<section id="mar-2023-7" class="level4" data-number="4.11.0.8">
<h4 data-number="4.11.0.8" class="anchored" data-anchor-id="mar-2023-7"><span class="header-section-number">4.11.0.8</span> 14-MAR-2023</h4>
<ul>
<li>out sick</li>
<li>talked with Hailey Dockery about shadowing RNA extractions</li>
</ul>
</section>
<section id="mar-2023-8" class="level4" data-number="4.11.0.9">
<h4 data-number="4.11.0.9" class="anchored" data-anchor-id="mar-2023-8"><span class="header-section-number">4.11.0.9</span> 13-MAR-2023</h4>
<ul>
<li>worked on SMEA/FISH 539 final paper</li>
<li>updated R &amp; RStudio to latest versions</li>
</ul>
</section>
<section id="mar-2023-9" class="level4" data-number="4.11.0.10">
<h4 data-number="4.11.0.10" class="anchored" data-anchor-id="mar-2023-9"><span class="header-section-number">4.11.0.10</span> 10-MAR-2023</h4>
<ul>
<li>finished SMEA/FISH 1page reflection</li>
<li>worked on SMEA/FISH 539 final paper</li>
<li>met with GSIS speaker on Zoom</li>
<li>made hotel reservations for GSIS</li>
</ul>
</section>
<section id="mar-2023-10" class="level4" data-number="4.11.0.11">
<h4 data-number="4.11.0.11" class="anchored" data-anchor-id="mar-2023-10"><span class="header-section-number">4.11.0.11</span> 09-MAR-2023</h4>
<ul>
<li><p>Attended Coastal Ecology candidate chalk talk</p></li>
<li><p>Shadowed Grace for new RNA extraction protocol &amp; Qubit QC for RNA quality</p></li>
<li><p>Met with Emma Strand about M cap DNA/RNA extractions… notes from Zoom chat:</p>
<ul>
<li>email Erin Chille <a href="mailto:eec72@scarletmail.rutgers.edu" class="email">eec72@scarletmail.rutgers.edu</a> about 16S extraction kit</li>
<li>https://emmastrand.github.io/EmmaStrand_Notebook/Kbay-Bleaching-2019-DNA-RNA-Extractions/</li>
<li>host DNA (looked at DNA methylation), symbiont DNA (ITS2), microbiome DNA (16S not so good), host RNA + symbiont RNA (Illumina Nova Seq RNAseq)</li>
<li>coral issues- mucus clogging spin-column filter (avoid mucucs, avoid skeleton)</li>
<li>how big of a fragment do I put in the bead tube? 1/3 bead, 1/3 coral, 1/3 shield – https://emmastrand.github.io/EmmaStrand_Notebook/Kbay-Bleaching-2019-DNA-RNA-Extractions/</li>
<li>optimize time to vortex bead tube (2mins for m. cap) *vortex machine, holds 8 tubes at once</li>
<li>https://www.southernlabware.com/mortexertm-vortex-mixer-115v.html?gclid=Cj0KCQiApKagBhC1ARIsAFc7Mc5LH-QjqtDR98g9ZgE9iAOs-y35m7X5rr7pHCqfPGRtTJVPEeQWbLYaAtCpEALw_wcB</li>
<li>100uL total DNA material end product from kit</li>
<li>12ng of input for ITS2 about 5uL</li>
<li>aim for 10 - 70 ng/uL DNA quantity (Qubit)</li>
<li>DNA quality (gel) *important for methylation studies, but may not be for 16S/ITS2</li>
<li>100uL total RNA material end product, send 20-30uL for RNAseq</li>
<li>https://github.com/hputnam/HI_Bleaching_Timeseries/blob/main/Dec-July-2019-analysis/data/Molecular-labwork.xlsx</li>
<li>sample ID, use a separate extraction ID (in the event of multiple extractions from the same sample!)</li>
</ul></li>
</ul>
</section>
<section id="mar-2023-11" class="level4" data-number="4.11.0.12">
<h4 data-number="4.11.0.12" class="anchored" data-anchor-id="mar-2023-11"><span class="header-section-number">4.11.0.12</span> 07-MAR-2023</h4>
<ul>
<li>worked on SMEA/FISH 539 final paper</li>
<li>met to make GSIS budget w/ Juliette</li>
<li>met with Grace from Robert’s Lab to practice RNA extraction</li>
<li>participated in SAC interview for ADDEI</li>
</ul>
</section>
<section id="mar-2023-12" class="level4" data-number="4.11.0.13">
<h4 data-number="4.11.0.13" class="anchored" data-anchor-id="mar-2023-12"><span class="header-section-number">4.11.0.13</span> 06-MAR-2023</h4>
<ul>
<li>presented final group project in SMEA/FISH 539</li>
<li>worked on SMEA/FISH 539 final paper</li>
<li>attended Coastal Ecology faculty candidate seminar</li>
</ul>
</section>
<section id="mar-2023-13" class="level4" data-number="4.11.0.14">
<h4 data-number="4.11.0.14" class="anchored" data-anchor-id="mar-2023-13"><span class="header-section-number">4.11.0.14</span> 03-MAR-2023</h4>
<ul>
<li>I swear I’ve done stuff since January / how has winter quarter gone by so fast?!</li>
<li>submitted <a href="https://pcc.uw.edu/research/funding-opportunities/">PCC Climate Solutions</a> grant funding application this morning</li>
<li>worked on FSMEA/FISH 539 final project + presentation</li>
</ul>
</section>
<section id="mar-2023-14" class="level4" data-number="4.11.0.15">
<h4 data-number="4.11.0.15" class="anchored" data-anchor-id="mar-2023-14"><span class="header-section-number">4.11.0.15</span> 02-MAR-2023</h4>
<ul>
<li>submitted FHL TA application for summer A term marine botany</li>
<li>helped interview ADDEI candidate</li>
<li>attended Carter’s Coastal Ecology candidate chalk talk</li>
<li>met with Steven on Zoom and got permission to use Bioanalyzer</li>
<li>worked on sequencing quotes, DNA/RNA extraction protocol, reached out to Emma Strand and Grace</li>
</ul>
</section>
</section>
<section id="january-2023" class="level2" data-number="4.12">
<h2 data-number="4.12" class="anchored" data-anchor-id="january-2023"><span class="header-section-number">4.12</span> January 2023</h2>
<section id="jan-2023" class="level4" data-number="4.12.0.1">
<h4 data-number="4.12.0.1" class="anchored" data-anchor-id="jan-2023"><span class="header-section-number">4.12.0.1</span> 31-JAN-2023</h4>
<ul>
<li>prepared 1st committee meeting blog post</li>
</ul>
</section>
<section id="jan-2023-1" class="level4" data-number="4.12.0.2">
<h4 data-number="4.12.0.2" class="anchored" data-anchor-id="jan-2023-1"><span class="header-section-number">4.12.0.2</span> 20-JAN-2023</h4>
<ul>
<li>cleaned PBB</li>
<li>strategized chapter 1: Low Dose Effects &amp; chapter 2: Multiple Stressors for thesis</li>
<li>arranged to meet with Miranda to talk about summer kelp gametophyte project</li>
<li>completed lab safety trainings</li>
</ul>
</section>
<section id="jan-2023-2" class="level4" data-number="4.12.0.3">
<h4 data-number="4.12.0.3" class="anchored" data-anchor-id="jan-2023-2"><span class="header-section-number">4.12.0.3</span> 19-JAN-2023</h4>
<ul>
<li><p>read <a href="https://ahuffmyer.github.io/ASH_Putnam_Lab_Notebook/">Ariana Huffmyer’s</a> lab posts on extraction protocols</p></li>
<li><p>read <a href="https://www.nature.com/articles/ismej201539.pdf">Ainsworth et al 2015</a> &amp; <a href="https://onlinelibrary.wiley.com/doi/abs/10.1111/mec.15899">Brown et al 2021</a></p>
<ul>
<li>MarBio Plant DNA extraction Kit for whole coral holobiont 16S rRNA Bacteria</li>
<li>Pure Link Plant Total DNA purification systems (Invitrogen) for 16S rRNA Bacteria</li>
</ul></li>
<li><p>attended JPG lab meeting and gave feedback on Eileen Bates’ pinto abalone presentation for Alaska Science Sympsosium</p></li>
<li><p>designed graphical methods in Canva for pae-temp experiment</p></li>
</ul>
</section>
<section id="jan-2023-3" class="level4" data-number="4.12.0.4">
<h4 data-number="4.12.0.4" class="anchored" data-anchor-id="jan-2023-3"><span class="header-section-number">4.12.0.4</span> 18-JAN-2023</h4>
<ul>
<li>worked with <a href="https://github.com/norahess">norahess</a> to setup collaboration on ibis repo!</li>
<li>talking about setting up Aiptasia spawning tanks to run experiments with cnidarian model organism reproduction</li>
</ul>
</section>
<section id="jan-2023-4" class="level4" data-number="4.12.0.5">
<h4 data-number="4.12.0.5" class="anchored" data-anchor-id="jan-2023-4"><span class="header-section-number">4.12.0.5</span> 17-JAN-2023</h4>
<ul>
<li>attended Roberts lab meeting, discussed curriculum development for Maritime Highschool</li>
</ul>
</section>
<section id="jan-2023-5" class="level4" data-number="4.12.0.6">
<h4 data-number="4.12.0.6" class="anchored" data-anchor-id="jan-2023-5"><span class="header-section-number">4.12.0.6</span> 13-JAN-2023</h4>
<ul>
<li>Read up on ‘Omics’, started a post on definitions and basics of the study of DNA &amp; RNA</li>
<li>Found <a href="https://www.ncbi.nlm.nih.gov/genome/browse/#!/overview/montipora%20capitata">whole genome for <em>M. capitata</em> on NCBI</a></li>
<li>Helmkampf, M., Bellinger, M. R., Geib, S. M., Sim, S. B., &amp; Takabayashi, M. (2019). Draft Genome of the Rice Coral Montipora capitata Obtained from Linked-Read Sequencing. <em>Genome Biology and Evolution</em>, <em>11</em>(7), 2045–2054. https://doi.org/10.1093/gbe/evz135</li>
<li>Williams, A., Pathmanathan, J. S., Stephens, T. G., Su, X., Chiles, E. N., Conetta, D., Putnam, H. M., &amp; Bhattacharya, D. (2021). Multi-omic characterization of the thermal stress phenome in the stony coral Montipora capitata. <em>PeerJ</em>, <em>9</em>, e12335. https://doi.org/10.7717/peerj.12335</li>
<li>Shumaker, A., Putnam, H. M., Qiu, H., Price, D. C., Zelzion, E., Harel, A., Wagner, N. E., Gates, R. D., Yoon, H. S., &amp; Bhattacharya, D. (2019). Genome analysis of the rice coral Montipora capitata. <em>Scientific Reports</em>, <em>9</em>(1), 2571. https://doi.org/10.1038/s41598-019-39274-3</li>
</ul>
</section>
<section id="jan-2023-6" class="level4" data-number="4.12.0.7">
<h4 data-number="4.12.0.7" class="anchored" data-anchor-id="jan-2023-6"><span class="header-section-number">4.12.0.7</span> 12-JAN-2023</h4>
<ul>
<li><p>Explored the use of purple sea urchin and/or Aiptasia for phthalate low-dose effect gametogenesis study (proposal brainstorming) for the <a href="https://environment.uw.edu/students/current-students/identity-belonging-and-inquiry-in-science-ibis-program/">IBIS research mentor program</a></p></li>
<li><p>Created <a href="https://github.com/sarahtanja/ibis">ibis repo</a> to organize project</p></li>
<li><p>Completed <a href="https://github.com/sarahtanja/ibis/blob/main/ehslabsafetytrainmatrix.pdf">lab safety training matrix</a> for myself and NH, identified required and recommended lab safety trainings</p></li>
<li><p>Potential to use rack in PBB near interior door for Aiptasia spawning incubator, setup workflow with dissecting scope and camera all in PBB for IBIS project</p></li>
<li><p>Inventoried equipment we have in lab 236 for molecular work:</p>
<ul>
<li>Beckman Coulter Allegra X-30R Centrifuge</li>
<li>Thermo Scientific NanoDrop 2000c Spectrophotometer</li>
<li>Fisherbrand Isotemp heat block</li>
<li>Beckman Coulter Microfuge 16 Centrifuge</li>
<li>Eppendorf Mastercycler nexus gradient</li>
</ul></li>
</ul>
</section>
<section id="jan-2023-7" class="level4" data-number="4.12.0.8">
<h4 data-number="4.12.0.8" class="anchored" data-anchor-id="jan-2023-7"><span class="header-section-number">4.12.0.8</span> 09-JAN-2023</h4>
<ul>
<li><p>Today I setup my GitHub ‘notebook’! This is the beginning of my exploration of using GitHub to jot down thoughts, record ‘daily bits’, and organize the procedures and methods I’ll use in my thesis research</p></li>
<li><p><a href="https://github.com/zbengt">zbengt</a> introduced me to ‘<a href="https://elicit.org/">elicit</a>’, a lit review search AI. I searched “What is the role of endocrine signaling in cnidarian development” and found an exciting paper titled <a href="https://www.frontiersin.org/articles/10.3389/fgene.2021.793306/full"><strong>“Sea Anemones Responding to Sex Hormones, Oxybenzone, and Benzyl Butyl Phthalate: Transcriptional Profiling and <em>in Silico</em> Modelling Provide Clues to Decipher Endocrine Disruption in Cnidarians”</strong></a> authored by Morgan et al.&nbsp;in 2022</p></li>
<li><p>I explored another AI, <a href="https://openai.com/blog/chatgpt/">ChatGPT</a>, and gave it the prompt “write me a poem about corals and oysters and mothers” and it returned:</p>
<ul>
<li><blockquote class="blockquote">
<p>Coral reefs, a vibrant sea of life, A home for countless creatures, rife. Oysters, clams, and mussels too, All nestled safe, their shells in view.</p>
<p>But danger lurks, the waters warm, The coral bleaches, it’s colors torn. Mother Nature’s wrath, a sight to see, But we can help, if only we believe.</p>
<p>For mothers, fierce and loving beasts, Will do what it takes, to protect at least. Their young and home, with all their might, They’ll fight and strive, through day and night.</p>
<p>So let us join, in their noble quest, To save the reefs, and all the rest. For oysters, corals, and mothers too, Deserve a chance, to make it through.</p>
</blockquote></li>
</ul></li>
<li><p>I followed up with a ChatGPT prompt “what are examples of endocrine signaling in cnidarians?” and it returned:</p>
<ul>
<li><blockquote class="blockquote">
<p>Endocrine signaling is the process of communicating information from one cell to another by releasing hormones into the circulatory system or through direct contact. In cnidarians, endocrine signaling can be used for a variety of functions, including the regulation of growth, development, and behavior.</p>
<p>One example of endocrine signaling in cnidarians is the release of growth hormones, which stimulate the growth and development of cells and tissues. For example, in some species of cnidarians, the release of growth hormones is regulated by the action of a hormone called insulin-like growth factor (IGF).</p>
<p>Another example of endocrine signaling in cnidarians is the regulation of reproductive function. In some species of cnidarians, hormones called gonadotropins are released by the endocrine cells of the gonads (sex organs) to stimulate the production and release of gametes (sex cells).</p>
<p>Endocrine signaling is also involved in the regulation of behavior in cnidarians. For example, some species of cnidarians release hormones called neuropeptides that act on the nervous system to regulate behaviors such as feeding, movement, and response to stimuli.</p>
<p>Overall, endocrine signaling plays a vital role in the regulation of various functions in cnidarians, including growth, development, and behavior.</p>
</blockquote></li>
</ul></li>
</ul>
<p>​**Is this is all true!? I’ll have to follow-up with my own dive into primary literature…</p>
<ul>
<li><p>Another ChatGPT AI response, when asked the same questions, but prompted to provide references:</p>
<ul>
<li><blockquote class="blockquote">
<p>Examples of endocrine signaling in cnidarians include the release of hormones in response to stimuli such as touch, temperature changes, and changes in light intensity.</p>
<p>For example, the hormone adenylate cyclase-activating polypeptide (ACAP) is released from cnidocytes in response to a variety of stimuli and activates adenylate cyclase, leading to the production of cyclic adenosine monophosphate (cAMP) and the regulation of various physiological processes.</p>
<p>Other signaling molecules that have been identified in cnidarians include cyclic guanosine monophosphate (cGMP), inositol triphosphate (IP3), and Ca2+-dependent signaling pathways.</p>
<p>References:</p>
<p>Levitan, D. R., &amp; Kohn, A. B. (2001). The Endocrine System of Cnidarians. Annual Review of Physiology, 63(1), 319-344.</p>
<p>Schmid, V., &amp; Pfründner, R. (1995). Adenylate cyclase-activating polypeptide in cnidarians: a new class of peptides with signal transducing properties. Regulatory Peptides, 61(2), 149-157.</p>
</blockquote></li>
</ul></li>
</ul>
<p>​</p>


<!-- -->


</section>
</section>
</section>

</main> <!-- /main -->
<script id="quarto-html-after-body" type="application/javascript">
window.document.addEventListener("DOMContentLoaded", function (event) {
  const toggleBodyColorMode = (bsSheetEl) => {
    const mode = bsSheetEl.getAttribute("data-mode");
    const bodyEl = window.document.querySelector("body");
    if (mode === "dark") {
      bodyEl.classList.add("quarto-dark");
      bodyEl.classList.remove("quarto-light");
    } else {
      bodyEl.classList.add("quarto-light");
      bodyEl.classList.remove("quarto-dark");
    }
  }
  const toggleBodyColorPrimary = () => {
    const bsSheetEl = window.document.querySelector("link#quarto-bootstrap");
    if (bsSheetEl) {
      toggleBodyColorMode(bsSheetEl);
    }
  }
  toggleBodyColorPrimary();  
  const disableStylesheet = (stylesheets) => {
    for (let i=0; i < stylesheets.length; i++) {
      const stylesheet = stylesheets[i];
      stylesheet.rel = 'prefetch';
    }
  }
  const enableStylesheet = (stylesheets) => {
    for (let i=0; i < stylesheets.length; i++) {
      const stylesheet = stylesheets[i];
      stylesheet.rel = 'stylesheet';
    }
  }
  const manageTransitions = (selector, allowTransitions) => {
    const els = window.document.querySelectorAll(selector);
    for (let i=0; i < els.length; i++) {
      const el = els[i];
      if (allowTransitions) {
        el.classList.remove('notransition');
      } else {
        el.classList.add('notransition');
      }
    }
  }
  const toggleGiscusIfUsed = (isAlternate, darkModeDefault) => {
    const baseTheme = document.querySelector('#giscus-base-theme')?.value ?? 'light';
    const alternateTheme = document.querySelector('#giscus-alt-theme')?.value ?? 'dark';
    let newTheme = '';
    if(darkModeDefault) {
      newTheme = isAlternate ? baseTheme : alternateTheme;
    } else {
      newTheme = isAlternate ? alternateTheme : baseTheme;
    }
    const changeGiscusTheme = () => {
      // From: https://github.com/giscus/giscus/issues/336
      const sendMessage = (message) => {
        const iframe = document.querySelector('iframe.giscus-frame');
        if (!iframe) return;
        iframe.contentWindow.postMessage({ giscus: message }, 'https://giscus.app');
      }
      sendMessage({
        setConfig: {
          theme: newTheme
        }
      });
    }
    const isGiscussLoaded = window.document.querySelector('iframe.giscus-frame') !== null;
    if (isGiscussLoaded) {
      changeGiscusTheme();
    }
  }
  const toggleColorMode = (alternate) => {
    // Switch the stylesheets
    const alternateStylesheets = window.document.querySelectorAll('link.quarto-color-scheme.quarto-color-alternate');
    manageTransitions('#quarto-margin-sidebar .nav-link', false);
    if (alternate) {
      enableStylesheet(alternateStylesheets);
      for (const sheetNode of alternateStylesheets) {
        if (sheetNode.id === "quarto-bootstrap") {
          toggleBodyColorMode(sheetNode);
        }
      }
    } else {
      disableStylesheet(alternateStylesheets);
      toggleBodyColorPrimary();
    }
    manageTransitions('#quarto-margin-sidebar .nav-link', true);
    // Switch the toggles
    const toggles = window.document.querySelectorAll('.quarto-color-scheme-toggle');
    for (let i=0; i < toggles.length; i++) {
      const toggle = toggles[i];
      if (toggle) {
        if (alternate) {
          toggle.classList.add("alternate");     
        } else {
          toggle.classList.remove("alternate");
        }
      }
    }
    // Hack to workaround the fact that safari doesn't
    // properly recolor the scrollbar when toggling (#1455)
    if (navigator.userAgent.indexOf('Safari') > 0 && navigator.userAgent.indexOf('Chrome') == -1) {
      manageTransitions("body", false);
      window.scrollTo(0, 1);
      setTimeout(() => {
        window.scrollTo(0, 0);
        manageTransitions("body", true);
      }, 40);  
    }
  }
  const isFileUrl = () => { 
    return window.location.protocol === 'file:';
  }
  const hasAlternateSentinel = () => {  
    let styleSentinel = getColorSchemeSentinel();
    if (styleSentinel !== null) {
      return styleSentinel === "alternate";
    } else {
      return false;
    }
  }
  const setStyleSentinel = (alternate) => {
    const value = alternate ? "alternate" : "default";
    if (!isFileUrl()) {
      window.localStorage.setItem("quarto-color-scheme", value);
    } else {
      localAlternateSentinel = value;
    }
  }
  const getColorSchemeSentinel = () => {
    if (!isFileUrl()) {
      const storageValue = window.localStorage.getItem("quarto-color-scheme");
      return storageValue != null ? storageValue : localAlternateSentinel;
    } else {
      return localAlternateSentinel;
    }
  }
  const darkModeDefault = false;
  let localAlternateSentinel = darkModeDefault ? 'alternate' : 'default';
  // Dark / light mode switch
  window.quartoToggleColorScheme = () => {
    // Read the current dark / light value 
    let toAlternate = !hasAlternateSentinel();
    toggleColorMode(toAlternate);
    setStyleSentinel(toAlternate);
    toggleGiscusIfUsed(toAlternate, darkModeDefault);
  };
  // Ensure there is a toggle, if there isn't float one in the top right
  if (window.document.querySelector('.quarto-color-scheme-toggle') === null) {
    const a = window.document.createElement('a');
    a.classList.add('top-right');
    a.classList.add('quarto-color-scheme-toggle');
    a.href = "";
    a.onclick = function() { try { window.quartoToggleColorScheme(); } catch {} return false; };
    const i = window.document.createElement("i");
    i.classList.add('bi');
    a.appendChild(i);
    window.document.body.appendChild(a);
  }
  // Switch to dark mode if need be
  if (hasAlternateSentinel()) {
    toggleColorMode(true);
  } else {
    toggleColorMode(false);
  }
  const icon = "";
  const anchorJS = new window.AnchorJS();
  anchorJS.options = {
    placement: 'right',
    icon: icon
  };
  anchorJS.add('.anchored');
  const isCodeAnnotation = (el) => {
    for (const clz of el.classList) {
      if (clz.startsWith('code-annotation-')) {                     
        return true;
      }
    }
    return false;
  }
  const clipboard = new window.ClipboardJS('.code-copy-button', {
    text: function(trigger) {
      const codeEl = trigger.previousElementSibling.cloneNode(true);
      for (const childEl of codeEl.children) {
        if (isCodeAnnotation(childEl)) {
          childEl.remove();
        }
      }
      return codeEl.innerText;
    }
  });
  clipboard.on('success', function(e) {
    // button target
    const button = e.trigger;
    // don't keep focus
    button.blur();
    // flash "checked"
    button.classList.add('code-copy-button-checked');
    var currentTitle = button.getAttribute("title");
    button.setAttribute("title", "Copied!");
    let tooltip;
    if (window.bootstrap) {
      button.setAttribute("data-bs-toggle", "tooltip");
      button.setAttribute("data-bs-placement", "left");
      button.setAttribute("data-bs-title", "Copied!");
      tooltip = new bootstrap.Tooltip(button, 
        { trigger: "manual", 
          customClass: "code-copy-button-tooltip",
          offset: [0, -8]});
      tooltip.show();    
    }
    setTimeout(function() {
      if (tooltip) {
        tooltip.hide();
        button.removeAttribute("data-bs-title");
        button.removeAttribute("data-bs-toggle");
        button.removeAttribute("data-bs-placement");
      }
      button.setAttribute("title", currentTitle);
      button.classList.remove('code-copy-button-checked');
    }, 1000);
    // clear code selection
    e.clearSelection();
  });
  const viewSource = window.document.getElementById('quarto-view-source') ||
                     window.document.getElementById('quarto-code-tools-source');
  if (viewSource) {
    const sourceUrl = viewSource.getAttribute("data-quarto-source-url");
    viewSource.addEventListener("click", function(e) {
      if (sourceUrl) {
        // rstudio viewer pane
        if (/\bcapabilities=\b/.test(window.location)) {
          window.open(sourceUrl);
        } else {
          window.location.href = sourceUrl;
        }
      } else {
        const modal = new bootstrap.Modal(document.getElementById('quarto-embedded-source-code-modal'));
        modal.show();
      }
      return false;
    });
  }
  function toggleCodeHandler(show) {
    return function(e) {
      const detailsSrc = window.document.querySelectorAll(".cell > details > .sourceCode");
      for (let i=0; i<detailsSrc.length; i++) {
        const details = detailsSrc[i].parentElement;
        if (show) {
          details.open = true;
        } else {
          details.removeAttribute("open");
        }
      }
      const cellCodeDivs = window.document.querySelectorAll(".cell > .sourceCode");
      const fromCls = show ? "hidden" : "unhidden";
      const toCls = show ? "unhidden" : "hidden";
      for (let i=0; i<cellCodeDivs.length; i++) {
        const codeDiv = cellCodeDivs[i];
        if (codeDiv.classList.contains(fromCls)) {
          codeDiv.classList.remove(fromCls);
          codeDiv.classList.add(toCls);
        } 
      }
      return false;
    }
  }
  const hideAllCode = window.document.getElementById("quarto-hide-all-code");
  if (hideAllCode) {
    hideAllCode.addEventListener("click", toggleCodeHandler(false));
  }
  const showAllCode = window.document.getElementById("quarto-show-all-code");
  if (showAllCode) {
    showAllCode.addEventListener("click", toggleCodeHandler(true));
  }
    var localhostRegex = new RegExp(/^(?:http|https):\/\/localhost\:?[0-9]*\//);
    var mailtoRegex = new RegExp(/^mailto:/);
      var filterRegex = new RegExp("https:\/\/sarahtanja\.github\.io\/quarto-blog\/");
    var isInternal = (href) => {
        return filterRegex.test(href) || localhostRegex.test(href) || mailtoRegex.test(href);
    }
    // Inspect non-navigation links and adorn them if external
 	var links = window.document.querySelectorAll('a[href]:not(.nav-link):not(.navbar-brand):not(.toc-action):not(.sidebar-link):not(.sidebar-item-toggle):not(.pagination-link):not(.no-external):not([aria-hidden]):not(.dropdown-item):not(.quarto-navigation-tool)');
    for (var i=0; i<links.length; i++) {
      const link = links[i];
      if (!isInternal(link.href)) {
          // target, if specified
          link.setAttribute("target", "_blank");
          if (link.getAttribute("rel") === null) {
            link.setAttribute("rel", "noopener");
          }
          // default icon
          link.classList.add("external");
      }
    }
  function tippyHover(el, contentFn, onTriggerFn, onUntriggerFn) {
    const config = {
      allowHTML: true,
      maxWidth: 500,
      delay: 100,
      arrow: false,
      appendTo: function(el) {
          return el.parentElement;
      },
      interactive: true,
      interactiveBorder: 10,
      theme: 'quarto',
      placement: 'bottom-start',
    };
    if (contentFn) {
      config.content = contentFn;
    }
    if (onTriggerFn) {
      config.onTrigger = onTriggerFn;
    }
    if (onUntriggerFn) {
      config.onUntrigger = onUntriggerFn;
    }
    window.tippy(el, config); 
  }
  const noterefs = window.document.querySelectorAll('a[role="doc-noteref"]');
  for (var i=0; i<noterefs.length; i++) {
    const ref = noterefs[i];
    tippyHover(ref, function() {
      // use id or data attribute instead here
      let href = ref.getAttribute('data-footnote-href') || ref.getAttribute('href');
      try { href = new URL(href).hash; } catch {}
      const id = href.replace(/^#\/?/, "");
      const note = window.document.getElementById(id);
      return note.innerHTML;
    });
  }
  const xrefs = window.document.querySelectorAll('a.quarto-xref');
  const processXRef = (id, note) => {
    // Strip column container classes
    const stripColumnClz = (el) => {
      el.classList.remove("page-full", "page-columns");
      if (el.children) {
        for (const child of el.children) {
          stripColumnClz(child);
        }
      }
    }
    stripColumnClz(note)
    if (id === null || id.startsWith('sec-')) {
      // Special case sections, only their first couple elements
      const container = document.createElement("div");
      if (note.children && note.children.length > 2) {
        container.appendChild(note.children[0].cloneNode(true));
        for (let i = 1; i < note.children.length; i++) {
          const child = note.children[i];
          if (child.tagName === "P" && child.innerText === "") {
            continue;
          } else {
            container.appendChild(child.cloneNode(true));
            break;
          }
        }
        if (window.Quarto?.typesetMath) {
          window.Quarto.typesetMath(container);
        }
        return container.innerHTML
      } else {
        if (window.Quarto?.typesetMath) {
          window.Quarto.typesetMath(note);
        }
        return note.innerHTML;
      }
    } else {
      // Remove any anchor links if they are present
      const anchorLink = note.querySelector('a.anchorjs-link');
      if (anchorLink) {
        anchorLink.remove();
      }
      if (window.Quarto?.typesetMath) {
        window.Quarto.typesetMath(note);
      }
      // TODO in 1.5, we should make sure this works without a callout special case
      if (note.classList.contains("callout")) {
        return note.outerHTML;
      } else {
        return note.innerHTML;
      }
    }
  }
  for (var i=0; i<xrefs.length; i++) {
    const xref = xrefs[i];
    tippyHover(xref, undefined, function(instance) {
      instance.disable();
      let url = xref.getAttribute('href');
      let hash = undefined; 
      if (url.startsWith('#')) {
        hash = url;
      } else {
        try { hash = new URL(url).hash; } catch {}
      }
      if (hash) {
        const id = hash.replace(/^#\/?/, "");
        const note = window.document.getElementById(id);
        if (note !== null) {
          try {
            const html = processXRef(id, note.cloneNode(true));
            instance.setContent(html);
          } finally {
            instance.enable();
            instance.show();
          }
        } else {
          // See if we can fetch this
          fetch(url.split('#')[0])
          .then(res => res.text())
          .then(html => {
            const parser = new DOMParser();
            const htmlDoc = parser.parseFromString(html, "text/html");
            const note = htmlDoc.getElementById(id);
            if (note !== null) {
              const html = processXRef(id, note);
              instance.setContent(html);
            } 
          }).finally(() => {
            instance.enable();
            instance.show();
          });
        }
      } else {
        // See if we can fetch a full url (with no hash to target)
        // This is a special case and we should probably do some content thinning / targeting
        fetch(url)
        .then(res => res.text())
        .then(html => {
          const parser = new DOMParser();
          const htmlDoc = parser.parseFromString(html, "text/html");
          const note = htmlDoc.querySelector('main.content');
          if (note !== null) {
            // This should only happen for chapter cross references
            // (since there is no id in the URL)
            // remove the first header
            if (note.children.length > 0 && note.children[0].tagName === "HEADER") {
              note.children[0].remove();
            }
            const html = processXRef(null, note);
            instance.setContent(html);
          } 
        }).finally(() => {
          instance.enable();
          instance.show();
        });
      }
    }, function(instance) {
    });
  }
      let selectedAnnoteEl;
      const selectorForAnnotation = ( cell, annotation) => {
        let cellAttr = 'data-code-cell="' + cell + '"';
        let lineAttr = 'data-code-annotation="' +  annotation + '"';
        const selector = 'span[' + cellAttr + '][' + lineAttr + ']';
        return selector;
      }
      const selectCodeLines = (annoteEl) => {
        const doc = window.document;
        const targetCell = annoteEl.getAttribute("data-target-cell");
        const targetAnnotation = annoteEl.getAttribute("data-target-annotation");
        const annoteSpan = window.document.querySelector(selectorForAnnotation(targetCell, targetAnnotation));
        const lines = annoteSpan.getAttribute("data-code-lines").split(",");
        const lineIds = lines.map((line) => {
          return targetCell + "-" + line;
        })
        let top = null;
        let height = null;
        let parent = null;
        if (lineIds.length > 0) {
            //compute the position of the single el (top and bottom and make a div)
            const el = window.document.getElementById(lineIds[0]);
            top = el.offsetTop;
            height = el.offsetHeight;
            parent = el.parentElement.parentElement;
          if (lineIds.length > 1) {
            const lastEl = window.document.getElementById(lineIds[lineIds.length - 1]);
            const bottom = lastEl.offsetTop + lastEl.offsetHeight;
            height = bottom - top;
          }
          if (top !== null && height !== null && parent !== null) {
            // cook up a div (if necessary) and position it 
            let div = window.document.getElementById("code-annotation-line-highlight");
            if (div === null) {
              div = window.document.createElement("div");
              div.setAttribute("id", "code-annotation-line-highlight");
              div.style.position = 'absolute';
              parent.appendChild(div);
            }
            div.style.top = top - 2 + "px";
            div.style.height = height + 4 + "px";
            div.style.left = 0;
            let gutterDiv = window.document.getElementById("code-annotation-line-highlight-gutter");
            if (gutterDiv === null) {
              gutterDiv = window.document.createElement("div");
              gutterDiv.setAttribute("id", "code-annotation-line-highlight-gutter");
              gutterDiv.style.position = 'absolute';
              const codeCell = window.document.getElementById(targetCell);
              const gutter = codeCell.querySelector('.code-annotation-gutter');
              gutter.appendChild(gutterDiv);
            }
            gutterDiv.style.top = top - 2 + "px";
            gutterDiv.style.height = height + 4 + "px";
          }
          selectedAnnoteEl = annoteEl;
        }
      };
      const unselectCodeLines = () => {
        const elementsIds = ["code-annotation-line-highlight", "code-annotation-line-highlight-gutter"];
        elementsIds.forEach((elId) => {
          const div = window.document.getElementById(elId);
          if (div) {
            div.remove();
          }
        });
        selectedAnnoteEl = undefined;
      };
        // Handle positioning of the toggle
    window.addEventListener(
      "resize",
      throttle(() => {
        elRect = undefined;
        if (selectedAnnoteEl) {
          selectCodeLines(selectedAnnoteEl);
        }
      }, 10)
    );
    function throttle(fn, ms) {
    let throttle = false;
    let timer;
      return (...args) => {
        if(!throttle) { // first call gets through
            fn.apply(this, args);
            throttle = true;
        } else { // all the others get throttled
            if(timer) clearTimeout(timer); // cancel #2
            timer = setTimeout(() => {
              fn.apply(this, args);
              timer = throttle = false;
            }, ms);
        }
      };
    }
      // Attach click handler to the DT
      const annoteDls = window.document.querySelectorAll('dt[data-target-cell]');
      for (const annoteDlNode of annoteDls) {
        annoteDlNode.addEventListener('click', (event) => {
          const clickedEl = event.target;
          if (clickedEl !== selectedAnnoteEl) {
            unselectCodeLines();
            const activeEl = window.document.querySelector('dt[data-target-cell].code-annotation-active');
            if (activeEl) {
              activeEl.classList.remove('code-annotation-active');
            }
            selectCodeLines(clickedEl);
            clickedEl.classList.add('code-annotation-active');
          } else {
            // Unselect the line
            unselectCodeLines();
            clickedEl.classList.remove('code-annotation-active');
          }
        });
      }
  const findCites = (el) => {
    const parentEl = el.parentElement;
    if (parentEl) {
      const cites = parentEl.dataset.cites;
      if (cites) {
        return {
          el,
          cites: cites.split(' ')
        };
      } else {
        return findCites(el.parentElement)
      }
    } else {
      return undefined;
    }
  };
  var bibliorefs = window.document.querySelectorAll('a[role="doc-biblioref"]');
  for (var i=0; i<bibliorefs.length; i++) {
    const ref = bibliorefs[i];
    const citeInfo = findCites(ref);
    if (citeInfo) {
      tippyHover(citeInfo.el, function() {
        var popup = window.document.createElement('div');
        citeInfo.cites.forEach(function(cite) {
          var citeDiv = window.document.createElement('div');
          citeDiv.classList.add('hanging-indent');
          citeDiv.classList.add('csl-entry');
          var biblioDiv = window.document.getElementById('ref-' + cite);
          if (biblioDiv) {
            citeDiv.innerHTML = biblioDiv.innerHTML;
          }
          popup.appendChild(citeDiv);
        });
        return popup.innerHTML;
      });
    }
  }
});
</script><div class="modal fade" id="quarto-embedded-source-code-modal" tabindex="-1" aria-labelledby="quarto-embedded-source-code-modal-label" aria-hidden="true"><div class="modal-dialog modal-dialog-scrollable"><div class="modal-content"><div class="modal-header"><h5 class="modal-title" id="quarto-embedded-source-code-modal-label">Source Code</h5><button class="btn-close" data-bs-dismiss="modal"></button></div><div class="modal-body"><div class="">
<div class="sourceCode" id="cb34" data-shortcodes="false"><pre class="sourceCode markdown code-with-copy"><code class="sourceCode markdown"><span id="cb34-1"><a href="#cb34-1" aria-hidden="true" tabindex="-1"></a><span class="co">---</span></span>
<span id="cb34-2"><a href="#cb34-2" aria-hidden="true" tabindex="-1"></a><span class="an">title:</span><span class="co"> "Daily Lab Log"</span></span>
<span id="cb34-3"><a href="#cb34-3" aria-hidden="true" tabindex="-1"></a><span class="an">subtitle:</span><span class="co"> "Enter at your peril; This one is for me, not you"</span></span>
<span id="cb34-4"><a href="#cb34-4" aria-hidden="true" tabindex="-1"></a><span class="an">author:</span><span class="co"> "Sarah Tanja"</span></span>
<span id="cb34-5"><a href="#cb34-5" aria-hidden="true" tabindex="-1"></a><span class="an">date:</span><span class="co"> 02/22/2024</span></span>
<span id="cb34-6"><a href="#cb34-6" aria-hidden="true" tabindex="-1"></a><span class="an">date-format:</span><span class="co"> long</span></span>
<span id="cb34-7"><a href="#cb34-7" aria-hidden="true" tabindex="-1"></a><span class="an">date-modified:</span><span class="co"> today</span></span>
<span id="cb34-8"><a href="#cb34-8" aria-hidden="true" tabindex="-1"></a><span class="an">categories:</span><span class="co"> [daily log]</span></span>
<span id="cb34-9"><a href="#cb34-9" aria-hidden="true" tabindex="-1"></a><span class="an">toc:</span><span class="co"> true</span></span>
<span id="cb34-10"><a href="#cb34-10" aria-hidden="true" tabindex="-1"></a><span class="an">toc-expand:</span><span class="co"> true</span></span>
<span id="cb34-11"><a href="#cb34-11" aria-hidden="true" tabindex="-1"></a><span class="an">toc-title:</span><span class="co"> Contents &lt;i class="bi bi-bookmark-heart"&gt;&lt;/i&gt;</span></span>
<span id="cb34-12"><a href="#cb34-12" aria-hidden="true" tabindex="-1"></a><span class="an">toc-depth:</span><span class="co"> 5</span></span>
<span id="cb34-13"><a href="#cb34-13" aria-hidden="true" tabindex="-1"></a><span class="an">toc-location:</span><span class="co"> left</span></span>
<span id="cb34-14"><a href="#cb34-14" aria-hidden="true" tabindex="-1"></a><span class="an">reference-location:</span><span class="co"> margin</span></span>
<span id="cb34-15"><a href="#cb34-15" aria-hidden="true" tabindex="-1"></a><span class="an">citation-location:</span><span class="co"> margin</span></span>
<span id="cb34-16"><a href="#cb34-16" aria-hidden="true" tabindex="-1"></a><span class="an">bibliography:</span><span class="co"> references.bib</span></span>
<span id="cb34-17"><a href="#cb34-17" aria-hidden="true" tabindex="-1"></a><span class="an">image:</span><span class="co"> log.png</span></span>
<span id="cb34-18"><a href="#cb34-18" aria-hidden="true" tabindex="-1"></a><span class="co">---</span></span>
<span id="cb34-19"><a href="#cb34-19" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb34-20"><a href="#cb34-20" aria-hidden="true" tabindex="-1"></a><span class="fu"># March 2025</span></span>
<span id="cb34-21"><a href="#cb34-21" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb34-22"><a href="#cb34-22" aria-hidden="true" tabindex="-1"></a><span class="fu">#### 02-APR-2025</span></span>
<span id="cb34-23"><a href="#cb34-23" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb34-24"><a href="#cb34-24" aria-hidden="true" tabindex="-1"></a>Migrated references from Zotero to Paperpile, paid subscription.</span>
<span id="cb34-25"><a href="#cb34-25" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb34-26"><a href="#cb34-26" aria-hidden="true" tabindex="-1"></a>You can use **Quarto's project root referencing** to flexibly specify the bibliography file relative to your project directory, no matter where the Quarto document is located.</span>
<span id="cb34-27"><a href="#cb34-27" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb34-28"><a href="#cb34-28" aria-hidden="true" tabindex="-1"></a><span class="fu">### **Solution: Use `{project}` in Your Path**</span></span>
<span id="cb34-29"><a href="#cb34-29" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb34-30"><a href="#cb34-30" aria-hidden="true" tabindex="-1"></a>Quarto allows you to reference files relative to the **project root** using <span class="in">`{project}`</span>. If <span class="in">`references.bib`</span> is in your project’s root directory (e.g., inside your Quarto blog repository), use this in your YAML:</span>
<span id="cb34-31"><a href="#cb34-31" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb34-32"><a href="#cb34-32" aria-hidden="true" tabindex="-1"></a><span class="in">```         </span></span>
<span id="cb34-33"><a href="#cb34-33" aria-hidden="true" tabindex="-1"></a><span class="in">yaml</span></span>
<span id="cb34-34"><a href="#cb34-34" aria-hidden="true" tabindex="-1"></a><span class="in">```</span></span>
<span id="cb34-35"><a href="#cb34-35" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb34-36"><a href="#cb34-36" aria-hidden="true" tabindex="-1"></a>CopyEdit</span>
<span id="cb34-37"><a href="#cb34-37" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb34-38"><a href="#cb34-38" aria-hidden="true" tabindex="-1"></a><span class="in">`bibliography: {project}/references.bib`</span></span>
<span id="cb34-39"><a href="#cb34-39" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb34-40"><a href="#cb34-40" aria-hidden="true" tabindex="-1"></a><span class="fu">### **How It Works**</span></span>
<span id="cb34-41"><a href="#cb34-41" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb34-42"><a href="#cb34-42" aria-hidden="true" tabindex="-1"></a><span class="ss">-   </span><span class="in">`{project}`</span> refers to the **root directory of your Quarto project**, which is usually where your <span class="in">`.qmd`</span> or <span class="in">`_quarto.yml`</span> file is located.</span>
<span id="cb34-43"><a href="#cb34-43" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb34-44"><a href="#cb34-44" aria-hidden="true" tabindex="-1"></a>    This ensures that no matter where the <span class="in">`.qmd`</span> file is located within subdirectories, it will always find <span class="in">`references.bib`</span> at the project root.</span>
<span id="cb34-45"><a href="#cb34-45" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb34-46"><a href="#cb34-46" aria-hidden="true" tabindex="-1"></a><span class="fu">### **Example Project Structure**</span></span>
<span id="cb34-47"><a href="#cb34-47" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb34-48"><a href="#cb34-48" aria-hidden="true" tabindex="-1"></a><span class="in">```         </span></span>
<span id="cb34-49"><a href="#cb34-49" aria-hidden="true" tabindex="-1"></a><span class="in">lua</span></span>
<span id="cb34-50"><a href="#cb34-50" aria-hidden="true" tabindex="-1"></a><span class="in">```</span></span>
<span id="cb34-51"><a href="#cb34-51" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb34-52"><a href="#cb34-52" aria-hidden="true" tabindex="-1"></a>CopyEdit</span>
<span id="cb34-53"><a href="#cb34-53" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb34-54"><a href="#cb34-54" aria-hidden="true" tabindex="-1"></a><span class="in">`C:\Users\Minerva\Documents\GitHub\quarto-blog\ │-- _quarto.yml │-- references.bib │-- index.qmd │-- posts\ │   ├── post1\ │   │   ├── analysis.qmd  &lt;-- This can still find references.bib! │   ├── post2\ │-- styles\`</span></span>
<span id="cb34-55"><a href="#cb34-55" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb34-56"><a href="#cb34-56" aria-hidden="true" tabindex="-1"></a>Even if <span class="in">`analysis.qmd`</span> is inside a nested folder (<span class="in">`posts/post1/`</span>), using:</span>
<span id="cb34-57"><a href="#cb34-57" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb34-58"><a href="#cb34-58" aria-hidden="true" tabindex="-1"></a><span class="in">```         </span></span>
<span id="cb34-59"><a href="#cb34-59" aria-hidden="true" tabindex="-1"></a><span class="in">yaml</span></span>
<span id="cb34-60"><a href="#cb34-60" aria-hidden="true" tabindex="-1"></a><span class="in">```</span></span>
<span id="cb34-61"><a href="#cb34-61" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb34-62"><a href="#cb34-62" aria-hidden="true" tabindex="-1"></a>CopyEdit</span>
<span id="cb34-63"><a href="#cb34-63" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb34-64"><a href="#cb34-64" aria-hidden="true" tabindex="-1"></a><span class="in">`bibliography: {project}/references.bib`</span></span>
<span id="cb34-65"><a href="#cb34-65" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb34-66"><a href="#cb34-66" aria-hidden="true" tabindex="-1"></a>will always point to <span class="in">`C:\Users\Minerva\Documents\GitHub\quarto-blog\references.bib`</span>.</span>
<span id="cb34-67"><a href="#cb34-67" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb34-68"><a href="#cb34-68" aria-hidden="true" tabindex="-1"></a><span class="fu">### **Why This Is Better**</span></span>
<span id="cb34-69"><a href="#cb34-69" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb34-70"><a href="#cb34-70" aria-hidden="true" tabindex="-1"></a>✅ No need for absolute paths\</span>
<span id="cb34-71"><a href="#cb34-71" aria-hidden="true" tabindex="-1"></a>✅ Works across different systems (Windows, macOS, Linux)\</span>
<span id="cb34-72"><a href="#cb34-72" aria-hidden="true" tabindex="-1"></a>✅ More flexible than using <span class="in">`../../`</span> for relative paths\</span>
<span id="cb34-73"><a href="#cb34-73" aria-hidden="true" tabindex="-1"></a>✅ Avoids symlink issues</span>
<span id="cb34-74"><a href="#cb34-74" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb34-75"><a href="#cb34-75" aria-hidden="true" tabindex="-1"></a><span class="fu"># September 2024</span></span>
<span id="cb34-76"><a href="#cb34-76" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb34-77"><a href="#cb34-77" aria-hidden="true" tabindex="-1"></a><span class="fu">#### 03-SEP-2024</span></span>
<span id="cb34-78"><a href="#cb34-78" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb34-79"><a href="#cb34-79" aria-hidden="true" tabindex="-1"></a><span class="ss">-   </span>Received confirmation that RNA is in line for QAQC</span>
<span id="cb34-80"><a href="#cb34-80" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb34-81"><a href="#cb34-81" aria-hidden="true" tabindex="-1"></a><span class="ss">-   </span>Read manual for Nikon compound scope, setup for making embryology preservation slides &amp; taking photos</span>
<span id="cb34-82"><a href="#cb34-82" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb34-83"><a href="#cb34-83" aria-hidden="true" tabindex="-1"></a><span class="fu">#### 02-SEP-2024</span></span>
<span id="cb34-84"><a href="#cb34-84" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb34-85"><a href="#cb34-85" aria-hidden="true" tabindex="-1"></a><span class="ss">-   </span>Submitted Sample Form to Azenta</span>
<span id="cb34-86"><a href="#cb34-86" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb34-87"><a href="#cb34-87" aria-hidden="true" tabindex="-1"></a><span class="ss">-   </span>Writing &amp; editing chapter 1 methods</span>
<span id="cb34-88"><a href="#cb34-88" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb34-89"><a href="#cb34-89" aria-hidden="true" tabindex="-1"></a><span class="fu"># August 2024</span></span>
<span id="cb34-90"><a href="#cb34-90" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb34-91"><a href="#cb34-91" aria-hidden="true" tabindex="-1"></a><span class="fu">#### 28-AUG-2024</span></span>
<span id="cb34-92"><a href="#cb34-92" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb34-93"><a href="#cb34-93" aria-hidden="true" tabindex="-1"></a><span class="ss">-   </span>Received email confirmation of receipt of RNA at Azenta!</span>
<span id="cb34-94"><a href="#cb34-94" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb34-95"><a href="#cb34-95" aria-hidden="true" tabindex="-1"></a><span class="fu">#### 27-AUG-2024</span></span>
<span id="cb34-96"><a href="#cb34-96" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb34-97"><a href="#cb34-97" aria-hidden="true" tabindex="-1"></a><span class="ss">-   </span>Shipped Total RNA to Azenta in 96 well plate on \~16lbs of dry ice, FedEx Priority Overnight from SeaTac FedEx Office Ship Center</span>
<span id="cb34-98"><a href="#cb34-98" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb34-99"><a href="#cb34-99" aria-hidden="true" tabindex="-1"></a><span class="ss">-   </span>Tracking #: 278809181237</span>
<span id="cb34-100"><a href="#cb34-100" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb34-101"><a href="#cb34-101" aria-hidden="true" tabindex="-1"></a><span class="fu">#### 15-AUG-2024</span></span>
<span id="cb34-102"><a href="#cb34-102" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb34-103"><a href="#cb34-103" aria-hidden="true" tabindex="-1"></a><span class="ss">-   </span>Met with Jefferey Kim of Azenta GeneWiz</span>
<span id="cb34-104"><a href="#cb34-104" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb34-105"><a href="#cb34-105" aria-hidden="true" tabindex="-1"></a><span class="ss">    -   </span>Got a quote for 60 TotalRNA-seq with PolyA selection, 20M reads per sample</span>
<span id="cb34-106"><a href="#cb34-106" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb34-107"><a href="#cb34-107" aria-hidden="true" tabindex="-1"></a><span class="ss">    -   </span>For Total RNA in nuclease-free water on dry ice, need 100ng of Total RNA... so if I have 10ng/uL in 30 uL... that means there is 300 ng of Total RNA in that sample. Jeff said 10ng/uL is usually the cutoff. I can send Total eluted RNA in 30uL without normalizing or Nanodropping... they will run RIN scores when they receive the samples. I just have to extract and Qubit!</span>
<span id="cb34-108"><a href="#cb34-108" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb34-109"><a href="#cb34-109" aria-hidden="true" tabindex="-1"></a><span class="ss">    -   </span>23-28 business days turnaround time</span>
<span id="cb34-110"><a href="#cb34-110" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb34-111"><a href="#cb34-111" aria-hidden="true" tabindex="-1"></a><span class="ss">    -   </span>With discount, price per sample is \$98.50! Total cost is \$5,910.00 in quote :D</span>
<span id="cb34-112"><a href="#cb34-112" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb34-113"><a href="#cb34-113" aria-hidden="true" tabindex="-1"></a><span class="fu"># March 2024</span></span>
<span id="cb34-114"><a href="#cb34-114" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb34-115"><a href="#cb34-115" aria-hidden="true" tabindex="-1"></a><span class="fu">#### 28-MAR-2024</span></span>
<span id="cb34-116"><a href="#cb34-116" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb34-117"><a href="#cb34-117" aria-hidden="true" tabindex="-1"></a><span class="ss">-   </span>met with Kip and made calendar of assignments due</span>
<span id="cb34-118"><a href="#cb34-118" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb34-119"><a href="#cb34-119" aria-hidden="true" tabindex="-1"></a><span class="ss">-   </span>changed out filters on seawater filter rack in PBB</span>
<span id="cb34-120"><a href="#cb34-120" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb34-121"><a href="#cb34-121" aria-hidden="true" tabindex="-1"></a><span class="at">    &gt; To measure impacts of microplastic exposure on anemone size, oral disk diameters were calculated using the average of two perpendicular measurements across the oral surface intersecting at the mouth. Measurements were made using Image J software (Rasband 19972018) with digital photographs of each anemone .... - Beck, 2022 WWU Dissertation</span></span>
<span id="cb34-122"><a href="#cb34-122" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb34-123"><a href="#cb34-123" aria-hidden="true" tabindex="-1"></a><span class="ss">-   </span>How do we get scaled images of the oral disc?</span>
<span id="cb34-124"><a href="#cb34-124" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb34-125"><a href="#cb34-125" aria-hidden="true" tabindex="-1"></a><span class="co">&lt;!-- --&gt;</span></span>
<span id="cb34-126"><a href="#cb34-126" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb34-127"><a href="#cb34-127" aria-hidden="true" tabindex="-1"></a><span class="ss">-   </span>downloaded AmScope software for MU503B that is attached to the dissecting scope in PBB: <span class="ot">&lt;https://amscope.com/pages/software-downloads&gt;</span></span>
<span id="cb34-128"><a href="#cb34-128" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb34-129"><a href="#cb34-129" aria-hidden="true" tabindex="-1"></a><span class="ss">-   </span>downloaded AmScope manual: <span class="ot">&lt;https://storage.googleapis.com/software-download-d79bb.appspot.com/Manual%20Download%20Files/Manual%20Files/Cameras/MU_MD_Series_Camera_Software_Manual.pdf&gt;</span></span>
<span id="cb34-130"><a href="#cb34-130" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb34-131"><a href="#cb34-131" aria-hidden="true" tabindex="-1"></a><span class="fu">#### 21-MAR-2024</span></span>
<span id="cb34-132"><a href="#cb34-132" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb34-133"><a href="#cb34-133" aria-hidden="true" tabindex="-1"></a><span class="ss">-   </span>spent nearly all day drying an edge of each petri dish, and gluing bee-tags to them with superglue, then letting it dry by floating the petri dish on a floating rack in the seatable. Took all day..</span>
<span id="cb34-134"><a href="#cb34-134" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb34-135"><a href="#cb34-135" aria-hidden="true" tabindex="-1"></a><span class="ss">-   </span>swapped anemones from upper seatable to lower, changed water in upper, and letting it come down to temp (when doing water changes, cannot do it directly, as header water is at room temp and seatables are at 10C.... How will I deal with this during the experiment when each anemone is in a 200mL jar and I'll be doing water changes every other day</span>
<span id="cb34-136"><a href="#cb34-136" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb34-137"><a href="#cb34-137" aria-hidden="true" tabindex="-1"></a>    <span class="in">```{r}</span></span>
<span id="cb34-138"><a href="#cb34-138" aria-hidden="true" tabindex="-1"></a><span class="in">    # 72 anemones each in 200mL volume... how much volume I will need to change every other day</span></span>
<span id="cb34-139"><a href="#cb34-139" aria-hidden="true" tabindex="-1"></a><span class="in">    ml &lt;- 72*200</span></span>
<span id="cb34-140"><a href="#cb34-140" aria-hidden="true" tabindex="-1"></a><span class="in">    print(ml)</span></span>
<span id="cb34-141"><a href="#cb34-141" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb34-142"><a href="#cb34-142" aria-hidden="true" tabindex="-1"></a><span class="in">    # how much of each treatment (seawater vs. leachate)</span></span>
<span id="cb34-143"><a href="#cb34-143" aria-hidden="true" tabindex="-1"></a><span class="in">    ml/2</span></span>
<span id="cb34-144"><a href="#cb34-144" aria-hidden="true" tabindex="-1"></a><span class="in">    ```</span></span>
<span id="cb34-145"><a href="#cb34-145" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb34-146"><a href="#cb34-146" aria-hidden="true" tabindex="-1"></a>    The amount of microplastic may be our limiting factor for how long the exposure is.</span>
<span id="cb34-147"><a href="#cb34-147" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb34-148"><a href="#cb34-148" aria-hidden="true" tabindex="-1"></a>We have 3 grams of each microplastic type</span>
<span id="cb34-149"><a href="#cb34-149" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb34-150"><a href="#cb34-150" aria-hidden="true" tabindex="-1"></a>To make a batch of leachate according to the Saliu Lab protocol, we have 250 mg of microplastic.</span>
<span id="cb34-151"><a href="#cb34-151" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb34-154"><a href="#cb34-154" aria-hidden="true" tabindex="-1"></a><span class="in">```{r}</span></span>
<span id="cb34-155"><a href="#cb34-155" aria-hidden="true" tabindex="-1"></a><span class="co"># 3 grams is 3000 mg, how many batches of leachate will each microplastic type make?</span></span>
<span id="cb34-156"><a href="#cb34-156" aria-hidden="true" tabindex="-1"></a><span class="dv">3000</span><span class="sc">/</span><span class="dv">250</span></span>
<span id="cb34-157"><a href="#cb34-157" aria-hidden="true" tabindex="-1"></a><span class="in">```</span></span>
<span id="cb34-158"><a href="#cb34-158" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb34-159"><a href="#cb34-159" aria-hidden="true" tabindex="-1"></a>12 batches... that's a lot, and should be plenty since it is a highly concentrated amount that will be diluted.</span>
<span id="cb34-160"><a href="#cb34-160" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb34-161"><a href="#cb34-161" aria-hidden="true" tabindex="-1"></a>How will it be diluted?</span>
<span id="cb34-162"><a href="#cb34-162" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb34-163"><a href="#cb34-163" aria-hidden="true" tabindex="-1"></a><span class="fu">#### 20-MAR-2024</span></span>
<span id="cb34-164"><a href="#cb34-164" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb34-165"><a href="#cb34-165" aria-hidden="true" tabindex="-1"></a><span class="ss">-   </span>cleaned anemones by picking rocks and debris from them, and squirting with chilled seawater from a lab squeezie bottle. Two were 'escapees', one was fully out of the petri dish! I popped it back in. The ones that are nudging around the edge/lip of the petri dish I use my finger to just push their pedal disc back inside the dish... not sure what I can use to deter them from escape that won't expose them to more chemicals (i.e. tape ahdesive, etc.)</span>
<span id="cb34-166"><a href="#cb34-166" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb34-167"><a href="#cb34-167" aria-hidden="true" tabindex="-1"></a><span class="fu"># February 2024</span></span>
<span id="cb34-168"><a href="#cb34-168" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb34-169"><a href="#cb34-169" aria-hidden="true" tabindex="-1"></a><span class="fu">#### 22-FEB-2024</span></span>
<span id="cb34-170"><a href="#cb34-170" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb34-171"><a href="#cb34-171" aria-hidden="true" tabindex="-1"></a><span class="ss">-   </span>Roberts Lab meeting, received feedback on anemone research proposal: suggested I do initial response curves to find at what temp A. elegantissima experience stress, at what microplastic leachate concentration do they experience stress? Measure response variables 'quickly' by respirometry and PAM. Can I make the single stress, multiple response variable, the first chapter of the Master's thesis? Then, two stressors together with <span class="sc">\`</span>omics response variables (once we find the inflection points of stress) can be the second chapter.... Ariana pointed out that the paper reviewers are going to want to know why I chose certain concentrations of microplastic leachate. I have to justify that by doing a response curve first. Learned about Roberts Lab pubathon.</span>
<span id="cb34-172"><a href="#cb34-172" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb34-173"><a href="#cb34-173" aria-hidden="true" tabindex="-1"></a><span class="ss">-   </span>Took FireSting O2 to IT to try and get correct drivers on <span class="in">`mariscos`</span> ruggedized laptop to run the program...</span>
<span id="cb34-174"><a href="#cb34-174" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb34-175"><a href="#cb34-175" aria-hidden="true" tabindex="-1"></a><span class="ss">-   </span>pyroscience FireStingO2 deemed 'dead' by IT specialist Michael Shriver... the unit won't connect. I sent the following to the Walz product support team Todd Kana (kana\@bayinstruments.com) &amp; Erich Keim (ekeim\@walz.com):</span>
<span id="cb34-176"><a href="#cb34-176" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb34-177"><a href="#cb34-177" aria-hidden="true" tabindex="-1"></a><span class="at">    &gt; Good afternoon&nbsp; Erich and Todd,&nbsp;</span></span>
<span id="cb34-178"><a href="#cb34-178" aria-hidden="true" tabindex="-1"></a><span class="at">    &gt;</span></span>
<span id="cb34-179"><a href="#cb34-179" aria-hidden="true" tabindex="-1"></a><span class="at">    &gt; I'm a graduate student researcher in the Padilla-Gamino Lab at the University of Washington, and I'm working with a Walz Mini PAM-II and FireStingO2 oxygen meter that our lab purchased from you as a set. The instrument was previously used by Miranda Roethler, who passed your contact information on to me.&nbsp;</span></span>
<span id="cb34-180"><a href="#cb34-180" aria-hidden="true" tabindex="-1"></a><span class="at">    &gt;</span></span>
<span id="cb34-181"><a href="#cb34-181" aria-hidden="true" tabindex="-1"></a><span class="at">    &gt; The FireStingO2 oxygen meter is not powering up, and won't connect to any computer that we've tried it on. It was just with our IT specialist for the last few hours (he was attempting to get it to connect by downloading&nbsp;old drivers), and he shared that the laser-cut </span><span class="sc">\`</span><span class="at">FiresStingO2</span><span class="sc">\`</span><span class="at"> light on the instrument would 'pulse' red and then remain off. He deemed the instrument 'dead'...&nbsp;</span></span>
<span id="cb34-182"><a href="#cb34-182" aria-hidden="true" tabindex="-1"></a><span class="at">    &gt;</span></span>
<span id="cb34-183"><a href="#cb34-183" aria-hidden="true" tabindex="-1"></a><span class="at">    &gt; This instrument has only been used a handful of times, and it has been kept in good condition (not dropped, or been exposed/submerged in water etc.).&nbsp;</span></span>
<span id="cb34-184"><a href="#cb34-184" aria-hidden="true" tabindex="-1"></a><span class="at">    &gt;</span></span>
<span id="cb34-185"><a href="#cb34-185" aria-hidden="true" tabindex="-1"></a><span class="at">    &gt; Have you had this issue before? If so, are there any solutions you can help provide?</span></span>
<span id="cb34-186"><a href="#cb34-186" aria-hidden="true" tabindex="-1"></a><span class="at">    &gt;</span></span>
<span id="cb34-187"><a href="#cb34-187" aria-hidden="true" tabindex="-1"></a><span class="at">    &gt; Kindy,&nbsp;</span></span>
<span id="cb34-188"><a href="#cb34-188" aria-hidden="true" tabindex="-1"></a><span class="at">    &gt;</span></span>
<span id="cb34-189"><a href="#cb34-189" aria-hidden="true" tabindex="-1"></a><span class="at">    &gt; </span><span class="sc">\-</span><span class="at"> Sarah Tanja&nbsp;</span></span>
<span id="cb34-190"><a href="#cb34-190" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb34-191"><a href="#cb34-191" aria-hidden="true" tabindex="-1"></a><span class="fu">#### 21-FEB-2024</span></span>
<span id="cb34-192"><a href="#cb34-192" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb34-193"><a href="#cb34-193" aria-hidden="true" tabindex="-1"></a><span class="ss">-   </span>pyroscience FireStingO2 only works on Windows 10!?</span>
<span id="cb34-194"><a href="#cb34-194" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb34-195"><a href="#cb34-195" aria-hidden="true" tabindex="-1"></a><span class="ss">-   </span>Ammonia levels in PBB header tank are at 1ppm, something green is growing in there. When was the last time we checked that the UV sterilizer was working? I should change out all the filters on that seawater purification rig, and the reverse osmosis deionized water filter cascade.</span>
<span id="cb34-196"><a href="#cb34-196" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb34-197"><a href="#cb34-197" aria-hidden="true" tabindex="-1"></a><span class="ss">-   </span>There is algae growing in the PBB header/holding tank</span>
<span id="cb34-198"><a href="#cb34-198" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb34-199"><a href="#cb34-199" aria-hidden="true" tabindex="-1"></a><span class="ss">-   </span>Met with Kip, Added Kip to WDFW scientific collection permit</span>
<span id="cb34-200"><a href="#cb34-200" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb34-201"><a href="#cb34-201" aria-hidden="true" tabindex="-1"></a><span class="ss">-   </span>9 permit locations!</span>
<span id="cb34-202"><a href="#cb34-202" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb34-203"><a href="#cb34-203" aria-hidden="true" tabindex="-1"></a><span class="ss">-   </span>Messed around with mapping the collection sites, from R Graph Gallery, <span class="ot">&lt;https://r-graph-gallery.com/180-change-background-in-leaflet-map.html&gt;</span></span>
<span id="cb34-204"><a href="#cb34-204" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb34-205"><a href="#cb34-205" aria-hidden="true" tabindex="-1"></a><span class="ss">-   </span>All map background by leaflet can be found here: <span class="ot">&lt;https://leaflet-extras.github.io/leaflet-providers/preview/index.html&gt;</span></span>
<span id="cb34-206"><a href="#cb34-206" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb34-207"><a href="#cb34-207" aria-hidden="true" tabindex="-1"></a>Interactive Map with <span class="in">`leaflet`</span></span>
<span id="cb34-208"><a href="#cb34-208" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb34-209"><a href="#cb34-209" aria-hidden="true" tabindex="-1"></a><span class="in">```{r echo=TRUE, cache=TRUE}</span></span>
<span id="cb34-210"><a href="#cb34-210" aria-hidden="true" tabindex="-1"></a><span class="in">## set CRAN mirror</span></span>
<span id="cb34-211"><a href="#cb34-211" aria-hidden="true" tabindex="-1"></a><span class="in">options(repos = c(CRAN = "https://cran.r-project.org/"))</span></span>
<span id="cb34-212"><a href="#cb34-212" aria-hidden="true" tabindex="-1"></a><span class="in">```</span></span>
<span id="cb34-213"><a href="#cb34-213" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb34-214"><a href="#cb34-214" aria-hidden="true" tabindex="-1"></a><span class="in">```{r echo=TRUE, cache=TRUE}</span></span>
<span id="cb34-215"><a href="#cb34-215" aria-hidden="true" tabindex="-1"></a><span class="in">## install packages</span></span>
<span id="cb34-216"><a href="#cb34-216" aria-hidden="true" tabindex="-1"></a><span class="in">if (!require("tidyverse", character.only = TRUE)) { install.packages("tidyverse")}</span></span>
<span id="cb34-217"><a href="#cb34-217" aria-hidden="true" tabindex="-1"></a><span class="in">if (!require("leaflet", character.only = TRUE)){ install.packages("leaflet")}</span></span>
<span id="cb34-218"><a href="#cb34-218" aria-hidden="true" tabindex="-1"></a><span class="in">```</span></span>
<span id="cb34-219"><a href="#cb34-219" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb34-220"><a href="#cb34-220" aria-hidden="true" tabindex="-1"></a><span class="in">```{r echo=TRUE, cache=TRUE}</span></span>
<span id="cb34-221"><a href="#cb34-221" aria-hidden="true" tabindex="-1"></a><span class="in">## load libraries</span></span>
<span id="cb34-222"><a href="#cb34-222" aria-hidden="true" tabindex="-1"></a><span class="in">library(tidyverse)</span></span>
<span id="cb34-223"><a href="#cb34-223" aria-hidden="true" tabindex="-1"></a><span class="in">library(leaflet)</span></span>
<span id="cb34-224"><a href="#cb34-224" aria-hidden="true" tabindex="-1"></a><span class="in">```</span></span>
<span id="cb34-225"><a href="#cb34-225" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb34-226"><a href="#cb34-226" aria-hidden="true" tabindex="-1"></a><span class="in">```{r message=FALSE, warning=FALSE}</span></span>
<span id="cb34-227"><a href="#cb34-227" aria-hidden="true" tabindex="-1"></a><span class="in">grey &lt;- leaflet() %&gt;% </span></span>
<span id="cb34-228"><a href="#cb34-228" aria-hidden="true" tabindex="-1"></a><span class="in">   addTiles() %&gt;% </span></span>
<span id="cb34-229"><a href="#cb34-229" aria-hidden="true" tabindex="-1"></a><span class="in">   setView( lng = -122.8, lat = 47.9, zoom = 8 ) %&gt;% </span></span>
<span id="cb34-230"><a href="#cb34-230" aria-hidden="true" tabindex="-1"></a><span class="in">   addProviderTiles("Esri.WorldGrayCanvas")</span></span>
<span id="cb34-231"><a href="#cb34-231" aria-hidden="true" tabindex="-1"></a><span class="in">grey</span></span>
<span id="cb34-232"><a href="#cb34-232" aria-hidden="true" tabindex="-1"></a><span class="in">```</span></span>
<span id="cb34-233"><a href="#cb34-233" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb34-234"><a href="#cb34-234" aria-hidden="true" tabindex="-1"></a><span class="in">```{r message=FALSE, warning=FALSE}</span></span>
<span id="cb34-235"><a href="#cb34-235" aria-hidden="true" tabindex="-1"></a><span class="in">terrain &lt;- leaflet() %&gt;% </span></span>
<span id="cb34-236"><a href="#cb34-236" aria-hidden="true" tabindex="-1"></a><span class="in">   addTiles() %&gt;% </span></span>
<span id="cb34-237"><a href="#cb34-237" aria-hidden="true" tabindex="-1"></a><span class="in">   setView( lng = -122.8, lat = 47.9, zoom = 8 ) %&gt;% </span></span>
<span id="cb34-238"><a href="#cb34-238" aria-hidden="true" tabindex="-1"></a><span class="in">   addProviderTiles("Esri.WorldTerrain")</span></span>
<span id="cb34-239"><a href="#cb34-239" aria-hidden="true" tabindex="-1"></a><span class="in">terrain</span></span>
<span id="cb34-240"><a href="#cb34-240" aria-hidden="true" tabindex="-1"></a><span class="in">```</span></span>
<span id="cb34-241"><a href="#cb34-241" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb34-242"><a href="#cb34-242" aria-hidden="true" tabindex="-1"></a><span class="in">```{r message=FALSE, warning=FALSE}</span></span>
<span id="cb34-243"><a href="#cb34-243" aria-hidden="true" tabindex="-1"></a><span class="in">Stadia.AlidadeSmooth &lt;- leaflet() %&gt;% </span></span>
<span id="cb34-244"><a href="#cb34-244" aria-hidden="true" tabindex="-1"></a><span class="in">   addTiles() %&gt;% </span></span>
<span id="cb34-245"><a href="#cb34-245" aria-hidden="true" tabindex="-1"></a><span class="in">   setView( lng = -122.8, lat = 47.9, zoom = 8 ) %&gt;% </span></span>
<span id="cb34-246"><a href="#cb34-246" aria-hidden="true" tabindex="-1"></a><span class="in">   addProviderTiles("Stadia.AlidadeSmooth")</span></span>
<span id="cb34-247"><a href="#cb34-247" aria-hidden="true" tabindex="-1"></a><span class="in">Stadia.AlidadeSmooth</span></span>
<span id="cb34-248"><a href="#cb34-248" aria-hidden="true" tabindex="-1"></a><span class="in">```</span></span>
<span id="cb34-249"><a href="#cb34-249" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb34-250"><a href="#cb34-250" aria-hidden="true" tabindex="-1"></a><span class="in">```{r message=FALSE, warning=FALSE}</span></span>
<span id="cb34-251"><a href="#cb34-251" aria-hidden="true" tabindex="-1"></a><span class="in">Stadia.OSMBright &lt;- leaflet() %&gt;% </span></span>
<span id="cb34-252"><a href="#cb34-252" aria-hidden="true" tabindex="-1"></a><span class="in">   addTiles() %&gt;% </span></span>
<span id="cb34-253"><a href="#cb34-253" aria-hidden="true" tabindex="-1"></a><span class="in">   setView( lng = -122.8, lat = 47.9, zoom = 8 ) %&gt;% </span></span>
<span id="cb34-254"><a href="#cb34-254" aria-hidden="true" tabindex="-1"></a><span class="in">   addProviderTiles("Stadia.OSMBright")</span></span>
<span id="cb34-255"><a href="#cb34-255" aria-hidden="true" tabindex="-1"></a><span class="in">Stadia.OSMBright</span></span>
<span id="cb34-256"><a href="#cb34-256" aria-hidden="true" tabindex="-1"></a><span class="in">```</span></span>
<span id="cb34-257"><a href="#cb34-257" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb34-258"><a href="#cb34-258" aria-hidden="true" tabindex="-1"></a><span class="in">```{r message=FALSE, warning=FALSE}</span></span>
<span id="cb34-259"><a href="#cb34-259" aria-hidden="true" tabindex="-1"></a><span class="in">Jawg.Lagoon &lt;- leaflet() %&gt;% </span></span>
<span id="cb34-260"><a href="#cb34-260" aria-hidden="true" tabindex="-1"></a><span class="in">   addTiles() %&gt;% </span></span>
<span id="cb34-261"><a href="#cb34-261" aria-hidden="true" tabindex="-1"></a><span class="in">   setView( lng = -122.8, lat = 47.9, zoom = 8 ) %&gt;% </span></span>
<span id="cb34-262"><a href="#cb34-262" aria-hidden="true" tabindex="-1"></a><span class="in">   addProviderTiles("Jawg.Lagoon")</span></span>
<span id="cb34-263"><a href="#cb34-263" aria-hidden="true" tabindex="-1"></a><span class="in">Jawg.Lagoon</span></span>
<span id="cb34-264"><a href="#cb34-264" aria-hidden="true" tabindex="-1"></a><span class="in">```</span></span>
<span id="cb34-265"><a href="#cb34-265" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb34-268"><a href="#cb34-268" aria-hidden="true" tabindex="-1"></a><span class="in">```{r}</span></span>
<span id="cb34-269"><a href="#cb34-269" aria-hidden="true" tabindex="-1"></a>Stadia.StamenWatercolor <span class="ot">&lt;-</span> <span class="fu">leaflet</span>() <span class="sc">%&gt;%</span> </span>
<span id="cb34-270"><a href="#cb34-270" aria-hidden="true" tabindex="-1"></a>   <span class="fu">addTiles</span>() <span class="sc">%&gt;%</span> </span>
<span id="cb34-271"><a href="#cb34-271" aria-hidden="true" tabindex="-1"></a>   <span class="fu">setView</span>( <span class="at">lng =</span> <span class="sc">-</span><span class="fl">122.8</span>, <span class="at">lat =</span> <span class="fl">47.9</span>, <span class="at">zoom =</span> <span class="dv">8</span> ) <span class="sc">%&gt;%</span> </span>
<span id="cb34-272"><a href="#cb34-272" aria-hidden="true" tabindex="-1"></a>   <span class="fu">addProviderTiles</span>(<span class="st">"Stadia.StamenWatercolor"</span>)</span>
<span id="cb34-273"><a href="#cb34-273" aria-hidden="true" tabindex="-1"></a>Stadia.StamenWatercolor</span>
<span id="cb34-274"><a href="#cb34-274" aria-hidden="true" tabindex="-1"></a><span class="in">```</span></span>
<span id="cb34-275"><a href="#cb34-275" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb34-276"><a href="#cb34-276" aria-hidden="true" tabindex="-1"></a><span class="ss">-   </span></span>
<span id="cb34-277"><a href="#cb34-277" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb34-278"><a href="#cb34-278" aria-hidden="true" tabindex="-1"></a><span class="ss">-   </span>emailed Hannah at Seattle Aquarium about seawater pickup for Friday FEB 23 from 10-4</span>
<span id="cb34-279"><a href="#cb34-279" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb34-280"><a href="#cb34-280" aria-hidden="true" tabindex="-1"></a><span class="ss">-   </span>reserved SAFS Truck for seawater pickup</span>
<span id="cb34-281"><a href="#cb34-281" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb34-282"><a href="#cb34-282" aria-hidden="true" tabindex="-1"></a><span class="fu">#### 20-FEB-2024</span></span>
<span id="cb34-283"><a href="#cb34-283" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb34-284"><a href="#cb34-284" aria-hidden="true" tabindex="-1"></a><span class="ss">-   </span>Downloaded pyroscience FireStingO2 Pyro Oxygen Logger</span>
<span id="cb34-285"><a href="#cb34-285" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb34-286"><a href="#cb34-286" aria-hidden="true" tabindex="-1"></a><span class="ss">    -   </span>First tried on my computer Owl, USB was not recognizing, so we're trying it on QuelpoToo</span>
<span id="cb34-287"><a href="#cb34-287" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb34-288"><a href="#cb34-288" aria-hidden="true" tabindex="-1"></a><span class="ss">    -   </span>The USB connection doesn't connect to QuelpoToo either (is it the cable?)</span>
<span id="cb34-289"><a href="#cb34-289" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb34-290"><a href="#cb34-290" aria-hidden="true" tabindex="-1"></a><span class="fu">#### **02-FEB-2024**</span></span>
<span id="cb34-291"><a href="#cb34-291" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb34-292"><a href="#cb34-292" aria-hidden="true" tabindex="-1"></a><span class="ss">-   </span>Follow-up with UW office of veterans affairs about debt</span>
<span id="cb34-293"><a href="#cb34-293" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb34-294"><a href="#cb34-294" aria-hidden="true" tabindex="-1"></a><span class="ss">-   </span>Lab meeting notes:</span>
<span id="cb34-295"><a href="#cb34-295" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb34-296"><a href="#cb34-296" aria-hidden="true" tabindex="-1"></a><span class="ss">    -   </span>move shaker table to lab 242 hood</span>
<span id="cb34-297"><a href="#cb34-297" aria-hidden="true" tabindex="-1"></a><span class="ss">    -   </span>move nitrogen needle lipid extractor to methanol/chloroform cabinet in 242</span>
<span id="cb34-298"><a href="#cb34-298" aria-hidden="true" tabindex="-1"></a><span class="ss">    -   </span>ok to buy organizational bins for PBB, keep receipts &amp; get reimbursed</span>
<span id="cb34-299"><a href="#cb34-299" aria-hidden="true" tabindex="-1"></a><span class="ss">    -   </span>respirometry chamber battery operation problematic... reduced flow, battery drain over time (30mins); Nuria left a plug-in chamber somewhere in PBB</span>
<span id="cb34-300"><a href="#cb34-300" aria-hidden="true" tabindex="-1"></a><span class="ss">    -   </span></span>
<span id="cb34-301"><a href="#cb34-301" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb34-302"><a href="#cb34-302" aria-hidden="true" tabindex="-1"></a><span class="ss">-   </span>Writing your resume for federal job webinar</span>
<span id="cb34-303"><a href="#cb34-303" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb34-304"><a href="#cb34-304" aria-hidden="true" tabindex="-1"></a><span class="ss">    -   </span>schedule A: disability accommodation</span>
<span id="cb34-305"><a href="#cb34-305" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb34-306"><a href="#cb34-306" aria-hidden="true" tabindex="-1"></a><span class="ss">    -   </span>Federal hiring resume, there is no page limit</span>
<span id="cb34-307"><a href="#cb34-307" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb34-308"><a href="#cb34-308" aria-hidden="true" tabindex="-1"></a><span class="ss">    -   </span>The resume gets you to the hiring manager, show in the resume that you qualify before it goes to a hiring manager</span>
<span id="cb34-309"><a href="#cb34-309" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb34-310"><a href="#cb34-310" aria-hidden="true" tabindex="-1"></a><span class="ss">    -   </span>hiring managers in Federal hiring do not click on links (github, linkedIn, etc.) they won't be viewed...</span>
<span id="cb34-311"><a href="#cb34-311" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb34-312"><a href="#cb34-312" aria-hidden="true" tabindex="-1"></a><span class="ss">    -   </span>'Your resume is the movie trailer...' Highlight the level of complexity of your work</span>
<span id="cb34-313"><a href="#cb34-313" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb34-314"><a href="#cb34-314" aria-hidden="true" tabindex="-1"></a><span class="ss">    -   </span>Itemize how how manage something, on top of listing your role... you have to explain what you did in that role. The role 'president of xx' doesn't translate to the person reviewing the resume knowing what you can do... leave nothing to the imagination. Be explicit.</span>
<span id="cb34-315"><a href="#cb34-315" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb34-316"><a href="#cb34-316" aria-hidden="true" tabindex="-1"></a><span class="ss">    -   </span>Spoken presentations: how many presentations you have given, what &amp; when</span>
<span id="cb34-317"><a href="#cb34-317" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb34-318"><a href="#cb34-318" aria-hidden="true" tabindex="-1"></a><span class="ss">    -   </span>The first person that looks at your resume is a human resources specialist, not a scientist!</span>
<span id="cb34-319"><a href="#cb34-319" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb34-320"><a href="#cb34-320" aria-hidden="true" tabindex="-1"></a><span class="ss">    -   </span>show that you are an 'innovator' and that when you see a problem, you take initiative to fix it.</span>
<span id="cb34-321"><a href="#cb34-321" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb34-322"><a href="#cb34-322" aria-hidden="true" tabindex="-1"></a><span class="ss">    -   </span>where you go, people are impacted positively, there is recognition, things change or improve, things get better, you save money, you meet deadlines, you bring about conflict resolution.</span>
<span id="cb34-323"><a href="#cb34-323" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb34-324"><a href="#cb34-324" aria-hidden="true" tabindex="-1"></a><span class="ss">    -   </span>the cover letter is not used to qualify you, ONLY THE RESUME!</span>
<span id="cb34-325"><a href="#cb34-325" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb34-326"><a href="#cb34-326" aria-hidden="true" tabindex="-1"></a><span class="ss">    -   </span>Vet team to assist navigating federal hiring process: fedshirevets\@opm.gov</span>
<span id="cb34-327"><a href="#cb34-327" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb34-328"><a href="#cb34-328" aria-hidden="true" tabindex="-1"></a><span class="ss">    -   </span>You get shuttled into qualified, well-qualified, best-qualified. Qualifying doesn't mean that you get referred to the hiring manager... you can qualify but not get referred to the hiring manager. You can be referred, but not get an interview.</span>
<span id="cb34-329"><a href="#cb34-329" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb34-330"><a href="#cb34-330" aria-hidden="true" tabindex="-1"></a><span class="ss">    -   </span>wrp workforce recruitment program</span>
<span id="cb34-331"><a href="#cb34-331" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb34-332"><a href="#cb34-332" aria-hidden="true" tabindex="-1"></a><span class="fu">#### 08-FEB-2024</span></span>
<span id="cb34-333"><a href="#cb34-333" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb34-334"><a href="#cb34-334" aria-hidden="true" tabindex="-1"></a><span class="ss">-   </span>attended Roberts Lab meeting</span>
<span id="cb34-335"><a href="#cb34-335" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb34-336"><a href="#cb34-336" aria-hidden="true" tabindex="-1"></a><span class="ss">    -   </span>Will be presenting experiment design &amp; proposal on Feb 22nd</span>
<span id="cb34-337"><a href="#cb34-337" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb34-338"><a href="#cb34-338" aria-hidden="true" tabindex="-1"></a><span class="ss">-   </span>worked on dropbox-uploader app in raven server to try and get data from dropbox directly to raven <span class="sc">\*</span>not working</span>
<span id="cb34-339"><a href="#cb34-339" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb34-340"><a href="#cb34-340" aria-hidden="true" tabindex="-1"></a><span class="fu">#### 07-FEB-2024</span></span>
<span id="cb34-341"><a href="#cb34-341" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb34-342"><a href="#cb34-342" aria-hidden="true" tabindex="-1"></a><span class="ss">-   </span>Identified anemones from Alki Beach have *Symbiodinium muscatinei*</span>
<span id="cb34-343"><a href="#cb34-343" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb34-344"><a href="#cb34-344" aria-hidden="true" tabindex="-1"></a><span class="ss">-   </span>Identified anemones from Teddy Bear Cove have *Symbiodinium muscatinei*</span>
<span id="cb34-345"><a href="#cb34-345" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb34-346"><a href="#cb34-346" aria-hidden="true" tabindex="-1"></a><span class="fu">#### 05-FEB-2024</span></span>
<span id="cb34-347"><a href="#cb34-347" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb34-348"><a href="#cb34-348" aria-hidden="true" tabindex="-1"></a><span class="ss">-   </span>Lab meeting cancelled</span>
<span id="cb34-349"><a href="#cb34-349" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb34-350"><a href="#cb34-350" aria-hidden="true" tabindex="-1"></a><span class="ss">-   </span>emailed Francesco Saliu asking about sending their lab SPME fibers for analysis</span>
<span id="cb34-351"><a href="#cb34-351" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb34-352"><a href="#cb34-352" aria-hidden="true" tabindex="-1"></a><span class="ss">    -   </span>still no package or protocol, starting to think about plan B/C/D...etc</span>
<span id="cb34-353"><a href="#cb34-353" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb34-354"><a href="#cb34-354" aria-hidden="true" tabindex="-1"></a><span class="ss">-   </span>found rnoaa package and zenodo by Wendell Raymond https://zenodo.org/records/6555076 from paper "<span class="co">[</span><span class="ot">Assessment of the impacts of an unprecedented heatwave on intertidal shellfish of the Salish Sea</span><span class="co">](https://esajournals.onlinelibrary.wiley.com/doi/10.1002/ecy.3798)</span>" recommended by Bob Oxborrow</span>
<span id="cb34-355"><a href="#cb34-355" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb34-356"><a href="#cb34-356" aria-hidden="true" tabindex="-1"></a><span class="ss">    -   </span>explored rnoaa package</span>
<span id="cb34-357"><a href="#cb34-357" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb34-358"><a href="#cb34-358" aria-hidden="true" tabindex="-1"></a><span class="ss">-   </span>13:30 - shadow fish 427 lab on photosynthesis and respiration with Miranda</span>
<span id="cb34-359"><a href="#cb34-359" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb34-360"><a href="#cb34-360" aria-hidden="true" tabindex="-1"></a>Goal for the week:</span>
<span id="cb34-361"><a href="#cb34-361" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb34-362"><a href="#cb34-362" aria-hidden="true" tabindex="-1"></a><span class="ss">-   </span>Get PAM photosynthesis initial data</span>
<span id="cb34-363"><a href="#cb34-363" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb34-364"><a href="#cb34-364" aria-hidden="true" tabindex="-1"></a><span class="fu">## January 2024</span></span>
<span id="cb34-365"><a href="#cb34-365" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb34-366"><a href="#cb34-366" aria-hidden="true" tabindex="-1"></a><span class="fu">## December 2023</span></span>
<span id="cb34-367"><a href="#cb34-367" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb34-368"><a href="#cb34-368" aria-hidden="true" tabindex="-1"></a><span class="fu">## November 2023</span></span>
<span id="cb34-369"><a href="#cb34-369" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb34-370"><a href="#cb34-370" aria-hidden="true" tabindex="-1"></a><span class="fu">#### 27-NOV-2023</span></span>
<span id="cb34-371"><a href="#cb34-371" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb34-372"><a href="#cb34-372" aria-hidden="true" tabindex="-1"></a><span class="ss">-   </span>Lab Meeting:</span>
<span id="cb34-373"><a href="#cb34-373" aria-hidden="true" tabindex="-1"></a><span class="ss">    -   </span>Requested to order cultured abalone sea urchins</span>
<span id="cb34-374"><a href="#cb34-374" aria-hidden="true" tabindex="-1"></a><span class="ss">    -   </span>Reminded JPG about meeting with Alison Gardell Tues Nov 28th at 1:30PM via Zoom</span>
<span id="cb34-375"><a href="#cb34-375" aria-hidden="true" tabindex="-1"></a><span class="ss">    -   </span>JPG requested I put together slides for meeting with Alison Gardell</span>
<span id="cb34-376"><a href="#cb34-376" aria-hidden="true" tabindex="-1"></a><span class="ss">-   </span>Got seawater from Seattle Aquarium</span>
<span id="cb34-377"><a href="#cb34-377" aria-hidden="true" tabindex="-1"></a><span class="ss">-   </span>Met with Kip and got them setup with safety trainings</span>
<span id="cb34-378"><a href="#cb34-378" aria-hidden="true" tabindex="-1"></a><span class="ss">-   </span>Must order sea urchins on Mon Dec 11th for a Mon Dec 18th/Tues Dec 19th delivery</span>
<span id="cb34-379"><a href="#cb34-379" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb34-380"><a href="#cb34-380" aria-hidden="true" tabindex="-1"></a><span class="fu">#### **21-NOV-2023**</span></span>
<span id="cb34-381"><a href="#cb34-381" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb34-382"><a href="#cb34-382" aria-hidden="true" tabindex="-1"></a><span class="ss">-   </span><span class="co">[</span><span class="ot">2nd Master's Thesis Committee Meeting</span><span class="co">](../committee-2/index.qmd)</span></span>
<span id="cb34-383"><a href="#cb34-383" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb34-384"><a href="#cb34-384" aria-hidden="true" tabindex="-1"></a><span class="ss">    -   </span>Trying to come up with a solution to lack of solvent control.. how to move forward</span>
<span id="cb34-385"><a href="#cb34-385" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb34-386"><a href="#cb34-386" aria-hidden="true" tabindex="-1"></a><span class="fu">#### 11-NOV-2023</span></span>
<span id="cb34-387"><a href="#cb34-387" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb34-388"><a href="#cb34-388" aria-hidden="true" tabindex="-1"></a><span class="ss">-   </span>WSN day 4</span>
<span id="cb34-389"><a href="#cb34-389" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb34-390"><a href="#cb34-390" aria-hidden="true" tabindex="-1"></a><span class="fu">#### 10-NOV-2023</span></span>
<span id="cb34-391"><a href="#cb34-391" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb34-392"><a href="#cb34-392" aria-hidden="true" tabindex="-1"></a><span class="ss">-   </span>WSN day 3</span>
<span id="cb34-393"><a href="#cb34-393" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb34-394"><a href="#cb34-394" aria-hidden="true" tabindex="-1"></a><span class="fu">#### 09-NOV 2023</span></span>
<span id="cb34-395"><a href="#cb34-395" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb34-396"><a href="#cb34-396" aria-hidden="true" tabindex="-1"></a><span class="ss">-   </span>WSN day 2</span>
<span id="cb34-397"><a href="#cb34-397" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb34-398"><a href="#cb34-398" aria-hidden="true" tabindex="-1"></a><span class="fu">#### 08-NOV-2023</span></span>
<span id="cb34-399"><a href="#cb34-399" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb34-400"><a href="#cb34-400" aria-hidden="true" tabindex="-1"></a><span class="ss">-   </span>WSN day 1</span>
<span id="cb34-401"><a href="#cb34-401" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb34-402"><a href="#cb34-402" aria-hidden="true" tabindex="-1"></a><span class="fu">#### 05-NOV-2023</span></span>
<span id="cb34-403"><a href="#cb34-403" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb34-404"><a href="#cb34-404" aria-hidden="true" tabindex="-1"></a><span class="ss">-   </span>Undoing large file commit on coral-pae-temp for \~2hrs 🤬, tried to just undo the one commit, but couldn't get it to work. Had to execute <span class="in">`git reset –hard origin/main`</span></span>
<span id="cb34-405"><a href="#cb34-405" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb34-406"><a href="#cb34-406" aria-hidden="true" tabindex="-1"></a><span class="ss">-   </span>Added the following to the <span class="in">`git/hooks/pre-commit`</span> file:</span>
<span id="cb34-407"><a href="#cb34-407" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb34-408"><a href="#cb34-408" aria-hidden="true" tabindex="-1"></a>    <span class="in">```         </span></span>
<span id="cb34-409"><a href="#cb34-409" aria-hidden="true" tabindex="-1"></a><span class="in">    #!/bin/bash</span></span>
<span id="cb34-410"><a href="#cb34-410" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb34-411"><a href="#cb34-411" aria-hidden="true" tabindex="-1"></a><span class="in">    echo "automatically ignoring large files"</span></span>
<span id="cb34-412"><a href="#cb34-412" aria-hidden="true" tabindex="-1"></a><span class="in">    find . -size 5M | sed 's|^\./||g' &gt;&gt; .gitignore</span></span>
<span id="cb34-413"><a href="#cb34-413" aria-hidden="true" tabindex="-1"></a><span class="in">    cat .gitignore | sort | uniq &gt; .gitignore</span></span>
<span id="cb34-414"><a href="#cb34-414" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb34-415"><a href="#cb34-415" aria-hidden="true" tabindex="-1"></a><span class="in">    git diff --exit-code .gitignore</span></span>
<span id="cb34-416"><a href="#cb34-416" aria-hidden="true" tabindex="-1"></a><span class="in">    exit_status=$?</span></span>
<span id="cb34-417"><a href="#cb34-417" aria-hidden="true" tabindex="-1"></a><span class="in">    if [ $exit_status -eq 1 ]</span></span>
<span id="cb34-418"><a href="#cb34-418" aria-hidden="true" tabindex="-1"></a><span class="in">    then</span></span>
<span id="cb34-419"><a href="#cb34-419" aria-hidden="true" tabindex="-1"></a><span class="in">        set +e</span></span>
<span id="cb34-420"><a href="#cb34-420" aria-hidden="true" tabindex="-1"></a><span class="in">        for i in `cat .gitignore`</span></span>
<span id="cb34-421"><a href="#cb34-421" aria-hidden="true" tabindex="-1"></a><span class="in">        do</span></span>
<span id="cb34-422"><a href="#cb34-422" aria-hidden="true" tabindex="-1"></a><span class="in">        set +e</span></span>
<span id="cb34-423"><a href="#cb34-423" aria-hidden="true" tabindex="-1"></a><span class="in">            git rm --cached $i</span></span>
<span id="cb34-424"><a href="#cb34-424" aria-hidden="true" tabindex="-1"></a><span class="in">        done</span></span>
<span id="cb34-425"><a href="#cb34-425" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb34-426"><a href="#cb34-426" aria-hidden="true" tabindex="-1"></a><span class="in">        git add .gitignore</span></span>
<span id="cb34-427"><a href="#cb34-427" aria-hidden="true" tabindex="-1"></a><span class="in">        git commit .gitignore --no-verify -m"ignoring large files"</span></span>
<span id="cb34-428"><a href="#cb34-428" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb34-429"><a href="#cb34-429" aria-hidden="true" tabindex="-1"></a><span class="in">        echo "ignored new large files"</span></span>
<span id="cb34-430"><a href="#cb34-430" aria-hidden="true" tabindex="-1"></a><span class="in">    fi</span></span>
<span id="cb34-431"><a href="#cb34-431" aria-hidden="true" tabindex="-1"></a><span class="in">    ```</span></span>
<span id="cb34-432"><a href="#cb34-432" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb34-433"><a href="#cb34-433" aria-hidden="true" tabindex="-1"></a><span class="fu">#### 03-NOV-2023</span></span>
<span id="cb34-434"><a href="#cb34-434" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb34-435"><a href="#cb34-435" aria-hidden="true" tabindex="-1"></a><span class="ss">-   </span>Finished extraction, nanodrop &amp; qubit for 12 samples</span>
<span id="cb34-436"><a href="#cb34-436" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb34-437"><a href="#cb34-437" aria-hidden="true" tabindex="-1"></a><span class="ss">    -   </span>6MH1a</span>
<span id="cb34-438"><a href="#cb34-438" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb34-439"><a href="#cb34-439" aria-hidden="true" tabindex="-1"></a><span class="ss">    -   </span>10OH1a - greenish, bad! no DNA/RNA, degraded!</span>
<span id="cb34-440"><a href="#cb34-440" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb34-441"><a href="#cb34-441" aria-hidden="true" tabindex="-1"></a><span class="ss">    -   </span>6PA2a</span>
<span id="cb34-442"><a href="#cb34-442" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb34-443"><a href="#cb34-443" aria-hidden="true" tabindex="-1"></a><span class="ss">    -   </span>5HH1a</span>
<span id="cb34-444"><a href="#cb34-444" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb34-445"><a href="#cb34-445" aria-hidden="true" tabindex="-1"></a><span class="ss">    -   </span>5HA1a</span>
<span id="cb34-446"><a href="#cb34-446" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb34-447"><a href="#cb34-447" aria-hidden="true" tabindex="-1"></a><span class="ss">    -   </span>9OA2a</span>
<span id="cb34-448"><a href="#cb34-448" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb34-449"><a href="#cb34-449" aria-hidden="true" tabindex="-1"></a><span class="ss">    -   </span>7HA1a</span>
<span id="cb34-450"><a href="#cb34-450" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb34-451"><a href="#cb34-451" aria-hidden="true" tabindex="-1"></a><span class="ss">    -   </span>3CA1a</span>
<span id="cb34-452"><a href="#cb34-452" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb34-453"><a href="#cb34-453" aria-hidden="true" tabindex="-1"></a><span class="ss">    -   </span>8CA1a</span>
<span id="cb34-454"><a href="#cb34-454" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb34-455"><a href="#cb34-455" aria-hidden="true" tabindex="-1"></a><span class="ss">    -   </span>2OA1a</span>
<span id="cb34-456"><a href="#cb34-456" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb34-457"><a href="#cb34-457" aria-hidden="true" tabindex="-1"></a><span class="ss">    -   </span>2MH1a</span>
<span id="cb34-458"><a href="#cb34-458" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb34-459"><a href="#cb34-459" aria-hidden="true" tabindex="-1"></a><span class="ss">    -   </span>5CA2a</span>
<span id="cb34-460"><a href="#cb34-460" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb34-461"><a href="#cb34-461" aria-hidden="true" tabindex="-1"></a><span class="fu">#### 02-NOV-2023</span></span>
<span id="cb34-462"><a href="#cb34-462" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb34-463"><a href="#cb34-463" aria-hidden="true" tabindex="-1"></a><span class="ss">-   </span>Started extraction of 12 samples, paused after adding 350uL of DNA/RNA Lysis Buffer to 350uL of PK-Digested sample supernatent, and frozen in -80 freezer:</span>
<span id="cb34-464"><a href="#cb34-464" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb34-465"><a href="#cb34-465" aria-hidden="true" tabindex="-1"></a><span class="ss">    -   </span>6MH1a</span>
<span id="cb34-466"><a href="#cb34-466" aria-hidden="true" tabindex="-1"></a><span class="ss">    -   </span>10OH1a - green</span>
<span id="cb34-467"><a href="#cb34-467" aria-hidden="true" tabindex="-1"></a><span class="ss">    -   </span>6PA2a</span>
<span id="cb34-468"><a href="#cb34-468" aria-hidden="true" tabindex="-1"></a><span class="ss">    -   </span>5HH1a</span>
<span id="cb34-469"><a href="#cb34-469" aria-hidden="true" tabindex="-1"></a><span class="ss">    -   </span>5HA1a</span>
<span id="cb34-470"><a href="#cb34-470" aria-hidden="true" tabindex="-1"></a><span class="ss">    -   </span>9OA2a</span>
<span id="cb34-471"><a href="#cb34-471" aria-hidden="true" tabindex="-1"></a><span class="ss">    -   </span>7HA1a</span>
<span id="cb34-472"><a href="#cb34-472" aria-hidden="true" tabindex="-1"></a><span class="ss">    -   </span>3CA1a</span>
<span id="cb34-473"><a href="#cb34-473" aria-hidden="true" tabindex="-1"></a><span class="ss">    -   </span>8CA1a</span>
<span id="cb34-474"><a href="#cb34-474" aria-hidden="true" tabindex="-1"></a><span class="ss">    -   </span>2OA1a</span>
<span id="cb34-475"><a href="#cb34-475" aria-hidden="true" tabindex="-1"></a><span class="ss">    -   </span>2MH1a</span>
<span id="cb34-476"><a href="#cb34-476" aria-hidden="true" tabindex="-1"></a><span class="ss">    -   </span>5CA2a - green</span>
<span id="cb34-477"><a href="#cb34-477" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb34-478"><a href="#cb34-478" aria-hidden="true" tabindex="-1"></a><span class="ss">-   </span>Finished qubit of 10 samples:</span>
<span id="cb34-479"><a href="#cb34-479" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb34-480"><a href="#cb34-480" aria-hidden="true" tabindex="-1"></a><span class="ss">    -   </span>1MH2a</span>
<span id="cb34-481"><a href="#cb34-481" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb34-482"><a href="#cb34-482" aria-hidden="true" tabindex="-1"></a><span class="ss">    -   </span>2LA1a</span>
<span id="cb34-483"><a href="#cb34-483" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb34-484"><a href="#cb34-484" aria-hidden="true" tabindex="-1"></a><span class="ss">    -   </span>1MA1a</span>
<span id="cb34-485"><a href="#cb34-485" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb34-486"><a href="#cb34-486" aria-hidden="true" tabindex="-1"></a><span class="ss">    -   </span>1OH2a</span>
<span id="cb34-487"><a href="#cb34-487" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb34-488"><a href="#cb34-488" aria-hidden="true" tabindex="-1"></a><span class="ss">    -   </span>6PH2a</span>
<span id="cb34-489"><a href="#cb34-489" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb34-490"><a href="#cb34-490" aria-hidden="true" tabindex="-1"></a><span class="ss">    -   </span>10LH1a</span>
<span id="cb34-491"><a href="#cb34-491" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb34-492"><a href="#cb34-492" aria-hidden="true" tabindex="-1"></a><span class="ss">    -   </span>9LA2a</span>
<span id="cb34-493"><a href="#cb34-493" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb34-494"><a href="#cb34-494" aria-hidden="true" tabindex="-1"></a><span class="ss">    -   </span>8CH2a</span>
<span id="cb34-495"><a href="#cb34-495" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb34-496"><a href="#cb34-496" aria-hidden="true" tabindex="-1"></a><span class="ss">    -   </span>8OA1a</span>
<span id="cb34-497"><a href="#cb34-497" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb34-498"><a href="#cb34-498" aria-hidden="true" tabindex="-1"></a><span class="ss">    -   </span>5OH1a</span>
<span id="cb34-499"><a href="#cb34-499" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb34-500"><a href="#cb34-500" aria-hidden="true" tabindex="-1"></a><span class="fu">#### 01-NOV-2023</span></span>
<span id="cb34-501"><a href="#cb34-501" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb34-502"><a href="#cb34-502" aria-hidden="true" tabindex="-1"></a><span class="ss">-   </span>Finished extraction &amp; nanodrop of 10 samples:</span>
<span id="cb34-503"><a href="#cb34-503" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb34-504"><a href="#cb34-504" aria-hidden="true" tabindex="-1"></a><span class="ss">    -   </span>1MH2a</span>
<span id="cb34-505"><a href="#cb34-505" aria-hidden="true" tabindex="-1"></a><span class="ss">    -   </span>2LA1a</span>
<span id="cb34-506"><a href="#cb34-506" aria-hidden="true" tabindex="-1"></a><span class="ss">    -   </span>1MA1a</span>
<span id="cb34-507"><a href="#cb34-507" aria-hidden="true" tabindex="-1"></a><span class="ss">    -   </span>1OH2a</span>
<span id="cb34-508"><a href="#cb34-508" aria-hidden="true" tabindex="-1"></a><span class="ss">    -   </span>6PH2a</span>
<span id="cb34-509"><a href="#cb34-509" aria-hidden="true" tabindex="-1"></a><span class="ss">    -   </span>10LH1a</span>
<span id="cb34-510"><a href="#cb34-510" aria-hidden="true" tabindex="-1"></a><span class="ss">    -   </span>9LA2a</span>
<span id="cb34-511"><a href="#cb34-511" aria-hidden="true" tabindex="-1"></a><span class="ss">    -   </span>8CH2a</span>
<span id="cb34-512"><a href="#cb34-512" aria-hidden="true" tabindex="-1"></a><span class="ss">    -   </span>8OA1a</span>
<span id="cb34-513"><a href="#cb34-513" aria-hidden="true" tabindex="-1"></a><span class="ss">    -   </span>5OH1a</span>
<span id="cb34-514"><a href="#cb34-514" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb34-515"><a href="#cb34-515" aria-hidden="true" tabindex="-1"></a><span class="ss">-   </span>GSS Presentation Design Workshop with Tim Essington</span>
<span id="cb34-516"><a href="#cb34-516" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb34-517"><a href="#cb34-517" aria-hidden="true" tabindex="-1"></a><span class="ss">    -   </span>Presentations should have ONE THING they are communicating: 'Here's a nugget'</span>
<span id="cb34-518"><a href="#cb34-518" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb34-519"><a href="#cb34-519" aria-hidden="true" tabindex="-1"></a><span class="ss">    -   </span>One Word Theme</span>
<span id="cb34-520"><a href="#cb34-520" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb34-521"><a href="#cb34-521" aria-hidden="true" tabindex="-1"></a><span class="ss">    -   </span>One Sentence Summary</span>
<span id="cb34-522"><a href="#cb34-522" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb34-523"><a href="#cb34-523" aria-hidden="true" tabindex="-1"></a><span class="ss">    -   </span>Add motion to a few slides to break things up visually</span>
<span id="cb34-524"><a href="#cb34-524" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb34-525"><a href="#cb34-525" aria-hidden="true" tabindex="-1"></a><span class="ss">    -   </span>Use a blank slide and pause at a key pivot in your story</span>
<span id="cb34-526"><a href="#cb34-526" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb34-527"><a href="#cb34-527" aria-hidden="true" tabindex="-1"></a><span class="ss">    -   </span>build the slide iteratively when you have a complex figure</span>
<span id="cb34-528"><a href="#cb34-528" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb34-529"><a href="#cb34-529" aria-hidden="true" tabindex="-1"></a><span class="ss">    -   </span>story arc:</span>
<span id="cb34-530"><a href="#cb34-530" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb34-531"><a href="#cb34-531" aria-hidden="true" tabindex="-1"></a><span class="ss">        -   </span>ordinary world</span>
<span id="cb34-532"><a href="#cb34-532" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb34-533"><a href="#cb34-533" aria-hidden="true" tabindex="-1"></a><span class="ss">        -   </span>challenge</span>
<span id="cb34-534"><a href="#cb34-534" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb34-535"><a href="#cb34-535" aria-hidden="true" tabindex="-1"></a><span class="ss">        -   </span>series of actions to overcome challenge</span>
<span id="cb34-536"><a href="#cb34-536" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb34-537"><a href="#cb34-537" aria-hidden="true" tabindex="-1"></a><span class="ss">        -   </span>resolution</span>
<span id="cb34-538"><a href="#cb34-538" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb34-539"><a href="#cb34-539" aria-hidden="true" tabindex="-1"></a><span class="ss">        -   </span>new world</span>
<span id="cb34-540"><a href="#cb34-540" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb34-541"><a href="#cb34-541" aria-hidden="true" tabindex="-1"></a><span class="ss">    -   </span>'Lean into your 'buts'</span>
<span id="cb34-542"><a href="#cb34-542" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb34-543"><a href="#cb34-543" aria-hidden="true" tabindex="-1"></a><span class="ss">    -   </span>leave silence + space between story arcs</span>
<span id="cb34-544"><a href="#cb34-544" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb34-545"><a href="#cb34-545" aria-hidden="true" tabindex="-1"></a><span class="fu">## October 2023</span></span>
<span id="cb34-546"><a href="#cb34-546" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb34-547"><a href="#cb34-547" aria-hidden="true" tabindex="-1"></a><span class="fu">#### **27-OCT-2023**</span></span>
<span id="cb34-548"><a href="#cb34-548" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb34-549"><a href="#cb34-549" aria-hidden="true" tabindex="-1"></a><span class="ss">-   </span>QA/QC'd samples from 26-OCT-2023</span>
<span id="cb34-550"><a href="#cb34-550" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb34-551"><a href="#cb34-551" aria-hidden="true" tabindex="-1"></a><span class="ss">-   </span>Met with Kip &amp; moved experiment setup to PBB</span>
<span id="cb34-552"><a href="#cb34-552" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb34-553"><a href="#cb34-553" aria-hidden="true" tabindex="-1"></a><span class="ss">-   </span>Collected glycerol from Jon Bakker at Horticultural Center Merill Hall</span>
<span id="cb34-554"><a href="#cb34-554" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb34-555"><a href="#cb34-555" aria-hidden="true" tabindex="-1"></a><span class="fu">#### **26-OCT-2023**</span></span>
<span id="cb34-556"><a href="#cb34-556" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb34-557"><a href="#cb34-557" aria-hidden="true" tabindex="-1"></a><span class="ss">-   </span>extracted samples:</span>
<span id="cb34-558"><a href="#cb34-558" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb34-559"><a href="#cb34-559" aria-hidden="true" tabindex="-1"></a><span class="ss">    -   </span>6LH1a</span>
<span id="cb34-560"><a href="#cb34-560" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb34-561"><a href="#cb34-561" aria-hidden="true" tabindex="-1"></a><span class="ss">    -   </span>8LA2a</span>
<span id="cb34-562"><a href="#cb34-562" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb34-563"><a href="#cb34-563" aria-hidden="true" tabindex="-1"></a><span class="ss">    -   </span>10CH2a</span>
<span id="cb34-564"><a href="#cb34-564" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb34-565"><a href="#cb34-565" aria-hidden="true" tabindex="-1"></a><span class="ss">    -   </span>5CH2a</span>
<span id="cb34-566"><a href="#cb34-566" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb34-567"><a href="#cb34-567" aria-hidden="true" tabindex="-1"></a><span class="ss">    -   </span>4HA2a</span>
<span id="cb34-568"><a href="#cb34-568" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb34-569"><a href="#cb34-569" aria-hidden="true" tabindex="-1"></a><span class="ss">    -   </span>7OA2a</span>
<span id="cb34-570"><a href="#cb34-570" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb34-571"><a href="#cb34-571" aria-hidden="true" tabindex="-1"></a><span class="ss">    -   </span>2MA2a</span>
<span id="cb34-572"><a href="#cb34-572" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb34-573"><a href="#cb34-573" aria-hidden="true" tabindex="-1"></a><span class="ss">    -   </span>3LH1a</span>
<span id="cb34-574"><a href="#cb34-574" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb34-575"><a href="#cb34-575" aria-hidden="true" tabindex="-1"></a><span class="ss">    -   </span>5LA1a</span>
<span id="cb34-576"><a href="#cb34-576" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb34-577"><a href="#cb34-577" aria-hidden="true" tabindex="-1"></a><span class="ss">    -   </span>8MH2a</span>
<span id="cb34-578"><a href="#cb34-578" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb34-579"><a href="#cb34-579" aria-hidden="true" tabindex="-1"></a><span class="fu">#### 24-OCT-2023</span></span>
<span id="cb34-580"><a href="#cb34-580" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb34-581"><a href="#cb34-581" aria-hidden="true" tabindex="-1"></a><span class="ss">-   </span>working on WSN presentation!</span>
<span id="cb34-582"><a href="#cb34-582" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb34-583"><a href="#cb34-583" aria-hidden="true" tabindex="-1"></a><span class="fu">#### 23-OCT-2023</span></span>
<span id="cb34-584"><a href="#cb34-584" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb34-585"><a href="#cb34-585" aria-hidden="true" tabindex="-1"></a><span class="ss">-   </span>JPG 1on1</span>
<span id="cb34-586"><a href="#cb34-586" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb34-587"><a href="#cb34-587" aria-hidden="true" tabindex="-1"></a><span class="ss">-   </span>Bought Amtrak tickets to WSN (refundable)</span>
<span id="cb34-588"><a href="#cb34-588" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb34-589"><a href="#cb34-589" aria-hidden="true" tabindex="-1"></a><span class="fu">#### 16-OCT-2023</span></span>
<span id="cb34-590"><a href="#cb34-590" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb34-591"><a href="#cb34-591" aria-hidden="true" tabindex="-1"></a><span class="ss">-   </span>attended lab meeting via Zoom</span>
<span id="cb34-592"><a href="#cb34-592" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb34-593"><a href="#cb34-593" aria-hidden="true" tabindex="-1"></a>"Total RNA was extracted using the Duet DNA/RNA Miniprep Plus Kit (Zymo Research, Irvine, CA, USA, Cat #D7003) with modifications to the manufacturer's sample preparation steps, as described here. Sterile clippers were used to remove an approximately 1-cm diameter region from the coral tissue. The tissue clipping was placed in 500&nbsp;μL of DNA/RNA shield (Zymo Cat #R1100-50) with 0.5&nbsp;mm glass beads (Fisher Scientific Cat #15-340-152) and homogenized by vortexing for 1&nbsp;minute. The supernatant (450&nbsp;μl) was transferred to a new tube and centrifuged at 9000&nbsp;rcf for 3&nbsp;min to remove any debris. The supernatant (300&nbsp;μl) was mixed with 30&nbsp;μl of Proteinase K (Zymo Cat #D3001-2-20) digestion buffer and 15&nbsp;μl of Proteinase K in a new tube using a brief vortex step. Equal volume of Zymo lysis buffer (Zymo Cat #D7001-1-200) was added (345&nbsp;μl) and extraction was completed as outlined in the manufacturer's protocol. RNA concentration was quantified using the Qubit RNA Broad Range kit (Invitrogen Cat #Q10211) and integrity was assayed with an Agilent TapeStation 4200 system and the RNA Integrity Number (RIN) values ranged from 7.0--8.9. RNA libraries were prepared using the Zymo-Seq RiboFree Total RNA Library Kit (Zymo Cat #R3000) according to the manufacturer's instructions, starting with 125&nbsp;ng of RNA as input. Prepared libraries were sent to Zymo Research Corporation for sequencing using the Illumina HiSeq for paired end sequencing (2&nbsp;×&nbsp;100&nbsp;base pairs <span class="sc">\[</span>bp<span class="sc">\]</span>)." - Wong &amp; Putnam 2022 <span class="co">[</span><span class="ot">@wong2022</span><span class="co">]</span></span>
<span id="cb34-594"><a href="#cb34-594" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb34-595"><a href="#cb34-595" aria-hidden="true" tabindex="-1"></a><span class="fu">#### 11-OCT-2023</span></span>
<span id="cb34-596"><a href="#cb34-596" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb34-597"><a href="#cb34-597" aria-hidden="true" tabindex="-1"></a><span class="ss">-   </span>Attended <span class="co">[</span><span class="ot">ISB</span><span class="co">](https://isbscience.org/)</span> workshop day 1</span>
<span id="cb34-598"><a href="#cb34-598" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb34-599"><a href="#cb34-599" aria-hidden="true" tabindex="-1"></a><span class="ss">    -   </span>Gibbons-Lab github repo: <span class="ot">&lt;https://github.com/Gibbons-Lab/isb_course_2023/tree/main&gt;</span></span>
<span id="cb34-600"><a href="#cb34-600" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb34-601"><a href="#cb34-601" aria-hidden="true" tabindex="-1"></a><span class="ss">    -   </span>Course presentation slides: <span class="ot">&lt;https://gibbons-lab.github.io/isb_course_2023/16S/&gt;</span></span>
<span id="cb34-602"><a href="#cb34-602" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb34-603"><a href="#cb34-603" aria-hidden="true" tabindex="-1"></a><span class="ss">    -   </span>Multinomial Naive Bayes <span class="in">`qiime feature-classifier classify-sklearn`</span> trained on V4 region of the 16S marker gene</span>
<span id="cb34-604"><a href="#cb34-604" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb34-605"><a href="#cb34-605" aria-hidden="true" tabindex="-1"></a><span class="ss">    -   </span>center log ratio transform abundance counts</span>
<span id="cb34-606"><a href="#cb34-606" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb34-607"><a href="#cb34-607" aria-hidden="true" tabindex="-1"></a><span class="ss">    -   </span>recommended papers:</span>
<span id="cb34-608"><a href="#cb34-608" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb34-609"><a href="#cb34-609" aria-hidden="true" tabindex="-1"></a><span class="ss">        -   </span><span class="co">[</span><span class="ot">**Microbiome differential abundance methods produce different results across 38 datasets**</span><span class="co">](https://www.nature.com/articles/s41467-022-28034-z)</span></span>
<span id="cb34-610"><a href="#cb34-610" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb34-611"><a href="#cb34-611" aria-hidden="true" tabindex="-1"></a><span class="ss">        -   </span><span class="co">[</span><span class="ot">**Beta diversity (or Ordination) analysis**</span><span class="co">](https://www.futurelearn.com/info/courses/exploring-the-landscape-of-antibiotic-resistance-in-microbiomes/0/steps/272960#:~:text=PCoA%20and%20NMDS%20methods%20take,to%20fit%20a%20normal%20distribution)</span>"PCoA and NMDS methods take the distance matrix as an input and are sensitive to distance method used. The PCoA maximises the linear correlation between samples, wherein NMDS maximises the rank-order correlation between samples. Additionally, in case of NMDS, data is not required to fit a normal distribution." This link has some good visualizations: <span class="ot">&lt;https://www.davidzeleny.net/anadat-r/doku.php/en:pcoa_nmds&gt;</span></span>
<span id="cb34-612"><a href="#cb34-612" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb34-613"><a href="#cb34-613" aria-hidden="true" tabindex="-1"></a><span class="ss">        -   </span>Why is the mean being used instead of the median or the modal? Bacteria distribution is not a normal distribution across populations.... This is a log ratio of (relative abundance/geometric mean). It's a common method for transforming compositional data. For sequencing data normalization does not mean making anything normally distributed but rather putting samples on the same scale... Here's a review on the topic: <span class="ot">&lt;https://www.nature.com/articles/s41522-020-00160-w&gt;</span></span>
<span id="cb34-614"><a href="#cb34-614" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb34-615"><a href="#cb34-615" aria-hidden="true" tabindex="-1"></a>            <span class="al">![Nature](https://slack-imgs.com/?c=1&amp;o1=wi32.he32.si&amp;url=https%253A%252F%252Fwww.nature.com%252Fstatic%252Fimages%252Ffavicons%252Fnature%252Fapple-touch-icon-f39cb19454.png)</span>**Nature**</span>
<span id="cb34-616"><a href="#cb34-616" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb34-617"><a href="#cb34-617" aria-hidden="true" tabindex="-1"></a>            <span class="co">[</span><span class="ot">**Analysis of microbial compositions: a review of normalization and differential abundance analysis**</span><span class="co">](https://www.nature.com/articles/s41522-020-00160-w)</span></span>
<span id="cb34-618"><a href="#cb34-618" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb34-619"><a href="#cb34-619" aria-hidden="true" tabindex="-1"></a><span class="fu">#### 10-OCT-2023</span></span>
<span id="cb34-620"><a href="#cb34-620" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb34-621"><a href="#cb34-621" aria-hidden="true" tabindex="-1"></a><span class="ss">-   </span>Bought WSN flights</span>
<span id="cb34-622"><a href="#cb34-622" aria-hidden="true" tabindex="-1"></a><span class="ss">-   </span>Sent list of 123 samples to UW mim_c for formal quote</span>
<span id="cb34-623"><a href="#cb34-623" aria-hidden="true" tabindex="-1"></a><span class="ss">-   </span>filled out GI Bill enrollment verification</span>
<span id="cb34-624"><a href="#cb34-624" aria-hidden="true" tabindex="-1"></a><span class="ss">-   </span>updated all extraction notebook posts from handwritten notes</span>
<span id="cb34-625"><a href="#cb34-625" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb34-626"><a href="#cb34-626" aria-hidden="true" tabindex="-1"></a><span class="fu">#### 03-OCT-2023</span></span>
<span id="cb34-627"><a href="#cb34-627" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb34-628"><a href="#cb34-628" aria-hidden="true" tabindex="-1"></a><span class="ss">-   </span>Submitted abstract to WSN, registered for conference!</span>
<span id="cb34-629"><a href="#cb34-629" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb34-630"><a href="#cb34-630" aria-hidden="true" tabindex="-1"></a><span class="fu">#### 02-OCT-2023</span></span>
<span id="cb34-631"><a href="#cb34-631" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb34-632"><a href="#cb34-632" aria-hidden="true" tabindex="-1"></a><span class="ss">-   </span>Abstratc accepted to SICB for Jan 3rd!</span>
<span id="cb34-633"><a href="#cb34-633" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb34-634"><a href="#cb34-634" aria-hidden="true" tabindex="-1"></a><span class="ss">-   </span>&lt;div&gt;</span>
<span id="cb34-635"><a href="#cb34-635" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb34-636"><a href="#cb34-636" aria-hidden="true" tabindex="-1"></a><span class="at">    &gt; Your abstract Corals in a hot plastic ocean: How heat and phthalates shift a Montipora capitata coral microbiome has been accepted for the 2024 Annual Meeting in Seattle, WA,&nbsp;on&nbsp;2-6 January 2024. Your abstract id is 1230.\</span></span>
<span id="cb34-637"><a href="#cb34-637" aria-hidden="true" tabindex="-1"></a><span class="at">    &gt; \</span></span>
<span id="cb34-638"><a href="#cb34-638" aria-hidden="true" tabindex="-1"></a><span class="at">    &gt; You presentation is in the&nbsp;Response to anthropogenic environmental changes Concurrent session on&nbsp;Wednesday,&nbsp;January 03, 2024.</span></span>
<span id="cb34-639"><a href="#cb34-639" aria-hidden="true" tabindex="-1"></a><span class="at">    &gt;</span></span>
<span id="cb34-640"><a href="#cb34-640" aria-hidden="true" tabindex="-1"></a><span class="at">    &gt; The online program can be found </span><span class="co">[</span><span class="ot">HERE</span><span class="co">](https://urldefense.com/v3/__https://mandrillapp.com/track/click/30774040/www.xcdsystem.com?p=eyJzIjoiVHg1NGFGcEZRZmMyYWdNaEdPOTREWjRMcGQwIiwidiI6MSwicCI6IntcInVcIjozMDc3NDA0MCxcInZcIjoxLFwidXJsXCI6XCJodHRwczpcXFwvXFxcL3d3dy54Y2RzeXN0ZW0uY29tXFxcL3NpY2JcXFwvcHJvZ3JhbVxcXC9JMUtyMjN0XFxcL2luZGV4LmNmbT9wZ2lkPTEzNDImUnVuUmVtb3ZlU2Vzc2lvbkZpbHRlcj0xXCIsXCJpZFwiOlwiZWZiNjZmNTg1MGQ2NDFjMzk1NjU5ZjQ2MjJjMmMzZTZcIixcInVybF9pZHNcIjpbXCI4MDZmOTNlZTNiZmY5ZTM3ODhhMDM1MmE4YzhkODkzMmM1OTFlOTgxXCJdfSJ9__;!!K-Hz7m0Vt54!h-p1pLv7irLQW3sPow2nFi6Q2BxoJRfyoVmE8uN9VbspkSR__iihdpN1iPfG-49elEJFsqnVnF5IW-lY$)</span><span class="at">. If you need to make a change to your abstract or cancel it you MUST fill out this </span><span class="co">[</span><span class="ot">FORM</span><span class="co">](https://urldefense.com/v3/__https://mandrillapp.com/track/click/30774040/docs.google.com?p=eyJzIjoiZ0VMSHdvazI5OGJDRWMyMV81NEtlcHZuMWgwIiwidiI6MSwicCI6IntcInVcIjozMDc3NDA0MCxcInZcIjoxLFwidXJsXCI6XCJodHRwczpcXFwvXFxcL2RvY3MuZ29vZ2xlLmNvbVxcXC9mb3Jtc1xcXC9kXFxcL2VcXFwvMUZBSXBRTFNkbmRxdTBwRERoLUhySVBZUXdIZUp6S29fYlRldzJsb2xVeEdSV1Q3STZ1ZWwwZndcXFwvdmlld2Zvcm1cIixcImlkXCI6XCJlZmI2NmY1ODUwZDY0MWMzOTU2NTlmNDYyMmMyYzNlNlwiLFwidXJsX2lkc1wiOltcIjNhOTc5YWRhMTZmM2RiNDliODQ4NDdlOTQ2ZmI2NDk5MzllZDY3MDNcIl19In0__;!!K-Hz7m0Vt54!h-p1pLv7irLQW3sPow2nFi6Q2BxoJRfyoVmE8uN9VbspkSR__iihdpN1iPfG-49elEJFsqnVnJHbFgyn$)</span><span class="at">.&nbsp; This is the only way changes and cancellations will be accepted.\</span></span>
<span id="cb34-641"><a href="#cb34-641" aria-hidden="true" tabindex="-1"></a><span class="at">    &gt; \</span></span>
<span id="cb34-642"><a href="#cb34-642" aria-hidden="true" tabindex="-1"></a><span class="at">    &gt; Information regarding presentations at the SICB meeting can be found </span><span class="co">[</span><span class="ot">HERE</span><span class="co">](https://urldefense.com/v3/__https://mandrillapp.com/track/click/30774040/www.xcdsystem.com?p=eyJzIjoiTjJ3UFUxQ25vcmR1RWZGN093bUtaZXpBU3l3IiwidiI6MSwicCI6IntcInVcIjozMDc3NDA0MCxcInZcIjoxLFwidXJsXCI6XCJodHRwczpcXFwvXFxcL3d3dy54Y2RzeXN0ZW0uY29tXFxcL3NpY2JcXFwvcHJvZ3JhbVxcXC9JMUtyMjN0XFxcL2luZGV4LmNmbT9wZ2lkPTEzNjJcIixcImlkXCI6XCJlZmI2NmY1ODUwZDY0MWMzOTU2NTlmNDYyMmMyYzNlNlwiLFwidXJsX2lkc1wiOltcIjgwNmY5M2VlM2JmZjllMzc4OGEwMzUyYThjOGQ4OTMyYzU5MWU5ODFcIl19In0__;!!K-Hz7m0Vt54!h-p1pLv7irLQW3sPow2nFi6Q2BxoJRfyoVmE8uN9VbspkSR__iihdpN1iPfG-49elEJFsqnVnMuFu7ui$)</span><span class="at">.</span></span>
<span id="cb34-643"><a href="#cb34-643" aria-hidden="true" tabindex="-1"></a><span class="at">    &gt;</span></span>
<span id="cb34-644"><a href="#cb34-644" aria-hidden="true" tabindex="-1"></a><span class="at">    &gt; Please note that all presenters must register for the meeting by the presenter registration deadline of 1 November 2023, otherwise your talk may be cancelled. Click </span><span class="co">[</span><span class="ot">HERE</span><span class="co">](https://urldefense.com/v3/__https://mandrillapp.com/track/click/30774040/www.xcdsystem.com?p=eyJzIjoiaVNESXlTUjZ6anQ1Mi1FaS1Fa1dBZG50VjRJIiwidiI6MSwicCI6IntcInVcIjozMDc3NDA0MCxcInZcIjoxLFwidXJsXCI6XCJodHRwczpcXFwvXFxcL3d3dy54Y2RzeXN0ZW0uY29tXFxcL3NpY2JcXFwvcHJvZ3JhbVxcXC81WDlPSWJVXFxcL2luZGV4LmNmbT9wZ2lkPTQ0OFwiLFwiaWRcIjpcImVmYjY2ZjU4NTBkNjQxYzM5NTY1OWY0NjIyYzJjM2U2XCIsXCJ1cmxfaWRzXCI6W1wiNzY1MmM2ZTg0NzcxMDQ3MWFiZmNiZDQ0MGE1OGQ2NTFkNTg5MTgyNFwiXX0ifQ__;!!K-Hz7m0Vt54!h-p1pLv7irLQW3sPow2nFi6Q2BxoJRfyoVmE8uN9VbspkSR__iihdpN1iPfG-49elEJFsqnVnACQAtql$)</span><span class="at"> to register. Please visit the meeting web site for complete information on registering and conference activities. If you have registration questions email </span><span class="co">[</span><span class="ot">registration\@sicb.org</span><span class="co">](mailto:MScarbrough@burkinc.com)</span><span class="at">, for meeting questions email </span><span class="co">[</span><span class="ot">lstrong\@burkinc.com</span><span class="co">](mailto:lstrong@burkinc.com)</span><span class="at">.</span></span>
<span id="cb34-645"><a href="#cb34-645" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb34-646"><a href="#cb34-646" aria-hidden="true" tabindex="-1"></a>    &lt;/div&gt;</span>
<span id="cb34-647"><a href="#cb34-647" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb34-648"><a href="#cb34-648" aria-hidden="true" tabindex="-1"></a><span class="fu">## September 2023</span></span>
<span id="cb34-649"><a href="#cb34-649" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb34-650"><a href="#cb34-650" aria-hidden="true" tabindex="-1"></a><span class="fu">#### 20-SEP-2023</span></span>
<span id="cb34-651"><a href="#cb34-651" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb34-652"><a href="#cb34-652" aria-hidden="true" tabindex="-1"></a><span class="ss">-   </span><span class="co">[</span><span class="ot">Hidden Toxic Histories</span><span class="co">](https://theconversation.com/the-importance-of-shining-a-light-on-hidden-toxic-histories-211657?utm_medium=email&amp;utm_campaign=Daily%20newsletter%20September%2014%202023&amp;utm_content=Daily%20newsletter%20September%2014%202023+CID_1c20b166b5da888ab34925d477cebf8d&amp;utm_source=campaign_monitor_us&amp;utm_term=the%20artists%20and%20activists%20who%20are%20calling%20attention%20to%20toxic%20legacies)</span></span>
<span id="cb34-653"><a href="#cb34-653" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb34-654"><a href="#cb34-654" aria-hidden="true" tabindex="-1"></a><span class="fu">#### 05-SEP-2023</span></span>
<span id="cb34-655"><a href="#cb34-655" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb34-656"><a href="#cb34-656" aria-hidden="true" tabindex="-1"></a><span class="ss">-   </span>WSN newsletter says abstracts close Sep 29th 2023</span>
<span id="cb34-657"><a href="#cb34-657" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb34-658"><a href="#cb34-658" aria-hidden="true" tabindex="-1"></a><span class="ss">-   </span>5 min lightning talk or 15 minute talk</span>
<span id="cb34-659"><a href="#cb34-659" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb34-660"><a href="#cb34-660" aria-hidden="true" tabindex="-1"></a><span class="ss">-   </span><span class="co">[</span><span class="ot">WSN presenter guidelines</span><span class="co">](https://www.wsn-online.org/presenter-guidelines/)</span></span>
<span id="cb34-661"><a href="#cb34-661" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb34-662"><a href="#cb34-662" aria-hidden="true" tabindex="-1"></a><span class="ss">-   </span>Abstract Guidelines</span>
<span id="cb34-663"><a href="#cb34-663" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb34-664"><a href="#cb34-664" aria-hidden="true" tabindex="-1"></a><span class="co">&lt;!-- --&gt;</span></span>
<span id="cb34-665"><a href="#cb34-665" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb34-666"><a href="#cb34-666" aria-hidden="true" tabindex="-1"></a><span class="in">```         </span></span>
<span id="cb34-667"><a href="#cb34-667" aria-hidden="true" tabindex="-1"></a><span class="in">The abstract should briefly introduce the project, the goals, the results, and the significance</span></span>
<span id="cb34-668"><a href="#cb34-668" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb34-669"><a href="#cb34-669" aria-hidden="true" tabindex="-1"></a><span class="in">The presentation title should have no more than 130 characters max (including spaces) in ALL UPPERCASE</span></span>
<span id="cb34-670"><a href="#cb34-670" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb34-671"><a href="#cb34-671" aria-hidden="true" tabindex="-1"></a><span class="in">The abstract should have no more than 1600 characters max (including spaces); use standard HTML tags for special formatting (e.g., \&lt;b\&gt;**bold text**\&lt;/b\&gt;, \&lt;i\&gt;*italic text*\&lt;/i\&gt;, etc.)</span></span>
<span id="cb34-672"><a href="#cb34-672" aria-hidden="true" tabindex="-1"></a><span class="in">```</span></span>
<span id="cb34-673"><a href="#cb34-673" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb34-674"><a href="#cb34-674" aria-hidden="true" tabindex="-1"></a><span class="fu">#### 01-SEP-2023</span></span>
<span id="cb34-675"><a href="#cb34-675" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb34-676"><a href="#cb34-676" aria-hidden="true" tabindex="-1"></a><span class="ss">-   </span>crashed my bike and badly injured my hands... can't do lab work 😭</span>
<span id="cb34-677"><a href="#cb34-677" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb34-678"><a href="#cb34-678" aria-hidden="true" tabindex="-1"></a><span class="ss">-   </span>met with UW mim_c Steven Salipantes</span>
<span id="cb34-679"><a href="#cb34-679" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb34-680"><a href="#cb34-680" aria-hidden="true" tabindex="-1"></a><span class="ss">    -   </span>I need to send him a sample list for a final quote, but should be \$2K for \~100-120 samples (cost of sequencing lane + reagents) .. averages to \~\$20/sample, much better than Mr. DNA \~\$85/sample, plus no shipping charges.</span>
<span id="cb34-681"><a href="#cb34-681" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb34-682"><a href="#cb34-682" aria-hidden="true" tabindex="-1"></a><span class="ss">    -   </span>bring all 100-120 samples together</span>
<span id="cb34-683"><a href="#cb34-683" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb34-684"><a href="#cb34-684" aria-hidden="true" tabindex="-1"></a><span class="ss">    -   </span>1.5mL microcentrifuge tubes</span>
<span id="cb34-685"><a href="#cb34-685" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb34-686"><a href="#cb34-686" aria-hidden="true" tabindex="-1"></a><span class="ss">    -   </span>50uL volume , 20ng/uL quantity concentration</span>
<span id="cb34-687"><a href="#cb34-687" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb34-688"><a href="#cb34-688" aria-hidden="true" tabindex="-1"></a><span class="fu">## August 2023</span></span>
<span id="cb34-689"><a href="#cb34-689" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb34-690"><a href="#cb34-690" aria-hidden="true" tabindex="-1"></a><span class="fu">#### 31-AUG-2023</span></span>
<span id="cb34-691"><a href="#cb34-691" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb34-692"><a href="#cb34-692" aria-hidden="true" tabindex="-1"></a><span class="ss">-   </span>purchased additional kits, qubit assays, tubes, etc. to spend down Hall Grant</span>
<span id="cb34-693"><a href="#cb34-693" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb34-694"><a href="#cb34-694" aria-hidden="true" tabindex="-1"></a><span class="ss">-   </span>received Qubit and UV-C Sterilizer!</span>
<span id="cb34-695"><a href="#cb34-695" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb34-696"><a href="#cb34-696" aria-hidden="true" tabindex="-1"></a><span class="ss">-   </span>extracted:</span>
<span id="cb34-697"><a href="#cb34-697" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb34-698"><a href="#cb34-698" aria-hidden="true" tabindex="-1"></a><span class="ss">    -   </span>mock</span>
<span id="cb34-699"><a href="#cb34-699" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb34-700"><a href="#cb34-700" aria-hidden="true" tabindex="-1"></a><span class="ss">    -   </span>blank</span>
<span id="cb34-701"><a href="#cb34-701" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb34-702"><a href="#cb34-702" aria-hidden="true" tabindex="-1"></a><span class="ss">    -   </span>4-LA2a</span>
<span id="cb34-703"><a href="#cb34-703" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb34-704"><a href="#cb34-704" aria-hidden="true" tabindex="-1"></a><span class="ss">    -   </span>1-HH2a</span>
<span id="cb34-705"><a href="#cb34-705" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb34-706"><a href="#cb34-706" aria-hidden="true" tabindex="-1"></a><span class="ss">    -   </span>2-HH2a</span>
<span id="cb34-707"><a href="#cb34-707" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb34-708"><a href="#cb34-708" aria-hidden="true" tabindex="-1"></a><span class="ss">    -   </span>3-HA2a</span>
<span id="cb34-709"><a href="#cb34-709" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb34-710"><a href="#cb34-710" aria-hidden="true" tabindex="-1"></a><span class="ss">    -   </span>3-MA2a</span>
<span id="cb34-711"><a href="#cb34-711" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb34-712"><a href="#cb34-712" aria-hidden="true" tabindex="-1"></a><span class="ss">    -   </span>1-Ha2a</span>
<span id="cb34-713"><a href="#cb34-713" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb34-714"><a href="#cb34-714" aria-hidden="true" tabindex="-1"></a><span class="ss">    -   </span>4-MA2a</span>
<span id="cb34-715"><a href="#cb34-715" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb34-716"><a href="#cb34-716" aria-hidden="true" tabindex="-1"></a><span class="ss">    -   </span>3-HA2a</span>
<span id="cb34-717"><a href="#cb34-717" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb34-718"><a href="#cb34-718" aria-hidden="true" tabindex="-1"></a><span class="fu">#### 29-AUG-2023</span></span>
<span id="cb34-719"><a href="#cb34-719" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb34-720"><a href="#cb34-720" aria-hidden="true" tabindex="-1"></a><span class="ss">-   </span>submitted SICB abstract</span>
<span id="cb34-721"><a href="#cb34-721" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb34-722"><a href="#cb34-722" aria-hidden="true" tabindex="-1"></a><span class="ss">-   </span>submitted Kip's capstone proposal to Greg Jensen</span>
<span id="cb34-723"><a href="#cb34-723" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb34-724"><a href="#cb34-724" aria-hidden="true" tabindex="-1"></a><span class="fu">#### 28-AUG-2023</span></span>
<span id="cb34-725"><a href="#cb34-725" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb34-726"><a href="#cb34-726" aria-hidden="true" tabindex="-1"></a><span class="ss">-   </span>Look into the <span class="co">[</span><span class="ot">Margaret A. Davidson Graduate Fellowship</span><span class="co">](https://coast.noaa.gov/nerrs/research/davidson-fellowship.html)</span></span>
<span id="cb34-727"><a href="#cb34-727" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb34-728"><a href="#cb34-728" aria-hidden="true" tabindex="-1"></a><span class="ss">-   </span>Wrote abstract for <span class="co">[</span><span class="ot">SICB</span><span class="co">](https://sicb.org/)</span> , Seattle Jan 2-6 ... Deadline is Aug 29th!</span>
<span id="cb34-729"><a href="#cb34-729" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb34-730"><a href="#cb34-730" aria-hidden="true" tabindex="-1"></a><span class="ss">-   </span>Met with UW mim_c Kyle Asfahl</span>
<span id="cb34-731"><a href="#cb34-731" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb34-732"><a href="#cb34-732" aria-hidden="true" tabindex="-1"></a><span class="ss">-   </span>balanced Hall budget, have \~\$5K left to spend</span>
<span id="cb34-733"><a href="#cb34-733" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb34-734"><a href="#cb34-734" aria-hidden="true" tabindex="-1"></a><span class="ss">-   </span>Michael Shriever reset medusaFTIR and downloaded Nanodrop software to it</span>
<span id="cb34-735"><a href="#cb34-735" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb34-736"><a href="#cb34-736" aria-hidden="true" tabindex="-1"></a><span class="ss">-   </span>Take Jeremy's samples out of freezer to thaw at 4PM</span>
<span id="cb34-737"><a href="#cb34-737" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb34-738"><a href="#cb34-738" aria-hidden="true" tabindex="-1"></a><span class="fu">#### 24-AUG-2023</span></span>
<span id="cb34-739"><a href="#cb34-739" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb34-740"><a href="#cb34-740" aria-hidden="true" tabindex="-1"></a><span class="ss">-   </span>Trial with 2 ambient samples of modified extraction protocol and RNA Clean &amp; Concentrator. I did 2 samples because I had a kit that had 2 preps left over. Most of the time spent is in lab bench sterilizing, prepping, labelling, getting ice! Which takes literally a 30 minute hike across campus... But I started at 930AM and finished with Qubit in the FTR Roberts Lab by 4PM. With 2 samples! Extractions take ALL DAY.</span>
<span id="cb34-741"><a href="#cb34-741" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb34-742"><a href="#cb34-742" aria-hidden="true" tabindex="-1"></a><span class="fu">#### 17-AUG-2023</span></span>
<span id="cb34-743"><a href="#cb34-743" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb34-744"><a href="#cb34-744" aria-hidden="true" tabindex="-1"></a><span class="ss">-   </span>Are we sure we want to try TriZol for RNA? It isn't a phenol-free kit like the Quick DNA/RNA Miniprep, and we have RNA, we just need to purify it. Tomorrow lets run all eluted RNA through RNA Clean &amp; Concentrator, and check it on Nanodrop! LFG.</span>
<span id="cb34-745"><a href="#cb34-745" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb34-746"><a href="#cb34-746" aria-hidden="true" tabindex="-1"></a><span class="ss">-   </span>**THE BLANK HAD 'TOO LOW' DNA IN QUBIT!**</span>
<span id="cb34-747"><a href="#cb34-747" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb34-748"><a href="#cb34-748" aria-hidden="true" tabindex="-1"></a><span class="ss">-   </span>Organized extraction notes (handwritten) for extractions 1-8</span>
<span id="cb34-749"><a href="#cb34-749" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb34-750"><a href="#cb34-750" aria-hidden="true" tabindex="-1"></a><span class="ss">-   </span>Reserved 1 King Bed in Hotel Block for WSN</span>
<span id="cb34-751"><a href="#cb34-751" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb34-752"><a href="#cb34-752" aria-hidden="true" tabindex="-1"></a><span class="ss">-   </span>Applied for WSN Student Travel Grant</span>
<span id="cb34-753"><a href="#cb34-753" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb34-754"><a href="#cb34-754" aria-hidden="true" tabindex="-1"></a><span class="at">    &gt; **How will attending the WSN meeting help you advance your career as a naturalist and/or scientist (75 words max)?**</span></span>
<span id="cb34-755"><a href="#cb34-755" aria-hidden="true" tabindex="-1"></a><span class="at">    &gt;</span></span>
<span id="cb34-756"><a href="#cb34-756" aria-hidden="true" tabindex="-1"></a><span class="at">    &gt; Attending WSN would be my first opportunity to share my Masters thesis with the scientific community. As an undergrad, I had a bad first conference experience where I was ridiculed for not having results in a methods poster. I hope to attend WSN because its reputation as a welcoming environment for early-career scientists is the kind of scientific community I wish to be a part of and perpetuate in my own career.</span></span>
<span id="cb34-757"><a href="#cb34-757" aria-hidden="true" tabindex="-1"></a><span class="at">    &gt;</span></span>
<span id="cb34-758"><a href="#cb34-758" aria-hidden="true" tabindex="-1"></a><span class="at">    &gt; **What are you working on to improve diversity, equity and inclusion in your community (75 words max)? (Examples may relate to personal identify, interest, experience, etc.)**</span></span>
<span id="cb34-759"><a href="#cb34-759" aria-hidden="true" tabindex="-1"></a><span class="at">    &gt;</span></span>
<span id="cb34-760"><a href="#cb34-760" aria-hidden="true" tabindex="-1"></a><span class="at">    &gt; I recently took part of the Identity, Belonging, and Inquiry in Science Program, and received special training to mentor students from diverse backgrounds. My mentee worked hard to complete research that she will present at the upcoming SACNAS conference, and I couldn't be more proud. The training gave me confidence to take on another mentee, this time a Capstone student. Mentorship is a key part of DEI efforts that I find extremely rewarding.</span></span>
<span id="cb34-761"><a href="#cb34-761" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb34-762"><a href="#cb34-762" aria-hidden="true" tabindex="-1"></a><span class="fu">#### 16-AUG-2023</span></span>
<span id="cb34-763"><a href="#cb34-763" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb34-764"><a href="#cb34-764" aria-hidden="true" tabindex="-1"></a><span class="ss">-   </span>asked JPG to buy benchtop UV Sterilizer for me &amp; Callum, got approval! Purchased by Kat same day :D</span>
<span id="cb34-765"><a href="#cb34-765" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb34-766"><a href="#cb34-766" aria-hidden="true" tabindex="-1"></a><span class="ss">-   </span>NMDS (non-metric multidimensional scaling) ordination plots</span>
<span id="cb34-767"><a href="#cb34-767" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb34-768"><a href="#cb34-768" aria-hidden="true" tabindex="-1"></a><span class="ss">-   </span>walked with Callum, plan to ask JPG to buy Qubit too!</span>
<span id="cb34-769"><a href="#cb34-769" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb34-770"><a href="#cb34-770" aria-hidden="true" tabindex="-1"></a><span class="ss">-   </span>Made some taxonomy bar plots and heatmap dendrograms... still working to understand what they mean but they don't look like there is separation between treatments. Haven't gotten any pvalues yet from the composition analyses...</span>
<span id="cb34-771"><a href="#cb34-771" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb34-772"><a href="#cb34-772" aria-hidden="true" tabindex="-1"></a><span class="ss">-   </span>Wondering if I should break up the Notebook into diversity &amp; composition? It's getting really long and overwhelming with all the analyses moshed together</span>
<span id="cb34-773"><a href="#cb34-773" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb34-774"><a href="#cb34-774" aria-hidden="true" tabindex="-1"></a><span class="ss">-   </span>Still can't <span class="in">`quarto render`</span> in Jupyter Lab on raven server, need to figure out how to get those interactive jupyter plots in my notebook!</span>
<span id="cb34-775"><a href="#cb34-775" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb34-776"><a href="#cb34-776" aria-hidden="true" tabindex="-1"></a><span class="fu">#### 15-AUG-2023</span></span>
<span id="cb34-777"><a href="#cb34-777" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb34-778"><a href="#cb34-778" aria-hidden="true" tabindex="-1"></a><span class="ss">-   </span>met with Dr. Amy VanCise about microbiome analysis:</span>
<span id="cb34-779"><a href="#cb34-779" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb34-780"><a href="#cb34-780" aria-hidden="true" tabindex="-1"></a><span class="ss">    -   </span>Look into work by WHOI Amy Apprill, Laura Weber, Cynthia Becker</span>
<span id="cb34-781"><a href="#cb34-781" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb34-782"><a href="#cb34-782" aria-hidden="true" tabindex="-1"></a><span class="ss">    -   </span>Focus analysis on **community composition** over **diversity** statistics</span>
<span id="cb34-783"><a href="#cb34-783" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb34-784"><a href="#cb34-784" aria-hidden="true" tabindex="-1"></a><span class="at">        &gt; different sampling depths across different samples ... subsample the counts without replacement so that each sample has the same number of features</span></span>
<span id="cb34-785"><a href="#cb34-785" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb34-786"><a href="#cb34-786" aria-hidden="true" tabindex="-1"></a><span class="ss">    -   </span>Differential abundance analysis on microbial community is similar to differential gene expression analysis!</span>
<span id="cb34-787"><a href="#cb34-787" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb34-788"><a href="#cb34-788" aria-hidden="true" tabindex="-1"></a><span class="ss">    -   </span>Sampling depth is normalizing between samples</span>
<span id="cb34-789"><a href="#cb34-789" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb34-790"><a href="#cb34-790" aria-hidden="true" tabindex="-1"></a><span class="ss">    -   </span>Look into getting a UV Sterilizer</span>
<span id="cb34-791"><a href="#cb34-791" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb34-792"><a href="#cb34-792" aria-hidden="true" tabindex="-1"></a><span class="ss">    -   </span>Do the analysis without blank, mock, &amp; env samples included... get a clean look at only treatments</span>
<span id="cb34-793"><a href="#cb34-793" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb34-794"><a href="#cb34-794" aria-hidden="true" tabindex="-1"></a><span class="ss">    -   </span>Look into R Vegan package</span>
<span id="cb34-795"><a href="#cb34-795" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb34-796"><a href="#cb34-796" aria-hidden="true" tabindex="-1"></a><span class="ss">    -   </span>Phred score: spatial, 30 (or 25) is good to keep</span>
<span id="cb34-797"><a href="#cb34-797" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb34-798"><a href="#cb34-798" aria-hidden="true" tabindex="-1"></a><span class="fu">#### 03-AUG-2023</span></span>
<span id="cb34-799"><a href="#cb34-799" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb34-800"><a href="#cb34-800" aria-hidden="true" tabindex="-1"></a><span class="ss">-   </span>Received RNA samples back from UT GSAF</span>
<span id="cb34-801"><a href="#cb34-801" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb34-802"><a href="#cb34-802" aria-hidden="true" tabindex="-1"></a><span class="ss">-   </span>Used PCR Inhibitor Removal Kit on five of the samples (see RNA Woes post <span class="co">[</span><span class="ot">here</span><span class="co">](https://sarahtanja.github.io/quarto-blog/posts/rna-woes/#:~:text=A260%20value.%E2%80%9D-,Zymo%20OneStep%20PCR%20Inhibitor%20Removal%20Kit,-I%20first%20tried)</span>) :</span>
<span id="cb34-803"><a href="#cb34-803" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb34-804"><a href="#cb34-804" aria-hidden="true" tabindex="-1"></a><span class="ss">    -   </span>1-CA2a</span>
<span id="cb34-805"><a href="#cb34-805" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb34-806"><a href="#cb34-806" aria-hidden="true" tabindex="-1"></a><span class="ss">    -   </span>1-CH2a</span>
<span id="cb34-807"><a href="#cb34-807" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb34-808"><a href="#cb34-808" aria-hidden="true" tabindex="-1"></a><span class="ss">    -   </span>1-Ea</span>
<span id="cb34-809"><a href="#cb34-809" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb34-810"><a href="#cb34-810" aria-hidden="true" tabindex="-1"></a><span class="ss">    -   </span>1-PA2a</span>
<span id="cb34-811"><a href="#cb34-811" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb34-812"><a href="#cb34-812" aria-hidden="true" tabindex="-1"></a><span class="ss">    -   </span>1-PH1a</span>
<span id="cb34-813"><a href="#cb34-813" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb34-814"><a href="#cb34-814" aria-hidden="true" tabindex="-1"></a><span class="ss">-   </span>PCR Inhibitor Removal Kit said to add 50-200uL of RNA, but I only could add 30uL. I took the 5 samples from the 96-well plate and augmented them with all the leftover diluted normalized RNA.</span>
<span id="cb34-815"><a href="#cb34-815" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb34-816"><a href="#cb34-816" aria-hidden="true" tabindex="-1"></a><span class="ss">-   </span>Nanodrop note: blank should result in a spectrum that varies no more than 0.050A in the 1mm path. For this batch I used the Roberts Lab Nanodrop and the blank was -0.009/-0.008 10mm path.</span>
<span id="cb34-817"><a href="#cb34-817" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb34-818"><a href="#cb34-818" aria-hidden="true" tabindex="-1"></a><span class="fu">## July 2023</span></span>
<span id="cb34-819"><a href="#cb34-819" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb34-820"><a href="#cb34-820" aria-hidden="true" tabindex="-1"></a><span class="fu">#### 20-JUL-2023</span></span>
<span id="cb34-821"><a href="#cb34-821" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb34-822"><a href="#cb34-822" aria-hidden="true" tabindex="-1"></a><span class="ss">-   </span><span class="co">[</span><span class="ot">Integrating QIIME2 and R for data viz</span><span class="co">](https://forum.qiime2.org/t/tutorial-integrating-qiime2-and-r-for-data-visualization-and-analysis-using-qiime2r/4121)</span></span>
<span id="cb34-823"><a href="#cb34-823" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb34-824"><a href="#cb34-824" aria-hidden="true" tabindex="-1"></a><span class="ss">-   </span>RStudio<span class="sc">\&gt;</span>Project Options<span class="sc">\&gt;</span> Python<span class="sc">\&gt;</span> Python interpreter</span>
<span id="cb34-825"><a href="#cb34-825" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb34-826"><a href="#cb34-826" aria-hidden="true" tabindex="-1"></a><span class="ss">-   </span><span class="co">[</span><span class="ot">Using Python with the RStudio IDE</span><span class="co">](https://support.posit.co/hc/en-us/articles/1500007929061-Using-Python-with-the-RStudio-IDE)</span></span>
<span id="cb34-827"><a href="#cb34-827" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb34-828"><a href="#cb34-828" aria-hidden="true" tabindex="-1"></a><span class="fu">#### 19-JUL-2023</span></span>
<span id="cb34-829"><a href="#cb34-829" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb34-830"><a href="#cb34-830" aria-hidden="true" tabindex="-1"></a><span class="ss">-   </span>Emailed Jackie about getting another P1000</span>
<span id="cb34-831"><a href="#cb34-831" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb34-832"><a href="#cb34-832" aria-hidden="true" tabindex="-1"></a><span class="ss">-   </span>Helped Bob Oxborrow with pipettes for stable isotope sample prep</span>
<span id="cb34-833"><a href="#cb34-833" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb34-834"><a href="#cb34-834" aria-hidden="true" tabindex="-1"></a><span class="ss">-   </span>Downloaded QIIME2 to Roberts Lab Raven server for coral-pae-temp repo</span>
<span id="cb34-835"><a href="#cb34-835" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb34-836"><a href="#cb34-836" aria-hidden="true" tabindex="-1"></a><span class="ss">-   </span>Shadowed Miranda to learn how to acid wash glassware in 10% HCl:</span>
<span id="cb34-837"><a href="#cb34-837" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb34-838"><a href="#cb34-838" aria-hidden="true" tabindex="-1"></a><span class="ss">    -   </span>Scrub and let soak 24hrs in Alconite bath</span>
<span id="cb34-839"><a href="#cb34-839" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb34-840"><a href="#cb34-840" aria-hidden="true" tabindex="-1"></a><span class="ss">    -   </span>Rinse 3X in Ultrapure water &amp; shift to 10% HCl bath, let soak for 24hrs</span>
<span id="cb34-841"><a href="#cb34-841" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb34-842"><a href="#cb34-842" aria-hidden="true" tabindex="-1"></a><span class="ss">    -   </span>Remove &amp; rinse 3X in Ultrapure water and let sit in Ultrapure water for 24hrs</span>
<span id="cb34-843"><a href="#cb34-843" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb34-844"><a href="#cb34-844" aria-hidden="true" tabindex="-1"></a><span class="ss">    -   </span>Remove &amp; let air dry, cover opening with foil</span>
<span id="cb34-845"><a href="#cb34-845" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb34-846"><a href="#cb34-846" aria-hidden="true" tabindex="-1"></a><span class="fu">## June 2023</span></span>
<span id="cb34-847"><a href="#cb34-847" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb34-848"><a href="#cb34-848" aria-hidden="true" tabindex="-1"></a><span class="fu">#### 07-JUN-2023</span></span>
<span id="cb34-849"><a href="#cb34-849" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb34-850"><a href="#cb34-850" aria-hidden="true" tabindex="-1"></a><span class="ss">-   </span>Shipped DNA to Mr. DNA and RNA to UT GSAF!</span>
<span id="cb34-851"><a href="#cb34-851" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb34-852"><a href="#cb34-852" aria-hidden="true" tabindex="-1"></a><span class="fu">#### 06-JUN-2023</span></span>
<span id="cb34-853"><a href="#cb34-853" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb34-854"><a href="#cb34-854" aria-hidden="true" tabindex="-1"></a><span class="ss">-   </span>Normalized all 20 RNA samples</span>
<span id="cb34-855"><a href="#cb34-855" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb34-856"><a href="#cb34-856" aria-hidden="true" tabindex="-1"></a><span class="fu">#### 05-JUN-2023</span></span>
<span id="cb34-857"><a href="#cb34-857" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb34-858"><a href="#cb34-858" aria-hidden="true" tabindex="-1"></a><span class="ss">-   </span>Re-normalized 2-CA2a DNA and successfully got it to 19.5, 19.2 :D</span>
<span id="cb34-859"><a href="#cb34-859" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb34-860"><a href="#cb34-860" aria-hidden="true" tabindex="-1"></a><span class="ss">-   </span>Worked all day to normalize RNA with mixed results. In the first batch, 7 out of 20 samples were 'off' of the target concentration of 25ng/uL by enough to make me want to redo them (one was way high at 230ng/uL, others were too low at \~10ng/uL)</span>
<span id="cb34-861"><a href="#cb34-861" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb34-862"><a href="#cb34-862" aria-hidden="true" tabindex="-1"></a><span class="ss">-   </span>I first re-ran the qubit (2nd qubit run) and got really poor results, with all the rerun samples too-low, (even the one that was previously 230ng/uL!?)</span>
<span id="cb34-863"><a href="#cb34-863" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb34-864"><a href="#cb34-864" aria-hidden="true" tabindex="-1"></a><span class="ss">-   </span>I then re-normalized the samples and ran them on qubit again (3rd qubit run). In this run 4-PH1b hit the target, the rest were still too low</span>
<span id="cb34-865"><a href="#cb34-865" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb34-866"><a href="#cb34-866" aria-hidden="true" tabindex="-1"></a><span class="ss">-   </span>I re-normalized the samples for a third time (with some increasing concentration modifications) and ran them on qubit for a 4th time. In this run 4-PA2a hit the target, the rest were still too low, and 2-PH2a even got worse despite increasing concentration.</span>
<span id="cb34-867"><a href="#cb34-867" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb34-868"><a href="#cb34-868" aria-hidden="true" tabindex="-1"></a><span class="ss">-   </span>Ended the day unsatisfied with the following samples too low for normalization despite 4 attempts:</span>
<span id="cb34-869"><a href="#cb34-869" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb34-870"><a href="#cb34-870" aria-hidden="true" tabindex="-1"></a><span class="ss">    -   </span>2-PH2a - too low</span>
<span id="cb34-871"><a href="#cb34-871" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb34-872"><a href="#cb34-872" aria-hidden="true" tabindex="-1"></a><span class="ss">    -   </span>2-CA2a - 15.0, 15.4</span>
<span id="cb34-873"><a href="#cb34-873" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb34-874"><a href="#cb34-874" aria-hidden="true" tabindex="-1"></a><span class="ss">    -   </span>1-PA2a - 10.2, 10.0</span>
<span id="cb34-875"><a href="#cb34-875" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb34-876"><a href="#cb34-876" aria-hidden="true" tabindex="-1"></a><span class="ss">    -   </span>1-PH1a - 17.6, 17.6</span>
<span id="cb34-877"><a href="#cb34-877" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb34-878"><a href="#cb34-878" aria-hidden="true" tabindex="-1"></a><span class="ss">    -   </span>3-CH2a - 15.0, 14.6</span>
<span id="cb34-879"><a href="#cb34-879" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb34-880"><a href="#cb34-880" aria-hidden="true" tabindex="-1"></a><span class="ss">    -   </span>4-CA1a - 14.0, 13.0</span>
<span id="cb34-881"><a href="#cb34-881" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb34-882"><a href="#cb34-882" aria-hidden="true" tabindex="-1"></a><span class="fu">#### 02-JUN-2023</span></span>
<span id="cb34-883"><a href="#cb34-883" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb34-884"><a href="#cb34-884" aria-hidden="true" tabindex="-1"></a><span class="ss">-   </span>Normalized DNA (still have RNA left to do)</span>
<span id="cb34-885"><a href="#cb34-885" aria-hidden="true" tabindex="-1"></a><span class="ss">-   </span>Tested normalized DNA on Qubit BR DNA assay, results overall were good but a smidge lower than my goal</span>
<span id="cb34-886"><a href="#cb34-886" aria-hidden="true" tabindex="-1"></a><span class="ss">-   </span>blank + microbial standard extraction</span>
<span id="cb34-887"><a href="#cb34-887" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb34-888"><a href="#cb34-888" aria-hidden="true" tabindex="-1"></a><span class="fu">#### 01-JUN-2023</span></span>
<span id="cb34-889"><a href="#cb34-889" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb34-890"><a href="#cb34-890" aria-hidden="true" tabindex="-1"></a><span class="ss">-   </span>Presented Bioinformatics <span class="co">[</span><span class="ot">final project for FISH 546</span><span class="co">](https://rpubs.com/sarah_tanja/1048844)</span></span>
<span id="cb34-891"><a href="#cb34-891" aria-hidden="true" tabindex="-1"></a><span class="ss">-   </span>Attended GSR Meeting</span>
<span id="cb34-892"><a href="#cb34-892" aria-hidden="true" tabindex="-1"></a><span class="ss">-   </span>Attended Seminar/TGIT</span>
<span id="cb34-893"><a href="#cb34-893" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb34-894"><a href="#cb34-894" aria-hidden="true" tabindex="-1"></a><span class="fu">## May 2023</span></span>
<span id="cb34-895"><a href="#cb34-895" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb34-896"><a href="#cb34-896" aria-hidden="true" tabindex="-1"></a><span class="fu">#### 31-MAY-2023</span></span>
<span id="cb34-897"><a href="#cb34-897" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb34-898"><a href="#cb34-898" aria-hidden="true" tabindex="-1"></a><span class="ss">-   </span>wrote <span class="in">`dilute`</span> function to normalize RNA &amp; DNA in <span class="co">[</span><span class="ot">coral-DNA-RNA-lab-extractions</span><span class="co">](https://github.com/sarahtanja/coral-DNA-RNA-lab-extractions/blob/main/code/03-normalizing.qmd)</span> GitHub repo:</span>
<span id="cb34-899"><a href="#cb34-899" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb34-902"><a href="#cb34-902" aria-hidden="true" tabindex="-1"></a><span class="in">```{r}</span></span>
<span id="cb34-903"><a href="#cb34-903" aria-hidden="true" tabindex="-1"></a>dilute <span class="ot">&lt;-</span> <span class="cf">function</span>(C1, C2) {</span>
<span id="cb34-904"><a href="#cb34-904" aria-hidden="true" tabindex="-1"></a>  <span class="co"># Initialize variables</span></span>
<span id="cb34-905"><a href="#cb34-905" aria-hidden="true" tabindex="-1"></a>  n <span class="ot">&lt;-</span> <span class="fu">length</span>(C1)</span>
<span id="cb34-906"><a href="#cb34-906" aria-hidden="true" tabindex="-1"></a>  C1 <span class="ot">&lt;-</span> C1</span>
<span id="cb34-907"><a href="#cb34-907" aria-hidden="true" tabindex="-1"></a>  V1 <span class="ot">&lt;-</span> <span class="fu">numeric</span>(n)  <span class="co"># Initialize V1 vector</span></span>
<span id="cb34-908"><a href="#cb34-908" aria-hidden="true" tabindex="-1"></a>  V2 <span class="ot">&lt;-</span> <span class="fu">numeric</span>(n)  <span class="co"># Initialize V2 vector</span></span>
<span id="cb34-909"><a href="#cb34-909" aria-hidden="true" tabindex="-1"></a>  </span>
<span id="cb34-910"><a href="#cb34-910" aria-hidden="true" tabindex="-1"></a>  <span class="co"># Loop through rows</span></span>
<span id="cb34-911"><a href="#cb34-911" aria-hidden="true" tabindex="-1"></a>  <span class="cf">for</span> (i <span class="cf">in</span> <span class="fu">seq_along</span>(C1)) {</span>
<span id="cb34-912"><a href="#cb34-912" aria-hidden="true" tabindex="-1"></a>    <span class="co"># Check for missing data</span></span>
<span id="cb34-913"><a href="#cb34-913" aria-hidden="true" tabindex="-1"></a>    <span class="cf">if</span> (C1[i] <span class="sc">&lt;=</span> <span class="dv">0</span>) {</span>
<span id="cb34-914"><a href="#cb34-914" aria-hidden="true" tabindex="-1"></a>      <span class="fu">cat</span>(<span class="st">"Skipping row "</span>, i, <span class="st">": Invalid data</span><span class="sc">\n</span><span class="st">"</span>)</span>
<span id="cb34-915"><a href="#cb34-915" aria-hidden="true" tabindex="-1"></a>      <span class="cf">next</span></span>
<span id="cb34-916"><a href="#cb34-916" aria-hidden="true" tabindex="-1"></a>    }</span>
<span id="cb34-917"><a href="#cb34-917" aria-hidden="true" tabindex="-1"></a>    </span>
<span id="cb34-918"><a href="#cb34-918" aria-hidden="true" tabindex="-1"></a>    <span class="co"># Initialize V1 and V2 for the current row</span></span>
<span id="cb34-919"><a href="#cb34-919" aria-hidden="true" tabindex="-1"></a>    V1_row <span class="ot">&lt;-</span> <span class="dv">1</span></span>
<span id="cb34-920"><a href="#cb34-920" aria-hidden="true" tabindex="-1"></a>    V2_row <span class="ot">&lt;-</span> (C1[i] <span class="sc">*</span> V1_row) <span class="sc">/</span> C2</span>
<span id="cb34-921"><a href="#cb34-921" aria-hidden="true" tabindex="-1"></a>    </span>
<span id="cb34-922"><a href="#cb34-922" aria-hidden="true" tabindex="-1"></a>    <span class="co"># Loop until conditions are satisfied</span></span>
<span id="cb34-923"><a href="#cb34-923" aria-hidden="true" tabindex="-1"></a>    <span class="cf">while</span> (<span class="sc">!</span><span class="fu">is.na</span>(V1_row) <span class="sc">&amp;&amp;</span> <span class="sc">!</span><span class="fu">is.na</span>(V2_row) <span class="sc">&amp;&amp;</span> (V1_row <span class="sc">&lt;</span> <span class="dv">1</span> <span class="sc">||</span> V2_row <span class="sc">&lt;</span> <span class="dv">50</span>)) {</span>
<span id="cb34-924"><a href="#cb34-924" aria-hidden="true" tabindex="-1"></a>      V1_row <span class="ot">&lt;-</span> V1_row <span class="sc">+</span> <span class="dv">1</span></span>
<span id="cb34-925"><a href="#cb34-925" aria-hidden="true" tabindex="-1"></a>      V2_row <span class="ot">&lt;-</span> (C1[i] <span class="sc">*</span> V1_row) <span class="sc">/</span> C2</span>
<span id="cb34-926"><a href="#cb34-926" aria-hidden="true" tabindex="-1"></a>      </span>
<span id="cb34-927"><a href="#cb34-927" aria-hidden="true" tabindex="-1"></a>      <span class="co"># Break the loop if V1 exceeds a threshold value</span></span>
<span id="cb34-928"><a href="#cb34-928" aria-hidden="true" tabindex="-1"></a>      <span class="cf">if</span>(V1_row <span class="sc">&gt;</span> <span class="dv">49</span>) {</span>
<span id="cb34-929"><a href="#cb34-929" aria-hidden="true" tabindex="-1"></a>        <span class="fu">cat</span>(<span class="st">"Skipping row "</span>, i, <span class="st">": No valid solution found</span><span class="sc">\n</span><span class="st">"</span>)</span>
<span id="cb34-930"><a href="#cb34-930" aria-hidden="true" tabindex="-1"></a>        <span class="cf">next</span></span>
<span id="cb34-931"><a href="#cb34-931" aria-hidden="true" tabindex="-1"></a>      }</span>
<span id="cb34-932"><a href="#cb34-932" aria-hidden="true" tabindex="-1"></a>    }</span>
<span id="cb34-933"><a href="#cb34-933" aria-hidden="true" tabindex="-1"></a>    </span>
<span id="cb34-934"><a href="#cb34-934" aria-hidden="true" tabindex="-1"></a>    <span class="co"># Store optimized V1 and V2 values in the corresponding vectors</span></span>
<span id="cb34-935"><a href="#cb34-935" aria-hidden="true" tabindex="-1"></a>    V1[i] <span class="ot">&lt;-</span> V1_row</span>
<span id="cb34-936"><a href="#cb34-936" aria-hidden="true" tabindex="-1"></a>    V2[i] <span class="ot">&lt;-</span> V2_row</span>
<span id="cb34-937"><a href="#cb34-937" aria-hidden="true" tabindex="-1"></a>  }</span>
<span id="cb34-938"><a href="#cb34-938" aria-hidden="true" tabindex="-1"></a>  </span>
<span id="cb34-939"><a href="#cb34-939" aria-hidden="true" tabindex="-1"></a><span class="co"># Create a data frame with C1 column</span></span>
<span id="cb34-940"><a href="#cb34-940" aria-hidden="true" tabindex="-1"></a>  result <span class="ot">&lt;-</span> <span class="fu">data.frame</span>(<span class="at">C1 =</span> C1)</span>
<span id="cb34-941"><a href="#cb34-941" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb34-942"><a href="#cb34-942" aria-hidden="true" tabindex="-1"></a><span class="co"># Add V1 and V2 columns to the result data frame</span></span>
<span id="cb34-943"><a href="#cb34-943" aria-hidden="true" tabindex="-1"></a>  result<span class="sc">$</span>V1 <span class="ot">&lt;-</span> V1</span>
<span id="cb34-944"><a href="#cb34-944" aria-hidden="true" tabindex="-1"></a>  result<span class="sc">$</span>V2 <span class="ot">&lt;-</span> V2</span>
<span id="cb34-945"><a href="#cb34-945" aria-hidden="true" tabindex="-1"></a>  result<span class="sc">$</span>C2 <span class="ot">&lt;-</span> C1<span class="sc">*</span>V1<span class="sc">/</span>V2</span>
<span id="cb34-946"><a href="#cb34-946" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb34-947"><a href="#cb34-947" aria-hidden="true" tabindex="-1"></a>  <span class="fu">return</span>(result)</span>
<span id="cb34-948"><a href="#cb34-948" aria-hidden="true" tabindex="-1"></a>}</span>
<span id="cb34-949"><a href="#cb34-949" aria-hidden="true" tabindex="-1"></a><span class="in">```</span></span>
<span id="cb34-950"><a href="#cb34-950" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb34-951"><a href="#cb34-951" aria-hidden="true" tabindex="-1"></a><span class="ss">-   </span>SAFS Spring Picnic!</span>
<span id="cb34-952"><a href="#cb34-952" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb34-953"><a href="#cb34-953" aria-hidden="true" tabindex="-1"></a><span class="fu">#### 29-MAY-2023</span></span>
<span id="cb34-954"><a href="#cb34-954" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb34-955"><a href="#cb34-955" aria-hidden="true" tabindex="-1"></a><span class="ss">-   </span>Worked on Bioinformatics Final Project, RNA expression heat map with Gene Ontology Terms (Protein Functions)</span>
<span id="cb34-956"><a href="#cb34-956" aria-hidden="true" tabindex="-1"></a><span class="ss">-   </span>IBIS Celebration of <span class="co">[</span><span class="ot">Nora's Blog</span><span class="co">](https://norahess.quarto.pub/uw-lab-note-book/)</span>!</span>
<span id="cb34-957"><a href="#cb34-957" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb34-958"><a href="#cb34-958" aria-hidden="true" tabindex="-1"></a><span class="fu">#### 25-MAY-2023</span></span>
<span id="cb34-959"><a href="#cb34-959" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb34-960"><a href="#cb34-960" aria-hidden="true" tabindex="-1"></a><span class="ss">-   </span>Hosted Dr. Nicola Smith as the GSIS</span>
<span id="cb34-961"><a href="#cb34-961" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb34-962"><a href="#cb34-962" aria-hidden="true" tabindex="-1"></a><span class="fu">#### 24-MAY-2023</span></span>
<span id="cb34-963"><a href="#cb34-963" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb34-964"><a href="#cb34-964" aria-hidden="true" tabindex="-1"></a><span class="ss">-   </span>University of Texas at Austin <span class="co">[</span><span class="ot">Genomic Sequencing and Analysis Facility (GSAF)</span><span class="co">](https://wikis.utexas.edu/display/GSAF/Home+Page)</span>:</span>
<span id="cb34-965"><a href="#cb34-965" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb34-966"><a href="#cb34-966" aria-hidden="true" tabindex="-1"></a><span class="ss">    -   </span><span class="co">[</span><span class="ot">Sample Input Guidelines</span><span class="co">](https://wikis.utexas.edu/display/GSAF/Sample+Input+Guidelines)</span></span>
<span id="cb34-967"><a href="#cb34-967" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb34-968"><a href="#cb34-968" aria-hidden="true" tabindex="-1"></a><span class="at">        &gt; TagSeq: 25ng/ul concentration in greater than or equal to 25ul volume (submitting RNA as low as 10ng/ul or as high as 100ng/ul is acceptable BUT ALL samples MUST be submitted at the same or very similar concentrations)&nbsp;</span></span>
<span id="cb34-969"><a href="#cb34-969" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb34-970"><a href="#cb34-970" aria-hidden="true" tabindex="-1"></a><span class="ss">    -   </span><span class="co">[</span><span class="ot">GSAF Requester Portal Login</span><span class="co">](https://lockbox-604.my.site.com/s/login/)</span></span>
<span id="cb34-971"><a href="#cb34-971" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb34-972"><a href="#cb34-972" aria-hidden="true" tabindex="-1"></a><span class="ss">    -   </span><span class="co">[</span><span class="ot">Illumina BaseSpace Login</span><span class="co">](https://platform.login.illumina.com/platform-services-manager/?rURL=https://basespace.login.illumina.com/platform-home/&amp;redirectMethod=GET&amp;clientId=ps-home#/)</span></span>
<span id="cb34-973"><a href="#cb34-973" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb34-974"><a href="#cb34-974" aria-hidden="true" tabindex="-1"></a><span class="fu">#### 23-MAY-2023</span></span>
<span id="cb34-975"><a href="#cb34-975" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb34-976"><a href="#cb34-976" aria-hidden="true" tabindex="-1"></a><span class="ss">-   </span>completed UW driver safety training</span>
<span id="cb34-977"><a href="#cb34-977" aria-hidden="true" tabindex="-1"></a><span class="ss">-   </span>met with Steven Roberts, confirmed that no blank/standard is used for RNA-seq</span>
<span id="cb34-978"><a href="#cb34-978" aria-hidden="true" tabindex="-1"></a><span class="ss">-   </span>emailed with Jesse Zaneveld, learned about Zymo mock <span class="co">[</span><span class="ot">microbial community standard</span><span class="co">](https://www.zymoresearch.com/collections/zymobiomics-microbial-community-standards/products/zymobiomics-microbial-community-standard)</span> and <span class="co">[</span><span class="ot">microbial community DNA standards</span><span class="co">](https://www.zymoresearch.com/collections/zymobiomics-microbial-community-standards/products/zymobiomics-microbial-community-dna-standard)</span> to be used as positive controls for the extraction (community standard cells) and the PCR library prep and sequencing (community standard DNA) to assess bias in each process.</span>
<span id="cb34-979"><a href="#cb34-979" aria-hidden="true" tabindex="-1"></a><span class="ss">-   </span>Put both Zymo standards on purcahsing log, emailed Kat and she said she would buy them tonight with the pcard</span>
<span id="cb34-980"><a href="#cb34-980" aria-hidden="true" tabindex="-1"></a><span class="ss">-   </span>Met with Nora, she has made an <span class="co">[</span><span class="ot">awesome quarto blog</span><span class="co">](https://norahess.quarto.pub/uw-lab-note-book/)</span> for the IBIS celebration to showcase what she has learned. We also talked a bit about stats and need to follow up with what to do when your data is not normally distributed (moving from paired T test to Wilcoxon paired-sample T test)</span>
<span id="cb34-981"><a href="#cb34-981" aria-hidden="true" tabindex="-1"></a><span class="ss">-   </span>Tried using <span class="in">`quarto publish gh-pages`</span> command to publish blog</span>
<span id="cb34-982"><a href="#cb34-982" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb34-983"><a href="#cb34-983" aria-hidden="true" tabindex="-1"></a><span class="fu">#### 19-MAY-2023</span></span>
<span id="cb34-984"><a href="#cb34-984" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb34-985"><a href="#cb34-985" aria-hidden="true" tabindex="-1"></a><span class="ss">-   </span>Ran Qubit on ex7 samples</span>
<span id="cb34-986"><a href="#cb34-986" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb34-987"><a href="#cb34-987" aria-hidden="true" tabindex="-1"></a><span class="ss">-   </span>Extraction #7 (the redo) on samples:</span>
<span id="cb34-988"><a href="#cb34-988" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb34-989"><a href="#cb34-989" aria-hidden="true" tabindex="-1"></a><span class="ss">    -   </span>3-Eb</span>
<span id="cb34-990"><a href="#cb34-990" aria-hidden="true" tabindex="-1"></a><span class="ss">    -   </span>3-PH1b</span>
<span id="cb34-991"><a href="#cb34-991" aria-hidden="true" tabindex="-1"></a><span class="ss">    -   </span>2-PA2b</span>
<span id="cb34-992"><a href="#cb34-992" aria-hidden="true" tabindex="-1"></a><span class="ss">    -   </span>2-CH1b</span>
<span id="cb34-993"><a href="#cb34-993" aria-hidden="true" tabindex="-1"></a><span class="ss">    -   </span>4-PH1b</span>
<span id="cb34-994"><a href="#cb34-994" aria-hidden="true" tabindex="-1"></a><span class="ss">    -   </span>3-CA1b</span>
<span id="cb34-995"><a href="#cb34-995" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb34-996"><a href="#cb34-996" aria-hidden="true" tabindex="-1"></a><span class="ss">-   </span>found out how to use quarto to do <span class="co">[</span><span class="ot">Margin References</span><span class="co">](https://quarto.org/docs/blog/posts/2022-02-17-advanced-layout/#:~:text=.-,Margin%20References,-Footnotes%20and%20the)</span>!</span>
<span id="cb34-997"><a href="#cb34-997" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb34-998"><a href="#cb34-998" aria-hidden="true" tabindex="-1"></a><span class="ss">-   </span>found cool article on <span class="co">[</span><span class="ot">Verbatim code chunks</span><span class="co">](https://themockup.blog/posts/2021-08-27-displaying-verbatim-code-chunks-in-xaringan-presentations/)</span> in R (want to render the code chunk options and show what engine is being used?)</span>
<span id="cb34-999"><a href="#cb34-999" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb34-1000"><a href="#cb34-1000" aria-hidden="true" tabindex="-1"></a><span class="ss">-   </span>dry-ice at biochem stockroom is low, have to check that there will be a replenishment next week so I have enough dry ice to ship samples!</span>
<span id="cb34-1001"><a href="#cb34-1001" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb34-1002"><a href="#cb34-1002" aria-hidden="true" tabindex="-1"></a><span class="fu">#### 18-MAY-2023</span></span>
<span id="cb34-1003"><a href="#cb34-1003" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb34-1004"><a href="#cb34-1004" aria-hidden="true" tabindex="-1"></a><span class="ss">-   </span>wrote script to normalize eluted RNA/DNA</span>
<span id="cb34-1005"><a href="#cb34-1005" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb34-1006"><a href="#cb34-1006" aria-hidden="true" tabindex="-1"></a><span class="fu">#### 04-MAY-2023</span></span>
<span id="cb34-1007"><a href="#cb34-1007" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb34-1008"><a href="#cb34-1008" aria-hidden="true" tabindex="-1"></a><span class="ss">-   </span>received Qubit DNA Broad Range (BR) Assay Kit!</span>
<span id="cb34-1009"><a href="#cb34-1009" aria-hidden="true" tabindex="-1"></a><span class="ss">-   </span>Gotta get P2 DNA/RNA free tips!</span>
<span id="cb34-1010"><a href="#cb34-1010" aria-hidden="true" tabindex="-1"></a><span class="ss">-   </span>ran qubit on DNA &amp; RNA, got great results!</span>
<span id="cb34-1011"><a href="#cb34-1011" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb34-1012"><a href="#cb34-1012" aria-hidden="true" tabindex="-1"></a><span class="fu">## April 2023</span></span>
<span id="cb34-1013"><a href="#cb34-1013" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb34-1014"><a href="#cb34-1014" aria-hidden="true" tabindex="-1"></a><span class="fu">#### 25-APR-2023</span></span>
<span id="cb34-1015"><a href="#cb34-1015" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb34-1016"><a href="#cb34-1016" aria-hidden="true" tabindex="-1"></a><span class="ss">-   </span>sorted pipettes with Callum</span>
<span id="cb34-1017"><a href="#cb34-1017" aria-hidden="true" tabindex="-1"></a><span class="ss">-   </span>submitted Bioinformatics Assignment</span>
<span id="cb34-1018"><a href="#cb34-1018" aria-hidden="true" tabindex="-1"></a><span class="ss">-   </span>ordered Qubit DNA BR assay kit</span>
<span id="cb34-1019"><a href="#cb34-1019" aria-hidden="true" tabindex="-1"></a><span class="ss">-   </span>ordered film sealing plate roller</span>
<span id="cb34-1020"><a href="#cb34-1020" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb34-1021"><a href="#cb34-1021" aria-hidden="true" tabindex="-1"></a><span class="fu">#### 24-APR-2023</span></span>
<span id="cb34-1022"><a href="#cb34-1022" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb34-1023"><a href="#cb34-1023" aria-hidden="true" tabindex="-1"></a><span class="ss">-   </span>sent draft letter for NAH experiential learning fund to JPG on Monday April 24th</span>
<span id="cb34-1024"><a href="#cb34-1024" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb34-1025"><a href="#cb34-1025" aria-hidden="true" tabindex="-1"></a><span class="ss">-   </span>found <span class="co">[</span><span class="ot">96-well PCR shipping guide</span><span class="co">](https://microbe.med.umich.edu/sites/default/files/downloads/2021_ConsolidatedShippingGuide_0.pdf)</span></span>
<span id="cb34-1026"><a href="#cb34-1026" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb34-1027"><a href="#cb34-1027" aria-hidden="true" tabindex="-1"></a><span class="ss">-   </span>emailed Mr.DNA about 20 16S samples</span>
<span id="cb34-1028"><a href="#cb34-1028" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb34-1029"><a href="#cb34-1029" aria-hidden="true" tabindex="-1"></a><span class="ss">-   </span>read up on NanoDrop vs. Qubit</span>
<span id="cb34-1030"><a href="#cb34-1030" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb34-1031"><a href="#cb34-1031" aria-hidden="true" tabindex="-1"></a><span class="ss">-   </span>reached out to Sam White about Ubuntu/Windows dual-boot computer in lab 236, which is connected to the Thermo NanoDrop 2000c Sam shared:</span>
<span id="cb34-1032"><a href="#cb34-1032" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb34-1033"><a href="#cb34-1033" aria-hidden="true" tabindex="-1"></a><span class="at">    &gt; FYI, if you're solely using the NanoDrop for quantification (i.e. not worried about 260/280), then I'd highly recommend just relying on the Qubit, with both DNA and RNA. The reason for this is that the NanoDrop will almost always over-"calculate" your sample concentrations (both RNA and DNA). The reason for this is that the NanoDrop is a spectrophotometer and measures light absorbance. With almost all DNA/RNA isolations, there will be residual DNA or RNA in your sample. The NanoDrop cannot distinguish between the two molecules, because they're both made up of the same material; nucleotides. Nucleotides, regardless of their of their origin will absorb light at an average wavelength of 260nm.Since the Qubit assays use molecule-specific dyes, the dye will only bind to DNA or only to RNA. The dyes will not bind to any carryover of other types of molecules in the sample, so the accuracy (and sensitivity) is much, much higher than the NanoDrop.</span></span>
<span id="cb34-1034"><a href="#cb34-1034" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb34-1035"><a href="#cb34-1035" aria-hidden="true" tabindex="-1"></a><span class="fu">#### 21-APR-2023</span></span>
<span id="cb34-1036"><a href="#cb34-1036" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb34-1037"><a href="#cb34-1037" aria-hidden="true" tabindex="-1"></a><span class="ss">-   </span>met with JPG April 1on1</span>
<span id="cb34-1038"><a href="#cb34-1038" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb34-1039"><a href="#cb34-1039" aria-hidden="true" tabindex="-1"></a><span class="ss">    -   </span>talked about travel from Sept. 12 - Oct 9th, missing first 2 weeks of Autumn '23 quarter, focusing that quarter on writing &amp; analysis</span>
<span id="cb34-1040"><a href="#cb34-1040" aria-hidden="true" tabindex="-1"></a><span class="ss">    -   </span>GOAL: have all sequences (16S &amp; Transcriptomes) by September 1st 2023</span>
<span id="cb34-1041"><a href="#cb34-1041" aria-hidden="true" tabindex="-1"></a><span class="ss">    -   </span>Initial 20 samples extracted &amp; sent for sequencing before FHL</span>
<span id="cb34-1042"><a href="#cb34-1042" aria-hidden="true" tabindex="-1"></a><span class="ss">    -   </span>2nd committee meeting to be held once initial 20 samples are run through 1st analysis for 16S &amp; DGE, get advice on strategy for selecting remaining samples</span>
<span id="cb34-1043"><a href="#cb34-1043" aria-hidden="true" tabindex="-1"></a><span class="ss">    -   </span>Follow-up samples extracted &amp; sent for sequencing in July/August</span>
<span id="cb34-1044"><a href="#cb34-1044" aria-hidden="true" tabindex="-1"></a><span class="ss">    -   </span>Send draft letter for NAH experiential learning fund to JPG on Monday April 24th</span>
<span id="cb34-1045"><a href="#cb34-1045" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb34-1046"><a href="#cb34-1046" aria-hidden="true" tabindex="-1"></a><span class="ss">-   </span>Initial test round 2 samples extracted!!!</span>
<span id="cb34-1047"><a href="#cb34-1047" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb34-1048"><a href="#cb34-1048" aria-hidden="true" tabindex="-1"></a><span class="ss">    -   </span>3-CA1a</span>
<span id="cb34-1049"><a href="#cb34-1049" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb34-1050"><a href="#cb34-1050" aria-hidden="true" tabindex="-1"></a><span class="ss">    -   </span>3-Ea</span>
<span id="cb34-1051"><a href="#cb34-1051" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb34-1052"><a href="#cb34-1052" aria-hidden="true" tabindex="-1"></a><span class="ss">        -   </span>Sampes selected from <span class="co">[</span><span class="ot">extr1</span><span class="co">](https://github.com/sarahtanja/coral-DNA-RNA-lab-extractions/blob/main/rand-sample-processing.Rmd)</span> using a random number generator</span>
<span id="cb34-1053"><a href="#cb34-1053" aria-hidden="true" tabindex="-1"></a><span class="ss">        -   </span>details can be found in the <span class="co">[</span><span class="ot">coral-DNA-RNA-lab-extractions</span><span class="co">](https://github.com/sarahtanja/coral-DNA-RNA-lab-extractions)</span> repository</span>
<span id="cb34-1054"><a href="#cb34-1054" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb34-1055"><a href="#cb34-1055" aria-hidden="true" tabindex="-1"></a><span class="fu">#### 20-APR-2023</span></span>
<span id="cb34-1056"><a href="#cb34-1056" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb34-1057"><a href="#cb34-1057" aria-hidden="true" tabindex="-1"></a><span class="ss">-   </span>downloaded fastq files for 'fake data' to continue script for bioinformatics workflow</span>
<span id="cb34-1058"><a href="#cb34-1058" aria-hidden="true" tabindex="-1"></a><span class="ss">-   </span>Received Mortexer! Whoo!</span>
<span id="cb34-1059"><a href="#cb34-1059" aria-hidden="true" tabindex="-1"></a><span class="ss">-   </span>Received cryo-gloves</span>
<span id="cb34-1060"><a href="#cb34-1060" aria-hidden="true" tabindex="-1"></a><span class="ss">-   </span>got 8lbs of dry-ice with NH</span>
<span id="cb34-1061"><a href="#cb34-1061" aria-hidden="true" tabindex="-1"></a><span class="ss">-   </span>re-organized cryo-vials to wax boxes, with all of the 'a' samples in 2 boxes labelled with green lab tape and 'b' samples in 2 boxes labelled with blue tape with NH</span>
<span id="cb34-1062"><a href="#cb34-1062" aria-hidden="true" tabindex="-1"></a><span class="ss">-   </span>setup lab bench for dry run dna/rna extraction with NH</span>
<span id="cb34-1063"><a href="#cb34-1063" aria-hidden="true" tabindex="-1"></a><span class="ss">-   </span>NH setup mortexer</span>
<span id="cb34-1064"><a href="#cb34-1064" aria-hidden="true" tabindex="-1"></a><span class="ss">-   </span>submitted 2-year extended warranty for mortexer s/n 22040448 via benchmark scientific in JPG's name</span>
<span id="cb34-1065"><a href="#cb34-1065" aria-hidden="true" tabindex="-1"></a><span class="ss">-   </span>placed signage on lab 236 to lint roll and lab coat (because I forgot to and that was embarassing)</span>
<span id="cb34-1066"><a href="#cb34-1066" aria-hidden="true" tabindex="-1"></a><span class="ss">-   </span>submitted purchase for mortexer attachment head that will homogenize 12 1.5-2mL tubes at a time horizontal orientation (even better because closer to original lysis protocol for Zymo) Sadly, I should have looked for something like this earlier, but I saw it on the mortexer instruction manual when it was delivered and realized that was what I needed</span>
<span id="cb34-1067"><a href="#cb34-1067" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb34-1068"><a href="#cb34-1068" aria-hidden="true" tabindex="-1"></a><span class="fu">#### 19-APR-2023</span></span>
<span id="cb34-1069"><a href="#cb34-1069" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb34-1070"><a href="#cb34-1070" aria-hidden="true" tabindex="-1"></a><span class="ss">-   </span>continued draft application for ONeill-Quinn</span>
<span id="cb34-1071"><a href="#cb34-1071" aria-hidden="true" tabindex="-1"></a><span class="ss">-   </span>continued editing extraction protocol</span>
<span id="cb34-1072"><a href="#cb34-1072" aria-hidden="true" tabindex="-1"></a><span class="ss">-   </span>expected delivery of mortexer by end of day</span>
<span id="cb34-1073"><a href="#cb34-1073" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb34-1074"><a href="#cb34-1074" aria-hidden="true" tabindex="-1"></a><span class="fu">#### 18-APR-2023</span></span>
<span id="cb34-1075"><a href="#cb34-1075" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb34-1076"><a href="#cb34-1076" aria-hidden="true" tabindex="-1"></a><span class="ss">-   </span>Met with Nora, selected samples for processing using <span class="co">[</span><span class="ot">R script to randomize</span><span class="co">](https://github.com/sarahtanja/coral-DNA-RNA-lab-extractions/blob/main/rand-sample-processing.Rmd)</span></span>
<span id="cb34-1077"><a href="#cb34-1077" aria-hidden="true" tabindex="-1"></a><span class="ss">-   </span>Setup workflow in Hyak</span>
<span id="cb34-1078"><a href="#cb34-1078" aria-hidden="true" tabindex="-1"></a><span class="ss">-   </span>added to extraction protocol</span>
<span id="cb34-1079"><a href="#cb34-1079" aria-hidden="true" tabindex="-1"></a><span class="ss">-   </span>waiting for mortexer delivery, expected by end of day, but FedEx package delivery delayed</span>
<span id="cb34-1080"><a href="#cb34-1080" aria-hidden="true" tabindex="-1"></a><span class="ss">-   </span>emailed asking for details about Neptune controller unit (it was returned after being sent away for RMA maintenance request, but nothing appears to have been done to it)</span>
<span id="cb34-1081"><a href="#cb34-1081" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb34-1082"><a href="#cb34-1082" aria-hidden="true" tabindex="-1"></a><span class="fu">#### 17-APR-2023</span></span>
<span id="cb34-1083"><a href="#cb34-1083" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb34-1084"><a href="#cb34-1084" aria-hidden="true" tabindex="-1"></a><span class="ss">-   </span>added to extraction protocol</span>
<span id="cb34-1085"><a href="#cb34-1085" aria-hidden="true" tabindex="-1"></a><span class="ss">-   </span>read about remote computing, Hyak Mox</span>
<span id="cb34-1086"><a href="#cb34-1086" aria-hidden="true" tabindex="-1"></a><span class="ss">-   </span>enabled Hyak account</span>
<span id="cb34-1087"><a href="#cb34-1087" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb34-1088"><a href="#cb34-1088" aria-hidden="true" tabindex="-1"></a><span class="fu">#### 14-APR-2023</span></span>
<span id="cb34-1089"><a href="#cb34-1089" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb34-1090"><a href="#cb34-1090" aria-hidden="true" tabindex="-1"></a><span class="ss">-   </span>attended Cat Fong chalk talk &amp; SAFS Cafe</span>
<span id="cb34-1091"><a href="#cb34-1091" aria-hidden="true" tabindex="-1"></a><span class="ss">-   </span>gave tour to Edith</span>
<span id="cb34-1092"><a href="#cb34-1092" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb34-1093"><a href="#cb34-1093" aria-hidden="true" tabindex="-1"></a><span class="fu">#### 11-APR-2023</span></span>
<span id="cb34-1094"><a href="#cb34-1094" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb34-1095"><a href="#cb34-1095" aria-hidden="true" tabindex="-1"></a><span class="ss">-   </span>NCBI SRA links downloading RNAseq fastq files</span>
<span id="cb34-1096"><a href="#cb34-1096" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb34-1097"><a href="#cb34-1097" aria-hidden="true" tabindex="-1"></a><span class="ss">-   </span>Looking for 'fake data' for M. capitata</span>
<span id="cb34-1098"><a href="#cb34-1098" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb34-1099"><a href="#cb34-1099" aria-hidden="true" tabindex="-1"></a><span class="ss">    -   </span>DNA, ITS2 to measure symbiont to host cell ratios for 'Patterns of bleaching and recovery of Montipora capitata in Kaneohe Bay' hosted at zenodo <span class="co">[</span><span class="ot">here</span><span class="co">](https://zenodo.org/record/53231#.ZDWE0HbMKTQ)</span> https://zenodo.org/record/53231#.ZDWE0HbMKTQ -RNAseq resources from A. Huffmeyer</span>
<span id="cb34-1100"><a href="#cb34-1100" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb34-1101"><a href="#cb34-1101" aria-hidden="true" tabindex="-1"></a><span class="ss">        -   </span>https://ahuffmyer.github.io/ASH_Putnam_Lab_Notebook/E5-Deep-Dive-RNAseq-Count-Matrix-Analysis/</span>
<span id="cb34-1102"><a href="#cb34-1102" aria-hidden="true" tabindex="-1"></a><span class="ss">        -   </span>https://github.com/AHuffmyer/EarlyLifeHistory_Energetics <span class="sc">\&gt;</span> Here is the spreadsheet of my NCBI accession numbers for my sequences: https://docs.google.com/spreadsheets/d/1qDGGpLFcmoO-fIFOPSUhPcxi4ErXIq2PkQbxvCCzI40/edit?pli=1#gid=457642286 If you look at the Mcapitata Ontogeny Timeseries tab it'll have all the SRR numbers you will need to download the TagSeq files. Instead of downloading the full data set you could just pick like three lifestages for a test comparison - AH</span>
<span id="cb34-1103"><a href="#cb34-1103" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb34-1104"><a href="#cb34-1104" aria-hidden="true" tabindex="-1"></a><span class="fu">#### 10-APR-2023</span></span>
<span id="cb34-1105"><a href="#cb34-1105" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb34-1106"><a href="#cb34-1106" aria-hidden="true" tabindex="-1"></a><span class="ss">-   </span>created script to randomize sample processing in <span class="co">[</span><span class="ot">coral-DNA-RNA-lab-extractions</span><span class="co">](https://github.com/sarahtanja/coral-DNA-RNA-lab-extractions)</span> repo</span>
<span id="cb34-1107"><a href="#cb34-1107" aria-hidden="true" tabindex="-1"></a><span class="ss">-   </span>called Zymo and learned that 'mix' in protocol step can either mean pipette up and down 10-12 times, or briefly touch to vortex mixer for a pulse</span>
<span id="cb34-1108"><a href="#cb34-1108" aria-hidden="true" tabindex="-1"></a><span class="ss">-   </span>Zymo tech also shared that it's best to follow protocol and add Lysis buffer to the sample supernatent in a separate rnase-free tube. If you do it in the filter tube some sample may start to percolate thru the filter before being fully buffered.</span>
<span id="cb34-1109"><a href="#cb34-1109" aria-hidden="true" tabindex="-1"></a><span class="ss">-   </span><span class="co">[</span><span class="ot">Horizontal Microtube Holder</span><span class="co">](https://www.zymoresearch.com/products/horizontal-microtube-holder)</span></span>
<span id="cb34-1110"><a href="#cb34-1110" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb34-1111"><a href="#cb34-1111" aria-hidden="true" tabindex="-1"></a><span class="fu">## March 2023</span></span>
<span id="cb34-1112"><a href="#cb34-1112" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb34-1113"><a href="#cb34-1113" aria-hidden="true" tabindex="-1"></a><span class="fu">#### 29-MAR-2023</span></span>
<span id="cb34-1114"><a href="#cb34-1114" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb34-1115"><a href="#cb34-1115" aria-hidden="true" tabindex="-1"></a><span class="ss">-   </span>started template for DNA/RNA data extraction sheet in new repo for extractions\</span>
<span id="cb34-1116"><a href="#cb34-1116" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb34-1117"><a href="#cb34-1117" aria-hidden="true" tabindex="-1"></a><span class="ss">-   </span>organized notes for samples in metadata</span>
<span id="cb34-1118"><a href="#cb34-1118" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb34-1119"><a href="#cb34-1119" aria-hidden="true" tabindex="-1"></a><span class="ss">-   </span>installed git on WSL via <span class="co">[</span><span class="ot">this tutorial</span><span class="co">](https://www.git-tower.com/blog/git-wsl/)</span></span>
<span id="cb34-1120"><a href="#cb34-1120" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb34-1121"><a href="#cb34-1121" aria-hidden="true" tabindex="-1"></a><span class="ss">-   </span>finished homework 1 for FISH546</span>
<span id="cb34-1122"><a href="#cb34-1122" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb34-1123"><a href="#cb34-1123" aria-hidden="true" tabindex="-1"></a><span class="ss">-   </span>lab-meeting notes</span>
<span id="cb34-1124"><a href="#cb34-1124" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb34-1125"><a href="#cb34-1125" aria-hidden="true" tabindex="-1"></a><span class="ss">    -   </span>purchasing</span>
<span id="cb34-1126"><a href="#cb34-1126" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb34-1127"><a href="#cb34-1127" aria-hidden="true" tabindex="-1"></a><span class="ss">        -   </span>no purchases in June!</span>
<span id="cb34-1128"><a href="#cb34-1128" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb34-1129"><a href="#cb34-1129" aria-hidden="true" tabindex="-1"></a><span class="at">            &gt; The University of Washington is transitioning from their old DOS-based system to Workday Finance, with a Go Live date of July 5th, 2023. During the transition period, there will be some disruptions to normal operations, and the continuity of research and teaching and minimizing the impact on faculty, staff, and students is the priority. There will be Restrictive Periods during which no one will be able to process purchase orders or obtain reimbursements in ARIBA. Exceptions may be made for urgent or emergency purchases. Travel reimbursements and non-travel-related reimbursements should be submitted before June 9th, and processing will resume on July 5th. Blanket Purchase Orders that are expected to be active past June 30th, 2023, will not be automatically transferred to Workday, and all continuing BPOs must be manually reentered by the College of the Environment. Navigating the transition will require patience and close partnerships between research and administrative groups</span></span>
<span id="cb34-1130"><a href="#cb34-1130" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb34-1131"><a href="#cb34-1131" aria-hidden="true" tabindex="-1"></a><span class="ss">        -   </span>space survey</span>
<span id="cb34-1132"><a href="#cb34-1132" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb34-1133"><a href="#cb34-1133" aria-hidden="true" tabindex="-1"></a><span class="ss">            -   </span>\% of time spent in spaces from July '22 - March '23</span>
<span id="cb34-1134"><a href="#cb34-1134" aria-hidden="true" tabindex="-1"></a><span class="ss">            -   </span>2% 242</span>
<span id="cb34-1135"><a href="#cb34-1135" aria-hidden="true" tabindex="-1"></a><span class="ss">            -   </span>8% 236</span>
<span id="cb34-1136"><a href="#cb34-1136" aria-hidden="true" tabindex="-1"></a><span class="ss">            -   </span>30% HIMB</span>
<span id="cb34-1137"><a href="#cb34-1137" aria-hidden="true" tabindex="-1"></a><span class="ss">            -   </span>60% 264, 264B</span>
<span id="cb34-1138"><a href="#cb34-1138" aria-hidden="true" tabindex="-1"></a><span class="ss">            -   </span>undergrad Nora Hessen 2hrs per week in 264, 264B since January '23</span>
<span id="cb34-1139"><a href="#cb34-1139" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb34-1140"><a href="#cb34-1140" aria-hidden="true" tabindex="-1"></a><span class="fu">#### 28-MAR-2023</span></span>
<span id="cb34-1141"><a href="#cb34-1141" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb34-1142"><a href="#cb34-1142" aria-hidden="true" tabindex="-1"></a><span class="ss">-   </span>read papers:</span>
<span id="cb34-1143"><a href="#cb34-1143" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb34-1144"><a href="#cb34-1144" aria-hidden="true" tabindex="-1"></a><span class="ss">    -   </span><span class="co">[</span><span class="ot">Bourne et al. 2016</span><span class="co">](https://www.annualreviews.org/doi/pdf/10.1146/annurev-micro-102215-095440)</span></span>
<span id="cb34-1145"><a href="#cb34-1145" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb34-1146"><a href="#cb34-1146" aria-hidden="true" tabindex="-1"></a><span class="ss">    -   </span><span class="co">[</span><span class="ot">Sogin et al. 2017</span><span class="co">](https://ami-journals.onlinelibrary.wiley.com/doi/epdf/10.1111/1758-2229.12541)</span></span>
<span id="cb34-1147"><a href="#cb34-1147" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb34-1148"><a href="#cb34-1148" aria-hidden="true" tabindex="-1"></a><span class="ss">    -   </span><span class="co">[</span><span class="ot">Kelly et al. 2014</span><span class="co">](https://www.ncbi.nlm.nih.gov/pmc/articles/PMC4104888/)</span></span>
<span id="cb34-1149"><a href="#cb34-1149" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb34-1150"><a href="#cb34-1150" aria-hidden="true" tabindex="-1"></a><span class="ss">-   </span>called Zymo tech Christian Camacho ccamacho\@zymoresearch.com to get advice on working with coral samples and the Zymo DNA/RNA miniprep kit:</span>
<span id="cb34-1151"><a href="#cb34-1151" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb34-1152"><a href="#cb34-1152" aria-hidden="true" tabindex="-1"></a><span class="ss">    -   </span>bead bash for 40 minutes for microbes on mortexer vortex-mixer! This is still tested to result in good RNA, though there may be a trade-off between bead-bashing for lysis and RNA interity... see Zymo optimized lysis protocol</span>
<span id="cb34-1153"><a href="#cb34-1153" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb34-1154"><a href="#cb34-1154" aria-hidden="true" tabindex="-1"></a><span class="ss">    -   </span>benchmark mortexer is not a tested method for the kit.. but it's closest to the vortex genie approved method for lysis</span>
<span id="cb34-1155"><a href="#cb34-1155" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb34-1156"><a href="#cb34-1156" aria-hidden="true" tabindex="-1"></a><span class="ss">        -   </span><span class="co">[</span><span class="ot">optimized lysis protocols</span><span class="co">](https://files.zymoresearch.com/documents/bead_beating_short_protocol_tables.pdf)</span></span>
<span id="cb34-1157"><a href="#cb34-1157" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb34-1158"><a href="#cb34-1158" aria-hidden="true" tabindex="-1"></a><span class="ss">    -   </span>mortar &amp; pestle with liquid nitrogen, then add powder to bead bashing tube with RNA shield</span>
<span id="cb34-1159"><a href="#cb34-1159" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb34-1160"><a href="#cb34-1160" aria-hidden="true" tabindex="-1"></a><span class="ss">    -   </span>lyophilized (freeze-dried) proteinase K is stable at room temp, it should be stored in the -20 freezer only after resuspending in the buffer</span>
<span id="cb34-1161"><a href="#cb34-1161" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb34-1162"><a href="#cb34-1162" aria-hidden="true" tabindex="-1"></a><span class="ss">    -   </span>the DNAse 1 is similarly a lyophilized powder, but a very small amount, such that the vial looks empty</span>
<span id="cb34-1163"><a href="#cb34-1163" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb34-1164"><a href="#cb34-1164" aria-hidden="true" tabindex="-1"></a><span class="ss">-   </span>Gathered lab supplies</span>
<span id="cb34-1165"><a href="#cb34-1165" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb34-1166"><a href="#cb34-1166" aria-hidden="true" tabindex="-1"></a><span class="ss">    -   </span>mortar &amp; pestles</span>
<span id="cb34-1167"><a href="#cb34-1167" aria-hidden="true" tabindex="-1"></a><span class="ss">    -   </span>alcohol burner for use with denatured ethyl alcohol</span>
<span id="cb34-1168"><a href="#cb34-1168" aria-hidden="true" tabindex="-1"></a><span class="ss">    -   </span>flint spark torch ignitor</span>
<span id="cb34-1169"><a href="#cb34-1169" aria-hidden="true" tabindex="-1"></a><span class="ss">    -   </span>found expired RNA later</span>
<span id="cb34-1170"><a href="#cb34-1170" aria-hidden="true" tabindex="-1"></a><span class="ss">    -   </span>DNAse away</span>
<span id="cb34-1171"><a href="#cb34-1171" aria-hidden="true" tabindex="-1"></a><span class="ss">    -   </span>DEPC water (does that expire?)</span>
<span id="cb34-1172"><a href="#cb34-1172" aria-hidden="true" tabindex="-1"></a><span class="ss">    -   </span>weigh papers (for powdered samples)</span>
<span id="cb34-1173"><a href="#cb34-1173" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb34-1174"><a href="#cb34-1174" aria-hidden="true" tabindex="-1"></a>    NEED</span>
<span id="cb34-1175"><a href="#cb34-1175" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb34-1176"><a href="#cb34-1176" aria-hidden="true" tabindex="-1"></a><span class="in">    -   [ ] scoopulas (to scoop powder from mortar to weigh paper)</span></span>
<span id="cb34-1177"><a href="#cb34-1177" aria-hidden="true" tabindex="-1"></a><span class="in">    -   [ ] denatured ethyl alcohol (disinfect labware between samples)</span></span>
<span id="cb34-1178"><a href="#cb34-1178" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb34-1179"><a href="#cb34-1179" aria-hidden="true" tabindex="-1"></a><span class="fu">#### 27-MAR-2023</span></span>
<span id="cb34-1180"><a href="#cb34-1180" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb34-1181"><a href="#cb34-1181" aria-hidden="true" tabindex="-1"></a><span class="ss">-   </span>read papers:</span>
<span id="cb34-1182"><a href="#cb34-1182" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb34-1183"><a href="#cb34-1183" aria-hidden="true" tabindex="-1"></a><span class="ss">    -   </span>https://environmentalevidencejournal.biomedcentral.com/articles/10.1186/s13750-023-00298-y</span>
<span id="cb34-1184"><a href="#cb34-1184" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb34-1185"><a href="#cb34-1185" aria-hidden="true" tabindex="-1"></a><span class="ss">-   </span>worked on coral collection report</span>
<span id="cb34-1186"><a href="#cb34-1186" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb34-1187"><a href="#cb34-1187" aria-hidden="true" tabindex="-1"></a><span class="fu">##### </span></span>
<span id="cb34-1188"><a href="#cb34-1188" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb34-1189"><a href="#cb34-1189" aria-hidden="true" tabindex="-1"></a><span class="fu">#### 21-MAR-2023</span></span>
<span id="cb34-1190"><a href="#cb34-1190" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb34-1191"><a href="#cb34-1191" aria-hidden="true" tabindex="-1"></a><span class="ss">-   </span>interviewed for WSG student assistant position</span>
<span id="cb34-1192"><a href="#cb34-1192" aria-hidden="true" tabindex="-1"></a><span class="ss">-   </span>began work on himb-2022 coral collection report</span>
<span id="cb34-1193"><a href="#cb34-1193" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb34-1194"><a href="#cb34-1194" aria-hidden="true" tabindex="-1"></a><span class="fu">#### 20-MAR-2023</span></span>
<span id="cb34-1195"><a href="#cb34-1195" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb34-1196"><a href="#cb34-1196" aria-hidden="true" tabindex="-1"></a><span class="ss">-   </span>installed Sophos anti-virus on Owl</span>
<span id="cb34-1197"><a href="#cb34-1197" aria-hidden="true" tabindex="-1"></a><span class="ss">-   </span>attended Robert's Lab meeting</span>
<span id="cb34-1198"><a href="#cb34-1198" aria-hidden="true" tabindex="-1"></a><span class="ss">-   </span>checked out Qubit buffer + reagents in Roberts lab with Olivia &amp; Zach</span>
<span id="cb34-1199"><a href="#cb34-1199" aria-hidden="true" tabindex="-1"></a><span class="ss">-   </span>put mortexer, DEPC-treated water, Qubit assay kit, on purchase log</span>
<span id="cb34-1200"><a href="#cb34-1200" aria-hidden="true" tabindex="-1"></a><span class="ss">-   </span>wrote <span class="co">[</span><span class="ot">Qubit RNA Broad Range Protocol</span><span class="co">](https://sarahtanja.github.io/lab-book/protocols/protocol-qbit/)</span> post</span>
<span id="cb34-1201"><a href="#cb34-1201" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb34-1202"><a href="#cb34-1202" aria-hidden="true" tabindex="-1"></a><span class="fu">#### 16-MAR-2023</span></span>
<span id="cb34-1203"><a href="#cb34-1203" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb34-1204"><a href="#cb34-1204" aria-hidden="true" tabindex="-1"></a><span class="ss">-   </span>finished SMEA/FISH 539 final paper</span>
<span id="cb34-1205"><a href="#cb34-1205" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb34-1206"><a href="#cb34-1206" aria-hidden="true" tabindex="-1"></a><span class="fu">#### 15-MAR-2023</span></span>
<span id="cb34-1207"><a href="#cb34-1207" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb34-1208"><a href="#cb34-1208" aria-hidden="true" tabindex="-1"></a><span class="ss">-   </span>out sick</span>
<span id="cb34-1209"><a href="#cb34-1209" aria-hidden="true" tabindex="-1"></a><span class="ss">-   </span>worked on SMEA/FISH 539 final paper</span>
<span id="cb34-1210"><a href="#cb34-1210" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb34-1211"><a href="#cb34-1211" aria-hidden="true" tabindex="-1"></a><span class="fu">#### 14-MAR-2023</span></span>
<span id="cb34-1212"><a href="#cb34-1212" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb34-1213"><a href="#cb34-1213" aria-hidden="true" tabindex="-1"></a><span class="ss">-   </span>out sick</span>
<span id="cb34-1214"><a href="#cb34-1214" aria-hidden="true" tabindex="-1"></a><span class="ss">-   </span>talked with Hailey Dockery about shadowing RNA extractions</span>
<span id="cb34-1215"><a href="#cb34-1215" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb34-1216"><a href="#cb34-1216" aria-hidden="true" tabindex="-1"></a><span class="fu">#### 13-MAR-2023</span></span>
<span id="cb34-1217"><a href="#cb34-1217" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb34-1218"><a href="#cb34-1218" aria-hidden="true" tabindex="-1"></a><span class="ss">-   </span>worked on SMEA/FISH 539 final paper</span>
<span id="cb34-1219"><a href="#cb34-1219" aria-hidden="true" tabindex="-1"></a><span class="ss">-   </span>updated R &amp; RStudio to latest versions</span>
<span id="cb34-1220"><a href="#cb34-1220" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb34-1221"><a href="#cb34-1221" aria-hidden="true" tabindex="-1"></a><span class="fu">#### 10-MAR-2023</span></span>
<span id="cb34-1222"><a href="#cb34-1222" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb34-1223"><a href="#cb34-1223" aria-hidden="true" tabindex="-1"></a><span class="ss">-   </span>finished SMEA/FISH 1page reflection</span>
<span id="cb34-1224"><a href="#cb34-1224" aria-hidden="true" tabindex="-1"></a><span class="ss">-   </span>worked on SMEA/FISH 539 final paper</span>
<span id="cb34-1225"><a href="#cb34-1225" aria-hidden="true" tabindex="-1"></a><span class="ss">-   </span>met with GSIS speaker on Zoom</span>
<span id="cb34-1226"><a href="#cb34-1226" aria-hidden="true" tabindex="-1"></a><span class="ss">-   </span>made hotel reservations for GSIS</span>
<span id="cb34-1227"><a href="#cb34-1227" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb34-1228"><a href="#cb34-1228" aria-hidden="true" tabindex="-1"></a><span class="fu">#### 09-MAR-2023</span></span>
<span id="cb34-1229"><a href="#cb34-1229" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb34-1230"><a href="#cb34-1230" aria-hidden="true" tabindex="-1"></a><span class="ss">-   </span>Attended Coastal Ecology candidate chalk talk</span>
<span id="cb34-1231"><a href="#cb34-1231" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb34-1232"><a href="#cb34-1232" aria-hidden="true" tabindex="-1"></a><span class="ss">-   </span>Shadowed Grace for new RNA extraction protocol &amp; Qubit QC for RNA quality</span>
<span id="cb34-1233"><a href="#cb34-1233" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb34-1234"><a href="#cb34-1234" aria-hidden="true" tabindex="-1"></a><span class="ss">-   </span>Met with Emma Strand about M cap DNA/RNA extractions... notes from Zoom chat:</span>
<span id="cb34-1235"><a href="#cb34-1235" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb34-1236"><a href="#cb34-1236" aria-hidden="true" tabindex="-1"></a><span class="ss">    -   </span>email Erin Chille <span class="co">[</span><span class="ot">eec72\@scarletmail.rutgers.edu</span><span class="co">](mailto:eec72@scarletmail.rutgers.edu)</span>{.email} about 16S extraction kit</span>
<span id="cb34-1237"><a href="#cb34-1237" aria-hidden="true" tabindex="-1"></a><span class="ss">    -   </span>https://emmastrand.github.io/EmmaStrand_Notebook/Kbay-Bleaching-2019-DNA-RNA-Extractions/</span>
<span id="cb34-1238"><a href="#cb34-1238" aria-hidden="true" tabindex="-1"></a><span class="ss">    -   </span>host DNA (looked at DNA methylation), symbiont DNA (ITS2), microbiome DNA (16S not so good), host RNA + symbiont RNA (Illumina Nova Seq RNAseq)</span>
<span id="cb34-1239"><a href="#cb34-1239" aria-hidden="true" tabindex="-1"></a><span class="ss">    -   </span>coral issues- mucus clogging spin-column filter (avoid mucucs, avoid skeleton)</span>
<span id="cb34-1240"><a href="#cb34-1240" aria-hidden="true" tabindex="-1"></a><span class="ss">    -   </span>how big of a fragment do I put in the bead tube? 1/3 bead, 1/3 coral, 1/3 shield -- https://emmastrand.github.io/EmmaStrand_Notebook/Kbay-Bleaching-2019-DNA-RNA-Extractions/</span>
<span id="cb34-1241"><a href="#cb34-1241" aria-hidden="true" tabindex="-1"></a><span class="ss">    -   </span>optimize time to vortex bead tube (2mins for m. cap) <span class="sc">\*</span>vortex machine, holds 8 tubes at once</span>
<span id="cb34-1242"><a href="#cb34-1242" aria-hidden="true" tabindex="-1"></a><span class="ss">    -   </span>https://www.southernlabware.com/mortexertm-vortex-mixer-115v.html?gclid=Cj0KCQiApKagBhC1ARIsAFc7Mc5LH-QjqtDR98g9ZgE9iAOs-y35m7X5rr7pHCqfPGRtTJVPEeQWbLYaAtCpEALw_wcB</span>
<span id="cb34-1243"><a href="#cb34-1243" aria-hidden="true" tabindex="-1"></a><span class="ss">    -   </span>100uL total DNA material end product from kit</span>
<span id="cb34-1244"><a href="#cb34-1244" aria-hidden="true" tabindex="-1"></a><span class="ss">    -   </span>12ng of input for ITS2 about 5uL</span>
<span id="cb34-1245"><a href="#cb34-1245" aria-hidden="true" tabindex="-1"></a><span class="ss">    -   </span>aim for 10 - 70 ng/uL DNA quantity (Qubit)</span>
<span id="cb34-1246"><a href="#cb34-1246" aria-hidden="true" tabindex="-1"></a><span class="ss">    -   </span>DNA quality (gel) <span class="sc">\*</span>important for methylation studies, but may not be for 16S/ITS2</span>
<span id="cb34-1247"><a href="#cb34-1247" aria-hidden="true" tabindex="-1"></a><span class="ss">    -   </span>100uL total RNA material end product, send 20-30uL for RNAseq</span>
<span id="cb34-1248"><a href="#cb34-1248" aria-hidden="true" tabindex="-1"></a><span class="ss">    -   </span>https://github.com/hputnam/HI_Bleaching_Timeseries/blob/main/Dec-July-2019-analysis/data/Molecular-labwork.xlsx</span>
<span id="cb34-1249"><a href="#cb34-1249" aria-hidden="true" tabindex="-1"></a><span class="ss">    -   </span>sample ID, use a separate extraction ID (in the event of multiple extractions from the same sample!)</span>
<span id="cb34-1250"><a href="#cb34-1250" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb34-1251"><a href="#cb34-1251" aria-hidden="true" tabindex="-1"></a><span class="fu">#### 07-MAR-2023</span></span>
<span id="cb34-1252"><a href="#cb34-1252" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb34-1253"><a href="#cb34-1253" aria-hidden="true" tabindex="-1"></a><span class="ss">-   </span>worked on SMEA/FISH 539 final paper</span>
<span id="cb34-1254"><a href="#cb34-1254" aria-hidden="true" tabindex="-1"></a><span class="ss">-   </span>met to make GSIS budget w/ Juliette</span>
<span id="cb34-1255"><a href="#cb34-1255" aria-hidden="true" tabindex="-1"></a><span class="ss">-   </span>met with Grace from Robert's Lab to practice RNA extraction</span>
<span id="cb34-1256"><a href="#cb34-1256" aria-hidden="true" tabindex="-1"></a><span class="ss">-   </span>participated in SAC interview for ADDEI</span>
<span id="cb34-1257"><a href="#cb34-1257" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb34-1258"><a href="#cb34-1258" aria-hidden="true" tabindex="-1"></a><span class="fu">#### 06-MAR-2023</span></span>
<span id="cb34-1259"><a href="#cb34-1259" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb34-1260"><a href="#cb34-1260" aria-hidden="true" tabindex="-1"></a><span class="ss">-   </span>presented final group project in SMEA/FISH 539</span>
<span id="cb34-1261"><a href="#cb34-1261" aria-hidden="true" tabindex="-1"></a><span class="ss">-   </span>worked on SMEA/FISH 539 final paper</span>
<span id="cb34-1262"><a href="#cb34-1262" aria-hidden="true" tabindex="-1"></a><span class="ss">-   </span>attended Coastal Ecology faculty candidate seminar</span>
<span id="cb34-1263"><a href="#cb34-1263" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb34-1264"><a href="#cb34-1264" aria-hidden="true" tabindex="-1"></a><span class="fu">#### 03-MAR-2023</span></span>
<span id="cb34-1265"><a href="#cb34-1265" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb34-1266"><a href="#cb34-1266" aria-hidden="true" tabindex="-1"></a><span class="ss">-   </span>I swear I've done stuff since January / how has winter quarter gone by so fast?!</span>
<span id="cb34-1267"><a href="#cb34-1267" aria-hidden="true" tabindex="-1"></a><span class="ss">-   </span>submitted <span class="co">[</span><span class="ot">PCC Climate Solutions</span><span class="co">](https://pcc.uw.edu/research/funding-opportunities/)</span> grant funding application this morning</span>
<span id="cb34-1268"><a href="#cb34-1268" aria-hidden="true" tabindex="-1"></a><span class="ss">-   </span>worked on FSMEA/FISH 539 final project + presentation</span>
<span id="cb34-1269"><a href="#cb34-1269" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb34-1270"><a href="#cb34-1270" aria-hidden="true" tabindex="-1"></a><span class="fu">#### 02-MAR-2023</span></span>
<span id="cb34-1271"><a href="#cb34-1271" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb34-1272"><a href="#cb34-1272" aria-hidden="true" tabindex="-1"></a><span class="ss">-   </span>submitted FHL TA application for summer A term marine botany</span>
<span id="cb34-1273"><a href="#cb34-1273" aria-hidden="true" tabindex="-1"></a><span class="ss">-   </span>helped interview ADDEI candidate</span>
<span id="cb34-1274"><a href="#cb34-1274" aria-hidden="true" tabindex="-1"></a><span class="ss">-   </span>attended Carter's Coastal Ecology candidate chalk talk</span>
<span id="cb34-1275"><a href="#cb34-1275" aria-hidden="true" tabindex="-1"></a><span class="ss">-   </span>met with Steven on Zoom and got permission to use Bioanalyzer</span>
<span id="cb34-1276"><a href="#cb34-1276" aria-hidden="true" tabindex="-1"></a><span class="ss">-   </span>worked on sequencing quotes, DNA/RNA extraction protocol, reached out to Emma Strand and Grace</span>
<span id="cb34-1277"><a href="#cb34-1277" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb34-1278"><a href="#cb34-1278" aria-hidden="true" tabindex="-1"></a><span class="fu">## January 2023</span></span>
<span id="cb34-1279"><a href="#cb34-1279" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb34-1280"><a href="#cb34-1280" aria-hidden="true" tabindex="-1"></a><span class="fu">#### 31-JAN-2023</span></span>
<span id="cb34-1281"><a href="#cb34-1281" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb34-1282"><a href="#cb34-1282" aria-hidden="true" tabindex="-1"></a><span class="ss">-   </span>prepared 1st committee meeting blog post</span>
<span id="cb34-1283"><a href="#cb34-1283" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb34-1284"><a href="#cb34-1284" aria-hidden="true" tabindex="-1"></a><span class="fu">#### 20-JAN-2023</span></span>
<span id="cb34-1285"><a href="#cb34-1285" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb34-1286"><a href="#cb34-1286" aria-hidden="true" tabindex="-1"></a><span class="ss">-   </span>cleaned PBB</span>
<span id="cb34-1287"><a href="#cb34-1287" aria-hidden="true" tabindex="-1"></a><span class="ss">-   </span>strategized chapter 1: Low Dose Effects &amp; chapter 2: Multiple Stressors for thesis</span>
<span id="cb34-1288"><a href="#cb34-1288" aria-hidden="true" tabindex="-1"></a><span class="ss">-   </span>arranged to meet with Miranda to talk about summer kelp gametophyte project</span>
<span id="cb34-1289"><a href="#cb34-1289" aria-hidden="true" tabindex="-1"></a><span class="ss">-   </span>completed lab safety trainings</span>
<span id="cb34-1290"><a href="#cb34-1290" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb34-1291"><a href="#cb34-1291" aria-hidden="true" tabindex="-1"></a><span class="fu">#### 19-JAN-2023</span></span>
<span id="cb34-1292"><a href="#cb34-1292" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb34-1293"><a href="#cb34-1293" aria-hidden="true" tabindex="-1"></a><span class="ss">-   </span>read <span class="co">[</span><span class="ot">Ariana Huffmyer's</span><span class="co">](https://ahuffmyer.github.io/ASH_Putnam_Lab_Notebook/)</span> lab posts on extraction protocols</span>
<span id="cb34-1294"><a href="#cb34-1294" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb34-1295"><a href="#cb34-1295" aria-hidden="true" tabindex="-1"></a><span class="ss">-   </span>read <span class="co">[</span><span class="ot">Ainsworth et al 2015</span><span class="co">](https://www.nature.com/articles/ismej201539.pdf)</span> &amp; <span class="co">[</span><span class="ot">Brown et al 2021</span><span class="co">](https://onlinelibrary.wiley.com/doi/abs/10.1111/mec.15899)</span></span>
<span id="cb34-1296"><a href="#cb34-1296" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb34-1297"><a href="#cb34-1297" aria-hidden="true" tabindex="-1"></a><span class="ss">    -   </span>MarBio Plant DNA extraction Kit for whole coral holobiont 16S rRNA Bacteria</span>
<span id="cb34-1298"><a href="#cb34-1298" aria-hidden="true" tabindex="-1"></a><span class="ss">    -   </span>Pure Link Plant Total DNA purification systems (Invitrogen) for 16S rRNA Bacteria</span>
<span id="cb34-1299"><a href="#cb34-1299" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb34-1300"><a href="#cb34-1300" aria-hidden="true" tabindex="-1"></a><span class="ss">-   </span>attended JPG lab meeting and gave feedback on Eileen Bates' pinto abalone presentation for Alaska Science Sympsosium</span>
<span id="cb34-1301"><a href="#cb34-1301" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb34-1302"><a href="#cb34-1302" aria-hidden="true" tabindex="-1"></a><span class="ss">-   </span>designed graphical methods in Canva for pae-temp experiment</span>
<span id="cb34-1303"><a href="#cb34-1303" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb34-1304"><a href="#cb34-1304" aria-hidden="true" tabindex="-1"></a><span class="fu">#### 18-JAN-2023</span></span>
<span id="cb34-1305"><a href="#cb34-1305" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb34-1306"><a href="#cb34-1306" aria-hidden="true" tabindex="-1"></a><span class="ss">-   </span>worked with <span class="co">[</span><span class="ot">norahess</span><span class="co">](https://github.com/norahess)</span> to setup collaboration on ibis repo!</span>
<span id="cb34-1307"><a href="#cb34-1307" aria-hidden="true" tabindex="-1"></a><span class="ss">-   </span>talking about setting up Aiptasia spawning tanks to run experiments with cnidarian model organism reproduction</span>
<span id="cb34-1308"><a href="#cb34-1308" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb34-1309"><a href="#cb34-1309" aria-hidden="true" tabindex="-1"></a><span class="fu">#### 17-JAN-2023</span></span>
<span id="cb34-1310"><a href="#cb34-1310" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb34-1311"><a href="#cb34-1311" aria-hidden="true" tabindex="-1"></a><span class="ss">-   </span>attended Roberts lab meeting, discussed curriculum development for Maritime Highschool</span>
<span id="cb34-1312"><a href="#cb34-1312" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb34-1313"><a href="#cb34-1313" aria-hidden="true" tabindex="-1"></a><span class="fu">#### 13-JAN-2023</span></span>
<span id="cb34-1314"><a href="#cb34-1314" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb34-1315"><a href="#cb34-1315" aria-hidden="true" tabindex="-1"></a><span class="ss">-   </span>Read up on 'Omics', started a post on definitions and basics of the study of DNA &amp; RNA</span>
<span id="cb34-1316"><a href="#cb34-1316" aria-hidden="true" tabindex="-1"></a><span class="ss">-   </span>Found <span class="co">[</span><span class="ot">whole genome for *M. capitata* on NCBI</span><span class="co">](https://www.ncbi.nlm.nih.gov/genome/browse/#!/overview/montipora%20capitata)</span></span>
<span id="cb34-1317"><a href="#cb34-1317" aria-hidden="true" tabindex="-1"></a><span class="ss">-   </span>Helmkampf, M., Bellinger, M. R., Geib, S. M., Sim, S. B., &amp; Takabayashi, M. (2019). Draft Genome of the Rice Coral Montipora capitata Obtained from Linked-Read Sequencing. *Genome Biology and Evolution*, *11*(7), 2045--2054. https://doi.org/10.1093/gbe/evz135</span>
<span id="cb34-1318"><a href="#cb34-1318" aria-hidden="true" tabindex="-1"></a><span class="ss">-   </span>Williams, A., Pathmanathan, J. S., Stephens, T. G., Su, X., Chiles, E. N., Conetta, D., Putnam, H. M., &amp; Bhattacharya, D. (2021). Multi-omic characterization of the thermal stress phenome in the stony coral Montipora capitata. *PeerJ*, *9*, e12335. https://doi.org/10.7717/peerj.12335</span>
<span id="cb34-1319"><a href="#cb34-1319" aria-hidden="true" tabindex="-1"></a><span class="ss">-   </span>Shumaker, A., Putnam, H. M., Qiu, H., Price, D. C., Zelzion, E., Harel, A., Wagner, N. E., Gates, R. D., Yoon, H. S., &amp; Bhattacharya, D. (2019). Genome analysis of the rice coral Montipora capitata. *Scientific Reports*, *9*(1), 2571. https://doi.org/10.1038/s41598-019-39274-3</span>
<span id="cb34-1320"><a href="#cb34-1320" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb34-1321"><a href="#cb34-1321" aria-hidden="true" tabindex="-1"></a><span class="fu">#### 12-JAN-2023</span></span>
<span id="cb34-1322"><a href="#cb34-1322" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb34-1323"><a href="#cb34-1323" aria-hidden="true" tabindex="-1"></a><span class="ss">-   </span>Explored the use of purple sea urchin and/or Aiptasia for phthalate low-dose effect gametogenesis study (proposal brainstorming) for the <span class="co">[</span><span class="ot">IBIS research mentor program</span><span class="co">](https://environment.uw.edu/students/current-students/identity-belonging-and-inquiry-in-science-ibis-program/)</span></span>
<span id="cb34-1324"><a href="#cb34-1324" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb34-1325"><a href="#cb34-1325" aria-hidden="true" tabindex="-1"></a><span class="ss">-   </span>Created <span class="co">[</span><span class="ot">ibis repo</span><span class="co">](https://github.com/sarahtanja/ibis)</span> to organize project</span>
<span id="cb34-1326"><a href="#cb34-1326" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb34-1327"><a href="#cb34-1327" aria-hidden="true" tabindex="-1"></a><span class="ss">-   </span>Completed <span class="co">[</span><span class="ot">lab safety training matrix</span><span class="co">](https://github.com/sarahtanja/ibis/blob/main/ehslabsafetytrainmatrix.pdf)</span> for myself and NH, identified required and recommended lab safety trainings</span>
<span id="cb34-1328"><a href="#cb34-1328" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb34-1329"><a href="#cb34-1329" aria-hidden="true" tabindex="-1"></a><span class="ss">-   </span>Potential to use rack in PBB near interior door for Aiptasia spawning incubator, setup workflow with dissecting scope and camera all in PBB for IBIS project</span>
<span id="cb34-1330"><a href="#cb34-1330" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb34-1331"><a href="#cb34-1331" aria-hidden="true" tabindex="-1"></a><span class="ss">-   </span>Inventoried equipment we have in lab 236 for molecular work:</span>
<span id="cb34-1332"><a href="#cb34-1332" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb34-1333"><a href="#cb34-1333" aria-hidden="true" tabindex="-1"></a><span class="ss">    -   </span>Beckman Coulter Allegra X-30R Centrifuge</span>
<span id="cb34-1334"><a href="#cb34-1334" aria-hidden="true" tabindex="-1"></a><span class="ss">    -   </span>Thermo Scientific NanoDrop 2000c Spectrophotometer</span>
<span id="cb34-1335"><a href="#cb34-1335" aria-hidden="true" tabindex="-1"></a><span class="ss">    -   </span>Fisherbrand Isotemp heat block</span>
<span id="cb34-1336"><a href="#cb34-1336" aria-hidden="true" tabindex="-1"></a><span class="ss">    -   </span>Beckman Coulter Microfuge 16 Centrifuge</span>
<span id="cb34-1337"><a href="#cb34-1337" aria-hidden="true" tabindex="-1"></a><span class="ss">    -   </span>Eppendorf Mastercycler nexus gradient</span>
<span id="cb34-1338"><a href="#cb34-1338" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb34-1339"><a href="#cb34-1339" aria-hidden="true" tabindex="-1"></a><span class="fu">#### 09-JAN-2023</span></span>
<span id="cb34-1340"><a href="#cb34-1340" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb34-1341"><a href="#cb34-1341" aria-hidden="true" tabindex="-1"></a><span class="ss">-   </span>Today I setup my GitHub 'notebook'! This is the beginning of my exploration of using GitHub to jot down thoughts, record 'daily bits', and organize the procedures and methods I'll use in my thesis research</span>
<span id="cb34-1342"><a href="#cb34-1342" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb34-1343"><a href="#cb34-1343" aria-hidden="true" tabindex="-1"></a><span class="ss">-   </span><span class="co">[</span><span class="ot">zbengt</span><span class="co">](https://github.com/zbengt)</span> introduced me to '<span class="co">[</span><span class="ot">elicit</span><span class="co">](https://elicit.org/)</span>', a lit review search AI. I searched "What is the role of endocrine signaling in cnidarian development" and found an exciting paper titled <span class="co">[</span><span class="ot">**"Sea Anemones Responding to Sex Hormones, Oxybenzone, and Benzyl Butyl Phthalate: Transcriptional Profiling and *in Silico* Modelling Provide Clues to Decipher Endocrine Disruption in Cnidarians"**</span><span class="co">](https://www.frontiersin.org/articles/10.3389/fgene.2021.793306/full)</span> authored by Morgan et al. in 2022</span>
<span id="cb34-1344"><a href="#cb34-1344" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb34-1345"><a href="#cb34-1345" aria-hidden="true" tabindex="-1"></a><span class="ss">-   </span>I explored another AI, <span class="co">[</span><span class="ot">ChatGPT</span><span class="co">](https://openai.com/blog/chatgpt/)</span>, and gave it the prompt "write me a poem about corals and oysters and mothers" and it returned:</span>
<span id="cb34-1346"><a href="#cb34-1346" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb34-1347"><a href="#cb34-1347" aria-hidden="true" tabindex="-1"></a><span class="ss">    -   </span></span>
<span id="cb34-1348"><a href="#cb34-1348" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb34-1349"><a href="#cb34-1349" aria-hidden="true" tabindex="-1"></a><span class="at">        &gt; Coral reefs, a vibrant sea of life, A home for countless creatures, rife. Oysters, clams, and mussels too, All nestled safe, their shells in view.</span></span>
<span id="cb34-1350"><a href="#cb34-1350" aria-hidden="true" tabindex="-1"></a><span class="at">        &gt;</span></span>
<span id="cb34-1351"><a href="#cb34-1351" aria-hidden="true" tabindex="-1"></a><span class="at">        &gt; But danger lurks, the waters warm, The coral bleaches, it's colors torn. Mother Nature's wrath, a sight to see, But we can help, if only we believe.</span></span>
<span id="cb34-1352"><a href="#cb34-1352" aria-hidden="true" tabindex="-1"></a><span class="at">        &gt;</span></span>
<span id="cb34-1353"><a href="#cb34-1353" aria-hidden="true" tabindex="-1"></a><span class="at">        &gt; For mothers, fierce and loving beasts, Will do what it takes, to protect at least. Their young and home, with all their might, They'll fight and strive, through day and night.</span></span>
<span id="cb34-1354"><a href="#cb34-1354" aria-hidden="true" tabindex="-1"></a><span class="at">        &gt;</span></span>
<span id="cb34-1355"><a href="#cb34-1355" aria-hidden="true" tabindex="-1"></a><span class="at">        &gt; So let us join, in their noble quest, To save the reefs, and all the rest. For oysters, corals, and mothers too, Deserve a chance, to make it through.</span></span>
<span id="cb34-1356"><a href="#cb34-1356" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb34-1357"><a href="#cb34-1357" aria-hidden="true" tabindex="-1"></a><span class="ss">-   </span>I followed up with a ChatGPT prompt "what are examples of endocrine signaling in cnidarians?" and it returned:</span>
<span id="cb34-1358"><a href="#cb34-1358" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb34-1359"><a href="#cb34-1359" aria-hidden="true" tabindex="-1"></a><span class="ss">    -   </span></span>
<span id="cb34-1360"><a href="#cb34-1360" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb34-1361"><a href="#cb34-1361" aria-hidden="true" tabindex="-1"></a><span class="at">        &gt; Endocrine signaling is the process of communicating information from one cell to another by releasing hormones into the circulatory system or through direct contact. In cnidarians, endocrine signaling can be used for a variety of functions, including the regulation of growth, development, and behavior.</span></span>
<span id="cb34-1362"><a href="#cb34-1362" aria-hidden="true" tabindex="-1"></a><span class="at">        &gt;</span></span>
<span id="cb34-1363"><a href="#cb34-1363" aria-hidden="true" tabindex="-1"></a><span class="at">        &gt; One example of endocrine signaling in cnidarians is the release of growth hormones, which stimulate the growth and development of cells and tissues. For example, in some species of cnidarians, the release of growth hormones is regulated by the action of a hormone called insulin-like growth factor (IGF).</span></span>
<span id="cb34-1364"><a href="#cb34-1364" aria-hidden="true" tabindex="-1"></a><span class="at">        &gt;</span></span>
<span id="cb34-1365"><a href="#cb34-1365" aria-hidden="true" tabindex="-1"></a><span class="at">        &gt; Another example of endocrine signaling in cnidarians is the regulation of reproductive function. In some species of cnidarians, hormones called gonadotropins are released by the endocrine cells of the gonads (sex organs) to stimulate the production and release of gametes (sex cells).</span></span>
<span id="cb34-1366"><a href="#cb34-1366" aria-hidden="true" tabindex="-1"></a><span class="at">        &gt;</span></span>
<span id="cb34-1367"><a href="#cb34-1367" aria-hidden="true" tabindex="-1"></a><span class="at">        &gt; Endocrine signaling is also involved in the regulation of behavior in cnidarians. For example, some species of cnidarians release hormones called neuropeptides that act on the nervous system to regulate behaviors such as feeding, movement, and response to stimuli.</span></span>
<span id="cb34-1368"><a href="#cb34-1368" aria-hidden="true" tabindex="-1"></a><span class="at">        &gt;</span></span>
<span id="cb34-1369"><a href="#cb34-1369" aria-hidden="true" tabindex="-1"></a><span class="at">        &gt; Overall, endocrine signaling plays a vital role in the regulation of various functions in cnidarians, including growth, development, and behavior.</span></span>
<span id="cb34-1370"><a href="#cb34-1370" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb34-1371"><a href="#cb34-1371" aria-hidden="true" tabindex="-1"></a>​<span class="sc">\*\*</span>Is this is all true!? I'll have to follow-up with my own dive into primary literature...</span>
<span id="cb34-1372"><a href="#cb34-1372" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb34-1373"><a href="#cb34-1373" aria-hidden="true" tabindex="-1"></a><span class="ss">-   </span>Another ChatGPT AI response, when asked the same questions, but prompted to provide references:</span>
<span id="cb34-1374"><a href="#cb34-1374" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb34-1375"><a href="#cb34-1375" aria-hidden="true" tabindex="-1"></a><span class="ss">    -   </span></span>
<span id="cb34-1376"><a href="#cb34-1376" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb34-1377"><a href="#cb34-1377" aria-hidden="true" tabindex="-1"></a><span class="at">        &gt; Examples of endocrine signaling in cnidarians include the release of hormones in response to stimuli such as touch, temperature changes, and changes in light intensity.</span></span>
<span id="cb34-1378"><a href="#cb34-1378" aria-hidden="true" tabindex="-1"></a><span class="at">        &gt;</span></span>
<span id="cb34-1379"><a href="#cb34-1379" aria-hidden="true" tabindex="-1"></a><span class="at">        &gt; For example, the hormone adenylate cyclase-activating polypeptide (ACAP) is released from cnidocytes in response to a variety of stimuli and activates adenylate cyclase, leading to the production of cyclic adenosine monophosphate (cAMP) and the regulation of various physiological processes.</span></span>
<span id="cb34-1380"><a href="#cb34-1380" aria-hidden="true" tabindex="-1"></a><span class="at">        &gt;</span></span>
<span id="cb34-1381"><a href="#cb34-1381" aria-hidden="true" tabindex="-1"></a><span class="at">        &gt; Other signaling molecules that have been identified in cnidarians include cyclic guanosine monophosphate (cGMP), inositol triphosphate (IP3), and Ca2+-dependent signaling pathways.</span></span>
<span id="cb34-1382"><a href="#cb34-1382" aria-hidden="true" tabindex="-1"></a><span class="at">        &gt;</span></span>
<span id="cb34-1383"><a href="#cb34-1383" aria-hidden="true" tabindex="-1"></a><span class="at">        &gt; References:</span></span>
<span id="cb34-1384"><a href="#cb34-1384" aria-hidden="true" tabindex="-1"></a><span class="at">        &gt;</span></span>
<span id="cb34-1385"><a href="#cb34-1385" aria-hidden="true" tabindex="-1"></a><span class="at">        &gt; Levitan, D. R., &amp; Kohn, A. B. (2001). The Endocrine System of Cnidarians. Annual Review of Physiology, 63(1), 319-344.</span></span>
<span id="cb34-1386"><a href="#cb34-1386" aria-hidden="true" tabindex="-1"></a><span class="at">        &gt;</span></span>
<span id="cb34-1387"><a href="#cb34-1387" aria-hidden="true" tabindex="-1"></a><span class="at">        &gt; Schmid, V., &amp; Pfründner, R. (1995). Adenylate cyclase-activating polypeptide in cnidarians: a new class of peptides with signal transducing properties. Regulatory Peptides, 61(2), 149-157.</span></span>
<span id="cb34-1388"><a href="#cb34-1388" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb34-1389"><a href="#cb34-1389" aria-hidden="true" tabindex="-1"></a>​</span>
</code><button title="Copy to Clipboard" class="code-copy-button"><i class="bi"></i></button></pre></div>
</div></div></div></div></div>
</div> <!-- /content -->
<footer class="footer">
  <div class="nav-footer">
    <div class="nav-footer-left">
<p>This page is built using <a href="https://quarto.org/">Quarto</a>❤️ and many pages and posts are under active development</p>
</div>   
    <div class="nav-footer-center">
      &nbsp;
    </div>
    <div class="nav-footer-right">
      &nbsp;
    </div>
  </div>
</footer>




<script src="../../site_libs/quarto-html/zenscroll-min.js"></script>
</body></html>