---
title: "The RNA Inhibitors"
subtitle: "A tragic saga of continued troubles"
author: "Sarah Tanja"
date: '08/22/2023'
categories: [lab records]
draft: false
toc: true
toc-title: Contents <i class="bi bi-bookmark-heart"></i>
toc-depth: 5
toc-location: left
bibliography: references.bib
reference-location: margin
citation-location: margin
quarto:
  package_options:
    cran: "https://cran.rstudio.com/"
---

```{r, include=FALSE}
knitr::opts_chunk$set(
  message=FALSE,
  warning=FALSE,
  include=TRUE,
  #cache=TRUE,
  echo=FALSE
  )
```

```{r, include=FALSE}
#install.packages("data.table")
#install.packages("dplyr")
#install.packages("formattable")
#install.packages("tidyverse")

library(data.table)
library(dplyr)
library(formattable)
library(tidyverse)

customGreen = "#71CA97"
customOrange = "#fda172"
customRed = "#ff7f7f"


# formattable functions

# salts are good indicated by green text between 2.0 and 2.2
salts <- formatter("span",
  style = x ~ style("font-weight" = ifelse(x > 2.0, "bold", "normal"),
                    "color" = ifelse(x > 2.2, customGreen,
                                  ifelse(x >= 2.0 & x <= 2.2, customGreen,
                                      ifelse(x < 2.0, customRed, "inherit")))
                    ))

# proteins are good indicated by green text between 1.8 and 2.0
proteins <- formatter("span",
  style = x ~ style("font-weight" = ifelse(x > 1.8, "bold", "normal"),
                    "color" = ifelse(x > 2.0, customGreen,
                                    ifelse(x >= 1.8 & x <= 2.0, customGreen,
                                          ifelse(x < 1.8, customOrange, 
                                                ifelse(x < 1.6, customRed, "inherit"))))
                    ))

# concentration (nanograms per microliter) is good indicated by green text between 15-35 ng/uL 
ngul <- formatter("span",
  style = x ~ style("font-weight" = ifelse(x > 15, "bold", "normal"),
                    "color" = ifelse(x >= 15, customGreen,
                                  ifelse(x < 15 & x >= 10, customOrange,
                                        ifelse(x < 10, customRed, "inherit")))
                    ))

```

## So your samples failed library prep ... 😰

After shipping 20 samples to the University of Texas at Austin GSAF for TagSeq on June 7th 2023, I got a message back from them on June 28th 2023:

> "Your samples failed in library prep. All 20 of them, the other samples on the 96 well plate as well as our positive control all worked great. We seen this before and it is almost always an indication that there are some type of inhibitors in your samples and it impacted the prep. Based on the quality of the RNA there is nothing more we can do for them, they will not work in the prep until they have been purified, I am not sure how they were cleaned up after the extraction but it needs to be more rigorous and be certain there are no contaminants in the RNA."

After initial panic and frustration 😨😭... I started researching reverse transcriptase[^1] inhibitors and reaching out to the [Roberts](https://robertslab.github.io/resources/) and [Putnam](http://putnamlab.com/) Labs to ask for advice (recorded in this Roberts Lab [github issue](https://github.com/RobertsLab/resources/issues/1670)).

[^1]: Reverse transcriptase is an enzyme that transcribes single-stranded RNA into complementary DNA (cDNA) during library prep for RNA sequencing. It was discovered in the 1970's by David Baltimore and Howard Temin when studying retroviruses (checkout this [YouTube video](https://www.youtube.com/watch?app=desktop&v=v9EL8V-6la4) for a brief history on reverse transcriptase, and this [YouTube video](https://www.youtube.com/watch?v=POEDYHKyYDk) for a brief explanation of how it is used to turn mRNA to cDNA).

So what did I find, and what does this mean for my samples?

The RNA quantity, measured by Qubit Fluorometer, is good. And the RNA quality, measured by UT GSAF Bioanalyzer, is also good. The issue is that there is *something else* in the sample that is preventing the reverse transcriptase from generating the cDNA.

## Assessing Purity

Nanodrop purity targets[^2]:

[^2]: Check out [Assessing RNA quantity and purity with Nanodrop](https://toptipbio.com/the-nanodrop-results-explained/) by TopTipBio

-   [^3]260/280 (protein purity) **1.80 - 2.00**

-   260/230 (salts purity) **2.00 - 2.20**

[^3]: 5 nucleotides have different 260:280 ratios:

    -   Guanine: 1.15

    -   Adenine: 4.50

    -   Cytosine: 1.51

    -   Uracil: 4.0

    -   Thymine: 1.47

    So RNA typically has a higher 260/280 ratio than DNA due to the higher 260/280 ratio of Uracil compared to Thymine

#### ⚠️ Phenol Contamination may manifest in a 260/280 ratio greater than 2.00 !

> "**A 260/280 ratio greater than 2.0 does NOT indicate the presence of especially high quality DNA, it most often reflects residual contaminants, such as phenol.**" -[@zotero-2042]
>
> " ... samples with purity ratios that range from 1.8 to 2.0 are typically considered pure. **Although purity ratios are very useful and informative, they do not always provide a complete picture of sample purity.**"
>
> "Residual chemicals from nucleic acid extraction processes (such as **guanidine salts or phenol**) or cell components that co-extract with the nucleic acids (such as protein) **can** **artificially inflate the A260 value**."

## Zymo OneStep PCR Inhibitor Removal Kit

I first tried purifying sample RNA using the [Zymo OneStep PCR Inhibitor Removal Kit](https://www.zymoresearch.com/collections/onestep-pcr-inhibitor-removal-kits) on a subset of the samples.

```{r load in data}
nanodrop18AUG <- read_csv("nanodrop_18AUG23.csv")
```

I followed the kit instructions, and assessed the samples for purity using a Nanodrop.

![Nanodrop plot results](nanodrop_plot_03AUG23.jpg)

:   *RNA Nanodrop absorbance plot after passing Zymo DNA/RNA Quick Miniprep Plus extracted RNA samples through the Zymo OneStep PCR Inhibitor Kit. Run 03AUG2023.*

```{r}
onestep <- nanodrop18AUG %>% 
  filter(kit == 'OneStepPCRInhibitor') %>% 
  arrange(`sample-ID`)

formattable(onestep,
              align = "l",
              list(
                '260-230'= salts,
                '260-280' = proteins)
                )
```

::: {.callout-warning appearance="simple"}
Even after running samples through the Zymo OneStep PCR Inhibitor Removal Kit, The 260/230 ratios are all below the target of 2.0, indicating contamination by salts. Any 260/280 protein ratios OVER 2.0 may indicate contamination by phenols, which absorb between 280-300nm wavelengths. So... be suspicious when you see at 260/280 ratio higher than 2.0!
:::

The Zymo rep responded with the following advice:

> "The guanidine salts in our DNA/RNA Shield and DNA/RNA Lysis Buffer could be the reason of increased A230. We see this happening most often when small impurities (e.g., cell debris, protein aggregation) were transferred from sample processing and got stuck on the column. They would absorb buffers and release them during elution. Ways to avoid this include increasing the amount of DNA/RNA Shield used during processing, reducing sample input, and transferring only a fraction of sample (e.g.,500 µl from 800 µl) from the BashingBead tube to avoid debris. You can also further purify your existing eluted samples with either the RNA Clean & Concentrator that is on the way to you, or by following the"Liquids/Reaction Clean-up" procedure included in the Appendices section Quick-DNA/RNA Miniprep Plus's protocol."

------------------------------------------------------------------------

## Zymo RNA Clean & Concentrator Kit

I next tried to purify the RNA with the Zymo RNA Clean & Concentrator Kit:

![Nanodrop plot results](18aug23_drop_plot.jpg)

:   *RNA Nanodrop absorbance plot after passing Zymo DNA/RNA Quick Miniprep Plus extracted RNA samples through the Zymo RNA Clean & Concentrator Kit. Run 18AUG2023.*

```{r}
rnaCC <- nanodrop18AUG %>% 
          filter(kit == 'RNAClean&Concentrate')

formattable(rnaCC, 
              align = "l", # aligns text to the left
              list(
                '260-230'= salts,
                '260-280' = proteins)
                )
```

::: column-margin
P.S. I made these pretty tables based off [this article by Little Miss Data](https://www.littlemissdata.com/blog/prettytables)
:::

::: {.callout-warning appearance="simple"}
The 260/280 ratios over 2.0 may indicate worse phenol contamination!
:::

### Did the RNA Clean & Concentrate Kit improve the samples that had already been through the OneStep PCR Inhibitor Removal Kit?

```{r}
#| column: margin
#| fig-cap: "Nanodrop Results from OneStep PCR Inhibitor Removal"
#| cap-location: margin
formattable(onestep,
              align = "l",
              list(
                '260-230'= salts,
                '260-280' = proteins)
                )
```

```{r}
#| fig-cap: Nanodrop Results from RNA Clean & Concentrate
onesamples <- onestep$`sample-ID`

compare <- nanodrop18AUG %>% 
  filter(kit == 'RNAClean&Concentrate') %>% 
  filter(`sample-ID` %in% onesamples) %>% 
  arrange(`sample-ID`)

formattable(compare, 
              align = "l", # aligns text to the left
              list(
                '260-230'= salts,
                '260-280' = proteins)
                )
```

::: {.callout-tip appearance="simple"}
The RNA Clean & Concentrator improved the 260/230 ratios for samples 1Ea, 1CH2a, 1CA2a. 1PA2a & 1PH1a which had already been passed through the Zymo OneStep PCR Inhibitor Removal Kit.
:::

::: {.callout-tip appearance="simple"}
The 260/280 ratios are suspiciously high, making me wonder about phenol contamination... However, the Zymo Kit is phenol-free, so the only source of phenol would be from the samples themselves. We'll find out for sure when we send in the purified samples for another attempt at TagSeq Library Prep by UT GSAF.
:::

------------------------------------------------------------------------

## RNA Clean & Concentrator follow-up tests

#### 24-AUG-2023

**Samples:**

-   10-Aa
-   6-Aa

```{r}
nanodrop24AUG <- read_csv("nanodrop_24AUG23.csv")

formattable(nanodrop24AUG, 
              align = "l", # aligns text to the left
              list(
                '260-230'= salts,
                '260-280' = proteins)
                )
```

#### 31-AUG-2023

::: {.callout-warning appearance="simple"}
*In this extraction, I accidentally eluted the RNA into the collection tube instead of a nuclease free tube in the RNA Clean & Concentrator elution step. This might have contaminated the eluted RNA, so I took all 10 samples and ran them through another round of RNA Clean & Concentrator prior to quantification on Nanodrop & Qubit.*
:::

**Samples:**

-   mock
-   blank
-   4-LA2a
-   1-HH2a
-   2-HH2a
-   3-HA2a
-   3-MA2a
-   1-HA2a
-   4-MA2a
-   3-HH2a

```{r}
nanodrop31AUG <- read_csv("31aug23_drop_table.csv")

formattable(nanodrop31AUG, 
              align = "l", # aligns text to the left
              list(
                '260/230'= salts,
                '260/280' = proteins)
                )

```

![RNA Nanodrop absorbance plots after conducting modified extraction protocol and passing samples through two rounds of RNA Clean & Concentrator](31aug23_drop_plots.png){fig-alt="Plots of RNA nucleic acid absorbance spectra for 10 samples"}

::: {.callout-important appearance="simple"}
The absorbance spectra show that samples **4-LA2a**, **3-MA2a** & **3-HA2a** have salts contamination and **3-HA2a** shows protein contamination. In the table they have 260/280 and 260/230 values below the desired targets. The other 5 RNA samples seem to pass the contamination check... as far as we can tell....
:::

#### 06-SEP-2023

These 12 samples were passed through the Zymo RNA Clean & Concentrator kit a 2nd (or 3rd!) time to see if doing the kit another round made further improvements to purity

Samples:

-   1-CA2a
-   1-CH2a
-   1-Ea
-   1-PH1a
-   2-CH1b
-   3-CH2a
-   3-MA2a
-   3-HA2a
-   4-CH1a
-   4-MA2a
-   4-PH1b
-   4-LA2a

I wasn't stoked on the results....

![Nanodrop of RNA from 06SEP2023](06sep23_drop_plot.png)

```{r}

# Import Nanodrop CSV
nanodrop06sepRNA <- read_csv("06sep23_drop_table.csv")

# Build qubit (qbit for short) data.frame 

## qbit sampleID
sampleID <- c('3HA2a', '2CH1b', '3MA2a', '1CA2a', 
              '1Ea', '3CH2a', '4CH1a', '1CH2a', 
              '1PH1a', '4LA2a', '4MA2a', '4PH1b')
## qbit 1st read              
qbit1 <- c(25.8, 0, 30.0, 22.6, 
           12.0, 16.8, 10.8, 11.6,
           13.8, 31.0, 13.0, 10.0)

## qbit 2nd read           
qbit2 <- c(25.0, 0, 29.6, 21.8, 
           11.0, 16.6, 10.4, 11.0,
           12.6, 30.2, 12.4, 13.2)

## make a qbit data.frame
qbit06sepRNA <- data_frame("Sample ID" = sampleID, qbit1 = qbit1, qbit2 = qbit2)
## make an average column from first and second qbit reads
qbit06sepRNA <- qbit06sepRNA %>% 
  mutate(qbitAVG = (qbit1 + qbit2)/2)

## combine dataframes
qaqc06sepRNA <- full_join(qbit06sepRNA, nanodrop06sepRNA, by = "Sample ID")
```

```{r}
# Table of QAQC results
formattable(qaqc06sepRNA, 
              align = "l", # aligns text to the left
              list(
                '260/230'= salts,
                '260/280' = proteins,
                'Nucleic Acid' = ngul,
                'qbitAVG'= ngul
                ))

```

::: column-margin
I want to pick out the samples that are the best purity to send to UT Austin GSAF to see if they will pass library prep.

The purified RNA samples that are 'green across the board' on both concentration and purity and are 'good to go' are:

-   3-HA2a (50.4ng/uL nanodrop \| 25.4 qubit)

-   1-CA2a (49.3ng/uL nanodrop \| 22.2 qubit)

The ones that are borderline.....

-   1-PH1a (27.4ng/uL nanodrop \| 13.2 qubit?)

-   3-CH2a (21.5ng/uL nanodrop \| 16.7 qubit?)

-   4-MA2a (22.1ng/uL nanodrop \| 12.7 qubit?)

-   4-LA2a ( 260/230: 1.42? \| 30.6 qubit)

-   10-Aa?

-   6Aa?
:::

::: {.callout-warning appearance="simple"}
Some of the sample purity was made WORSE by running them through RNA Clean & Concentrator a second or third time....

3MA2a on 31AUG2023:

260/280: 1.85

260/230: 1.63

vs.

3MA2a on 06SEP2023:

260/280: 1.72

260/230: 0.94

However other samples were made slightly better...

3HA2a on 31AUG2023

260/280: 1.85

260/230: 2.08

vs.

3HA2a on 06SEP2023

260/280: 2.04

260/230: 2.08
:::

Really still frustrated that I'm not getting consistent results... Does this kit work to purify the samples? Not in large batches.... few of the samples meet the purity targets. I'm also worried that the more I work with the RNA the more it degrades and I'm losing quantity (ng/uL) and possibly the quality (time will tell when the RIN scores come back from UT GSAF)
