<!DOCTYPE html>
<html xmlns="http://www.w3.org/1999/xhtml" lang="en" xml:lang="en"><head>

<meta charset="utf-8">
<meta name="generator" content="quarto-1.7.31">

<meta name="viewport" content="width=device-width, initial-scale=1.0, user-scalable=yes">

<meta name="author" content="Sarah Tanja">
<meta name="dcterms.date" content="2024-05-23">

<title>Extracting &amp; measuring chlorophyll a &amp; c2 concentrations – Sarah Tanja’s Open Lab Notebook</title>
<style>
code{white-space: pre-wrap;}
span.smallcaps{font-variant: small-caps;}
div.columns{display: flex; gap: min(4vw, 1.5em);}
div.column{flex: auto; overflow-x: auto;}
div.hanging-indent{margin-left: 1.5em; text-indent: -1.5em;}
ul.task-list{list-style: none;}
ul.task-list li input[type="checkbox"] {
  width: 0.8em;
  margin: 0 0.8em 0.2em -1em; /* quarto-specific, see https://github.com/quarto-dev/quarto-cli/issues/4556 */ 
  vertical-align: middle;
}
/* CSS for syntax highlighting */
html { -webkit-text-size-adjust: 100%; }
pre > code.sourceCode { white-space: pre; position: relative; }
pre > code.sourceCode > span { display: inline-block; line-height: 1.25; }
pre > code.sourceCode > span:empty { height: 1.2em; }
.sourceCode { overflow: visible; }
code.sourceCode > span { color: inherit; text-decoration: inherit; }
div.sourceCode { margin: 1em 0; }
pre.sourceCode { margin: 0; }
@media screen {
div.sourceCode { overflow: auto; }
}
@media print {
pre > code.sourceCode { white-space: pre-wrap; }
pre > code.sourceCode > span { text-indent: -5em; padding-left: 5em; }
}
pre.numberSource code
  { counter-reset: source-line 0; }
pre.numberSource code > span
  { position: relative; left: -4em; counter-increment: source-line; }
pre.numberSource code > span > a:first-child::before
  { content: counter(source-line);
    position: relative; left: -1em; text-align: right; vertical-align: baseline;
    border: none; display: inline-block;
    -webkit-touch-callout: none; -webkit-user-select: none;
    -khtml-user-select: none; -moz-user-select: none;
    -ms-user-select: none; user-select: none;
    padding: 0 4px; width: 4em;
  }
pre.numberSource { margin-left: 3em;  padding-left: 4px; }
div.sourceCode
  {   }
@media screen {
pre > code.sourceCode > span > a:first-child::before { text-decoration: underline; }
}
/* CSS for citations */
div.csl-bib-body { }
div.csl-entry {
  clear: both;
  margin-bottom: 0em;
}
.hanging-indent div.csl-entry {
  margin-left:2em;
  text-indent:-2em;
}
div.csl-left-margin {
  min-width:2em;
  float:left;
}
div.csl-right-inline {
  margin-left:2em;
  padding-left:1em;
}
div.csl-indent {
  margin-left: 2em;
}</style>


<script src="../../../../site_libs/quarto-nav/quarto-nav.js"></script>
<script src="../../../../site_libs/quarto-nav/headroom.min.js"></script>
<script src="../../../../site_libs/clipboard/clipboard.min.js"></script>
<script src="../../../../site_libs/quarto-search/autocomplete.umd.js"></script>
<script src="../../../../site_libs/quarto-search/fuse.min.js"></script>
<script src="../../../../site_libs/quarto-search/quarto-search.js"></script>
<meta name="quarto:offset" content="../../../../">
<link href="../../../../images/favicon.png" rel="icon" type="image/png">
<script src="../../../../site_libs/quarto-html/quarto.js" type="module"></script>
<script src="../../../../site_libs/quarto-html/tabsets/tabsets.js" type="module"></script>
<script src="../../../../site_libs/quarto-html/popper.min.js"></script>
<script src="../../../../site_libs/quarto-html/tippy.umd.min.js"></script>
<script src="../../../../site_libs/quarto-html/anchor.min.js"></script>
<link href="../../../../site_libs/quarto-html/tippy.css" rel="stylesheet">
<link href="../../../../site_libs/quarto-html/tippy.css" rel="stylesheet" class="quarto-color-scheme-extra">
<script src="../../../../site_libs/bootstrap/bootstrap.min.js"></script>
<link href="../../../../site_libs/bootstrap/bootstrap-icons.css" rel="stylesheet">
<link href="../../../../site_libs/bootstrap/bootstrap-869eb0ff6db83d51c3d8117e186cf13a.min.css" rel="stylesheet" append-hash="true" class="quarto-color-scheme" id="quarto-bootstrap" data-mode="light">
<link href="../../../../site_libs/bootstrap/bootstrap-dark-e2b38b794831ff325e66a6a88120557b.min.css" rel="stylesheet" append-hash="true" class="quarto-color-scheme quarto-color-alternate" id="quarto-bootstrap" data-mode="dark">
<link href="../../../../site_libs/bootstrap/bootstrap-869eb0ff6db83d51c3d8117e186cf13a.min.css" rel="stylesheet" append-hash="true" class="quarto-color-scheme-extra" id="quarto-bootstrap" data-mode="light">
<script id="quarto-search-options" type="application/json">{
  "location": "navbar",
  "copy-button": false,
  "collapse-after": 3,
  "panel-placement": "end",
  "type": "overlay",
  "limit": 50,
  "keyboard-shortcut": [
    "f",
    "/",
    "s"
  ],
  "show-item-context": false,
  "language": {
    "search-no-results-text": "No results",
    "search-matching-documents-text": "matching documents",
    "search-copy-link-title": "Copy link to search",
    "search-hide-matches-text": "Hide additional matches",
    "search-more-match-text": "more match in this document",
    "search-more-matches-text": "more matches in this document",
    "search-clear-button-title": "Clear",
    "search-text-placeholder": "",
    "search-detached-cancel-button-title": "Cancel",
    "search-submit-button-title": "Submit",
    "search-label": "Search"
  }
}</script>
<style>html{ scroll-behavior: smooth; }</style>

  <script src="https://cdnjs.cloudflare.com/polyfill/v3/polyfill.min.js?features=es6"></script>
  <script src="https://cdn.jsdelivr.net/npm/mathjax@3/es5/tex-chtml-full.js" type="text/javascript"></script>

<script type="text/javascript">
const typesetMath = (el) => {
  if (window.MathJax) {
    // MathJax Typeset
    window.MathJax.typeset([el]);
  } else if (window.katex) {
    // KaTeX Render
    var mathElements = el.getElementsByClassName("math");
    var macros = [];
    for (var i = 0; i < mathElements.length; i++) {
      var texText = mathElements[i].firstChild;
      if (mathElements[i].tagName == "SPAN") {
        window.katex.render(texText.data, mathElements[i], {
          displayMode: mathElements[i].classList.contains('display'),
          throwOnError: false,
          macros: macros,
          fleqn: false
        });
      }
    }
  }
}
window.Quarto = {
  typesetMath
};
</script>

<meta property="og:title" content="Extracting &amp; measuring chlorophyll a &amp; c2 concentrations – Sarah Tanja’s Open Lab Notebook">
<meta property="og:description" content="… from an anemone-hosted symbiotic dinoflagellate using a ThermoFischer Nanodrop 2000c">
<meta property="og:image" content="https://sarahtanja.github.io/quarto-blog/posts/projects/anemone/chlorophyll/steinberg_fig4.png">
<meta property="og:site_name" content="Sarah Tanja's Open Lab Notebook">
<meta property="og:image:height" content="1253">
<meta property="og:image:width" content="879">
</head>

<body class="floating nav-fixed slimcontent quarto-light"><script id="quarto-html-before-body" type="application/javascript">
    const toggleBodyColorMode = (bsSheetEl) => {
      const mode = bsSheetEl.getAttribute("data-mode");
      const bodyEl = window.document.querySelector("body");
      if (mode === "dark") {
        bodyEl.classList.add("quarto-dark");
        bodyEl.classList.remove("quarto-light");
      } else {
        bodyEl.classList.add("quarto-light");
        bodyEl.classList.remove("quarto-dark");
      }
    }
    const toggleBodyColorPrimary = () => {
      const bsSheetEl = window.document.querySelector("link#quarto-bootstrap:not([rel=disabled-stylesheet])");
      if (bsSheetEl) {
        toggleBodyColorMode(bsSheetEl);
      }
    }
    const setColorSchemeToggle = (alternate) => {
      const toggles = window.document.querySelectorAll('.quarto-color-scheme-toggle');
      for (let i=0; i < toggles.length; i++) {
        const toggle = toggles[i];
        if (toggle) {
          if (alternate) {
            toggle.classList.add("alternate");
          } else {
            toggle.classList.remove("alternate");
          }
        }
      }
    };
    const toggleColorMode = (alternate) => {
      // Switch the stylesheets
      const primaryStylesheets = window.document.querySelectorAll('link.quarto-color-scheme:not(.quarto-color-alternate)');
      const alternateStylesheets = window.document.querySelectorAll('link.quarto-color-scheme.quarto-color-alternate');
      manageTransitions('#quarto-margin-sidebar .nav-link', false);
      if (alternate) {
        // note: dark is layered on light, we don't disable primary!
        enableStylesheet(alternateStylesheets);
        for (const sheetNode of alternateStylesheets) {
          if (sheetNode.id === "quarto-bootstrap") {
            toggleBodyColorMode(sheetNode);
          }
        }
      } else {
        disableStylesheet(alternateStylesheets);
        enableStylesheet(primaryStylesheets)
        toggleBodyColorPrimary();
      }
      manageTransitions('#quarto-margin-sidebar .nav-link', true);
      // Switch the toggles
      setColorSchemeToggle(alternate)
      // Hack to workaround the fact that safari doesn't
      // properly recolor the scrollbar when toggling (#1455)
      if (navigator.userAgent.indexOf('Safari') > 0 && navigator.userAgent.indexOf('Chrome') == -1) {
        manageTransitions("body", false);
        window.scrollTo(0, 1);
        setTimeout(() => {
          window.scrollTo(0, 0);
          manageTransitions("body", true);
        }, 40);
      }
    }
    const disableStylesheet = (stylesheets) => {
      for (let i=0; i < stylesheets.length; i++) {
        const stylesheet = stylesheets[i];
        stylesheet.rel = 'disabled-stylesheet';
      }
    }
    const enableStylesheet = (stylesheets) => {
      for (let i=0; i < stylesheets.length; i++) {
        const stylesheet = stylesheets[i];
        if(stylesheet.rel !== 'stylesheet') { // for Chrome, which will still FOUC without this check
          stylesheet.rel = 'stylesheet';
        }
      }
    }
    const manageTransitions = (selector, allowTransitions) => {
      const els = window.document.querySelectorAll(selector);
      for (let i=0; i < els.length; i++) {
        const el = els[i];
        if (allowTransitions) {
          el.classList.remove('notransition');
        } else {
          el.classList.add('notransition');
        }
      }
    }
    const isFileUrl = () => {
      return window.location.protocol === 'file:';
    }
    const hasAlternateSentinel = () => {
      let styleSentinel = getColorSchemeSentinel();
      if (styleSentinel !== null) {
        return styleSentinel === "alternate";
      } else {
        return false;
      }
    }
    const setStyleSentinel = (alternate) => {
      const value = alternate ? "alternate" : "default";
      if (!isFileUrl()) {
        window.localStorage.setItem("quarto-color-scheme", value);
      } else {
        localAlternateSentinel = value;
      }
    }
    const getColorSchemeSentinel = () => {
      if (!isFileUrl()) {
        const storageValue = window.localStorage.getItem("quarto-color-scheme");
        return storageValue != null ? storageValue : localAlternateSentinel;
      } else {
        return localAlternateSentinel;
      }
    }
    const toggleGiscusIfUsed = (isAlternate, darkModeDefault) => {
      const baseTheme = document.querySelector('#giscus-base-theme')?.value ?? 'light';
      const alternateTheme = document.querySelector('#giscus-alt-theme')?.value ?? 'dark';
      let newTheme = '';
      if(authorPrefersDark) {
        newTheme = isAlternate ? baseTheme : alternateTheme;
      } else {
        newTheme = isAlternate ? alternateTheme : baseTheme;
      }
      const changeGiscusTheme = () => {
        // From: https://github.com/giscus/giscus/issues/336
        const sendMessage = (message) => {
          const iframe = document.querySelector('iframe.giscus-frame');
          if (!iframe) return;
          iframe.contentWindow.postMessage({ giscus: message }, 'https://giscus.app');
        }
        sendMessage({
          setConfig: {
            theme: newTheme
          }
        });
      }
      const isGiscussLoaded = window.document.querySelector('iframe.giscus-frame') !== null;
      if (isGiscussLoaded) {
        changeGiscusTheme();
      }
    };
    const authorPrefersDark = false;
    const queryPrefersDark = window.matchMedia('(prefers-color-scheme: dark)');
    const darkModeDefault = queryPrefersDark.matches;
      document.querySelector('link#quarto-text-highlighting-styles.quarto-color-scheme-extra').rel = 'disabled-stylesheet';
      document.querySelector('link#quarto-bootstrap.quarto-color-scheme-extra').rel = 'disabled-stylesheet';
    let localAlternateSentinel = darkModeDefault ? 'alternate' : 'default';
    // Dark / light mode switch
    window.quartoToggleColorScheme = () => {
      // Read the current dark / light value
      let toAlternate = !hasAlternateSentinel();
      toggleColorMode(toAlternate);
      setStyleSentinel(toAlternate);
      toggleGiscusIfUsed(toAlternate, darkModeDefault);
      window.dispatchEvent(new Event('resize'));
    };
    queryPrefersDark.addEventListener("change", e => {
      if(window.localStorage.getItem("quarto-color-scheme") !== null)
        return;
      const alternate = e.matches
      toggleColorMode(alternate);
      localAlternateSentinel = e.matches ? 'alternate' : 'default'; // this is used alongside local storage!
      toggleGiscusIfUsed(alternate, darkModeDefault);
    });
    // Switch to dark mode if need be
    if (hasAlternateSentinel()) {
      toggleColorMode(true);
    } else {
      toggleColorMode(false);
    }
  </script>

<div id="quarto-search-results"></div>
  <header id="quarto-header" class="headroom fixed-top quarto-banner">
    <nav class="navbar navbar-expand-lg " data-bs-theme="dark">
      <div class="navbar-container container-fluid">
      <div class="navbar-brand-container mx-auto">
    <a class="navbar-brand" href="../../../../index.html">
    <span class="navbar-title">Sarah Tanja’s Open Lab Notebook</span>
    </a>
  </div>
            <div id="quarto-search" class="" title="Search"></div>
          <button class="navbar-toggler" type="button" data-bs-toggle="collapse" data-bs-target="#navbarCollapse" aria-controls="navbarCollapse" role="menu" aria-expanded="false" aria-label="Toggle navigation" onclick="if (window.quartoToggleHeadroom) { window.quartoToggleHeadroom(); }">
  <span class="navbar-toggler-icon"></span>
</button>
          <div class="collapse navbar-collapse" id="navbarCollapse">
            <ul class="navbar-nav navbar-nav-scroll ms-auto">
  <li class="nav-item">
    <a class="nav-link" href="../../../../projects.html"> 
<span class="menu-text">Projects</span></a>
  </li>  
  <li class="nav-item">
    <a class="nav-link" href="../../../../posts.html"> 
<span class="menu-text">Posts</span></a>
  </li>  
  <li class="nav-item">
    <a class="nav-link" href="../../../../about.html"> 
<span class="menu-text">About</span></a>
  </li>  
  <li class="nav-item compact">
    <a class="nav-link" href="https://www.linkedin.com/in/sarah-tanja-595722146/"> <i class="bi bi-linkedin" role="img">
</i> 
<span class="menu-text"></span></a>
  </li>  
  <li class="nav-item dropdown ">
    <a class="nav-link dropdown-toggle" href="#" id="nav-menu-bi-github" role="link" data-bs-toggle="dropdown" aria-expanded="false">
      <i class="bi bi-github" role="img">
</i> 
 <span class="menu-text"></span>
    </a>
    <ul class="dropdown-menu dropdown-menu-end" aria-labelledby="nav-menu-bi-github">    
        <li>
    <a class="dropdown-item" href="https://github.com/sarahtanja">
 <span class="dropdown-text">Sarah’s GitHub</span></a>
  </li>  
        <li>
    <a class="dropdown-item" href="https://github.com/sarahtanja/coral-embryo-leachate">
 <span class="dropdown-text">Coral Embryology Project Repository</span></a>
  </li>  
        <li>
    <a class="dropdown-item" href="https://github.com/sarahtanja/coral-leachate-heat">
 <span class="dropdown-text">Coral Photophysiology Project Repository</span></a>
  </li>  
        <li>
    <a class="dropdown-item" href="https://github.com/sarahtanja/anemone-leachate-heat">
 <span class="dropdown-text">Anemone Photophysiology Project Repository</span></a>
  </li>  
        <li>
    <a class="dropdown-item" href="https://github.com/sarahtanja/coral-pae-temp">
 <span class="dropdown-text">Coral Microbiome Project Repository</span></a>
  </li>  
    </ul>
  </li>
  <li class="nav-item dropdown ">
    <a class="nav-link dropdown-toggle" href="#" id="nav-menu-bi-envelope-paper" role="link" data-bs-toggle="dropdown" aria-expanded="false">
      <i class="bi bi-envelope-paper" role="img">
</i> 
 <span class="menu-text"></span>
    </a>
    <ul class="dropdown-menu dropdown-menu-end" aria-labelledby="nav-menu-bi-envelope-paper">    
        <li class="dropdown-header">stanja@uw.edu</li>
        <li>
    <a class="dropdown-item" href="../../../../contact.html">
 <span class="dropdown-text">Contact</span></a>
  </li>  
    </ul>
  </li>
</ul>
          </div> <!-- /navcollapse -->
            <div class="quarto-navbar-tools">
  <a href="" class="quarto-color-scheme-toggle quarto-navigation-tool  px-1" onclick="window.quartoToggleColorScheme(); return false;" title="Toggle dark mode"><i class="bi"></i></a>
</div>
      </div> <!-- /container-fluid -->
    </nav>
</header>
<!-- content -->
<header id="title-block-header" class="quarto-title-block default toc-left page-columns page-full">
  <div class="quarto-title-banner page-columns page-full">
    <div class="quarto-title column-body">
      <div class="quarto-title-block"><div><h1 class="title">Extracting &amp; measuring chlorophyll a &amp; c2 concentrations</h1><button type="button" class="btn code-tools-button" id="quarto-code-tools-source"><i class="bi"></i> Code</button></div></div>
            <p class="subtitle lead">… from an anemone-hosted symbiotic dinoflagellate using a ThermoFischer Nanodrop 2000c</p>
                                <div class="quarto-categories">
                <div class="quarto-category">protocols</div>
                <div class="quarto-category">anemones</div>
              </div>
                  </div>
  </div>
    
  
  <div class="quarto-title-meta">

      <div>
      <div class="quarto-title-meta-heading">Author</div>
      <div class="quarto-title-meta-contents">
               <p>Sarah Tanja </p>
            </div>
    </div>
      
      <div>
      <div class="quarto-title-meta-heading">Published</div>
      <div class="quarto-title-meta-contents">
        <p class="date">May 23, 2024</p>
      </div>
    </div>
    
      <div>
      <div class="quarto-title-meta-heading">Modified</div>
      <div class="quarto-title-meta-contents">
        <p class="date-modified">August 19, 2025</p>
      </div>
    </div>
      
    </div>
    
  
  </header><div id="quarto-content" class="quarto-container page-columns page-rows-contents page-layout-article page-navbar">
<!-- sidebar -->
  <nav id="quarto-sidebar" class="sidebar collapse collapse-horizontal quarto-sidebar-collapse-item sidebar-navigation floating overflow-auto">
    <nav id="TOC" role="doc-toc" class="toc-active">
    <h2 id="toc-title">Contents <i class="bi bi-bookmark-heart"></i></h2>
   
  <ul>
  <li><a href="#overview" id="toc-overview" class="nav-link active" data-scroll-target="#overview">Overview</a></li>
  <li><a href="#materials-list" id="toc-materials-list" class="nav-link" data-scroll-target="#materials-list">Materials list</a>
  <ul>
  <li><a href="#lab-equipment" id="toc-lab-equipment" class="nav-link" data-scroll-target="#lab-equipment">Lab equipment</a></li>
  <li><a href="#lab-consumables" id="toc-lab-consumables" class="nav-link" data-scroll-target="#lab-consumables">Lab consumables</a></li>
  <li><a href="#chemicals-solvents" id="toc-chemicals-solvents" class="nav-link" data-scroll-target="#chemicals-solvents">Chemicals &amp; solvents</a></li>
  </ul></li>
  <li><a href="#objectives" id="toc-objectives" class="nav-link" data-scroll-target="#objectives">Objectives</a></li>
  <li><a href="#procedure" id="toc-procedure" class="nav-link" data-scroll-target="#procedure">Procedure</a>
  <ul>
  <li><a href="#separate-symbiont-protein-fractions" id="toc-separate-symbiont-protein-fractions" class="nav-link" data-scroll-target="#separate-symbiont-protein-fractions">Separate symbiont &amp; protein fractions</a></li>
  <li><a href="#protein-quantification" id="toc-protein-quantification" class="nav-link" data-scroll-target="#protein-quantification">Protein quantification</a></li>
  <li><a href="#symbiont-counts" id="toc-symbiont-counts" class="nav-link" data-scroll-target="#symbiont-counts">Symbiont counts</a></li>
  <li><a href="#chlorophyll-extraction" id="toc-chlorophyll-extraction" class="nav-link" data-scroll-target="#chlorophyll-extraction">Chlorophyll extraction</a></li>
  <li><a href="#measure-absorbance-using-a-nanodrop-2000c-spectrophotometer" id="toc-measure-absorbance-using-a-nanodrop-2000c-spectrophotometer" class="nav-link" data-scroll-target="#measure-absorbance-using-a-nanodrop-2000c-spectrophotometer">Measure absorbance using a NanoDrop 2000c Spectrophotometer</a>
  <ul>
  <li><a href="#program-nanodrop-2000c-method-editor" id="toc-program-nanodrop-2000c-method-editor" class="nav-link" data-scroll-target="#program-nanodrop-2000c-method-editor">Program NanoDrop 2000c Method Editor</a></li>
  <li><a href="#measure-cuvette-blanks-and-samples" id="toc-measure-cuvette-blanks-and-samples" class="nav-link" data-scroll-target="#measure-cuvette-blanks-and-samples">Measure cuvette blanks and samples</a></li>
  </ul></li>
  </ul></li>
  <li><a href="#calculate-concentrations" id="toc-calculate-concentrations" class="nav-link" data-scroll-target="#calculate-concentrations">Calculate concentrations</a></li>
  </ul>
</nav>
</nav>
<div id="quarto-sidebar-glass" class="quarto-sidebar-collapse-item" data-bs-toggle="collapse" data-bs-target=".quarto-sidebar-collapse-item"></div>
<!-- margin-sidebar -->
    <div id="quarto-margin-sidebar" class="sidebar margin-sidebar zindex-bottom">
    </div>
<!-- main -->
<main class="content quarto-banner-title-block page-columns page-full" id="quarto-document-content">






<section id="overview" class="level1 page-columns page-full">
<h1>Overview</h1>
<div class="callout callout-style-default callout-important callout-titled">
<div class="callout-header d-flex align-content-center">
<div class="callout-icon-container">
<i class="callout-icon"></i>
</div>
<div class="callout-title-container flex-fill">
Important
</div>
</div>
<div class="callout-body-container callout-body">
<p>This protocol draws <em>heavily</em> (and almost exclusively!) from the awesome method paper, <em>“Its What’s on the Inside That Counts: An Effective, Efficient, and Streamlined Method for Quantification of Octocoral Symbiodiniaceae and Chlorophyll”</em> published by Rosemary Kate Steinberg, Emma L. Johnston, Teresa Bednarek, Katherine A. Dafforn and Tracy D. Ainsworth. <span class="citation" data-cites="steinberg2021">(<a href="#ref-steinberg2021" role="doc-biblioref">Steinberg et al. 2021</a>)</span></p>
<p>THANK YOU to all the above listed authors for this amazingly clear methods paper. It is pure gold, and I wish more science was <em>actually reproducible</em> like this is.</p>
</div>
</div>
<div class="no-row-height column-margin column-container"></div><div class="quarto-figure quarto-figure-center">
<figure class="figure">
<p><img src="steinberg_fig4.png" class="img-fluid figure-img"></p>
<figcaption>This is from the Steinberg et a. 2021 paper, checkout more cool sci comm art from Melissa Pappas at <a href="https://www.emergingcreativesofscience.com/" class="uri">https://www.emergingcreativesofscience.com/</a></figcaption>
</figure>
</div>
<p>Here I detail out the method I aim to use for extracting and measuring chlorophyll a and c2 of the <em>Anthopleura elegantissima</em> brown algal symbiont, <em>Breviolum muscatinei</em>, relative to cnidarian host wet weight and calculated protein content.</p>
<p>What is chlorophyll a and c2?</p>
<ul>
<li>Chlorophyll
<ul>
<li><p>a</p></li>
<li><p>c2</p></li>
</ul></li>
</ul>
<p>What about other pigments?</p>
<ul>
<li>Xanthophylls : photoprotective xanthophyll cycle
<ul>
<li><p>peridinin</p></li>
<li><p>fucoxanthin</p></li>
<li><p>diadinoxanthin (Dn)</p></li>
<li><p>diatoxanthin (Dt)</p></li>
<li><p>dinoxanthin &amp; alteration product diadinochrome</p></li>
<li><p>phaeophytin a</p></li>
</ul></li>
<li>Carotenoids
<ul>
<li><p>peridinin? Ambarsari et al 1997</p></li>
<li><p>Dn? Ambarsari et al 1997</p></li>
<li><p>B carotene</p></li>
</ul></li>
</ul>
<div class="quarto-figure quarto-figure-center">
<figure class="figure">
<p><img src="venn_table1.png" class="img-fluid figure-img"></p>
<figcaption>A. A. Venn et al., 2006, Table 1</figcaption>
</figure>
</div>
<blockquote class="blockquote">
<p>“Ambarsari et al.&nbsp;(1997) and Brown et al.&nbsp;(1999) obtained evidence of a functional xanthophyll cycle in Symbiodinium, thought to operate as part of non-photochemical quenching (NPQ), also observed in other marine algae (Demers et al.&nbsp;1991), and analogous to the photoprotective xanthophylls cycle in terrestrial plants.” (Venn et al., 2006, p.&nbsp;2134)</p>
<p>“Xanthophyll cycling is likely to be one of several mechanisms that underpin the environmental tolerance of a coral–algal symbiosis. Symbiotic dinoflagellates, unlike free-living marine algae, exist in coral tissue equipped with additional photoprotective and thermoprotective mechanisms, for example, fluorescent proteins (Dove 2004) and heat shock proteins (Black, Voellmy &amp; Szmant 1995), which may complement or even surpass photoprotective mechanisms linked to the algal pigment profile and thus help maintain the integrity of the symbiosis (Brown et al.&nbsp;2002a).” (Venn et al., 2006, p.&nbsp;2140</p>
</blockquote>
<p>Why is chlorophyll content an important metric if we’ve already used the pulse-amplitude modulated (PAM) fluorometer to measure photosynthetic efficiency, quantum yield, etc.?</p>
</section>
<section id="materials-list" class="level1 page-columns page-full">
<h1>Materials list</h1>
<section id="lab-equipment" class="level3 page-columns page-full">
<h3 class="anchored" data-anchor-id="lab-equipment">Lab equipment</h3>
<ul>
<li>tissue homogenizer (like the <a href="https://us.omni-inc.com/omni-tissue-homogenizer-th.html">OMNI TH</a> used by <span class="citation" data-cites="steinberg2021">(<a href="#ref-steinberg2021" role="doc-biblioref">Steinberg et al. 2021</a>)</span> with a 10mm stainless steel bit, we have a Biospec Products Tissue Tearor Model 985-370, which is a slightly older model compared to <a href="https://www.coleparmer.com/i/tissue-tearor-985370-07-homogenizer-120-vac-1-2-a/0475050?PubID=UX&amp;persist=true&amp;ip=no&amp;gad_source=1&amp;gclid=CjwKCAjwx-CyBhAqEiwAeOcTdU_POATJBTiINJzb41uUa_jdZx9EuYzIDbyWBoKMXQRaQdXV3SPlgBoCGr4QAvD_BwE">this here</a>)</li>
<li>vortex</li>
<li>centrifuge</li>
<li>NanoDrop 2000c Spectrophotometer</li>
<li>2 quartz-glass Nanodrop Spec cuvettes</li>
<li><a href="https://www.emsdiasum.com/docs/technical/datasheet/68052-14">Neubauer Improved Heamocytometer</a></li>
<li>Microscope or Dissecting scope</li>
</ul>
<div class="no-row-height column-margin column-container"></div></section>
<section id="lab-consumables" class="level3">
<h3 class="anchored" data-anchor-id="lab-consumables">Lab consumables</h3>
<ul>
<li>microcentrifuge tubes</li>
<li>pipettes &amp; tips</li>
<li>nitrile gloves</li>
<li><a href="https://www.simbalion.com/product/detail/A/A10/A10/No.800">ethanol-proof lab markers</a></li>
</ul>
</section>
<section id="chemicals-solvents" class="level3">
<h3 class="anchored" data-anchor-id="chemicals-solvents">Chemicals &amp; solvents</h3>
<ul>
<li>Milli-Q Water</li>
<li>10x <a href="https://www.sigmaaldrich.com/US/en/product/sigma/p7059?icid=sharepdp-clipboard-copy-productdetailpage">Phosphate-Buffered Saline (PBS)</a> (room temp storage)</li>
<li>100% Acetone</li>
</ul>
</section>
</section>
<section id="objectives" class="level1 page-columns page-full">
<h1>Objectives</h1>
<p>From these samples we already have<a href="#fn1" class="footnote-ref" id="fnref1" role="doc-noteref"><sup>1</sup></a>:</p>
<div class="no-row-height column-margin column-container"><div id="fn1"><p><sup>1</sup>&nbsp;We also technically have respirometry data for oxygen consumption and production, however the duration of the respirometry run was not sufficiently long to</p></div></div><ul>
<li>wet weight</li>
<li>calculated protein content</li>
<li>PAM Rapid Light Curve data</li>
</ul>
<p>Our overarching goals with these samples are to potentially get the following from each sample:</p>
<ul>
<li>Microbial DNA</li>
<li>Symbiont RNA</li>
<li>Anemone RNA</li>
<li>Host protein content</li>
<li>Symbiont counts</li>
<li>Chlorophyll a &amp; c2 content</li>
</ul>
<p>Confirm that the wet weight range of the samples is within 0.17g to 0.75g</p>
<div class="page-columns page-full"><blockquote class="blockquote">
<p>“… the method developed here requires very small sample sizes but can be used successfully for a range of weights (0.17–0.75 g wet weight)”<span class="citation" data-cites="steinberg2021">(<a href="#ref-steinberg2021" role="doc-biblioref">Steinberg et al. 2021</a>)</span></p>
</blockquote><div class="no-row-height column-margin column-container"><div id="ref-steinberg2021" class="csl-entry" role="listitem">
Steinberg, Rosemary Kate, Emma L. Johnston, Teresa Bednarek, Katherine A. Dafforn, and Tracy D. Ainsworth. 2021. <span>“Its <span>What</span>’s on the <span>Inside That Counts</span>: <span>An Effective</span>, <span>Efficient</span>, and <span>Streamlined Method</span> for <span>Quantification</span> of <span>Octocoral Symbiodiniaceae</span> and <span>Chlorophyll</span>.”</span> <em>Frontiers in Marine Science</em> 8 (September). <a href="https://doi.org/10.3389/fmars.2021.710730">https://doi.org/10.3389/fmars.2021.710730</a>.
</div></div></div>
</section>
<section id="procedure" class="level1 page-columns page-full">
<h1>Procedure</h1>
<section id="separate-symbiont-protein-fractions" class="level2 page-columns page-full">
<h2 class="anchored" data-anchor-id="separate-symbiont-protein-fractions">Separate symbiont &amp; protein fractions</h2>
<ul>
<li>top off each frozen sample tube with <code>Ultrapure water</code>/ <code>DNA/RNA Shield</code></li>
<li>thaw on ice until able to remove from tube</li>
<li>remove and cut 2 small amounts of tissue</li>
<li>place one of the tissue cuts in 2mL microcentrifuge tube Cat No.&nbsp;02-682-004 (the kind with a blunted or rounded bottom so the homogenizer bit tip can get all the way down) and add 500uL of Ultrapure or RO water</li>
<li>place the other tissue cut in a nuclease free tube and add 500uL of DNA/RNA Sheild</li>
<li>homogenize each tube using a tissue-tearor for 30 seconds. Sterilize the tissue-tearor between each tube. Try finding the homogenizer teflon thing…</li>
<li>centrifuge at 200<a href="#fn2" class="footnote-ref" id="fnref2" role="doc-noteref"><sup>2</sup></a> xg for 2 minutes</li>
<li>remove supernatent with pipette<a href="#fn3" class="footnote-ref" id="fnref3" role="doc-noteref"><sup>3</sup></a> &amp; save in separate labelled 2mL tube<a href="#fn4" class="footnote-ref" id="fnref4" role="doc-noteref"><sup>4</sup></a></li>
<li>refill symbiont pellet tube with 1mL <code>Ultrapure water</code>/ <code>DNA/RNA Shield</code></li>
<li>vortex to re-suspend symbiont cells</li>
<li>genomics tubes can be frozen at -20C in RNAlater or DNA/RNA Shield until ready to be extracted</li>
<li>for the chlorophyll content/mitotic index tubes, centrifuge at 200 xg for 2 minutes</li>
<li>remove supernatent with pipette, add to previously separated (protein) tube</li>
<li>top off symbiont pellet with 1ml of 3X<a href="#fn5" class="footnote-ref" id="fnref5" role="doc-noteref"><sup>5</sup></a> <a href="https://www.sigmaaldrich.com/US/en/product/sigma/p7059?icid=sharepdp-clipboard-copy-productdetailpage">Phosphate-Buffered Saline (PBS)</a></li>
<li>you may freeze both the symbiont and supernatent (aka anemone tissue/protein) tubes until ready to use downstream in DNA/RNA extraction, protein quantification, symbiont counts and mitotic index, and chlorophyll quantification.</li>
</ul>
<div class="no-row-height column-margin column-container"><div id="fn2"><p><sup>2</sup>&nbsp;Low and slow.. I tried it at much higher (16,110xg) and the pellet caked up really tight, making it hard to re-suspend the symbionts. It also did draw the protein down into a top layer above the symbiont pellet rather than left suspended in the supernatent</p></div><div id="fn3"><p><sup>3</sup>&nbsp;The pellet is not very stable and therefore the supernatent must be carefully pipetted out instead of poured so as not to disturb the symbiont cell ‘pellet’ that is formed at the bottom of the tube</p></div><div id="fn4"><p><sup>4</sup>&nbsp;This tube is for running a protein assay!</p></div><div id="fn5"><p><sup>5</sup>&nbsp;<a href="https://www.sigmaaldrich.com/US/en/support/calculators-and-apps/1x-phosphate-buffered-saline">How to prepare</a> 100mL of 3X PBS from 10X PBS stock solution: <span class="math display">\[
C_{1}V_{1} = C_{2}V_{2}
\]</span> <span class="math display">\[
10xPBS \cdot V_{1} = 3xPBS \cdot 100mL
\]</span> <span class="math display">\[
V_{1} = \frac{3xPBS \cdot 100mL}{10xPBS}
\]</span> <span class="math display">\[
V_{1} = \frac{300}{10} = 30mL
\]</span> Therefore <span class="math inline">\(30mL\)</span> of <code>10xPBS</code> added to <span class="math inline">\(70mL\)</span> of <code>Milli-Q Water</code> will give a final concentration of <code>3xPBS</code> in a final volume of <span class="math inline">\(100mL\)</span>.</p></div></div></section>
<section id="protein-quantification" class="level2">
<h2 class="anchored" data-anchor-id="protein-quantification">Protein quantification</h2>
<ul>
<li>defrost supernatent (protein tube)</li>
<li>top up with RO water to standard volume</li>
<li>quantify protein against a bovine albumin standard using the Thermo Scientific Coomasie Plus (Bradford) kit and protocol</li>
<li>take all spectrophotometer measurements at <span class="math inline">\(595nm\)</span></li>
<li>report units in <span class="math inline">\(\mu g/mL\)</span></li>
</ul>
</section>
<section id="symbiont-counts" class="level2 page-columns page-full">
<h2 class="anchored" data-anchor-id="symbiont-counts">Symbiont counts</h2>
<ul>
<li>remove 500uL to a new 2mL microcentrifuge tube</li>
<li>Add 1mL of 3X PBS</li>
<li>vortex to resuspend</li>
<li>pipette out <span class="math inline">\(200 \mu L\)</span> subsample, and place on a Neubauer Improved heamocytometer</li>
<li>place the heamocytometer slide on a microscope, focus &amp; photograph<a href="#fn6" class="footnote-ref" id="fnref6" role="doc-noteref"><sup>6</sup></a></li>
<li>repeat 6 times per sample! Oof… I need an undergrad for this.</li>
</ul>
<div class="no-row-height column-margin column-container"><div id="fn6"><blockquote class="blockquote"><sup>6</sup>&nbsp;
<p>“For ease and speed of counting, if there were more than 30 cells per square in the test sample, the entire treatment was diluted in separate Eppendorf tubes by either mixing 500 μl Symbiodiniaceae with 500 μl 3× PBS (2× dilution) or 250 μl Symbiodiniaceae with 750 μl of 3× PBS (4× dilution).” (Steinberg et al., 2021, p.&nbsp;3)</p>
</blockquote>
</div></div><blockquote class="blockquote">
<p>“The final count was then multiplied by the dilution to attain correct counts. Each haemocytometer fill was considered a technical replicate, while the squares counted within the grid were considered a sub-replicate. Six technical replicates and 5 sub-replicates per technical replicate were performed, for a total of 30 replicates. Replicate numbers were chosen based on previous work in stony and octocorals and anemones (Brown et al., 1995; Hueerkamp et al., 2001; Hill and Scott, 2012; Hill et al., 2014; Pupier et al., 2018; Gierz et al., 2020).” (Steinberg et al., 2021, p.&nbsp;3)</p>
<p>“The number of Symbiodiniaceae cells was standardized to the total protein concentration of the supernatant. The dilution factor (DF) of the supernatant compared to the resuspended Symbiodiniaceae pellet must be considered when calculating Symbiodiniaceae concentrations. In this study, the supernatant had a volume of 40 ml and the resuspended Symbiodiniaceae had a volume of 5 ml, and as such the DF was 8. Symbiodiniaceae per ml was calculated using the following formula: count ×dilution 6.25 ×10−6 , where 6.25 × 10−6 is the volume of one haemocytometer grid square. Symbiodiniaceae per μg protein was calculated using the following formula: Symbiodiniceae ml DF ×protein( μg ml ) or count ×dilution 6.25 ×10−6 DF ×protein ( μg ml ).” (Steinberg et al., 2021, p.&nbsp;3)</p>
</blockquote>
</section>
<section id="chlorophyll-extraction" class="level2 page-columns page-full">
<h2 class="anchored" data-anchor-id="chlorophyll-extraction">Chlorophyll extraction</h2>
<ul>
<li>vortex the remaining contents of the Chlorophyll tube to re-suspend the symbionts</li>
<li>centrifuge for 10 minutes at 800 xg</li>
<li>remove supernatent &amp; discard</li>
<li>add 5mL of 100% acetone</li>
<li>vortex to re-suspend symbiont pellet</li>
<li>incubate tubes in the dark at <span class="math inline">\(4^{\circ}C\)</span> for 48 hours<a href="#fn7" class="footnote-ref" id="fnref7" role="doc-noteref"><sup>7</sup></a></li>
</ul>
<div class="no-row-height column-margin column-container"><div id="fn7"><p><sup>7</sup>&nbsp;In the paper Steinberg et al., 2021, they compared a 24hr and a 48hr Chlorophyll extraction incubation period… they found that 24hrs was perfectly fine for Chl-a extraction, but that 48hrs was better for Chl-c2 extraction. See Figure 3. from the paper below!</p></div></div><div class="image-container" style="width: 150%;">
<p><img src="steinberg_fig3.png" class="img-fluid"></p>
</div>
</section>
<section id="measure-absorbance-using-a-nanodrop-2000c-spectrophotometer" class="level2 page-columns page-full">
<h2 class="anchored" data-anchor-id="measure-absorbance-using-a-nanodrop-2000c-spectrophotometer">Measure absorbance using a NanoDrop 2000c Spectrophotometer</h2>
<section id="program-nanodrop-2000c-method-editor" class="level3 page-columns page-full">
<h3 class="anchored" data-anchor-id="program-nanodrop-2000c-method-editor">Program NanoDrop 2000c Method Editor</h3>
<p>I emailed<a href="#fn8" class="footnote-ref" id="fnref8" role="doc-noteref"><sup>8</sup></a> Thermo Fisher for some advice on this procedure and got the following instructions:</p>
<div class="no-row-height column-margin column-container"><div id="fn8"><p><sup>8</sup>&nbsp;Hello Sarah,&nbsp;<br>
<br>
Thank you for your email. Please see link to below to a protocol that we have published for measuring chlorophyll on the NanoDrop One. It is important to note that the pre-defined custom method mentioned in the document is only compatible with the NanoDrop One, and not with the NanoDrop 2000.<br>
<br>
<a href="https://urldefense.com/v3/__https://assets.thermofisher.com/TFS-Assets/MSD/Application-Notes/quantify-chlorophyll-a-and-chlorophyll-b-with-custom-method-T141.pdf__;!!K-Hz7m0Vt54!l4t_v1Q2wbByHGjWD4zmlHTr0MMd4mW_xkGoPnXgzBrR7vofzzyK3trG-Kjumc3-HiSX3eYSacfPiQdwpSk$">quantify-chlorophyll-a-and-chlorophyll-b-with-custom-method-T141.pdf (thermofisher.com)</a><br>
<br>
You can replicate the Custom Method on your NanoDrop 2000 by following the attached protocol and using the below guidelines in the “Method Editor” application:&nbsp;</p>
<ul>
<li><p>Wavelength range Visible (350–850 nm)</p></li>
<li><p>Extinction coefficient 74.8 g/L</p></li>
<li><p>Analysis wavelength 666 nm</p></li>
<li><p>Correction for analysis wavelength 750 nm</p></li>
<li><p>Baseline correction 750 nm</p></li>
<li><p>Automated pathlength On</p></li>
</ul>
<p>If you have any questions, please do not hesitate to contact me; it is my pleasure to assist you. I can best be reached by email at <a href="mailto:nanodrop@thermofisher.com">nanodrop@thermofisher.com</a>, and typing Attention: Moira in the subject bar. If I am unavailable, any member of our Technical Support Team would be happy to assist you at 877-724-7690.<br>
<br>
Best,<br>
Moira Abbott<br>
Technical Application Specialist II<br>
Thermo Scientific NanoDrop Products<br>
<br>
Thermo Fisher Scientific<br>
<a href="https://www.google.com/maps/search/3411+Silverside+Road?entry=gmail&amp;source=g">3411 Silverside Road</a> | Tatnall Building, Suite 100 | Wilmington, DE 19810<br>
Tel: +1 (302) 479-7707 | Toll Free: +1 (877) 724-7690 ext. 2400109 | Fax: +1 (302) 792-7155<br>
<a href="mailto:nanodrop@thermofisher.com">nanodrop@thermofisher.com</a> | <a href="https://urldefense.com/v3/__http://www.thermofisher.com/nanodrop__;!!K-Hz7m0Vt54!l4t_v1Q2wbByHGjWD4zmlHTr0MMd4mW_xkGoPnXgzBrR7vofzzyK3trG-Kjumc3-HiSX3eYSacfPgBMP5QU$">www.thermofisher.com/nanodrop</a><br>
&nbsp;</p></div></div><p>Follow procedure from the <a href="https://assets.thermofisher.com/TFS-Assets/MSD/Application-Notes/quantify-chlorophyll-a-and-chlorophyll-b-with-custom-method-T141.pdf">NanoDrop One</a>, but note the following differences for accomplishing this on the NanoDrop 2000c:</p>
<ol type="1">
<li>Open the NanoDrop 2000c desktop application</li>
<li>Click on the “Method Editor” button</li>
</ol>
<p><img src="01_method-editor.PNG" class="img-fluid"></p>
<ol start="2" type="1">
<li><p>Name and describe the custom measurement method. Here, select <code>Manually enetered factor / extinction coefficient</code> for the <code>Method type</code>. Click <code>Next</code>.<img src="02_name-type.PNG" class="img-fluid"></p></li>
<li><p>In this window type in the analysis wavelength in nanometers (nm) that you want to measure. Here we want to measure 630, 663, and 750. We want the extinction coefficient to be from <span class="citation" data-cites="jeffrey1975">Jeffrey and Humphrey (<a href="#ref-jeffrey1975" role="doc-biblioref">1975</a>)</span> table 2:</p></li>
</ol>
<div class="no-row-height column-margin column-container"></div><p><img src="jeffery_table2.png" class="img-fluid"></p>
<p>However, the extinction coefficients for 100% acetone are different for each type of chlorophyll… If I want to measure both chlorophyll a and c2, does this mean I need a different method for each one?</p>
<p><img src="03_measurement.PNG" class="img-fluid"></p>
<ol start="4" type="1">
<li>Correction for analysis wavelength should be 750 nm <em>and</em> a baseline correction of 750 nm. Click <code>Next</code>.</li>
</ol>
<p><img src="04_correction.PNG" class="img-fluid"></p>
<ol start="5" type="1">
<li>Optionally enter a formula table. Here I just want to absorbance at each of the following wavelengths: 630, 663, and 750. Click <code>Next</code>.</li>
</ol>
<p><img src="05_formula-table.PNG" class="img-fluid"></p>
<ol start="6" type="1">
<li><p>In instrument settings select the wavelength measurement range to Visible range (350nm- 840nm). Also check the box next to <code>Auto pathlength</code> indicating that it should be ‘On’. Click <code>Finish</code>.</p>
<p><img src="06_settings.PNG" class="img-fluid"></p></li>
<li><p>Load your blank cuvette…</p>
<p><img src="08_cuvette.PNG" class="img-fluid"></p></li>
</ol>
</section>
<section id="measure-cuvette-blanks-and-samples" class="level3">
<h3 class="anchored" data-anchor-id="measure-cuvette-blanks-and-samples">Measure cuvette blanks and samples</h3>
<ol start="6" type="1">
<li>after the 48hr incubation, pull tubes out of the 4C fridge and centrifuge for 10 minutes at 805 RCF</li>
<li>transfer 2 2mL aliquots of extracted chlorophyll in acetone from each symbiont tube into two optical glass cuvettes</li>
<li>measure absorbance at 630, 663, and 750nm using the NanoDrop cuvettes</li>
</ol>
</section>
</section>
</section>
<section id="calculate-concentrations" class="level1 page-columns page-full">
<h1>Calculate concentrations</h1>
<ul>
<li><p>calculate concentrations of chlorophyll <span class="math inline">\(a\)</span> and <span class="math inline">\(c_2\)</span> in <span class="math inline">\(\mu g/mL\)</span> using the following equations from <span class="citation" data-cites="jeffrey1975">(<a href="#ref-jeffrey1975" role="doc-biblioref">Jeffrey and Humphrey 1975</a>)</span> Jeffery and Humphrey (1975) for dinoflagellates:</p>
<p><span class="math display">\[ Chlorophyll a = 11.43E_{663} − 0.64E_{630} \]</span> <span class="math display">\[ Chlorophyll c2 = 27.09E_{630} − 3.63E_{663} \]</span></p></li>
</ul>
<div class="no-row-height column-margin column-container"><div id="ref-jeffrey1975" class="csl-entry" role="listitem">
Jeffrey, S. W., and G. F. Humphrey. 1975. <span>“New Spectrophotometric Equations for Determining Chlorophylls <span><em>a</em></span>, <span><em>b</em></span>, <span><em>C</em></span>1 and <span><em>C</em></span>2 in Higher Plants, Algae and Natural Phytoplankton.”</span> <em>Biochemie Und Physiologie Der Pflanzen</em> 167 (2): 191–94. <a href="https://doi.org/10.1016/S0015-3796(17)30778-3">https://doi.org/10.1016/S0015-3796(17)30778-3</a>.
</div></div><blockquote class="blockquote">
<p>” As 1 ml of resuspended Symbiodiniaceae pellet was extracted in 5 ml of acetone, chlorophyll concentrations were multiplied by five. To calculate chlorophyll per μg protein, the following equation was used: 5 × chlorophyll ( μg ml ) 8 ×protein ( μg ml ).” (Steinberg et al., 2021, p.&nbsp;4)</p>
</blockquote>


<!-- -->


</section>


</main> <!-- /main -->
<script id="quarto-html-after-body" type="application/javascript">
  window.document.addEventListener("DOMContentLoaded", function (event) {
    // Ensure there is a toggle, if there isn't float one in the top right
    if (window.document.querySelector('.quarto-color-scheme-toggle') === null) {
      const a = window.document.createElement('a');
      a.classList.add('top-right');
      a.classList.add('quarto-color-scheme-toggle');
      a.href = "";
      a.onclick = function() { try { window.quartoToggleColorScheme(); } catch {} return false; };
      const i = window.document.createElement("i");
      i.classList.add('bi');
      a.appendChild(i);
      window.document.body.appendChild(a);
    }
    setColorSchemeToggle(hasAlternateSentinel())
    const icon = "";
    const anchorJS = new window.AnchorJS();
    anchorJS.options = {
      placement: 'right',
      icon: icon
    };
    anchorJS.add('.anchored');
    const isCodeAnnotation = (el) => {
      for (const clz of el.classList) {
        if (clz.startsWith('code-annotation-')) {                     
          return true;
        }
      }
      return false;
    }
    const onCopySuccess = function(e) {
      // button target
      const button = e.trigger;
      // don't keep focus
      button.blur();
      // flash "checked"
      button.classList.add('code-copy-button-checked');
      var currentTitle = button.getAttribute("title");
      button.setAttribute("title", "Copied!");
      let tooltip;
      if (window.bootstrap) {
        button.setAttribute("data-bs-toggle", "tooltip");
        button.setAttribute("data-bs-placement", "left");
        button.setAttribute("data-bs-title", "Copied!");
        tooltip = new bootstrap.Tooltip(button, 
          { trigger: "manual", 
            customClass: "code-copy-button-tooltip",
            offset: [0, -8]});
        tooltip.show();    
      }
      setTimeout(function() {
        if (tooltip) {
          tooltip.hide();
          button.removeAttribute("data-bs-title");
          button.removeAttribute("data-bs-toggle");
          button.removeAttribute("data-bs-placement");
        }
        button.setAttribute("title", currentTitle);
        button.classList.remove('code-copy-button-checked');
      }, 1000);
      // clear code selection
      e.clearSelection();
    }
    const getTextToCopy = function(trigger) {
        const codeEl = trigger.previousElementSibling.cloneNode(true);
        for (const childEl of codeEl.children) {
          if (isCodeAnnotation(childEl)) {
            childEl.remove();
          }
        }
        return codeEl.innerText;
    }
    const clipboard = new window.ClipboardJS('.code-copy-button:not([data-in-quarto-modal])', {
      text: getTextToCopy
    });
    clipboard.on('success', onCopySuccess);
    if (window.document.getElementById('quarto-embedded-source-code-modal')) {
      const clipboardModal = new window.ClipboardJS('.code-copy-button[data-in-quarto-modal]', {
        text: getTextToCopy,
        container: window.document.getElementById('quarto-embedded-source-code-modal')
      });
      clipboardModal.on('success', onCopySuccess);
    }
    const viewSource = window.document.getElementById('quarto-view-source') ||
                       window.document.getElementById('quarto-code-tools-source');
    if (viewSource) {
      const sourceUrl = viewSource.getAttribute("data-quarto-source-url");
      viewSource.addEventListener("click", function(e) {
        if (sourceUrl) {
          // rstudio viewer pane
          if (/\bcapabilities=\b/.test(window.location)) {
            window.open(sourceUrl);
          } else {
            window.location.href = sourceUrl;
          }
        } else {
          const modal = new bootstrap.Modal(document.getElementById('quarto-embedded-source-code-modal'));
          modal.show();
        }
        return false;
      });
    }
    function toggleCodeHandler(show) {
      return function(e) {
        const detailsSrc = window.document.querySelectorAll(".cell > details > .sourceCode");
        for (let i=0; i<detailsSrc.length; i++) {
          const details = detailsSrc[i].parentElement;
          if (show) {
            details.open = true;
          } else {
            details.removeAttribute("open");
          }
        }
        const cellCodeDivs = window.document.querySelectorAll(".cell > .sourceCode");
        const fromCls = show ? "hidden" : "unhidden";
        const toCls = show ? "unhidden" : "hidden";
        for (let i=0; i<cellCodeDivs.length; i++) {
          const codeDiv = cellCodeDivs[i];
          if (codeDiv.classList.contains(fromCls)) {
            codeDiv.classList.remove(fromCls);
            codeDiv.classList.add(toCls);
          } 
        }
        return false;
      }
    }
    const hideAllCode = window.document.getElementById("quarto-hide-all-code");
    if (hideAllCode) {
      hideAllCode.addEventListener("click", toggleCodeHandler(false));
    }
    const showAllCode = window.document.getElementById("quarto-show-all-code");
    if (showAllCode) {
      showAllCode.addEventListener("click", toggleCodeHandler(true));
    }
      var localhostRegex = new RegExp(/^(?:http|https):\/\/localhost\:?[0-9]*\//);
      var mailtoRegex = new RegExp(/^mailto:/);
        var filterRegex = new RegExp("https:\/\/sarahtanja\.github\.io\/quarto-blog\/");
      var isInternal = (href) => {
          return filterRegex.test(href) || localhostRegex.test(href) || mailtoRegex.test(href);
      }
      // Inspect non-navigation links and adorn them if external
     var links = window.document.querySelectorAll('a[href]:not(.nav-link):not(.navbar-brand):not(.toc-action):not(.sidebar-link):not(.sidebar-item-toggle):not(.pagination-link):not(.no-external):not([aria-hidden]):not(.dropdown-item):not(.quarto-navigation-tool):not(.about-link)');
      for (var i=0; i<links.length; i++) {
        const link = links[i];
        if (!isInternal(link.href)) {
          // undo the damage that might have been done by quarto-nav.js in the case of
          // links that we want to consider external
          if (link.dataset.originalHref !== undefined) {
            link.href = link.dataset.originalHref;
          }
            // target, if specified
            link.setAttribute("target", "_blank");
            if (link.getAttribute("rel") === null) {
              link.setAttribute("rel", "noopener");
            }
            // default icon
            link.classList.add("external");
        }
      }
    function tippyHover(el, contentFn, onTriggerFn, onUntriggerFn) {
      const config = {
        allowHTML: true,
        maxWidth: 500,
        delay: 100,
        arrow: false,
        appendTo: function(el) {
            return el.parentElement;
        },
        interactive: true,
        interactiveBorder: 10,
        theme: 'quarto',
        placement: 'bottom-start',
      };
      if (contentFn) {
        config.content = contentFn;
      }
      if (onTriggerFn) {
        config.onTrigger = onTriggerFn;
      }
      if (onUntriggerFn) {
        config.onUntrigger = onUntriggerFn;
      }
      window.tippy(el, config); 
    }
    const noterefs = window.document.querySelectorAll('a[role="doc-noteref"]');
    for (var i=0; i<noterefs.length; i++) {
      const ref = noterefs[i];
      tippyHover(ref, function() {
        // use id or data attribute instead here
        let href = ref.getAttribute('data-footnote-href') || ref.getAttribute('href');
        try { href = new URL(href).hash; } catch {}
        const id = href.replace(/^#\/?/, "");
        const note = window.document.getElementById(id);
        if (note) {
          return note.innerHTML;
        } else {
          return "";
        }
      });
    }
    const xrefs = window.document.querySelectorAll('a.quarto-xref');
    const processXRef = (id, note) => {
      // Strip column container classes
      const stripColumnClz = (el) => {
        el.classList.remove("page-full", "page-columns");
        if (el.children) {
          for (const child of el.children) {
            stripColumnClz(child);
          }
        }
      }
      stripColumnClz(note)
      if (id === null || id.startsWith('sec-')) {
        // Special case sections, only their first couple elements
        const container = document.createElement("div");
        if (note.children && note.children.length > 2) {
          container.appendChild(note.children[0].cloneNode(true));
          for (let i = 1; i < note.children.length; i++) {
            const child = note.children[i];
            if (child.tagName === "P" && child.innerText === "") {
              continue;
            } else {
              container.appendChild(child.cloneNode(true));
              break;
            }
          }
          if (window.Quarto?.typesetMath) {
            window.Quarto.typesetMath(container);
          }
          return container.innerHTML
        } else {
          if (window.Quarto?.typesetMath) {
            window.Quarto.typesetMath(note);
          }
          return note.innerHTML;
        }
      } else {
        // Remove any anchor links if they are present
        const anchorLink = note.querySelector('a.anchorjs-link');
        if (anchorLink) {
          anchorLink.remove();
        }
        if (window.Quarto?.typesetMath) {
          window.Quarto.typesetMath(note);
        }
        if (note.classList.contains("callout")) {
          return note.outerHTML;
        } else {
          return note.innerHTML;
        }
      }
    }
    for (var i=0; i<xrefs.length; i++) {
      const xref = xrefs[i];
      tippyHover(xref, undefined, function(instance) {
        instance.disable();
        let url = xref.getAttribute('href');
        let hash = undefined; 
        if (url.startsWith('#')) {
          hash = url;
        } else {
          try { hash = new URL(url).hash; } catch {}
        }
        if (hash) {
          const id = hash.replace(/^#\/?/, "");
          const note = window.document.getElementById(id);
          if (note !== null) {
            try {
              const html = processXRef(id, note.cloneNode(true));
              instance.setContent(html);
            } finally {
              instance.enable();
              instance.show();
            }
          } else {
            // See if we can fetch this
            fetch(url.split('#')[0])
            .then(res => res.text())
            .then(html => {
              const parser = new DOMParser();
              const htmlDoc = parser.parseFromString(html, "text/html");
              const note = htmlDoc.getElementById(id);
              if (note !== null) {
                const html = processXRef(id, note);
                instance.setContent(html);
              } 
            }).finally(() => {
              instance.enable();
              instance.show();
            });
          }
        } else {
          // See if we can fetch a full url (with no hash to target)
          // This is a special case and we should probably do some content thinning / targeting
          fetch(url)
          .then(res => res.text())
          .then(html => {
            const parser = new DOMParser();
            const htmlDoc = parser.parseFromString(html, "text/html");
            const note = htmlDoc.querySelector('main.content');
            if (note !== null) {
              // This should only happen for chapter cross references
              // (since there is no id in the URL)
              // remove the first header
              if (note.children.length > 0 && note.children[0].tagName === "HEADER") {
                note.children[0].remove();
              }
              const html = processXRef(null, note);
              instance.setContent(html);
            } 
          }).finally(() => {
            instance.enable();
            instance.show();
          });
        }
      }, function(instance) {
      });
    }
        let selectedAnnoteEl;
        const selectorForAnnotation = ( cell, annotation) => {
          let cellAttr = 'data-code-cell="' + cell + '"';
          let lineAttr = 'data-code-annotation="' +  annotation + '"';
          const selector = 'span[' + cellAttr + '][' + lineAttr + ']';
          return selector;
        }
        const selectCodeLines = (annoteEl) => {
          const doc = window.document;
          const targetCell = annoteEl.getAttribute("data-target-cell");
          const targetAnnotation = annoteEl.getAttribute("data-target-annotation");
          const annoteSpan = window.document.querySelector(selectorForAnnotation(targetCell, targetAnnotation));
          const lines = annoteSpan.getAttribute("data-code-lines").split(",");
          const lineIds = lines.map((line) => {
            return targetCell + "-" + line;
          })
          let top = null;
          let height = null;
          let parent = null;
          if (lineIds.length > 0) {
              //compute the position of the single el (top and bottom and make a div)
              const el = window.document.getElementById(lineIds[0]);
              top = el.offsetTop;
              height = el.offsetHeight;
              parent = el.parentElement.parentElement;
            if (lineIds.length > 1) {
              const lastEl = window.document.getElementById(lineIds[lineIds.length - 1]);
              const bottom = lastEl.offsetTop + lastEl.offsetHeight;
              height = bottom - top;
            }
            if (top !== null && height !== null && parent !== null) {
              // cook up a div (if necessary) and position it 
              let div = window.document.getElementById("code-annotation-line-highlight");
              if (div === null) {
                div = window.document.createElement("div");
                div.setAttribute("id", "code-annotation-line-highlight");
                div.style.position = 'absolute';
                parent.appendChild(div);
              }
              div.style.top = top - 2 + "px";
              div.style.height = height + 4 + "px";
              div.style.left = 0;
              let gutterDiv = window.document.getElementById("code-annotation-line-highlight-gutter");
              if (gutterDiv === null) {
                gutterDiv = window.document.createElement("div");
                gutterDiv.setAttribute("id", "code-annotation-line-highlight-gutter");
                gutterDiv.style.position = 'absolute';
                const codeCell = window.document.getElementById(targetCell);
                const gutter = codeCell.querySelector('.code-annotation-gutter');
                gutter.appendChild(gutterDiv);
              }
              gutterDiv.style.top = top - 2 + "px";
              gutterDiv.style.height = height + 4 + "px";
            }
            selectedAnnoteEl = annoteEl;
          }
        };
        const unselectCodeLines = () => {
          const elementsIds = ["code-annotation-line-highlight", "code-annotation-line-highlight-gutter"];
          elementsIds.forEach((elId) => {
            const div = window.document.getElementById(elId);
            if (div) {
              div.remove();
            }
          });
          selectedAnnoteEl = undefined;
        };
          // Handle positioning of the toggle
      window.addEventListener(
        "resize",
        throttle(() => {
          elRect = undefined;
          if (selectedAnnoteEl) {
            selectCodeLines(selectedAnnoteEl);
          }
        }, 10)
      );
      function throttle(fn, ms) {
      let throttle = false;
      let timer;
        return (...args) => {
          if(!throttle) { // first call gets through
              fn.apply(this, args);
              throttle = true;
          } else { // all the others get throttled
              if(timer) clearTimeout(timer); // cancel #2
              timer = setTimeout(() => {
                fn.apply(this, args);
                timer = throttle = false;
              }, ms);
          }
        };
      }
        // Attach click handler to the DT
        const annoteDls = window.document.querySelectorAll('dt[data-target-cell]');
        for (const annoteDlNode of annoteDls) {
          annoteDlNode.addEventListener('click', (event) => {
            const clickedEl = event.target;
            if (clickedEl !== selectedAnnoteEl) {
              unselectCodeLines();
              const activeEl = window.document.querySelector('dt[data-target-cell].code-annotation-active');
              if (activeEl) {
                activeEl.classList.remove('code-annotation-active');
              }
              selectCodeLines(clickedEl);
              clickedEl.classList.add('code-annotation-active');
            } else {
              // Unselect the line
              unselectCodeLines();
              clickedEl.classList.remove('code-annotation-active');
            }
          });
        }
    const findCites = (el) => {
      const parentEl = el.parentElement;
      if (parentEl) {
        const cites = parentEl.dataset.cites;
        if (cites) {
          return {
            el,
            cites: cites.split(' ')
          };
        } else {
          return findCites(el.parentElement)
        }
      } else {
        return undefined;
      }
    };
    var bibliorefs = window.document.querySelectorAll('a[role="doc-biblioref"]');
    for (var i=0; i<bibliorefs.length; i++) {
      const ref = bibliorefs[i];
      const citeInfo = findCites(ref);
      if (citeInfo) {
        tippyHover(citeInfo.el, function() {
          var popup = window.document.createElement('div');
          citeInfo.cites.forEach(function(cite) {
            var citeDiv = window.document.createElement('div');
            citeDiv.classList.add('hanging-indent');
            citeDiv.classList.add('csl-entry');
            var biblioDiv = window.document.getElementById('ref-' + cite);
            if (biblioDiv) {
              citeDiv.innerHTML = biblioDiv.innerHTML;
            }
            popup.appendChild(citeDiv);
          });
          return popup.innerHTML;
        });
      }
    }
  });
  </script><div class="modal fade" id="quarto-embedded-source-code-modal" tabindex="-1" aria-labelledby="quarto-embedded-source-code-modal-label" aria-hidden="true"><div class="modal-dialog modal-dialog-scrollable"><div class="modal-content"><div class="modal-header"><h5 class="modal-title" id="quarto-embedded-source-code-modal-label">Source Code</h5><button class="btn-close" data-bs-dismiss="modal"></button></div><div class="modal-body"><div class="">
<div class="sourceCode" id="cb1" data-shortcodes="false"><pre class="sourceCode markdown code-with-copy"><code class="sourceCode markdown"><span id="cb1-1"><a href="#cb1-1" aria-hidden="true" tabindex="-1"></a><span class="co">---</span></span>
<span id="cb1-2"><a href="#cb1-2" aria-hidden="true" tabindex="-1"></a><span class="an">title:</span><span class="co"> "Extracting &amp; measuring chlorophyll a &amp; c2 concentrations"</span></span>
<span id="cb1-3"><a href="#cb1-3" aria-hidden="true" tabindex="-1"></a><span class="an">subtitle:</span><span class="co"> "... from an anemone-hosted symbiotic dinoflagellate using a ThermoFischer Nanodrop 2000c"</span></span>
<span id="cb1-4"><a href="#cb1-4" aria-hidden="true" tabindex="-1"></a><span class="an">author:</span><span class="co"> "Sarah Tanja"</span></span>
<span id="cb1-5"><a href="#cb1-5" aria-hidden="true" tabindex="-1"></a><span class="an">date:</span><span class="co"> '05/23/2024'</span></span>
<span id="cb1-6"><a href="#cb1-6" aria-hidden="true" tabindex="-1"></a><span class="an">categories:</span><span class="co"> [protocols, anemones]</span></span>
<span id="cb1-7"><a href="#cb1-7" aria-hidden="true" tabindex="-1"></a><span class="an">draft:</span><span class="co"> false</span></span>
<span id="cb1-8"><a href="#cb1-8" aria-hidden="true" tabindex="-1"></a><span class="an">toc:</span><span class="co"> true</span></span>
<span id="cb1-9"><a href="#cb1-9" aria-hidden="true" tabindex="-1"></a><span class="an">toc-title:</span><span class="co"> Contents &lt;i class="bi bi-bookmark-heart"&gt;&lt;/i&gt;</span></span>
<span id="cb1-10"><a href="#cb1-10" aria-hidden="true" tabindex="-1"></a><span class="an">toc-depth:</span><span class="co"> 5</span></span>
<span id="cb1-11"><a href="#cb1-11" aria-hidden="true" tabindex="-1"></a><span class="an">toc-location:</span><span class="co"> left</span></span>
<span id="cb1-12"><a href="#cb1-12" aria-hidden="true" tabindex="-1"></a><span class="an">bibliography:</span><span class="co"> photophysiology.bib</span></span>
<span id="cb1-13"><a href="#cb1-13" aria-hidden="true" tabindex="-1"></a><span class="an">reference-location:</span><span class="co"> margin</span></span>
<span id="cb1-14"><a href="#cb1-14" aria-hidden="true" tabindex="-1"></a><span class="an">citation-location:</span><span class="co"> margin</span></span>
<span id="cb1-15"><a href="#cb1-15" aria-hidden="true" tabindex="-1"></a><span class="an">link-external-icon:</span><span class="co"> true</span></span>
<span id="cb1-16"><a href="#cb1-16" aria-hidden="true" tabindex="-1"></a><span class="an">link-external-newwindow:</span><span class="co"> true</span></span>
<span id="cb1-17"><a href="#cb1-17" aria-hidden="true" tabindex="-1"></a><span class="co">---</span></span>
<span id="cb1-18"><a href="#cb1-18" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb1-19"><a href="#cb1-19" aria-hidden="true" tabindex="-1"></a><span class="fu"># Overview</span></span>
<span id="cb1-20"><a href="#cb1-20" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb1-21"><a href="#cb1-21" aria-hidden="true" tabindex="-1"></a>::: callout-important</span>
<span id="cb1-22"><a href="#cb1-22" aria-hidden="true" tabindex="-1"></a>This protocol draws *heavily* (and almost exclusively!) from the awesome method paper, *"Its What’s on the Inside That Counts: An Effective, Efficient, and Streamlined Method for Quantification of Octocoral Symbiodiniaceae and Chlorophyll"* published by Rosemary Kate Steinberg, Emma L. Johnston, Teresa Bednarek, Katherine A. Dafforn and Tracy D. Ainsworth. <span class="co">[</span><span class="ot">@steinberg2021</span><span class="co">]</span></span>
<span id="cb1-23"><a href="#cb1-23" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb1-24"><a href="#cb1-24" aria-hidden="true" tabindex="-1"></a>THANK YOU to all the above listed authors for this amazingly clear methods paper. It is pure gold, and I wish more science was *actually reproducible* like this is.</span>
<span id="cb1-25"><a href="#cb1-25" aria-hidden="true" tabindex="-1"></a>:::</span>
<span id="cb1-26"><a href="#cb1-26" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb1-27"><a href="#cb1-27" aria-hidden="true" tabindex="-1"></a><span class="al">![This is from the Steinberg et a. 2021 paper, checkout more cool sci comm art from Melissa Pappas at &lt;https://www.emergingcreativesofscience.com/&gt;](steinberg_fig4.png)</span></span>
<span id="cb1-28"><a href="#cb1-28" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb1-29"><a href="#cb1-29" aria-hidden="true" tabindex="-1"></a>Here I detail out the method I aim to use for extracting and measuring chlorophyll a and c2 of the *Anthopleura elegantissima* brown algal symbiont, *Breviolum muscatinei*, relative to cnidarian host wet weight and calculated protein content.</span>
<span id="cb1-30"><a href="#cb1-30" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb1-31"><a href="#cb1-31" aria-hidden="true" tabindex="-1"></a>What is chlorophyll a and c2?</span>
<span id="cb1-32"><a href="#cb1-32" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb1-33"><a href="#cb1-33" aria-hidden="true" tabindex="-1"></a><span class="ss">-   </span>Chlorophyll</span>
<span id="cb1-34"><a href="#cb1-34" aria-hidden="true" tabindex="-1"></a><span class="ss">    -   </span>a</span>
<span id="cb1-35"><a href="#cb1-35" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb1-36"><a href="#cb1-36" aria-hidden="true" tabindex="-1"></a><span class="ss">    -   </span>c2</span>
<span id="cb1-37"><a href="#cb1-37" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb1-38"><a href="#cb1-38" aria-hidden="true" tabindex="-1"></a>What about other pigments?</span>
<span id="cb1-39"><a href="#cb1-39" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb1-40"><a href="#cb1-40" aria-hidden="true" tabindex="-1"></a><span class="ss">-   </span>Xanthophylls : photoprotective xanthophyll cycle</span>
<span id="cb1-41"><a href="#cb1-41" aria-hidden="true" tabindex="-1"></a><span class="ss">    -   </span>peridinin</span>
<span id="cb1-42"><a href="#cb1-42" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb1-43"><a href="#cb1-43" aria-hidden="true" tabindex="-1"></a><span class="ss">    -   </span>fucoxanthin</span>
<span id="cb1-44"><a href="#cb1-44" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb1-45"><a href="#cb1-45" aria-hidden="true" tabindex="-1"></a><span class="ss">    -   </span>diadinoxanthin (Dn)</span>
<span id="cb1-46"><a href="#cb1-46" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb1-47"><a href="#cb1-47" aria-hidden="true" tabindex="-1"></a><span class="ss">    -   </span>diatoxanthin (Dt)</span>
<span id="cb1-48"><a href="#cb1-48" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb1-49"><a href="#cb1-49" aria-hidden="true" tabindex="-1"></a><span class="ss">    -   </span>dinoxanthin &amp; alteration product diadinochrome</span>
<span id="cb1-50"><a href="#cb1-50" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb1-51"><a href="#cb1-51" aria-hidden="true" tabindex="-1"></a><span class="ss">    -   </span>phaeophytin a</span>
<span id="cb1-52"><a href="#cb1-52" aria-hidden="true" tabindex="-1"></a><span class="ss">-   </span>Carotenoids</span>
<span id="cb1-53"><a href="#cb1-53" aria-hidden="true" tabindex="-1"></a><span class="ss">    -   </span>peridinin? Ambarsari et al 1997</span>
<span id="cb1-54"><a href="#cb1-54" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb1-55"><a href="#cb1-55" aria-hidden="true" tabindex="-1"></a><span class="ss">    -   </span>Dn? Ambarsari et al 1997</span>
<span id="cb1-56"><a href="#cb1-56" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb1-57"><a href="#cb1-57" aria-hidden="true" tabindex="-1"></a><span class="ss">    -   </span>B carotene</span>
<span id="cb1-58"><a href="#cb1-58" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb1-59"><a href="#cb1-59" aria-hidden="true" tabindex="-1"></a><span class="al">![A. A. Venn et al., 2006, Table 1](venn_table1.png)</span></span>
<span id="cb1-60"><a href="#cb1-60" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb1-61"><a href="#cb1-61" aria-hidden="true" tabindex="-1"></a><span class="at">&gt; “Ambarsari et al. (1997) and Brown et al. (1999) obtained evidence of a functional xanthophyll cycle in Symbiodinium, thought to operate as part of non-photochemical quenching (NPQ), also observed in other marine algae (Demers et al. 1991), and analogous to the photoprotective xanthophylls cycle in terrestrial plants.” (Venn et al., 2006, p. 2134)</span></span>
<span id="cb1-62"><a href="#cb1-62" aria-hidden="true" tabindex="-1"></a><span class="at">&gt;</span></span>
<span id="cb1-63"><a href="#cb1-63" aria-hidden="true" tabindex="-1"></a><span class="at">&gt; “Xanthophyll cycling is likely to be one of several mechanisms that underpin the environmental tolerance of a coral–algal symbiosis. Symbiotic dinoflagellates, unlike free-living marine algae, exist in coral tissue equipped with additional photoprotective and thermoprotective mechanisms, for example, fluorescent proteins (Dove 2004) and heat shock proteins (Black, Voellmy &amp; Szmant 1995), which may complement or even surpass photoprotective mechanisms linked to the algal pigment profile and thus help maintain the integrity of the symbiosis (Brown et al. 2002a).” (Venn et al., 2006, p. 2140</span></span>
<span id="cb1-64"><a href="#cb1-64" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb1-65"><a href="#cb1-65" aria-hidden="true" tabindex="-1"></a>Why is chlorophyll content an important metric if we've already used the pulse-amplitude modulated (PAM) fluorometer to measure photosynthetic efficiency, quantum yield, etc.?</span>
<span id="cb1-66"><a href="#cb1-66" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb1-67"><a href="#cb1-67" aria-hidden="true" tabindex="-1"></a><span class="fu"># Materials list</span></span>
<span id="cb1-68"><a href="#cb1-68" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb1-69"><a href="#cb1-69" aria-hidden="true" tabindex="-1"></a><span class="fu">### Lab equipment</span></span>
<span id="cb1-70"><a href="#cb1-70" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb1-71"><a href="#cb1-71" aria-hidden="true" tabindex="-1"></a><span class="ss">-   </span>tissue homogenizer (like the <span class="co">[</span><span class="ot">OMNI TH</span><span class="co">](https://us.omni-inc.com/omni-tissue-homogenizer-th.html)</span> used by <span class="co">[</span><span class="ot">@steinberg2021</span><span class="co">]</span> with a 10mm stainless steel bit, we have a Biospec Products Tissue Tearor Model 985-370, which is a slightly older model compared to <span class="co">[</span><span class="ot">this here</span><span class="co">](https://www.coleparmer.com/i/tissue-tearor-985370-07-homogenizer-120-vac-1-2-a/0475050?PubID=UX&amp;persist=true&amp;ip=no&amp;gad_source=1&amp;gclid=CjwKCAjwx-CyBhAqEiwAeOcTdU_POATJBTiINJzb41uUa_jdZx9EuYzIDbyWBoKMXQRaQdXV3SPlgBoCGr4QAvD_BwE)</span>)</span>
<span id="cb1-72"><a href="#cb1-72" aria-hidden="true" tabindex="-1"></a><span class="ss">-   </span>vortex</span>
<span id="cb1-73"><a href="#cb1-73" aria-hidden="true" tabindex="-1"></a><span class="ss">-   </span>centrifuge</span>
<span id="cb1-74"><a href="#cb1-74" aria-hidden="true" tabindex="-1"></a><span class="ss">-   </span>NanoDrop 2000c Spectrophotometer</span>
<span id="cb1-75"><a href="#cb1-75" aria-hidden="true" tabindex="-1"></a><span class="ss">-   </span>2 quartz-glass Nanodrop Spec cuvettes</span>
<span id="cb1-76"><a href="#cb1-76" aria-hidden="true" tabindex="-1"></a><span class="ss">-   </span><span class="co">[</span><span class="ot">Neubauer Improved Heamocytometer</span><span class="co">](https://www.emsdiasum.com/docs/technical/datasheet/68052-14)</span></span>
<span id="cb1-77"><a href="#cb1-77" aria-hidden="true" tabindex="-1"></a><span class="ss">-   </span>Microscope or Dissecting scope</span>
<span id="cb1-78"><a href="#cb1-78" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb1-79"><a href="#cb1-79" aria-hidden="true" tabindex="-1"></a><span class="fu">### Lab consumables</span></span>
<span id="cb1-80"><a href="#cb1-80" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb1-81"><a href="#cb1-81" aria-hidden="true" tabindex="-1"></a><span class="ss">-   </span>microcentrifuge tubes</span>
<span id="cb1-82"><a href="#cb1-82" aria-hidden="true" tabindex="-1"></a><span class="ss">-   </span>pipettes &amp; tips</span>
<span id="cb1-83"><a href="#cb1-83" aria-hidden="true" tabindex="-1"></a><span class="ss">-   </span>nitrile gloves</span>
<span id="cb1-84"><a href="#cb1-84" aria-hidden="true" tabindex="-1"></a><span class="ss">-   </span><span class="co">[</span><span class="ot">ethanol-proof lab markers</span><span class="co">](https://www.simbalion.com/product/detail/A/A10/A10/No.800)</span></span>
<span id="cb1-85"><a href="#cb1-85" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb1-86"><a href="#cb1-86" aria-hidden="true" tabindex="-1"></a><span class="fu">### Chemicals &amp; solvents</span></span>
<span id="cb1-87"><a href="#cb1-87" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb1-88"><a href="#cb1-88" aria-hidden="true" tabindex="-1"></a><span class="ss">-   </span>Milli-Q Water</span>
<span id="cb1-89"><a href="#cb1-89" aria-hidden="true" tabindex="-1"></a><span class="ss">-   </span>10x <span class="co">[</span><span class="ot">Phosphate-Buffered Saline (PBS)</span><span class="co">](https://www.sigmaaldrich.com/US/en/product/sigma/p7059?icid=sharepdp-clipboard-copy-productdetailpage)</span> (room temp storage)</span>
<span id="cb1-90"><a href="#cb1-90" aria-hidden="true" tabindex="-1"></a><span class="ss">-   </span>100% Acetone</span>
<span id="cb1-91"><a href="#cb1-91" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb1-92"><a href="#cb1-92" aria-hidden="true" tabindex="-1"></a><span class="fu"># Objectives</span></span>
<span id="cb1-93"><a href="#cb1-93" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb1-94"><a href="#cb1-94" aria-hidden="true" tabindex="-1"></a>From these samples we already have<span class="ot">[^1]</span>:</span>
<span id="cb1-95"><a href="#cb1-95" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb1-96"><a href="#cb1-96" aria-hidden="true" tabindex="-1"></a><span class="ot">[^1]: </span>We also technically have respirometry data for oxygen consumption and production, however the duration of the respirometry run was not sufficiently long to</span>
<span id="cb1-97"><a href="#cb1-97" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb1-98"><a href="#cb1-98" aria-hidden="true" tabindex="-1"></a><span class="ss">-   </span>wet weight</span>
<span id="cb1-99"><a href="#cb1-99" aria-hidden="true" tabindex="-1"></a><span class="ss">-   </span>calculated protein content</span>
<span id="cb1-100"><a href="#cb1-100" aria-hidden="true" tabindex="-1"></a><span class="ss">-   </span>PAM Rapid Light Curve data</span>
<span id="cb1-101"><a href="#cb1-101" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb1-102"><a href="#cb1-102" aria-hidden="true" tabindex="-1"></a>Our overarching goals with these samples are to potentially get the following from each sample:</span>
<span id="cb1-103"><a href="#cb1-103" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb1-104"><a href="#cb1-104" aria-hidden="true" tabindex="-1"></a><span class="ss">-   </span>Microbial DNA</span>
<span id="cb1-105"><a href="#cb1-105" aria-hidden="true" tabindex="-1"></a><span class="ss">-   </span>Symbiont RNA</span>
<span id="cb1-106"><a href="#cb1-106" aria-hidden="true" tabindex="-1"></a><span class="ss">-   </span>Anemone RNA</span>
<span id="cb1-107"><a href="#cb1-107" aria-hidden="true" tabindex="-1"></a><span class="ss">-   </span>Host protein content</span>
<span id="cb1-108"><a href="#cb1-108" aria-hidden="true" tabindex="-1"></a><span class="ss">-   </span>Symbiont counts</span>
<span id="cb1-109"><a href="#cb1-109" aria-hidden="true" tabindex="-1"></a><span class="ss">-   </span>Chlorophyll a &amp; c2 content</span>
<span id="cb1-110"><a href="#cb1-110" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb1-111"><a href="#cb1-111" aria-hidden="true" tabindex="-1"></a>Confirm that the wet weight range of the samples is within 0.17g to 0.75g</span>
<span id="cb1-112"><a href="#cb1-112" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb1-113"><a href="#cb1-113" aria-hidden="true" tabindex="-1"></a><span class="at">&gt; "... the method developed here requires very small sample sizes but can be used successfully for a range of weights (0.17–0.75 g wet weight)"</span><span class="co">[</span><span class="ot">@steinberg2021</span><span class="co">]</span></span>
<span id="cb1-114"><a href="#cb1-114" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb1-115"><a href="#cb1-115" aria-hidden="true" tabindex="-1"></a><span class="fu"># Procedure</span></span>
<span id="cb1-116"><a href="#cb1-116" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb1-117"><a href="#cb1-117" aria-hidden="true" tabindex="-1"></a><span class="fu">## Separate symbiont &amp; protein fractions</span></span>
<span id="cb1-118"><a href="#cb1-118" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb1-119"><a href="#cb1-119" aria-hidden="true" tabindex="-1"></a><span class="ss">-   </span>top off each frozen sample tube with <span class="in">`Ultrapure water`</span>/ <span class="in">`DNA/RNA Shield`</span></span>
<span id="cb1-120"><a href="#cb1-120" aria-hidden="true" tabindex="-1"></a><span class="ss">-   </span>thaw on ice until able to remove from tube</span>
<span id="cb1-121"><a href="#cb1-121" aria-hidden="true" tabindex="-1"></a><span class="ss">-   </span>remove and cut 2 small amounts of tissue</span>
<span id="cb1-122"><a href="#cb1-122" aria-hidden="true" tabindex="-1"></a><span class="ss">-   </span>place one of the tissue cuts in 2mL microcentrifuge tube Cat No. 02-682-004 (the kind with a blunted or rounded bottom so the homogenizer bit tip can get all the way down) and add 500uL of Ultrapure or RO water</span>
<span id="cb1-123"><a href="#cb1-123" aria-hidden="true" tabindex="-1"></a><span class="ss">-   </span>place the other tissue cut in a nuclease free tube and add 500uL of DNA/RNA Sheild</span>
<span id="cb1-124"><a href="#cb1-124" aria-hidden="true" tabindex="-1"></a><span class="ss">-   </span>homogenize each tube using a tissue-tearor for 30 seconds. Sterilize the tissue-tearor between each tube. Try finding the homogenizer teflon thing...</span>
<span id="cb1-125"><a href="#cb1-125" aria-hidden="true" tabindex="-1"></a><span class="ss">-   </span>centrifuge at 200<span class="ot">[^2]</span> xg for 2 minutes</span>
<span id="cb1-126"><a href="#cb1-126" aria-hidden="true" tabindex="-1"></a><span class="ss">-   </span>remove supernatent with pipette<span class="ot">[^3]</span> &amp; save in separate labelled 2mL tube<span class="ot">[^4]</span></span>
<span id="cb1-127"><a href="#cb1-127" aria-hidden="true" tabindex="-1"></a><span class="ss">-   </span>refill symbiont pellet tube with 1mL <span class="in">`Ultrapure water`</span>/ <span class="in">`DNA/RNA Shield`</span></span>
<span id="cb1-128"><a href="#cb1-128" aria-hidden="true" tabindex="-1"></a><span class="ss">-   </span>vortex to re-suspend symbiont cells</span>
<span id="cb1-129"><a href="#cb1-129" aria-hidden="true" tabindex="-1"></a><span class="ss">-   </span>genomics tubes can be frozen at -20C in RNAlater or DNA/RNA Shield until ready to be extracted</span>
<span id="cb1-130"><a href="#cb1-130" aria-hidden="true" tabindex="-1"></a><span class="ss">-   </span>for the chlorophyll content/mitotic index tubes, centrifuge at 200 xg for 2 minutes</span>
<span id="cb1-131"><a href="#cb1-131" aria-hidden="true" tabindex="-1"></a><span class="ss">-   </span>remove supernatent with pipette, add to previously separated (protein) tube</span>
<span id="cb1-132"><a href="#cb1-132" aria-hidden="true" tabindex="-1"></a><span class="ss">-   </span>top off symbiont pellet with 1ml of 3X<span class="ot">[^5]</span> <span class="co">[</span><span class="ot">Phosphate-Buffered Saline (PBS)</span><span class="co">](https://www.sigmaaldrich.com/US/en/product/sigma/p7059?icid=sharepdp-clipboard-copy-productdetailpage)</span></span>
<span id="cb1-133"><a href="#cb1-133" aria-hidden="true" tabindex="-1"></a><span class="ss">-   </span>you may freeze both the symbiont and supernatent (aka anemone tissue/protein) tubes until ready to use downstream in DNA/RNA extraction, protein quantification, symbiont counts and mitotic index, and chlorophyll quantification.</span>
<span id="cb1-134"><a href="#cb1-134" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb1-135"><a href="#cb1-135" aria-hidden="true" tabindex="-1"></a><span class="ot">[^2]: </span>Low and slow.. I tried it at much higher (16,110xg) and the pellet caked up really tight, making it hard to re-suspend the symbionts. It also did draw the protein down into a top layer above the symbiont pellet rather than left suspended in the supernatent</span>
<span id="cb1-136"><a href="#cb1-136" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb1-137"><a href="#cb1-137" aria-hidden="true" tabindex="-1"></a><span class="ot">[^3]: </span>The pellet is not very stable and therefore the supernatent must be carefully pipetted out instead of poured so as not to disturb the symbiont cell 'pellet' that is formed at the bottom of the tube</span>
<span id="cb1-138"><a href="#cb1-138" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb1-139"><a href="#cb1-139" aria-hidden="true" tabindex="-1"></a><span class="ot">[^4]: </span>This tube is for running a protein assay!</span>
<span id="cb1-140"><a href="#cb1-140" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb1-141"><a href="#cb1-141" aria-hidden="true" tabindex="-1"></a><span class="ot">[^5]: </span><span class="co">[</span><span class="ot">How to prepare</span><span class="co">](https://www.sigmaaldrich.com/US/en/support/calculators-and-apps/1x-phosphate-buffered-saline)</span> 100mL of 3X PBS from 10X PBS stock solution: $$</span>
<span id="cb1-142"><a href="#cb1-142" aria-hidden="true" tabindex="-1"></a>    C_{1}V_{1} = C_{2}V_{2} </span>
<span id="cb1-143"><a href="#cb1-143" aria-hidden="true" tabindex="-1"></a>    $$ $$</span>
<span id="cb1-144"><a href="#cb1-144" aria-hidden="true" tabindex="-1"></a>    10xPBS \cdot V_{1} = 3xPBS \cdot 100mL</span>
<span id="cb1-145"><a href="#cb1-145" aria-hidden="true" tabindex="-1"></a>    $$ $$</span>
<span id="cb1-146"><a href="#cb1-146" aria-hidden="true" tabindex="-1"></a>    V_{1} = \frac{3xPBS \cdot 100mL}{10xPBS}</span>
<span id="cb1-147"><a href="#cb1-147" aria-hidden="true" tabindex="-1"></a>    $$ $$</span>
<span id="cb1-148"><a href="#cb1-148" aria-hidden="true" tabindex="-1"></a>    V_{1} = \frac{300}{10} = 30mL</span>
<span id="cb1-149"><a href="#cb1-149" aria-hidden="true" tabindex="-1"></a>    $$ Therefore $30mL$ of <span class="in">`10xPBS`</span> added to $70mL$ of <span class="in">`Milli-Q Water`</span> will give a final concentration of <span class="in">`3xPBS`</span> in a final volume of $100mL$.</span>
<span id="cb1-150"><a href="#cb1-150" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb1-151"><a href="#cb1-151" aria-hidden="true" tabindex="-1"></a><span class="fu">## Protein quantification</span></span>
<span id="cb1-152"><a href="#cb1-152" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb1-153"><a href="#cb1-153" aria-hidden="true" tabindex="-1"></a><span class="ss">-   </span>defrost supernatent (protein tube)</span>
<span id="cb1-154"><a href="#cb1-154" aria-hidden="true" tabindex="-1"></a><span class="ss">-   </span>top up with RO water to standard volume</span>
<span id="cb1-155"><a href="#cb1-155" aria-hidden="true" tabindex="-1"></a><span class="ss">-   </span>quantify protein against a bovine albumin standard using the Thermo Scientific Coomasie Plus (Bradford) kit and protocol</span>
<span id="cb1-156"><a href="#cb1-156" aria-hidden="true" tabindex="-1"></a><span class="ss">-   </span>take all spectrophotometer measurements at $595nm$</span>
<span id="cb1-157"><a href="#cb1-157" aria-hidden="true" tabindex="-1"></a><span class="ss">-   </span>report units in $\mu g/mL$</span>
<span id="cb1-158"><a href="#cb1-158" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb1-159"><a href="#cb1-159" aria-hidden="true" tabindex="-1"></a><span class="fu">## Symbiont counts</span></span>
<span id="cb1-160"><a href="#cb1-160" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb1-161"><a href="#cb1-161" aria-hidden="true" tabindex="-1"></a><span class="ss">-   </span>remove 500uL to a new 2mL microcentrifuge tube</span>
<span id="cb1-162"><a href="#cb1-162" aria-hidden="true" tabindex="-1"></a><span class="ss">-   </span>Add 1mL of 3X PBS</span>
<span id="cb1-163"><a href="#cb1-163" aria-hidden="true" tabindex="-1"></a><span class="ss">-   </span>vortex to resuspend</span>
<span id="cb1-164"><a href="#cb1-164" aria-hidden="true" tabindex="-1"></a><span class="ss">-   </span>pipette out $200 \mu L$ subsample, and place on a Neubauer Improved heamocytometer</span>
<span id="cb1-165"><a href="#cb1-165" aria-hidden="true" tabindex="-1"></a><span class="ss">-   </span>place the heamocytometer slide on a microscope, focus &amp; photograph<span class="ot">[^6]</span></span>
<span id="cb1-166"><a href="#cb1-166" aria-hidden="true" tabindex="-1"></a><span class="ss">-   </span>repeat 6 times per sample! Oof... I need an undergrad for this.</span>
<span id="cb1-167"><a href="#cb1-167" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb1-168"><a href="#cb1-168" aria-hidden="true" tabindex="-1"></a><span class="ot">[^6]</span>:</span>
<span id="cb1-169"><a href="#cb1-169" aria-hidden="true" tabindex="-1"></a>    <span class="at">&gt; “For ease and speed of counting, if there were more than 30 cells per square in the test sample, the entire treatment was diluted in separate Eppendorf tubes by either mixing 500 μl Symbiodiniaceae with 500 μl 3× PBS (2× dilution) or 250 μl Symbiodiniaceae with 750 μl of 3× PBS (4× dilution).” (Steinberg et al., 2021, p. 3)</span></span>
<span id="cb1-170"><a href="#cb1-170" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb1-171"><a href="#cb1-171" aria-hidden="true" tabindex="-1"></a><span class="at">&gt; “The final count was then multiplied by the dilution to attain correct counts. Each haemocytometer fill was considered a technical replicate, while the squares counted within the grid were considered a sub-replicate. Six technical replicates and 5 sub-replicates per technical replicate were performed, for a total of 30 replicates. Replicate numbers were chosen based on previous work in stony and octocorals and anemones (Brown et al., 1995; Hueerkamp et al., 2001; Hill and Scott, 2012; Hill et al., 2014; Pupier et al., 2018; Gierz et al., 2020).” (Steinberg et al., 2021, p. 3)</span></span>
<span id="cb1-172"><a href="#cb1-172" aria-hidden="true" tabindex="-1"></a><span class="at">&gt;</span></span>
<span id="cb1-173"><a href="#cb1-173" aria-hidden="true" tabindex="-1"></a><span class="at">&gt; “The number of Symbiodiniaceae cells was standardized to the total protein concentration of the supernatant. The dilution factor (DF) of the supernatant compared to the resuspended Symbiodiniaceae pellet must be considered when calculating Symbiodiniaceae concentrations. In this study, the supernatant had a volume of 40 ml and the resuspended Symbiodiniaceae had a volume of 5 ml, and as such the DF was 8. Symbiodiniaceae per ml was calculated using the following formula: count ×dilution 6.25 ×10−6 , where 6.25 × 10−6 is the volume of one haemocytometer grid square. Symbiodiniaceae per μg protein was calculated using the following formula: Symbiodiniceae ml DF ×protein( μg ml ) or count ×dilution 6.25 ×10−6 DF ×protein ( μg ml ).” (Steinberg et al., 2021, p. 3)</span></span>
<span id="cb1-174"><a href="#cb1-174" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb1-175"><a href="#cb1-175" aria-hidden="true" tabindex="-1"></a><span class="fu">## Chlorophyll extraction</span></span>
<span id="cb1-176"><a href="#cb1-176" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb1-177"><a href="#cb1-177" aria-hidden="true" tabindex="-1"></a><span class="ss">-   </span>vortex the remaining contents of the Chlorophyll tube to re-suspend the symbionts</span>
<span id="cb1-178"><a href="#cb1-178" aria-hidden="true" tabindex="-1"></a><span class="ss">-   </span>centrifuge for 10 minutes at 800 xg</span>
<span id="cb1-179"><a href="#cb1-179" aria-hidden="true" tabindex="-1"></a><span class="ss">-   </span>remove supernatent &amp; discard</span>
<span id="cb1-180"><a href="#cb1-180" aria-hidden="true" tabindex="-1"></a><span class="ss">-   </span>add 5mL of 100% acetone</span>
<span id="cb1-181"><a href="#cb1-181" aria-hidden="true" tabindex="-1"></a><span class="ss">-   </span>vortex to re-suspend symbiont pellet</span>
<span id="cb1-182"><a href="#cb1-182" aria-hidden="true" tabindex="-1"></a><span class="ss">-   </span>incubate tubes in the dark at $4^{\circ}C$ for 48 hours<span class="ot">[^7]</span></span>
<span id="cb1-183"><a href="#cb1-183" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb1-184"><a href="#cb1-184" aria-hidden="true" tabindex="-1"></a><span class="ot">[^7]: </span>In the paper Steinberg et al., 2021, they compared a 24hr and a 48hr Chlorophyll extraction incubation period... they found that 24hrs was perfectly fine for Chl-a extraction, but that 48hrs was better for Chl-c2 extraction. See Figure 3. from the paper below!</span>
<span id="cb1-185"><a href="#cb1-185" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb1-186"><a href="#cb1-186" aria-hidden="true" tabindex="-1"></a>::: {.image-container style="width: 150%;"}</span>
<span id="cb1-187"><a href="#cb1-187" aria-hidden="true" tabindex="-1"></a><span class="al">![](steinberg_fig3.png)</span></span>
<span id="cb1-188"><a href="#cb1-188" aria-hidden="true" tabindex="-1"></a>:::</span>
<span id="cb1-189"><a href="#cb1-189" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb1-190"><a href="#cb1-190" aria-hidden="true" tabindex="-1"></a><span class="fu">## Measure absorbance using a NanoDrop 2000c Spectrophotometer</span></span>
<span id="cb1-191"><a href="#cb1-191" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb1-192"><a href="#cb1-192" aria-hidden="true" tabindex="-1"></a><span class="fu">### Program NanoDrop 2000c Method Editor</span></span>
<span id="cb1-193"><a href="#cb1-193" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb1-194"><a href="#cb1-194" aria-hidden="true" tabindex="-1"></a>I emailed<span class="ot">[^8]</span> Thermo Fisher for some advice on this procedure and got the following instructions:</span>
<span id="cb1-195"><a href="#cb1-195" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb1-196"><a href="#cb1-196" aria-hidden="true" tabindex="-1"></a><span class="ot">[^8]: </span>Hello Sarah,&nbsp;\</span>
<span id="cb1-197"><a href="#cb1-197" aria-hidden="true" tabindex="-1"></a>    \</span>
<span id="cb1-198"><a href="#cb1-198" aria-hidden="true" tabindex="-1"></a>    Thank you for your email. Please see link to below to a protocol that we have published for measuring chlorophyll on the NanoDrop One. It is important to note that the pre-defined custom method mentioned in the document is only compatible with the NanoDrop One, and not with the NanoDrop 2000.\</span>
<span id="cb1-199"><a href="#cb1-199" aria-hidden="true" tabindex="-1"></a>    \</span>
<span id="cb1-200"><a href="#cb1-200" aria-hidden="true" tabindex="-1"></a>    <span class="co">[</span><span class="ot">quantify-chlorophyll-a-and-chlorophyll-b-with-custom-method-T141.pdf (thermofisher.com)</span><span class="co">](https://urldefense.com/v3/__https://assets.thermofisher.com/TFS-Assets/MSD/Application-Notes/quantify-chlorophyll-a-and-chlorophyll-b-with-custom-method-T141.pdf__;!!K-Hz7m0Vt54!l4t_v1Q2wbByHGjWD4zmlHTr0MMd4mW_xkGoPnXgzBrR7vofzzyK3trG-Kjumc3-HiSX3eYSacfPiQdwpSk$)</span>\</span>
<span id="cb1-201"><a href="#cb1-201" aria-hidden="true" tabindex="-1"></a>    \</span>
<span id="cb1-202"><a href="#cb1-202" aria-hidden="true" tabindex="-1"></a>    You can replicate the Custom Method on your NanoDrop 2000 by following the attached protocol and using the below guidelines in the "Method Editor" application:&nbsp;</span>
<span id="cb1-203"><a href="#cb1-203" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb1-204"><a href="#cb1-204" aria-hidden="true" tabindex="-1"></a><span class="in">    -   Wavelength range Visible (350–850 nm)</span></span>
<span id="cb1-205"><a href="#cb1-205" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb1-206"><a href="#cb1-206" aria-hidden="true" tabindex="-1"></a><span class="in">    -   Extinction coefficient 74.8 g/L</span></span>
<span id="cb1-207"><a href="#cb1-207" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb1-208"><a href="#cb1-208" aria-hidden="true" tabindex="-1"></a><span class="in">    -   Analysis wavelength 666 nm</span></span>
<span id="cb1-209"><a href="#cb1-209" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb1-210"><a href="#cb1-210" aria-hidden="true" tabindex="-1"></a><span class="in">    -   Correction for analysis wavelength 750 nm</span></span>
<span id="cb1-211"><a href="#cb1-211" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb1-212"><a href="#cb1-212" aria-hidden="true" tabindex="-1"></a><span class="in">    -   Baseline correction 750 nm</span></span>
<span id="cb1-213"><a href="#cb1-213" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb1-214"><a href="#cb1-214" aria-hidden="true" tabindex="-1"></a><span class="in">    -   Automated pathlength On</span></span>
<span id="cb1-215"><a href="#cb1-215" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb1-216"><a href="#cb1-216" aria-hidden="true" tabindex="-1"></a><span class="in">    If you have any questions, please do not hesitate to contact me; it is my pleasure to assist you. I can best be reached by email at [nanodrop\@thermofisher.com](mailto:nanodrop@thermofisher.com), and typing Attention: Moira in the subject bar. If I am unavailable, any member of our Technical Support Team would be happy to assist you at 877-724-7690.\</span></span>
<span id="cb1-217"><a href="#cb1-217" aria-hidden="true" tabindex="-1"></a><span class="in">    \</span></span>
<span id="cb1-218"><a href="#cb1-218" aria-hidden="true" tabindex="-1"></a><span class="in">    Best,\</span></span>
<span id="cb1-219"><a href="#cb1-219" aria-hidden="true" tabindex="-1"></a><span class="in">    Moira Abbott\</span></span>
<span id="cb1-220"><a href="#cb1-220" aria-hidden="true" tabindex="-1"></a><span class="in">    Technical Application Specialist II\</span></span>
<span id="cb1-221"><a href="#cb1-221" aria-hidden="true" tabindex="-1"></a><span class="in">    Thermo Scientific NanoDrop Products\</span></span>
<span id="cb1-222"><a href="#cb1-222" aria-hidden="true" tabindex="-1"></a><span class="in">    \</span></span>
<span id="cb1-223"><a href="#cb1-223" aria-hidden="true" tabindex="-1"></a><span class="in">    Thermo Fisher Scientific\</span></span>
<span id="cb1-224"><a href="#cb1-224" aria-hidden="true" tabindex="-1"></a><span class="in">    [3411 Silverside Road](https://www.google.com/maps/search/3411+Silverside+Road?entry=gmail&amp;source=g) \| Tatnall Building, Suite 100 \| Wilmington, DE 19810\</span></span>
<span id="cb1-225"><a href="#cb1-225" aria-hidden="true" tabindex="-1"></a><span class="in">    Tel: +1 (302) 479-7707 \| Toll Free: +1 (877) 724-7690 ext. 2400109 \| Fax: +1 (302) 792-7155\</span></span>
<span id="cb1-226"><a href="#cb1-226" aria-hidden="true" tabindex="-1"></a><span class="in">    [nanodrop\@thermofisher.com](mailto:nanodrop@thermofisher.com) \| [www.thermofisher.com/nanodrop](https://urldefense.com/v3/__http://www.thermofisher.com/nanodrop__;!!K-Hz7m0Vt54!l4t_v1Q2wbByHGjWD4zmlHTr0MMd4mW_xkGoPnXgzBrR7vofzzyK3trG-Kjumc3-HiSX3eYSacfPgBMP5QU$)\</span></span>
<span id="cb1-227"><a href="#cb1-227" aria-hidden="true" tabindex="-1"></a><span class="in">    &nbsp;</span></span>
<span id="cb1-228"><a href="#cb1-228" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb1-229"><a href="#cb1-229" aria-hidden="true" tabindex="-1"></a>Follow procedure from the <span class="co">[</span><span class="ot">NanoDrop One</span><span class="co">](https://assets.thermofisher.com/TFS-Assets/MSD/Application-Notes/quantify-chlorophyll-a-and-chlorophyll-b-with-custom-method-T141.pdf)</span>, but note the following differences for accomplishing this on the NanoDrop 2000c:</span>
<span id="cb1-230"><a href="#cb1-230" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb1-231"><a href="#cb1-231" aria-hidden="true" tabindex="-1"></a><span class="ss">1.  </span>Open the NanoDrop 2000c desktop application</span>
<span id="cb1-232"><a href="#cb1-232" aria-hidden="true" tabindex="-1"></a><span class="ss">2.  </span>Click on the "Method Editor" button</span>
<span id="cb1-233"><a href="#cb1-233" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb1-234"><a href="#cb1-234" aria-hidden="true" tabindex="-1"></a><span class="al">![](01_method-editor.PNG)</span></span>
<span id="cb1-235"><a href="#cb1-235" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb1-236"><a href="#cb1-236" aria-hidden="true" tabindex="-1"></a><span class="ss">2.  </span>Name and describe the custom measurement method. Here, select <span class="in">`Manually enetered factor / extinction coefficient`</span> for the <span class="in">`Method type`</span>. Click <span class="in">`Next`</span>.<span class="al">![](02_name-type.PNG)</span></span>
<span id="cb1-237"><a href="#cb1-237" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb1-238"><a href="#cb1-238" aria-hidden="true" tabindex="-1"></a><span class="ss">3.  </span>In this window type in the analysis wavelength in nanometers (nm) that you want to measure. Here we want to measure 630, 663, and 750. We want the extinction coefficient to be from @jeffrey1975 table 2:</span>
<span id="cb1-239"><a href="#cb1-239" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb1-240"><a href="#cb1-240" aria-hidden="true" tabindex="-1"></a><span class="al">![](jeffery_table2.png)</span></span>
<span id="cb1-241"><a href="#cb1-241" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb1-242"><a href="#cb1-242" aria-hidden="true" tabindex="-1"></a>However, the extinction coefficients for 100% acetone are different for each type of chlorophyll... If I want to measure both chlorophyll a and c2, does this mean I need a different method for each one?</span>
<span id="cb1-243"><a href="#cb1-243" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb1-244"><a href="#cb1-244" aria-hidden="true" tabindex="-1"></a><span class="al">![](03_measurement.PNG)</span></span>
<span id="cb1-245"><a href="#cb1-245" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb1-246"><a href="#cb1-246" aria-hidden="true" tabindex="-1"></a><span class="ss">4.  </span>Correction for analysis wavelength should be 750 nm *and* a baseline correction of 750 nm. Click <span class="in">`Next`</span>.</span>
<span id="cb1-247"><a href="#cb1-247" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb1-248"><a href="#cb1-248" aria-hidden="true" tabindex="-1"></a><span class="al">![](04_correction.PNG)</span></span>
<span id="cb1-249"><a href="#cb1-249" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb1-250"><a href="#cb1-250" aria-hidden="true" tabindex="-1"></a><span class="ss">5.  </span>Optionally enter a formula table. Here I just want to absorbance at each of the following wavelengths: 630, 663, and 750. Click <span class="in">`Next`</span>.</span>
<span id="cb1-251"><a href="#cb1-251" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb1-252"><a href="#cb1-252" aria-hidden="true" tabindex="-1"></a><span class="al">![](05_formula-table.PNG)</span></span>
<span id="cb1-253"><a href="#cb1-253" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb1-254"><a href="#cb1-254" aria-hidden="true" tabindex="-1"></a><span class="ss">6.  </span>In instrument settings select the wavelength measurement range to Visible range (350nm- 840nm). Also check the box next to <span class="in">`Auto pathlength`</span> indicating that it should be 'On'. Click <span class="in">`Finish`</span>.</span>
<span id="cb1-255"><a href="#cb1-255" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb1-256"><a href="#cb1-256" aria-hidden="true" tabindex="-1"></a>    <span class="al">![](06_settings.PNG)</span></span>
<span id="cb1-257"><a href="#cb1-257" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb1-258"><a href="#cb1-258" aria-hidden="true" tabindex="-1"></a><span class="ss">7.  </span>Load your blank cuvette...</span>
<span id="cb1-259"><a href="#cb1-259" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb1-260"><a href="#cb1-260" aria-hidden="true" tabindex="-1"></a>    <span class="al">![](08_cuvette.PNG)</span></span>
<span id="cb1-261"><a href="#cb1-261" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb1-262"><a href="#cb1-262" aria-hidden="true" tabindex="-1"></a><span class="fu">### Measure cuvette blanks and samples</span></span>
<span id="cb1-263"><a href="#cb1-263" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb1-264"><a href="#cb1-264" aria-hidden="true" tabindex="-1"></a><span class="ss">6.  </span>after the 48hr incubation, pull tubes out of the 4C fridge and centrifuge for 10 minutes at 805 RCF</span>
<span id="cb1-265"><a href="#cb1-265" aria-hidden="true" tabindex="-1"></a><span class="ss">7.  </span>transfer 2 2mL aliquots of extracted chlorophyll in acetone from each symbiont tube into two optical glass cuvettes</span>
<span id="cb1-266"><a href="#cb1-266" aria-hidden="true" tabindex="-1"></a><span class="ss">8.  </span>measure absorbance at 630, 663, and 750nm using the NanoDrop cuvettes</span>
<span id="cb1-267"><a href="#cb1-267" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb1-268"><a href="#cb1-268" aria-hidden="true" tabindex="-1"></a><span class="fu"># Calculate concentrations</span></span>
<span id="cb1-269"><a href="#cb1-269" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb1-270"><a href="#cb1-270" aria-hidden="true" tabindex="-1"></a><span class="ss">-   </span>calculate concentrations of chlorophyll $a$ and $c_2$ in $\mu g/mL$ using the following equations from <span class="co">[</span><span class="ot">@jeffrey1975</span><span class="co">]</span> Jeffery and Humphrey (1975) for dinoflagellates:</span>
<span id="cb1-271"><a href="#cb1-271" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb1-272"><a href="#cb1-272" aria-hidden="true" tabindex="-1"></a>    $$ Chlorophyll a = 11.43E_{663} − 0.64E_{630} $$ $$ Chlorophyll c2 = 27.09E_{630} − 3.63E_{663} $$</span>
<span id="cb1-273"><a href="#cb1-273" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb1-274"><a href="#cb1-274" aria-hidden="true" tabindex="-1"></a><span class="at">&gt; " As 1 ml of resuspended Symbiodiniaceae pellet was extracted in 5 ml of acetone, chlorophyll concentrations were multiplied by five. To calculate chlorophyll per μg protein, the following equation was used: 5 × chlorophyll ( μg ml ) 8 ×protein ( μg ml )." (Steinberg et al., 2021, p. 4)</span></span></code><button title="Copy to Clipboard" class="code-copy-button" data-in-quarto-modal=""><i class="bi"></i></button></pre></div>
</div></div></div></div></div>
</div> <!-- /content -->
<footer class="footer">
  <div class="nav-footer">
    <div class="nav-footer-left">
<p>This page is built using <a href="https://quarto.org/">Quarto</a>❤️ and many pages and posts are under active development</p>
</div>   
    <div class="nav-footer-center">
      &nbsp;
    </div>
    <div class="nav-footer-right">
      &nbsp;
    </div>
  </div>
</footer>




<script src="../../../../site_libs/quarto-html/zenscroll-min.js"></script>
</body></html>