---
title: "MS Committee Meeting"
date: 2025-05-06
author: "Sarah Tanja" 
format: 
  revealjs: 
    theme: solarized
    smaller: true
    scrollable: true
    embed-resources: true
    fig-responsive: true
---

## MS Thesis Proposal

-   proposal is complete
-   submitted signed proposal pdf to SAFS milestones

![](proposal.jpg)

## Clarifications on experiment

::: notes
The samples are vials in which I placed two egg-sperm bundles from genetically distinct parent colonies, resulting in \~30-15 embryos per sample (depending on fertilization & survival). The microbiome and transcriptomics datasets are paired as the DNA/RNA were extracted from the same pooled samples.
:::

![](Experimental_Assay_Design.jpeg)

-   12 treatments across leachate exposure and developmental milestones (hpf)
-   Microscopy n=9
-   July 5th & 6th
-   Originally n=10, dropped 1 cross due to no fertilization
-   Microbiome n=5 & Transcriptomics n=5
-   July 5th, 6th & 7th
-   Originally n=15, pooled samples (3 per) to have enough material to extract

# Microscopy

## Annotations

7 Control 4, 9, & 14 hpf

::::: columns
::: {.column width="50%"}
![](7C4_4xstitch_00_anno.jpg) ![](7C9_001_anno.jpg)
:::

::: {.column width="50%"}
![](7C14_000_anno.jpg) ![](scope_csv.jpg)
:::
:::::

## Direction of cell division from 2 to 4 proceeds from the inside out

::::: columns
::: {.column width="50%"}
![](10H4_8.jpg) ![](10H4_21.jpg)
:::

::: {.column width="50%"}
![](10H4_10.jpg) ![](10M4_23.jpg)
:::
:::::

## Initial results

::::: columns
::: {.column width="50%"}
![](density_viable.png) ![](counts_viable_boxplot.png)
:::

::: {.column width="50%"}
![](proportion_stage_stackedbar.png) ![](proportion_status_stackedbar.png)
:::
:::::

# Microbiome

-   Partnered with mim_c for analysis
-   Shannon diversity non-significant across features

## Taxa barplot

![](level-7-bars.svg) ![](level-7-legend.svg)

## Maaslin2 significant differential features

:::::: columns
::: {.column width="30%"}
![](leachate_1.png) ![](leachate_2.png) ![](leachate_3.png)
:::

::: {.column width="30%"}
![](leachate_4.png) ![](leachate_5.png) ![](leachate_6.png)
:::

::: {.column width="30%"}
![](leachate_7.png) ![](leachate_8.png) ![](leachate_9.png)
:::
::::::

# Leachate decreased abundance in key coral microbes

-   Arcobacteraceae (mixotroph, carbon fixation)
-   Lactobacillus (beneficial probiotic, carbohydrate fermentation produces lactic acid)
-   Alteromonas (contribute to carbohydrate degradation, supports nutrient cycling, DMSP (dimethylsulfoniopropionate) metabolism --\> tolerance to thermal stress)

::: callout-important
PICRUSt2 : infer functional pathway abundances
:::

# Transcriptomics

## PCA

![](deg_pca.png)

## DEGs

-   Interaction term
```         
#Set DESeq2 design for 2 factors and their interaction

dds_i <- DESeqDataSetFromMatrix(countData = gcm_tidyfilt,
                                  colData = metadata,
                                  design = ~ embryonic_phase + pvc_leachate_level + embryonic_phase:pvc_leachate_level)
```

```
summary(results(dds_i_wald), alpha = 0.05)

out of 22634 with nonzero total read count
adjusted p-value < 0.05
LFC > 0 (up)       : 217, 0.96%
LFC < 0 (down)     : 24, 0.11%
outliers [1]       : 30, 0.13%
low counts [2]     : 4824, 21%
(mean count < 7)
```

- 4 hpf has 1 gene with a LFC > 0 
```
dds_4 <- DESeqDataSetFromMatrix(countData = gcm_tidyfilt_4,
                              colData = metadata_4,
                              design = ~ pvc_leachate_level)
```


```
summary(results(dds_4_wald), alpha = 0.05)
out of 14191 with nonzero total read count
adjusted p-value < 0.05
LFC > 0 (up)       : 1, 0.007%
LFC < 0 (down)     : 0, 0%
outliers [1]       : 198, 1.4%
low counts [2]     : 0, 0%
(mean count < 2)
```
 - 9 hpf & 14 hpf have zero pairwise DEGs

## Heatmaps

::::: columns
::: {.column-width="50%"}
![](deg_simple_heat.png)
:::

::: {.column-width="50%"}
![](deg_complex_heat.png)
:::
:::::

## GO functional enrichment

::: callout-important
-   Interaction: need to confirm model inputs for [multifactorial model of differential expression](https://marineomics.github.io/DGE_comparison_v2.html)
-   Pairwise:  true list of DEG's very very small... (0, or 1 differentially expressed gene!?)

-   Currently removing outliers, checking model fit assumptions as per MarineOmics tutorial

-   We will either have 1 really interesting gene to talk about, or show no differential expression due to leachate

-   Differential expression between embryonic stages this early has not previously been demonstrated
:::

## Remaining timeline

-   SP25 Finish Analyses (focus on Results section)
-   June 15th 2025 \| deadline to apply for SAFS finishing fellowship for AU25
-   On-Leave status for SU25 (no funding, GI Bill runs out at the end of SP25)
-   Writing Results & Discussion SU25
-   Back again AU25\
-   Sep 20th 2025 Submit Thesis Draft
-   Nov 20th 2025 Final Exam
-   Dec 12th 2025 Submit Thesis to UW

## Links

[coral-embryo-scope](https://github.com/sarahtanja/coral-embryo-scope)

[coral-embryo-microbiome](https://github.com/sarahtanja/coral-embryo-microbiome)

[coral-embryo-RNAseq](https://github.com/sarahtanja/coral-embryo-RNAseq)
